{
 "cells": [
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Create the datasets for LUKE entity pair classification \n",
    "1. Add a column to the original dataset with the entity spans"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found existing installation: pandas 1.3.5\n",
      "Uninstalling pandas-1.3.5:\n",
      "  Successfully uninstalled pandas-1.3.5\n",
      "Collecting pandas==1.3.5\n",
      "  Using cached pandas-1.3.5-cp39-cp39-macosx_10_9_universal2.whl\n",
      "Requirement already satisfied: python-dateutil>=2.7.3 in /Users/avarela/miniconda3/envs/preprocessing-for-luke-dlbnlp/lib/python3.9/site-packages (from pandas==1.3.5) (2.8.2)\n",
      "Requirement already satisfied: pytz>=2017.3 in /Users/avarela/miniconda3/envs/preprocessing-for-luke-dlbnlp/lib/python3.9/site-packages (from pandas==1.3.5) (2022.7)\n",
      "Requirement already satisfied: numpy>=1.20.0 in /Users/avarela/miniconda3/envs/preprocessing-for-luke-dlbnlp/lib/python3.9/site-packages (from pandas==1.3.5) (1.24.3)\n",
      "Requirement already satisfied: six>=1.5 in /Users/avarela/miniconda3/envs/preprocessing-for-luke-dlbnlp/lib/python3.9/site-packages (from python-dateutil>=2.7.3->pandas==1.3.5) (1.16.0)\n",
      "Installing collected packages: pandas\n",
      "Successfully installed pandas-1.3.5\n",
      "Package                       Version\n",
      "----------------------------- -------\n",
      "appnope                       0.1.3\n",
      "asttokens                     2.2.1\n",
      "backcall                      0.2.0\n",
      "backports.functools-lru-cache 1.6.4\n",
      "Bottleneck                    1.3.5\n",
      "debugpy                       1.5.1\n",
      "decorator                     5.1.1\n",
      "entrypoints                   0.4\n",
      "executing                     1.2.0\n",
      "ipykernel                     6.15.0\n",
      "ipython                       8.13.2\n",
      "jedi                          0.18.2\n",
      "jupyter-client                7.3.4\n",
      "jupyter_core                  5.3.0\n",
      "matplotlib-inline             0.1.6\n",
      "nest-asyncio                  1.5.6\n",
      "numexpr                       2.8.4\n",
      "numpy                         1.24.3\n",
      "packaging                     23.1\n",
      "pandas                        1.3.5\n",
      "parso                         0.8.3\n",
      "pexpect                       4.8.0\n",
      "pickleshare                   0.7.5\n",
      "pip                           23.0.1\n",
      "platformdirs                  3.5.1\n",
      "prompt-toolkit                3.0.38\n",
      "psutil                        5.9.0\n",
      "ptyprocess                    0.7.0\n",
      "pure-eval                     0.2.2\n",
      "Pygments                      2.15.1\n",
      "python-dateutil               2.8.2\n",
      "pytz                          2022.7\n",
      "pyzmq                         25.0.2\n",
      "setuptools                    66.0.0\n",
      "six                           1.16.0\n",
      "stack-data                    0.6.2\n",
      "tornado                       6.1\n",
      "traitlets                     5.9.0\n",
      "typing_extensions             4.5.0\n",
      "wcwidth                       0.2.6\n",
      "wheel                         0.38.4\n"
     ]
    }
   ],
   "source": [
    "!pip3 uninstall -y pandas\n",
    "!pip3 install pandas==1.3.5\n",
    "!pip3 list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip uninstall -y pandas\n",
    "!pip install pandas==1.3.5\n",
    "!pip list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'1.3.5'"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd \n",
    "pd.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['PMID', 'NUMSENT', 'REGULATOR', 'REGULATED', 'REGULATED_TYPE',\n",
       "       'REGULATED_GENE', 'REGULATED_TU', 'EFFECT', 'NORMALIZED_EFFECT',\n",
       "       'SENTENCE', 'regulator_regex', 'regulated_regex', 'regulator_count',\n",
       "       'regulated_count', 'regulator_plus_one', 'regulated_plus_one',\n",
       "       'regulator_regulated_plus_one', 'regulated_tag', 'regulator_tag',\n",
       "       'sentence_tagged'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# All curated datasets including multi regulator- multi regulated\n",
    "data= pd.read_csv(\"../results/ECO/curated_master_dataset/ECO_dataset_master_curated_dataset_v2.tsv\",sep=\"\\t\")\n",
    "data.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# curated datasets but multi regulator- multi regulated\n",
    "#data= pd.read_csv(\"../results/ECO/curated_master_dataset/ECO_dataset_master_curated_dataset_v1.tsv\",sep=\"\\t\")\n",
    "#data.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['PMID', 'NUMSENT', 'REGULATOR', 'REGULATED', 'REGULATED_TYPE',\n",
       "       'REGULATED_GENE', 'REGULATED_TU', 'EFFECT', 'NORMALIZED_EFFECT',\n",
       "       'SENTENCE', 'regulator_regex', 'regulated_regex', 'regulator_count',\n",
       "       'regulated_count', 'regulator_plus_one', 'regulated_plus_one',\n",
       "       'regulator_regulated_plus_one', 'regulated_tag', 'regulator_tag',\n",
       "       'sentence_tagged'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# For Reading the file with the correct tagged sentences only one regulator - one regulated\n",
    "#data_2 = pd.read_csv(\"../results/ECO/one_regulator_one_regulated/ECO_dataset_one_regulator_one_regulated_whole_tagging_info_v2.tsv\",sep = \"\\t\")\n",
    "#data_2.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0       Summary In vivo and in vitro analyses indicate...\n",
      "1       Summary In vivo and in vitro analyses indicate...\n",
      "2       To address the mechanisms of activation by @TF...\n",
      "3       To address the mechanisms of activation by CRP...\n",
      "4       Microarray analysis revealed that @TF$ repress...\n",
      "                              ...                        \n",
      "1557    Identification of the DNA - binding domain of ...\n",
      "1558    Identification of the DNA - binding domain of ...\n",
      "1559    DNA - protein recognition : demonstration of t...\n",
      "1560    @TF$ / cAMP and CytR regulated promoters in Es...\n",
      "1561    CRP / cAMP and @TF$ regulated promoters in Esc...\n",
      "Name: sentence_tagged, Length: 1562, dtype: object\n"
     ]
    }
   ],
   "source": [
    "print(data[\"sentence_tagged\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PMID                             int64\n",
      "NUMSENT                          int64\n",
      "REGULATOR                       object\n",
      "REGULATED                       object\n",
      "REGULATED_TYPE                  object\n",
      "REGULATED_GENE                  object\n",
      "REGULATED_TU                    object\n",
      "EFFECT                          object\n",
      "NORMALIZED_EFFECT               object\n",
      "SENTENCE                        object\n",
      "regulator_regex                 object\n",
      "regulated_regex                 object\n",
      "regulator_count                  int64\n",
      "regulated_count                  int64\n",
      "regulator_plus_one                bool\n",
      "regulated_plus_one                bool\n",
      "regulator_regulated_plus_one      bool\n",
      "regulated_tag                   object\n",
      "regulator_tag                   object\n",
      "sentence_tagged                 object\n",
      "dtype: object\n"
     ]
    }
   ],
   "source": [
    "print(data.dtypes)\n",
    "#print(data_2.dtypes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\" \n",
    "Get the start position and end position of a tagged string inside the original sentence.\n",
    "\n",
    ":param pos: int, index of the tagged word in the sentence \n",
    ":param list_sent: list, the original sentence splitted into words  \n",
    ":return: tuple, start idx and end idx of the tagged string\n",
    "\"\"\"\n",
    "def get_start_end(pos, list_sent):\n",
    "    start = 0\n",
    "    end = 0\n",
    "    token_pos = 0\n",
    "    for token in list_sent:\n",
    "        return_list = []\n",
    "        #print(\"token: {}\".format(token))\n",
    "        if pos == 0:\n",
    "            start = 0\n",
    "            end = start + len(token)\n",
    "            return_list.append(start)\n",
    "            return_list.append(end)\n",
    "            return return_list \n",
    "        if token_pos == pos:\n",
    "            # start += 1\n",
    "            end = start + len(token)\n",
    "            return_list.append(start)\n",
    "            return_list.append(end)\n",
    "            return return_list \n",
    "        start += len(token) + 1\n",
    "        #print(\"start: {}\".format(start))\n",
    "        token_pos += 1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Summary In vivo and in vitro analyses indicate that transcription of the argO gene coding for an arginine exporter is regulated by the global transcriptional regulator Lrp , an effect that went by unnoticed in previous genome - scale screenings of the Lrp regulatory network in Escherichia coli . Summary In vivo and in vitro analyses indicate that transcription of the @Regulated$ gene coding for an arginine exporter is regulated by the global transcriptional regulator @TF$ , an effect that went by unnoticed in previous genome - scale screenings of the Lrp regulatory network in Escherichia coli .\n",
      "12\n",
      "26\n",
      "Start y end del regulador: [168, 171]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: argO\n",
      "Summary In vivo and in vitro analyses indicate that transcription of the argO gene coding for an arginine exporter is regulated by the global transcriptional regulator Lrp , an effect that went by unnoticed in previous genome - scale screenings of the Lrp regulatory network in Escherichia coli . Summary In vivo and in vitro analyses indicate that transcription of the @Regulated$ gene coding for an arginine exporter is regulated by the global transcriptional regulator Lrp , an effect that went by unnoticed in previous genome - scale screenings of the @TF$ regulatory network in Escherichia coli .\n",
      "12\n",
      "42\n",
      "Start y end del regulador: [252, 255]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: argO\n",
      "To address the mechanisms of activation by CRP and the RNA polymerase - subunit C - terminal domain ( - CTD ) at rhaSR , we tested the effects of alanine substitutions in CRP activating regions 1 and 2 , overexpression of a truncated version of ( - 235 ) , and alanine substitutions throughout - CTD . To address the mechanisms of activation by @TF$ and the RNA polymerase - subunit C - terminal domain ( - CTD ) at @Regulated$ , we tested the effects of alanine substitutions in CRP activating regions 1 and 2 , overexpression of a truncated version of ( - 235 ) , and alanine substitutions throughout - CTD .\n",
      "23\n",
      "7\n",
      "Start y end del regulador: [43, 46]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [113, 118]\n",
      "Regulado: rhaSR\n",
      "To address the mechanisms of activation by CRP and the RNA polymerase - subunit C - terminal domain ( - CTD ) at rhaSR , we tested the effects of alanine substitutions in CRP activating regions 1 and 2 , overexpression of a truncated version of ( - 235 ) , and alanine substitutions throughout - CTD . To address the mechanisms of activation by CRP and the RNA polymerase - subunit C - terminal domain ( - CTD ) at @Regulated$ , we tested the effects of alanine substitutions in @TF$ activating regions 1 and 2 , overexpression of a truncated version of ( - 235 ) , and alanine substitutions throughout - CTD .\n",
      "23\n",
      "33\n",
      "Start y end del regulador: [171, 174]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [113, 118]\n",
      "Regulado: rhaSR\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that @TF$ represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the @Regulated$ , ytfE , and ygbA genes that were previously shown to be regulated by NsrR .\n",
      "18\n",
      "4\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [123, 127]\n",
      "Regulado: hmpA\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the @Regulated$ , ytfE , and ygbA genes that were previously shown to be regulated by @TF$ .\n",
      "18\n",
      "33\n",
      "Start y end del regulador: [198, 202]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [123, 127]\n",
      "Regulado: hmpA\n",
      "Plasmids overproducing either the NagC protein or the Mlc protein repress the expression of manX , but the effect of the Mlc protein is stronger . Plasmids overproducing either the NagC protein or the @TF$ protein repress the expression of @Regulated$ , but the effect of the Mlc protein is stronger .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [54, 57]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: manX\n",
      "Plasmids overproducing either the NagC protein or the Mlc protein repress the expression of manX , but the effect of the Mlc protein is stronger . Plasmids overproducing either the NagC protein or the Mlc protein repress the expression of @Regulated$ , but the effect of the @TF$ protein is stronger .\n",
      "14\n",
      "21\n",
      "Start y end del regulador: [121, 124]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: manX\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that @TF$ represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and @Regulated$ genes that were previously shown to be regulated by NsrR .\n",
      "23\n",
      "4\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: ygbA\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and @Regulated$ genes that were previously shown to be regulated by @TF$ .\n",
      "23\n",
      "33\n",
      "Start y end del regulador: [198, 202]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: ygbA\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that @TF$ represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , @Regulated$ , and ygbA genes that were previously shown to be regulated by NsrR .\n",
      "20\n",
      "4\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [130, 134]\n",
      "Regulado: ytfE\n",
      "Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , ytfE , and ygbA genes that were previously shown to be regulated by NsrR . Microarray analysis revealed that NsrR represses nine operons encoding 20 genes in Escherichia coli MG1655 , including the hmpA , @Regulated$ , and ygbA genes that were previously shown to be regulated by @TF$ .\n",
      "20\n",
      "33\n",
      "Start y end del regulador: [198, 202]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [130, 134]\n",
      "Regulado: ytfE\n",
      "In the present study , we first identified two positively charged amino acids in 70 , K593 and R599 , and three negatively charged amino acids in RhaR , D276 , E284 , and D285 , that were important for RhaR - mediated transcription activation of the rhaSR operon . In the present study , we first identified two positively charged amino acids in 70 , K593 and R599 , and three negatively charged amino acids in RhaR , D276 , E284 , and D285 , that were important for @TF$  - mediated transcription activation of the @Regulated$ operon .\n",
      "47\n",
      "40\n",
      "Start y end del regulador: [202, 206]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [250, 255]\n",
      "Regulado: rhaSR\n",
      "In the present study , we first identified two positively charged amino acids in 70 , K593 and R599 , and three negatively charged amino acids in RhaR , D276 , E284 , and D285 , that were important for RhaR - mediated transcription activation of the rhaSR operon . In the present study , we first identified two positively charged amino acids in 70 , K593 and R599 , and three negatively charged amino acids in @TF$ , D276 , E284 , and D285 , that were important for  RhaR - mediated transcription activation of the @Regulated$ operon .\n",
      "47\n",
      "27\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [250, 255]\n",
      "Regulado: rhaSR\n",
      "We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type fucPIK promoters , and we identify the CRP - binding site that is required for this activity . We show that the mutant @TF$ constitutively activates transcription from both the IS5 - disrupted and the wild - type @Regulated$ promoters , and we identify the CRP - binding site that is required for this activity .\n",
      "20\n",
      "5\n",
      "Start y end del regulador: [24, 27]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [117, 123]\n",
      "Regulado: fucPIK\n",
      "We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type fucPIK promoters , and we identify the CRP - binding site that is required for this activity . We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type @Regulated$ promoters , and we identify the @TF$ - binding site that is required for this activity .\n",
      "20\n",
      "27\n",
      "Start y end del regulador: [156, 159]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [117, 123]\n",
      "Regulado: fucPIK\n",
      "Our results show that the fucPIK promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase . Our results show that the @Regulated$ promoter , a complex promoter which ordinarily depends on both @TF$ and the fucose - specific regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase .\n",
      "5\n",
      "16\n",
      "Start y end del regulador: [96, 99]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [26, 32]\n",
      "Regulado: fucPIK\n",
      "Our results show that the fucPIK promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase . Our results show that the @Regulated$ promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation, can be activated in the absence of FucR by a mutant @TF$ that uses three , rather than two , activating regions to contact RNA polymerase .\n",
      "5\n",
      "38\n",
      "Start y end del regulador: [207, 213]\n",
      "Regulador: mutant\n",
      "Start y end del regulado: [26, 32]\n",
      "Regulado: fucPIK\n",
      "Our results show that the fucPIK promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase . Our results show that the @Regulated$ promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator @TF$ for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase .\n",
      "5\n",
      "23\n",
      "Start y end del regulador: [136, 140]\n",
      "Regulador: FucR\n",
      "Start y end del regulado: [26, 32]\n",
      "Regulado: fucPIK\n",
      "Our results show that the fucPIK promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation , can be activated in the absence of FucR by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase . Our results show that the @Regulated$ promoter , a complex promoter which ordinarily depends on both CRP and the fucose - specific regulator FucR for its activation , can be activated in the absence of @TF$ by a mutant CRP that uses three , rather than two , activating regions to contact RNA polymerase .\n",
      "5\n",
      "35\n",
      "Start y end del regulador: [197, 201]\n",
      "Regulador: FucR\n",
      "Start y end del regulado: [26, 32]\n",
      "Regulado: fucPIK\n",
      "Thus , although the region of the ydhY â€“ T promoter containing the â€ 16 and + 15 heptamers was recognized by NarL in vitro , mutation of these heptamers did not affect NarL - mediated repression in vivo . Thus , although the region of the @Regulated$ â€“ T promoter containing the â€ 16 and + 15 heptamers was recognized by @TF$ in vitro , mutation of these heptamers did not affect NarL - mediated repression in vivo .\n",
      "7\n",
      "22\n",
      "Start y end del regulador: [113, 117]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: ydhY\n",
      "Thus , although the region of the ydhY â€“ T promoter containing the â€ 16 and + 15 heptamers was recognized by NarL in vitro , mutation of these heptamers did not affect NarL - mediated repression in vivo . Thus , although the region of the @Regulated$ â€“ T promoter containing the â€ 16 and + 15 heptamers was recognized by NarL in vitro , mutation of these heptamers did not affect @TF$ - mediated repression in vivo .\n",
      "7\n",
      "33\n",
      "Start y end del regulador: [172, 176]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: ydhY\n",
      "MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose metabolism ( 7 ) . MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that @TF$ , together with the cyclic AMP receptor protein , CRP , regulates expression of the @Regulated$ operon that encodes products essential for melibiose metabolism ( 7 ) .\n",
      "41\n",
      "25\n",
      "Start y end del regulador: [128, 132]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [217, 222]\n",
      "Regulado: melAB\n",
      "MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose metabolism ( 7 ) .  @TF$ is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the @Regulated$ operon that encodes products essential for melibiose metabolism ( 7 ) .\n",
      "41\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [217, 222]\n",
      "Regulado: melAB\n",
      "In this report , we determined the cis - element responsible for the IciA - dependent activation of the dnaA promoter 1P and the binding mode of IciA protein to the IciA I site . In this report , we determined the cis - element responsible for the @TF$ - dependent activation of the @Regulated$ promoter 1P and the binding mode of IciA protein to the IciA I site .\n",
      "19\n",
      "13\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: dnaA\n",
      "In this report , we determined the cis - element responsible for the IciA - dependent activation of the dnaA promoter 1P and the binding mode of IciA protein to the IciA I site . In this report , we determined the cis - element responsible for the IciA - dependent activation of the @Regulated$ promoter 1P and the binding mode of @TF$ protein to the IciA I site .\n",
      "19\n",
      "27\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: dnaA\n",
      "In this report , we determined the cis - element responsible for the IciA - dependent activation of the dnaA promoter 1P and the binding mode of IciA protein to the IciA I site . In this report , we determined the cis - element responsible for the IciA - dependent activation of the @Regulated$ promoter 1P and the binding mode of IciA protein to the @TF$ I site .\n",
      "19\n",
      "31\n",
      "Start y end del regulador: [165, 169]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: dnaA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , @Regulated$ , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "19\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [124, 130]\n",
      "Regulado: marRAB\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , @Regulated$ , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "19\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [124, 130]\n",
      "Regulado: marRAB\n",
      "marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .  @Regulated$ expression is repressed by @TF$ and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .\n",
      "0\n",
      "5\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: marA\n",
      "marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .  @Regulated$ expression is repressed by MarR and is derepressed by the interaction of @TF$ with various phenolic compounds such as salicylate .\n",
      "0\n",
      "13\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: marA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , @Regulated$ , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "21\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: aslB\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , @Regulated$ , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "21\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: aslB\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and @Regulated$ ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "32\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [172, 176]\n",
      "Regulado: galT\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and @Regulated$ ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "32\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [172, 176]\n",
      "Regulado: galT\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely @Regulated$ , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "17\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: inaA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely @Regulated$ , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "17\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: inaA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , @Regulated$ , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "25\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: mdlA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , @Regulated$ , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "25\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: mdlA\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , @Regulated$ , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "23\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: ybaO\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , @Regulated$ , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "23\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: ybaO\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , @Regulated$ , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "29\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [161, 165]\n",
      "Regulado: ybiS\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , @Regulated$ , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "29\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [161, 165]\n",
      "Regulado: ybiS\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "27\n",
      "11\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: yfhD\n",
      "Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) . Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "27\n",
      "7\n",
      "Start y end del regulador: [57, 60]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: yfhD\n",
      "P1 drives cotranscription of edd and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work . P1 drives cotranscription of edd and @Regulated$ and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by @TF$ ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work .\n",
      "6\n",
      "24\n",
      "Start y end del regulador: [118, 122]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [37, 40]\n",
      "Regulado: eda\n",
      "P1 drives cotranscription of edd and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work . P1 drives cotranscription of edd and @Regulated$ and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of @TF$ to the P1 regulatory region was not established prior to this work .\n",
      "6\n",
      "36\n",
      "Start y end del regulador: [162, 166]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [37, 40]\n",
      "Regulado: eda\n",
      "P1 drives cotranscription of edd and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work . P1 drives cotranscription of @Regulated$ and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by @TF$ ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work .\n",
      "4\n",
      "24\n",
      "Start y end del regulador: [118, 122]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [29, 32]\n",
      "Regulado: edd\n",
      "P1 drives cotranscription of edd and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of GntR to the P1 regulatory region was not established prior to this work . P1 drives cotranscription of @Regulated$ and eda and is induced by growth on gluconate ( 7 ) ; P1 appears to be controlled by GntR ( 20 , 21 , 24 ) , although binding of @TF$ to the P1 regulatory region was not established prior to this work .\n",
      "4\n",
      "36\n",
      "Start y end del regulador: [162, 166]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [29, 32]\n",
      "Regulado: edd\n",
      "Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter . Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of @TF$ in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .\n",
      "23\n",
      "15\n",
      "Start y end del regulador: [102, 105]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [138, 144]\n",
      "Regulado: dmsABC\n",
      "Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter . Mutagenesis of the @TF$ recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .\n",
      "23\n",
      "3\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [138, 144]\n",
      "Regulado: dmsABC\n",
      "Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter . Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the @TF$ recognition site at the P1 promoter .\n",
      "23\n",
      "35\n",
      "Start y end del regulador: [219, 222]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [138, 144]\n",
      "Regulado: dmsABC\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that @TF$ overexpression activates expression of @Regulated$ , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: fumC\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that Rob overexpression activates expression of @Regulated$ , sodA , and inaA ( unknown function ) , but not zwf , and that purified @TF$ forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "10\n",
      "28\n",
      "Start y end del regulador: [148, 151]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: fumC\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that @TF$ overexpression activates expression of fumC , sodA , and @Regulated$ ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "15\n",
      "5\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [88, 92]\n",
      "Regulado: inaA\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that Rob overexpression activates expression of fumC , sodA , and @Regulated$ ( unknown function ) , but not zwf , and that purified @TF$ forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "15\n",
      "28\n",
      "Start y end del regulador: [148, 151]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [88, 92]\n",
      "Regulado: inaA\n",
      "Recent studies ( 11 ) have shown that the occupation of site 2 by MelR is the most crucial for the activation of the melAB promoter , but that , of the four target sites , site 2 binds MelR most weakly . Recent studies ( 11 ) have shown that the occupation of site 2 by @TF$ is the most crucial for the activation of the @Regulated$ promoter , but that , of the four target sites , site 2 binds MelR most weakly .\n",
      "24\n",
      "14\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [117, 122]\n",
      "Regulado: melAB\n",
      "Recent studies ( 11 ) have shown that the occupation of site 2 by MelR is the most crucial for the activation of the melAB promoter , but that , of the four target sites , site 2 binds MelR most weakly . Recent studies ( 11 ) have shown that the occupation of site 2 by MelR is the most crucial for the activation of the @Regulated$ promoter , but that , of the four target sites , site 2 binds @TF$ most weakly .\n",
      "24\n",
      "39\n",
      "Start y end del regulador: [185, 189]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [117, 122]\n",
      "Regulado: melAB\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by @TF$ ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "3\n",
      "25\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [14, 19]\n",
      "Regulado: gntKU\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "3\n",
      "28\n",
      "Start y end del regulador: [141, 145]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [14, 19]\n",
      "Regulado: gntKU\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two @TF$ - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "3\n",
      "45\n",
      "Start y end del regulador: [213, 217]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [14, 19]\n",
      "Regulado: gntKU\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and @Regulated$ operons is controlled by both the NarL and NarP proteins .\n",
      "36\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [190, 194]\n",
      "Regulado: fdnG\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and @Regulated$ operons is controlled by both the @TF$ and NarP proteins .\n",
      "36\n",
      "43\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [190, 194]\n",
      "Regulado: fdnG\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the @TF$ protein alone , such as the narG and @Regulated$ operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "17\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [85, 89]\n",
      "Regulado: frdA\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and @Regulated$ operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the @TF$ and NarP proteins .\n",
      "17\n",
      "43\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [85, 89]\n",
      "Regulado: frdA\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the @TF$ protein alone , such as the @Regulated$ and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "15\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [76, 80]\n",
      "Regulado: narG\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the @Regulated$ and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the @TF$ and NarP proteins .\n",
      "15\n",
      "43\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [76, 80]\n",
      "Regulado: narG\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "24\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [126, 130]\n",
      "Regulado: nirB\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the @TF$ and NarP proteins .\n",
      "24\n",
      "43\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [126, 130]\n",
      "Regulado: nirB\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , @Regulated$ and fdnG operons is controlled by both the @TF$ and NarP proteins .\n",
      "34\n",
      "43\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: nrfA\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , @Regulated$ and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "34\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: nrfA\n",
      "An isorepressor , GalS , has also been identified as Â© 2008 The Authors Journal compilation Â© 2008 Blackwell Publishing LtdNagC , GalR and GalS repression at galP 147 Fig . 1 . An isorepressor , GalS , has also been identified as Â© 2008 The Authors Journal compilation Â© 2008 Blackwell Publishing LtdNagC , GalR and @TF$ repression at @Regulated$ 147 Fig . 1 .\n",
      "27\n",
      "24\n",
      "Start y end del regulador: [141, 145]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: galP\n",
      "An isorepressor , GalS , has also been identified as Â© 2008 The Authors Journal compilation Â© 2008 Blackwell Publishing LtdNagC , GalR and GalS repression at galP 147 Fig . 1 . An isorepressor , @TF$ , has also been identified as Â© 2008 The Authors Journal compilation Â© 2008 Blackwell Publishing LtdNagC , GalR and GalS repression at @Regulated$ 147 Fig . 1 .\n",
      "27\n",
      "3\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: galP\n",
      "The enhanced ability of Fnr to activate dmsABC gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA . The enhanced ability of @TF$ to activate @Regulated$ gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .\n",
      "7\n",
      "4\n",
      "Start y end del regulador: [24, 27]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [40, 46]\n",
      "Regulado: dmsABC\n",
      "The enhanced ability of Fnr to activate dmsABC gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA . The enhanced ability of Fnr to activate @Regulated$ gene expression is possibly , by analogy to Crp , due to increased affinity of @TF$ at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .\n",
      "7\n",
      "23\n",
      "Start y end del regulador: [126, 129]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [40, 46]\n",
      "Regulado: dmsABC\n",
      "The enhanced ability of Fnr to activate dmsABC gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA . The enhanced ability of Fnr to activate @Regulated$ gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus @TF$ recognition sequence at dmsA .\n",
      "7\n",
      "33\n",
      "Start y end del regulador: [181, 184]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [40, 46]\n",
      "Regulado: dmsABC\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the @Regulated$ operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "27\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [137, 142]\n",
      "Regulado: guaBA\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the @Regulated$ operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by @TF$ protein ( 22 ) .\n",
      "27\n",
      "51\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [137, 142]\n",
      "Regulado: guaBA\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , @Regulated$ ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "21\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [119, 123]\n",
      "Regulado: mioC\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , @Regulated$ ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by @TF$ protein ( 22 ) .\n",
      "21\n",
      "51\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [119, 123]\n",
      "Regulado: mioC\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the @Regulated$ gene appeared to be enhanced by @TF$ protein ( 22 ) .\n",
      "44\n",
      "51\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [205, 208]\n",
      "Regulado: nrd\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the @Regulated$ gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "44\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [205, 208]\n",
      "Regulado: nrd\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including @Regulated$ ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "16\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: rpoH\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including @Regulated$ ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by @TF$ protein ( 22 ) .\n",
      "16\n",
      "51\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: rpoH\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and @Regulated$ ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "34\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [163, 167]\n",
      "Regulado: uvrB\n",
      "Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) . Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and @Regulated$ ( 21 ) , while the expression of the nrd gene appeared to be enhanced by @TF$ protein ( 22 ) .\n",
      "34\n",
      "51\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [163, 167]\n",
      "Regulado: uvrB\n",
      "Also , IciA protein binds to two sites , IciA I and IciA II , in the dnaA promoter region ( 25 ) . Also , @TF$ protein binds to two sites , IciA I and IciA II , in the @Regulated$ promoter region ( 25 ) .\n",
      "17\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: dnaA\n",
      "Also , IciA protein binds to two sites , IciA I and IciA II , in the dnaA promoter region ( 25 ) . Also , IciA protein binds to two sites , @TF$ I and IciA II , in the @Regulated$ promoter region ( 25 ) .\n",
      "17\n",
      "9\n",
      "Start y end del regulador: [41, 45]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: dnaA\n",
      "Also , IciA protein binds to two sites , IciA I and IciA II , in the dnaA promoter region ( 25 ) . Also , IciA protein binds to two sites , IciA I and @TF$ II , in the @Regulated$ promoter region ( 25 ) .\n",
      "17\n",
      "12\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: dnaA\n",
      "An additional 50 - fold activation of rhaBAD expression occurs when CRP occupies its binding site centered at 92.5 , which places CRP adjacent to RhaS ( 9 ) . An additional 50 - fold activation of @Regulated$ expression occurs when @TF$ occupies its binding site centered at 92.5 , which places CRP adjacent to RhaS ( 9 ) .\n",
      "7\n",
      "11\n",
      "Start y end del regulador: [68, 71]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [38, 44]\n",
      "Regulado: rhaBAD\n",
      "An additional 50 - fold activation of rhaBAD expression occurs when CRP occupies its binding site centered at 92.5 , which places CRP adjacent to RhaS ( 9 ) . An additional 50 - fold activation of @Regulated$ expression occurs when CRP occupies its binding site centered at 92.5 , which places @TF$ adjacent to RhaS ( 9 ) .\n",
      "7\n",
      "22\n",
      "Start y end del regulador: [130, 133]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [38, 44]\n",
      "Regulado: rhaBAD\n",
      "This investigation reveals that the similarity sufficient to guarantee homology is hardly found between the regulatory protein and the regulated protein , except for the relation between the repressor RbsR and the regulated operon rbsDACBK , where RbsR has been indicated to show considerable similarity to the ribose binding protein RbsB ( Mauzy &amp; Hermodson , 1992 ) , and for some of the regulatory proteins that are known to be self - regulated at the level of transcription . This investigation reveals that the similarity sufficient to guarantee homology is hardly found between the regulatory protein and the regulated protein , except for the relation between the repressor @TF$ and the regulated operon @Regulated$ , where RbsR has been indicated to show considerable similarity to the ribose binding protein RbsB ( Mauzy &amp; Hermodson , 1992 ) , and for some of the regulatory proteins that are known to be self - regulated at the level of transcription .\n",
      "34\n",
      "29\n",
      "Start y end del regulador: [201, 205]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [231, 239]\n",
      "Regulado: rbsDACBK\n",
      "This investigation reveals that the similarity sufficient to guarantee homology is hardly found between the regulatory protein and the regulated protein , except for the relation between the repressor RbsR and the regulated operon rbsDACBK , where RbsR has been indicated to show considerable similarity to the ribose binding protein RbsB ( Mauzy &amp; Hermodson , 1992 ) , and for some of the regulatory proteins that are known to be self - regulated at the level of transcription . This investigation reveals that the similarity sufficient to guarantee homology is hardly found between the regulatory protein and the regulated protein , except for the relation between the repressor RbsR and the regulated operon @Regulated$ , where @TF$ has been indicated to show considerable similarity to the ribose binding protein RbsB ( Mauzy &amp; Hermodson , 1992 ) , and for some of the regulatory proteins that are known to be self - regulated at the level of transcription .\n",
      "34\n",
      "37\n",
      "Start y end del regulador: [248, 252]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [231, 239]\n",
      "Regulado: rbsDACBK\n",
      "MarA upregulates expression of nfnB Both oxygen - insensitive nitroreductases nfsA and nfnB were shown to be upregulated by constitutive expression of MarA from plasmid pAS10 ( Barbosa and Levy , 2000 ) . MarA upregulates expression of nfnB Both oxygen - insensitive nitroreductases @Regulated$ and nfnB were shown to be upregulated by constitutive expression of @TF$ from plasmid pAS10 ( Barbosa and Levy , 2000 ) .\n",
      "10\n",
      "22\n",
      "Start y end del regulador: [151, 155]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: nfsA\n",
      "MarA upregulates expression of nfnB Both oxygen - insensitive nitroreductases nfsA and nfnB were shown to be upregulated by constitutive expression of MarA from plasmid pAS10 ( Barbosa and Levy , 2000 ) .  @TF$ upregulates expression of nfnB Both oxygen - insensitive nitroreductases @Regulated$ and nfnB were shown to be upregulated by constitutive expression of MarA from plasmid pAS10 ( Barbosa and Levy , 2000 ) .\n",
      "10\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: nfsA\n",
      "The 32 P - end - labelled fragments were incubated with the indicated amounts of IciA protein and subjected to gel - shift analysis as described in the Experimental procedures . # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 â€“ 396Transcriptional activation of the dnaA gene by IciA protein 391 Fig . 3 . The 32 P - end - labelled fragments were incubated with the indicated amounts of IciA protein and subjected to gel - shift analysis as described in the Experimental procedures . # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 â€“ 396Transcriptional activation of the @Regulated$ gene by @TF$ protein 391 Fig . 3 .\n",
      "48\n",
      "51\n",
      "Start y end del regulador: [297, 301]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [284, 288]\n",
      "Regulado: dnaA\n",
      "The 32 P - end - labelled fragments were incubated with the indicated amounts of IciA protein and subjected to gel - shift analysis as described in the Experimental procedures . # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 â€“ 396Transcriptional activation of the dnaA gene by IciA protein 391 Fig . 3 . The 32 P - end - labelled fragments were incubated with the indicated amounts of @TF$ protein and subjected to gel - shift analysis as described in the Experimental procedures . # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 â€“ 396Transcriptional activation of the @Regulated$ gene by IciA protein 391 Fig . 3 .\n",
      "48\n",
      "15\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [284, 288]\n",
      "Regulado: dnaA\n",
      "We show that the melR promoter is repressed by MelR and that this autoregulation requires MelR binding to Site R . We show that the @Regulated$ promoter is repressed by @TF$ and that this autoregulation requires MelR binding to Site R .\n",
      "4\n",
      "9\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [17, 21]\n",
      "Regulado: melR\n",
      "We show that the melR promoter is repressed by MelR and that this autoregulation requires MelR binding to Site R . We show that the @Regulated$ promoter is repressed by MelR and that this autoregulation requires @TF$ binding to Site R .\n",
      "4\n",
      "15\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [17, 21]\n",
      "Regulado: melR\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , @Regulated$ , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "29\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [191, 195]\n",
      "Regulado: araA\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , @Regulated$ , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "29\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [191, 195]\n",
      "Regulado: araA\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( @Regulated$ , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "27\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [184, 188]\n",
      "Regulado: araB\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( @Regulated$ , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "27\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [184, 188]\n",
      "Regulado: araB\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , @Regulated$ , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "31\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [198, 202]\n",
      "Regulado: araD\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , @Regulated$ , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "31\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [198, 202]\n",
      "Regulado: araD\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , @Regulated$ , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "33\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: araE\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , @Regulated$ , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "33\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: araE\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , @Regulated$ , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "35\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [212, 216]\n",
      "Regulado: araF\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , @Regulated$ , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "35\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [212, 216]\n",
      "Regulado: araF\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , @Regulated$ , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "37\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: araG\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , @Regulated$ , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "37\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: araG\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and @Regulated$ ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "40\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [230, 234]\n",
      "Regulado: araH\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and @Regulated$ ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "40\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [230, 234]\n",
      "Regulado: araH\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , @Regulated$ , and araU ) .\n",
      "55\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [301, 305]\n",
      "Regulado: araT\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , @Regulated$ , and araU ) .\n",
      "55\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [301, 305]\n",
      "Regulado: araT\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and @Regulated$ ) .\n",
      "58\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [312, 316]\n",
      "Regulado: araU\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and @Regulated$ ) .\n",
      "58\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [312, 316]\n",
      "Regulado: araU\n",
      "To confirm that the nfnB activation observed in the array study was specifically mediated by MarA rather than by a cell stress response to the constitutive expression of the protein from a multicopy number plasmid , Northern blot analysis was performed on cells in which MarA production was inducible . To confirm that the @Regulated$ activation observed in the array study was specifically mediated by @TF$ rather than by a cell stress response to the constitutive expression of the protein from a multicopy number plasmid , Northern blot analysis was performed on cells in which MarA production was inducible .\n",
      "4\n",
      "15\n",
      "Start y end del regulador: [93, 97]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [20, 24]\n",
      "Regulado: nfnB\n",
      "To confirm that the nfnB activation observed in the array study was specifically mediated by MarA rather than by a cell stress response to the constitutive expression of the protein from a multicopy number plasmid , Northern blot analysis was performed on cells in which MarA production was inducible . To confirm that the @Regulated$ activation observed in the array study was specifically mediated by MarA rather than by a cell stress response to the constitutive expression of the protein from a multicopy number plasmid , Northern blot analysis was performed on cells in which @TF$ production was inducible .\n",
      "4\n",
      "45\n",
      "Start y end del regulador: [271, 275]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [20, 24]\n",
      "Regulado: nfnB\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( @Regulated$ , araT , and araU ) .\n",
      "53\n",
      "22\n",
      "Start y end del regulador: [159, 163]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [294, 298]\n",
      "Regulado: ytfQ\n",
      "Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) . Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved @TF$ regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( @Regulated$ , araT , and araU ) .\n",
      "53\n",
      "15\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [294, 298]\n",
      "Regulado: ytfQ\n",
      "This domain alone could activate transcription of araBAD up to 15 % as well as full - length AraC when alone or to the same level as fulllength AraC when fused to an unrelated dimerization domain ( 7 , 36 ) . This domain alone could activate transcription of @Regulated$ up to 15 % as well as full - length @TF$ when alone or to the same level as fulllength AraC when fused to an unrelated dimerization domain ( 7 , 36 ) .\n",
      "7\n",
      "18\n",
      "Start y end del regulador: [93, 97]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [50, 56]\n",
      "Regulado: araBAD\n",
      "This domain alone could activate transcription of araBAD up to 15 % as well as full - length AraC when alone or to the same level as fulllength AraC when fused to an unrelated dimerization domain ( 7 , 36 ) . This domain alone could activate transcription of @Regulated$ up to 15 % as well as full - length AraC when alone or to the same level as fulllength @TF$ when fused to an unrelated dimerization domain ( 7 , 36 ) .\n",
      "7\n",
      "28\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [50, 56]\n",
      "Regulado: araBAD\n",
      "We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter . We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for @TF$ - mediated transcription activation at the @Regulated$ promoter .\n",
      "27\n",
      "20\n",
      "Start y end del regulador: [102, 106]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [150, 155]\n",
      "Regulado: rhaSR\n",
      "We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter . We identified amino acid residues in the C - terminal domain of 70 and in @TF$ that are important for RhaR - mediated transcription activation at the @Regulated$ promoter .\n",
      "27\n",
      "15\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [150, 155]\n",
      "Regulado: rhaSR\n",
      "It should be noted ( lanes 1 - 4 ) that although plasmid pRSE562 directs the synthesis of the MetR protein there is still a large stimulation of metE expression by added MetR protein . It should be noted ( lanes 1 - 4 ) that although plasmid pRSE562 directs the synthesis of the MetR protein there is still a large stimulation of @Regulated$ expression by added @TF$ protein .\n",
      "28\n",
      "32\n",
      "Start y end del regulador: [170, 174]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: metE\n",
      "It should be noted ( lanes 1 - 4 ) that although plasmid pRSE562 directs the synthesis of the MetR protein there is still a large stimulation of metE expression by added MetR protein . It should be noted ( lanes 1 - 4 ) that although plasmid pRSE562 directs the synthesis of the @TF$ protein there is still a large stimulation of @Regulated$ expression by added MetR protein .\n",
      "28\n",
      "19\n",
      "Start y end del regulador: [94, 98]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: metE\n",
      "As shown previously ( 16 ) , IciA protein activated transcription from the dnaA promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) . As shown previously ( 16 ) , @TF$ protein activated transcription from the @Regulated$ promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: dnaA\n",
      "As shown previously ( 16 ) , IciA protein activated transcription from the dnaA promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) . As shown previously ( 16 ) , IciA protein activated transcription from the @Regulated$ promoter 1P in the 488 bp EcoRI / HindIII fragment ( + @TF$ I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .\n",
      "13\n",
      "26\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: dnaA\n",
      "As shown previously ( 16 ) , IciA protein activated transcription from the dnaA promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) . As shown previously ( 16 ) , IciA protein activated transcription from the @Regulated$ promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + @TF$ II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .\n",
      "13\n",
      "30\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: dnaA\n",
      "From these results , we conclude that the IciA I site is essential and alone sufficient for the activation of transcription from the dnaA promoter 1P by IciA protein . From these results , we conclude that the IciA I site is essential and alone sufficient for the activation of transcription from the @Regulated$ promoter 1P by @TF$ protein .\n",
      "23\n",
      "27\n",
      "Start y end del regulador: [153, 157]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: dnaA\n",
      "From these results , we conclude that the IciA I site is essential and alone sufficient for the activation of transcription from the dnaA promoter 1P by IciA protein . From these results , we conclude that the @TF$ I site is essential and alone sufficient for the activation of transcription from the @Regulated$ promoter 1P by IciA protein .\n",
      "23\n",
      "8\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: dnaA\n",
      "As earlier , with ArgP alone added , all radioactivity was shifted to a sharp band of retarded mobility , indicative of formation of a binary complex that ArgP - mediated regulation of E . coli argO Figure 2 . As earlier , with ArgP alone added , all radioactivity was shifted to a sharp band of retarded mobility , indicative of formation of a binary complex that @TF$ - mediated regulation of E . coli @Regulated$ Figure 2 .\n",
      "36\n",
      "28\n",
      "Start y end del regulador: [155, 159]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [194, 198]\n",
      "Regulado: argO\n",
      "As earlier , with ArgP alone added , all radioactivity was shifted to a sharp band of retarded mobility , indicative of formation of a binary complex that ArgP - mediated regulation of E . coli argO Figure 2 . As earlier , with @TF$ alone added , all radioactivity was shifted to a sharp band of retarded mobility , indicative of formation of a binary complex that ArgP - mediated regulation of E . coli @Regulated$ Figure 2 .\n",
      "36\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [194, 198]\n",
      "Regulado: argO\n",
      "Thus , the melR promoter is not efficiently repressed by MelR , and MelR is over - expressed . Thus , the @Regulated$ promoter is not efficiently repressed by MelR , and @TF$ is over - expressed .\n",
      "3\n",
      "13\n",
      "Start y end del regulador: [68, 72]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: melR\n",
      "Thus , the melR promoter is not efficiently repressed by MelR , and MelR is over - expressed . Thus , the @Regulated$ promoter is not efficiently repressed by @TF$ , and MelR is over - expressed .\n",
      "3\n",
      "10\n",
      "Start y end del regulador: [57, 61]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: melR\n",
      "When reactions were carried out in the presence of RNAP alone as compared with 200 nM purified His 6 - MarA , we detected MarA upregulation of nfnB transcription ( Fig . 4 ) . When reactions were carried out in the presence of RNAP alone as compared with 200 nM purified His 6 - MarA , we detected @TF$ upregulation of @Regulated$ transcription ( Fig . 4 ) .\n",
      "27\n",
      "24\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [143, 147]\n",
      "Regulado: nfnB\n",
      "When reactions were carried out in the presence of RNAP alone as compared with 200 nM purified His 6 - MarA , we detected MarA upregulation of nfnB transcription ( Fig . 4 ) . When reactions were carried out in the presence of RNAP alone as compared with 200 nM purified His 6 - @TF$ , we detected MarA upregulation of @Regulated$ transcription ( Fig . 4 ) .\n",
      "27\n",
      "20\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [143, 147]\n",
      "Regulado: nfnB\n",
      "In our previous work with the tri - alanine scanning library , we also determined that several members of the library reduced SoxS - dependent activation of transcription from the class I fpr promoter , as expected , because the SoxS binding site of class I promoters lies upstream of the 35 hexamer , which is therefore exposed and available for binding by 70 R4 . In our previous work with the tri - alanine scanning library , we also determined that several members of the library reduced @TF$ - dependent activation of transcription from the class I @Regulated$ promoter , as expected , because the SoxS binding site of class I promoters lies upstream of the 35 hexamer , which is therefore exposed and available for binding by 70 R4 .\n",
      "32\n",
      "22\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [188, 191]\n",
      "Regulado: fpr\n",
      "In our previous work with the tri - alanine scanning library , we also determined that several members of the library reduced SoxS - dependent activation of transcription from the class I fpr promoter , as expected , because the SoxS binding site of class I promoters lies upstream of the 35 hexamer , which is therefore exposed and available for binding by 70 R4 . In our previous work with the tri - alanine scanning library , we also determined that several members of the library reduced SoxS - dependent activation of transcription from the class I @Regulated$ promoter , as expected , because the @TF$ binding site of class I promoters lies upstream of the 35 hexamer , which is therefore exposed and available for binding by 70 R4 .\n",
      "32\n",
      "40\n",
      "Start y end del regulador: [229, 233]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [188, 191]\n",
      "Regulado: fpr\n",
      "These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA . These results show that the decrease in purA and @Regulated$ expression was caused , directly or indirectly , by transiently synthesized @TF$ rather than by a stress reaction due to constitutively overabundant MarA .\n",
      "9\n",
      "21\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: hdeA\n",
      "These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA . These results show that the decrease in purA and @Regulated$ expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant @TF$ .\n",
      "9\n",
      "32\n",
      "Start y end del regulador: [203, 207]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: hdeA\n",
      "These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA . These results show that the decrease in @Regulated$ and hdeA expression was caused , directly or indirectly , by transiently synthesized @TF$ rather than by a stress reaction due to constitutively overabundant MarA .\n",
      "7\n",
      "21\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: purA\n",
      "These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA . These results show that the decrease in @Regulated$ and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant @TF$ .\n",
      "7\n",
      "32\n",
      "Start y end del regulador: [203, 207]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: purA\n",
      "From this , we conclude that the MelR - binding Site R plays an essential role in MelR - dependent repression of the melR promoter . From this , we conclude that the MelR - binding Site R plays an essential role in @TF$ - dependent repression of the @Regulated$ promoter .\n",
      "23\n",
      "17\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: melR\n",
      "From this , we conclude that the MelR - binding Site R plays an essential role in MelR - dependent repression of the melR promoter . From this , we conclude that the @TF$ - binding Site R plays an essential role in MelR - dependent repression of the @Regulated$ promoter .\n",
      "23\n",
      "7\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: melR\n",
      "The promoter for the MarA - activated gene nfnB ( 20 ) , also mixed with gnd DNA , was tested in parallel as a positive control for MarA activity . The promoter for the @TF$ - activated gene @Regulated$ ( 20 ) , also mixed with gnd DNA , was tested in parallel as a positive control for MarA activity .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [21, 25]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: nfnB\n",
      "The promoter for the MarA - activated gene nfnB ( 20 ) , also mixed with gnd DNA , was tested in parallel as a positive control for MarA activity . The promoter for the MarA - activated gene @Regulated$ ( 20 ) , also mixed with gnd DNA , was tested in parallel as a positive control for @TF$ activity .\n",
      "8\n",
      "28\n",
      "Start y end del regulador: [132, 136]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: nfnB\n",
      "This result implies that the binding of two dimers of IciA protein to the IciA I site is required for the activation of RNA polymerase binding to the dnaA promoter 1P . This result implies that the binding of two dimers of IciA protein to the @TF$ I site is required for the activation of RNA polymerase binding to the @Regulated$ promoter 1P .\n",
      "28\n",
      "14\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [150, 154]\n",
      "Regulado: dnaA\n",
      "This result implies that the binding of two dimers of IciA protein to the IciA I site is required for the activation of RNA polymerase binding to the dnaA promoter 1P . This result implies that the binding of two dimers of @TF$ protein to the IciA I site is required for the activation of RNA polymerase binding to the @Regulated$ promoter 1P .\n",
      "28\n",
      "10\n",
      "Start y end del regulador: [54, 58]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [150, 154]\n",
      "Regulado: dnaA\n",
      "We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of dnaA promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site . We demonstrate in this paper that the mechanism of stimulation of transcription by @TF$ protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of @Regulated$ promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site .\n",
      "49\n",
      "13\n",
      "Start y end del regulador: [83, 87]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: dnaA\n",
      "We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of dnaA promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site . We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of @TF$ protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of @Regulated$ promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site .\n",
      "49\n",
      "27\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: dnaA\n",
      "We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of dnaA promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site . We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the @TF$ I site , which is located â€“ 50 to â€“ 63 from the transcription start site of @Regulated$ promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site .\n",
      "49\n",
      "31\n",
      "Start y end del regulador: [165, 169]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: dnaA\n",
      "We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of dnaA promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site . We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of @Regulated$ promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two @TF$ protein dimers on the IciA I site .\n",
      "49\n",
      "68\n",
      "Start y end del regulador: [345, 349]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: dnaA\n",
      "We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of dnaA promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the IciA I site . We demonstrate in this paper that the mechanism of stimulation of transcription by IciA protein may include : ( i ) the binding of two dimers of IciA protein to the IciA I site , which is located â€“ 50 to â€“ 63 from the transcription start site of @Regulated$ promoter 1P ; and ( ii ) the stimulation of RNA polymerase binding to the promoter by two IciA protein dimers on the @TF$ I site .\n",
      "49\n",
      "73\n",
      "Start y end del regulador: [372, 376]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: dnaA\n",
      "This complex binding pattern and its Lrp concentration - dependent profile strongly suggest the cooperative binding of Lrp dimers to multiple binding sites in the argO control region and the formation of complexes with a different stoichiometry . This complex binding pattern and its Lrp concentration - dependent profile strongly suggest the cooperative binding of @TF$ dimers to multiple binding sites in the @Regulated$ control region and the formation of complexes with a different stoichiometry .\n",
      "25\n",
      "17\n",
      "Start y end del regulador: [119, 122]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [163, 167]\n",
      "Regulado: argO\n",
      "This complex binding pattern and its Lrp concentration - dependent profile strongly suggest the cooperative binding of Lrp dimers to multiple binding sites in the argO control region and the formation of complexes with a different stoichiometry . This complex binding pattern and its @TF$ concentration - dependent profile strongly suggest the cooperative binding of Lrp dimers to multiple binding sites in the @Regulated$ control region and the formation of complexes with a different stoichiometry .\n",
      "25\n",
      "6\n",
      "Start y end del regulador: [37, 40]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [163, 167]\n",
      "Regulado: argO\n",
      "We believe that the low - affinity CAP site located between - 130 and - 140 probably does not constitute the site for CAP - stimulation of - the araC promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other CAP sites ( 16 , 17 ) , but also be - A A FIG . 5 . We believe that the low - affinity CAP site located between - 130 and - 140 probably does not constitute the site for @TF$ - stimulation of - the @Regulated$ promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other CAP sites ( 16 , 17 ) , but also be - A A FIG . 5 .\n",
      "29\n",
      "23\n",
      "Start y end del regulador: [118, 121]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: araC\n",
      "We believe that the low - affinity CAP site located between - 130 and - 140 probably does not constitute the site for CAP - stimulation of - the araC promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other CAP sites ( 16 , 17 ) , but also be - A A FIG . 5 . We believe that the low - affinity @TF$ site located between - 130 and - 140 probably does not constitute the site for CAP - stimulation of - the @Regulated$ promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other CAP sites ( 16 , 17 ) , but also be - A A FIG . 5 .\n",
      "29\n",
      "7\n",
      "Start y end del regulador: [35, 38]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: araC\n",
      "We believe that the low - affinity CAP site located between - 130 and - 140 probably does not constitute the site for CAP - stimulation of - the araC promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other CAP sites ( 16 , 17 ) , but also be - A A FIG . 5 . We believe that the low - affinity CAP site located between - 130 and - 140 probably does not constitute the site for CAP - stimulation of - the @Regulated$ promoter , not only because its position ( in relation to the polymerase binding site and transcriptional start point ) is at variance with other @TF$ sites ( 16 , 17 ) , but also be - A A FIG . 5 .\n",
      "29\n",
      "55\n",
      "Start y end del regulador: [296, 299]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: araC\n",
      "In contrast , and in full agreement with the previous results of Wade et al . ( 17 ) , the longer deletion in the TB23 fragment , that removes MelR binding site 2 , results in a sharp reduction in the repression of the melR promoter by MelR . In contrast , and in full agreement with the previous results of Wade et al . ( 17 ) , the longer deletion in the TB23 fragment , that removes MelR binding site 2 , results in a sharp reduction in the repression of the @Regulated$ promoter by @TF$ .\n",
      "45\n",
      "48\n",
      "Start y end del regulador: [236, 240]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: melR\n",
      "In contrast , and in full agreement with the previous results of Wade et al . ( 17 ) , the longer deletion in the TB23 fragment , that removes MelR binding site 2 , results in a sharp reduction in the repression of the melR promoter by MelR . In contrast , and in full agreement with the previous results of Wade et al . ( 17 ) , the longer deletion in the TB23 fragment , that removes @TF$ binding site 2 , results in a sharp reduction in the repression of the @Regulated$ promoter by MelR .\n",
      "45\n",
      "30\n",
      "Start y end del regulador: [143, 147]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: melR\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that @TF$ activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "12\n",
      "10\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [53, 56]\n",
      "Regulado: mar\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that SoxS activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - @TF$ or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "12\n",
      "41\n",
      "Start y end del regulador: [220, 224]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [53, 56]\n",
      "Regulado: mar\n",
      "By analogy with Crp - dependent promoters ( 3 ) and synthetic Fnr - dependent promoters ( 29 , 30 ) , we hypothesized that the mechanism by which napF expression is activated by Fnr is distinct from the mechanism involved in the activation of most4194 DARWIN ET AL . J . By analogy with Crp - dependent promoters ( 3 ) and synthetic Fnr - dependent promoters ( 29 , 30 ) , we hypothesized that the mechanism by which @Regulated$ expression is activated by @TF$ is distinct from the mechanism involved in the activation of most4194 DARWIN ET AL . J .\n",
      "29\n",
      "34\n",
      "Start y end del regulador: [178, 181]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [146, 150]\n",
      "Regulado: napF\n",
      "By analogy with Crp - dependent promoters ( 3 ) and synthetic Fnr - dependent promoters ( 29 , 30 ) , we hypothesized that the mechanism by which napF expression is activated by Fnr is distinct from the mechanism involved in the activation of most4194 DARWIN ET AL . J . By analogy with Crp - dependent promoters ( 3 ) and synthetic @TF$ - dependent promoters ( 29 , 30 ) , we hypothesized that the mechanism by which @Regulated$ expression is activated by Fnr is distinct from the mechanism involved in the activation of most4194 DARWIN ET AL . J .\n",
      "29\n",
      "12\n",
      "Start y end del regulador: [62, 65]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [146, 150]\n",
      "Regulado: napF\n",
      "To further investigate ArgP - mediated activation of dapB transcription , we purified His - tagged ArgP protein ( 17 ) and used it to perform EMSE . To further investigate @TF$ - mediated activation of @Regulated$ transcription , we purified His - tagged ArgP protein ( 17 ) and used it to perform EMSE .\n",
      "8\n",
      "3\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: dapB\n",
      "To further investigate ArgP - mediated activation of dapB transcription , we purified His - tagged ArgP protein ( 17 ) and used it to perform EMSE . To further investigate ArgP - mediated activation of @Regulated$ transcription , we purified His - tagged @TF$ protein ( 17 ) and used it to perform EMSE .\n",
      "8\n",
      "16\n",
      "Start y end del regulador: [99, 103]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: dapB\n",
      "Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with Fis binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli nirB promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) . Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with @TF$ binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli @Regulated$ promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .\n",
      "28\n",
      "11\n",
      "Start y end del regulador: [69, 72]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [161, 165]\n",
      "Regulado: nirB\n",
      "Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with Fis binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli nirB promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) . Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with Fis binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli @Regulated$ promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , @TF$ and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .\n",
      "28\n",
      "45\n",
      "Start y end del regulador: [273, 276]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [161, 165]\n",
      "Regulado: nirB\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( @Regulated$ , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "19\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [115, 118]\n",
      "Regulado: asd\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( @Regulated$ , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "19\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [115, 118]\n",
      "Regulado: asd\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( @Regulated$ , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "19\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [115, 118]\n",
      "Regulado: asd\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "21\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: dapB\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "21\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: dapB\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "21\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: dapB\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , @Regulated$ , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "23\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: dapD\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , @Regulated$ , gdhA , lysA , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "23\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: dapD\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , @Regulated$ , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "23\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: dapD\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , @Regulated$ , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "25\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: gdhA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , @Regulated$ , lysA , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "25\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: gdhA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , @Regulated$ , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "25\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: gdhA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , @Regulated$ , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "27\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: lysA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , @Regulated$ , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "27\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: lysA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , @Regulated$ , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "27\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: lysA\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , @Regulated$ , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "29\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [149, 153]\n",
      "Regulado: lysC\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , @Regulated$ , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "29\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [149, 153]\n",
      "Regulado: lysC\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , @Regulated$ , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "29\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [149, 153]\n",
      "Regulado: lysC\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and @Regulated$ ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "32\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: lysP\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and @Regulated$ ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "32\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: lysP\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and @Regulated$ ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "32\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: lysP\n",
      "gntR gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by GntR negatively regulated by GntR .  @Regulated$ gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by GntR negatively regulated by @TF$ .\n",
      "0\n",
      "26\n",
      "Start y end del regulador: [136, 140]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: gntR\n",
      "gntR gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by GntR negatively regulated by GntR .  @Regulated$ gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by @TF$ negatively regulated by GntR .\n",
      "0\n",
      "22\n",
      "Start y end del regulador: [107, 111]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: gntR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for @TF$ - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "30\n",
      "24\n",
      "Start y end del regulador: [107, 111]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of @TF$ bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "30\n",
      "2\n",
      "Start y end del regulador: [15, 19]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that @TF$ - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "30\n",
      "16\n",
      "Start y end del regulador: [68, 72]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A @TF$ RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "30\n",
      "32\n",
      "Start y end del regulador: [149, 153]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 @TF$ + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "30\n",
      "41\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 @TF$ + MelR TB22 Figure 2 .\n",
      "30\n",
      "52\n",
      "Start y end del regulador: [232, 236]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 . Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + @TF$ TB22 Figure 2 .\n",
      "30\n",
      "54\n",
      "Start y end del regulador: [239, 243]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "To investigate how IHF activates fimB expression further , the IHF binding site mutation described above was combined with those in O NC1 , O NC2 and O NR alone , and O NC1 O NR in combination , and the expression of the FimB â€“ LacZ chimera was measured ( Fig . 6 ) . To investigate how @TF$ activates @Regulated$ expression further , the IHF binding site mutation described above was combined with those in O NC1 , O NC2 and O NR alone , and O NC1 O NR in combination , and the expression of the FimB â€“ LacZ chimera was measured ( Fig . 6 ) .\n",
      "5\n",
      "3\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [33, 37]\n",
      "Regulado: fimB\n",
      "To investigate how IHF activates fimB expression further , the IHF binding site mutation described above was combined with those in O NC1 , O NC2 and O NR alone , and O NC1 O NR in combination , and the expression of the FimB â€“ LacZ chimera was measured ( Fig . 6 ) . To investigate how IHF activates @Regulated$ expression further , the @TF$ binding site mutation described above was combined with those in O NC1 , O NC2 and O NR alone , and O NC1 O NR in combination , and the expression of the FimB â€“ LacZ chimera was measured ( Fig . 6 ) .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [33, 37]\n",
      "Regulado: fimB\n",
      "For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by Rob , the presence of Rob increases the transcription of the chromosomal marRAB promoter and therefore only increases the repression of the mutant fusions . For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by Rob , the presence of @TF$ increases the transcription of the chromosomal @Regulated$ promoter and therefore only increases the repression of the mutant fusions .\n",
      "30\n",
      "23\n",
      "Start y end del regulador: [115, 118]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [166, 172]\n",
      "Regulado: marRAB\n",
      "For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by Rob , the presence of Rob increases the transcription of the chromosomal marRAB promoter and therefore only increases the repression of the mutant fusions . For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by @TF$ , the presence of Rob increases the transcription of the chromosomal @Regulated$ promoter and therefore only increases the repression of the mutant fusions .\n",
      "30\n",
      "18\n",
      "Start y end del regulador: [93, 96]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [166, 172]\n",
      "Regulado: marRAB\n",
      "The binding of two dimers of IciA protein to the IciA I site is responsible for the activation of dnaA promoter 1P . The binding of two dimers of IciA protein to the @TF$ I site is responsible for the activation of @Regulated$ promoter 1P .\n",
      "19\n",
      "10\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [98, 102]\n",
      "Regulado: dnaA\n",
      "The binding of two dimers of IciA protein to the IciA I site is responsible for the activation of dnaA promoter 1P . The binding of two dimers of @TF$ protein to the IciA I site is responsible for the activation of @Regulated$ promoter 1P .\n",
      "19\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [98, 102]\n",
      "Regulado: dnaA\n",
      "Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the dapB promoter region . Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than @TF$ is able to bind significantly to the @Regulated$ promoter region .\n",
      "70\n",
      "62\n",
      "Start y end del regulador: [321, 325]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [363, 367]\n",
      "Regulado: dapB\n",
      "Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the dapB promoter region . Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - @TF$ protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the @Regulated$ promoter region .\n",
      "70\n",
      "31\n",
      "Start y end del regulador: [173, 177]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [363, 367]\n",
      "Regulado: dapB\n",
      "Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the dapB promoter region . Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of @TF$ gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the @Regulated$ promoter region .\n",
      "70\n",
      "39\n",
      "Start y end del regulador: [221, 225]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [363, 367]\n",
      "Regulado: dapB\n",
      "Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , zwf by SoxS or MarA but not by Rob ) . Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , @Regulated$ by SoxS or @TF$ but not by Rob ) .\n",
      "41\n",
      "45\n",
      "Start y end del regulador: [241, 245]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [226, 229]\n",
      "Regulado: zwf\n",
      "Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , zwf by SoxS or MarA but not by Rob ) . Despite overlapping regulation by SoxS , @TF$ , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , @Regulated$ by SoxS or MarA but not by Rob ) .\n",
      "41\n",
      "6\n",
      "Start y end del regulador: [41, 45]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [226, 229]\n",
      "Regulado: zwf\n",
      "Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , zwf by SoxS or MarA but not by Rob ) . Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , @Regulated$ by @TF$ or MarA but not by Rob ) .\n",
      "41\n",
      "43\n",
      "Start y end del regulador: [233, 237]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [226, 229]\n",
      "Regulado: zwf\n",
      "Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , zwf by SoxS or MarA but not by Rob ) . Despite overlapping regulation by @TF$ , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e . g . , @Regulated$ by SoxS or MarA but not by Rob ) .\n",
      "41\n",
      "4\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [226, 229]\n",
      "Regulado: zwf\n",
      "Taken together , our results argue that residues Thr158 , Pro160 , Gln164 and Lys166 of CRP participate in transcription activation at the melAB promoter : these residues identify a surface of CRP that overlaps with AR1 ( Figure 4 ) . Taken together , our results argue that residues Thr158 , Pro160 , Gln164 and Lys166 of @TF$ participate in transcription activation at the @Regulated$ promoter : these residues identify a surface of CRP that overlaps with AR1 ( Figure 4 ) .\n",
      "23\n",
      "16\n",
      "Start y end del regulador: [88, 91]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [139, 144]\n",
      "Regulado: melAB\n",
      "Taken together , our results argue that residues Thr158 , Pro160 , Gln164 and Lys166 of CRP participate in transcription activation at the melAB promoter : these residues identify a surface of CRP that overlaps with AR1 ( Figure 4 ) . Taken together , our results argue that residues Thr158 , Pro160 , Gln164 and Lys166 of CRP participate in transcription activation at the @Regulated$ promoter : these residues identify a surface of @TF$ that overlaps with AR1 ( Figure 4 ) .\n",
      "23\n",
      "32\n",
      "Start y end del regulador: [193, 196]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [139, 144]\n",
      "Regulado: melAB\n",
      "We previously demonstrated that in vitro transcriptional activation by MarA at the zwf promoter requires interaction with the carboxy - terminal domain of the a subunit of RNA polymerase ( 21 ) and inferred that MarA activates the mar promoter by enhancing the binding of RNA polymerase ( 27 ) . We previously demonstrated that in vitro transcriptional activation by @TF$ at the @Regulated$ promoter requires interaction with the carboxy - terminal domain of the a subunit of RNA polymerase ( 21 ) and inferred that MarA activates the mar promoter by enhancing the binding of RNA polymerase ( 27 ) .\n",
      "12\n",
      "9\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [83, 86]\n",
      "Regulado: zwf\n",
      "We previously demonstrated that in vitro transcriptional activation by MarA at the zwf promoter requires interaction with the carboxy - terminal domain of the a subunit of RNA polymerase ( 21 ) and inferred that MarA activates the mar promoter by enhancing the binding of RNA polymerase ( 27 ) . We previously demonstrated that in vitro transcriptional activation by MarA at the @Regulated$ promoter requires interaction with the carboxy - terminal domain of the a subunit of RNA polymerase ( 21 ) and inferred that @TF$ activates the mar promoter by enhancing the binding of RNA polymerase ( 27 ) .\n",
      "12\n",
      "35\n",
      "Start y end del regulador: [212, 216]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [83, 86]\n",
      "Regulado: zwf\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the dnaA promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase . DISCUSSION Physically and functionally heterogeneous complexes containing @TF$ protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .\n",
      "12\n",
      "7\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the dnaA promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase . DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the mode of binding of @TF$ protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .\n",
      "12\n",
      "27\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the dnaA promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase . DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single @TF$ box and how this binding inhibits transcription by RNA polymerase .\n",
      "12\n",
      "34\n",
      "Start y end del regulador: [224, 228]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type melAB promoter . This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 @TF$ - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type @Regulated$ promoter .\n",
      "41\n",
      "17\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [243, 248]\n",
      "Regulado: melAB\n",
      "This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type melAB promoter . This shows that we had failed to identify @TF$ mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type @Regulated$ promoter .\n",
      "41\n",
      "8\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [243, 248]\n",
      "Regulado: melAB\n",
      "The fact that the same level of CRP - dependent transcription activation is seen in vitro with both RNAP preparations argues that CRP activates transcription at the melAB promoter independently of aCTD . The fact that the same level of CRP - dependent transcription activation is seen in vitro with both RNAP preparations argues that @TF$ activates transcription at the @Regulated$ promoter independently of aCTD .\n",
      "27\n",
      "22\n",
      "Start y end del regulador: [130, 133]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [165, 170]\n",
      "Regulado: melAB\n",
      "The fact that the same level of CRP - dependent transcription activation is seen in vitro with both RNAP preparations argues that CRP activates transcription at the melAB promoter independently of aCTD . The fact that the same level of @TF$ - dependent transcription activation is seen in vitro with both RNAP preparations argues that CRP activates transcription at the @Regulated$ promoter independently of aCTD .\n",
      "27\n",
      "7\n",
      "Start y end del regulador: [32, 35]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [165, 170]\n",
      "Regulado: melAB\n",
      "The finding of only one MetR binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of metE as well as the autoregulation of metR ( 20 , 36 ) . The finding of only one MetR binding site suggests that the binding of @TF$ to this region may be responsible for both the activation of expression of @Regulated$ as well as the autoregulation of metR ( 20 , 36 ) .\n",
      "27\n",
      "13\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [151, 155]\n",
      "Regulado: metE\n",
      "The finding of only one MetR binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of metE as well as the autoregulation of metR ( 20 , 36 ) . The finding of only one @TF$ binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of @Regulated$ as well as the autoregulation of metR ( 20 , 36 ) .\n",
      "27\n",
      "5\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [151, 155]\n",
      "Regulado: metE\n",
      "The finding of only one MetR binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of metE as well as the autoregulation of metR ( 20 , 36 ) . The finding of only one MetR binding site suggests that the binding of @TF$ to this region may be responsible for both the activation of expression of metE as well as the autoregulation of @Regulated$ ( 20 , 36 ) .\n",
      "34\n",
      "13\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: metR\n",
      "The finding of only one MetR binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of metE as well as the autoregulation of metR ( 20 , 36 ) . The finding of only one @TF$ binding site suggests that the binding of MetR to this region may be responsible for both the activation of expression of metE as well as the autoregulation of @Regulated$ ( 20 , 36 ) .\n",
      "34\n",
      "5\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: metR\n",
      "However , this idea was contradicted when a single mutation at position 16 ( C to G ) of RE 2 , which has been shown to establish direct contacts with the second of two HTH motifs of MarA ( Rhee et al . , 1998 ; Dangi et al . , 2001 ) , eliminated MarA - mediated activation of the nfnB gene . However , this idea was contradicted when a single mutation at position 16 ( C to G ) of RE 2 , which has been shown to establish direct contacts with the second of two HTH motifs of MarA ( Rhee et al . , 1998 ; Dangi et al . , 2001 ) , eliminated @TF$ - mediated activation of the @Regulated$ gene .\n",
      "62\n",
      "56\n",
      "Start y end del regulador: [248, 252]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [282, 286]\n",
      "Regulado: nfnB\n",
      "However , this idea was contradicted when a single mutation at position 16 ( C to G ) of RE 2 , which has been shown to establish direct contacts with the second of two HTH motifs of MarA ( Rhee et al . , 1998 ; Dangi et al . , 2001 ) , eliminated MarA - mediated activation of the nfnB gene . However , this idea was contradicted when a single mutation at position 16 ( C to G ) of RE 2 , which has been shown to establish direct contacts with the second of two HTH motifs of @TF$ ( Rhee et al . , 1998 ; Dangi et al . , 2001 ) , eliminated MarA - mediated activation of the @Regulated$ gene .\n",
      "62\n",
      "38\n",
      "Start y end del regulador: [183, 187]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [282, 286]\n",
      "Regulado: nfnB\n",
      "Finally , either NarL or NarP was able to activate nrfA - lacZ expression independently of the other , while only NarL could repress expression . Finally , either @TF$ or NarP was able to activate @Regulated$ - lacZ expression independently of the other , while only NarL could repress expression .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: nrfA\n",
      "Finally , either NarL or NarP was able to activate nrfA - lacZ expression independently of the other , while only NarL could repress expression . Finally , either NarL or NarP was able to activate @Regulated$ - lacZ expression independently of the other , while only @TF$ could repress expression .\n",
      "10\n",
      "21\n",
      "Start y end del regulador: [114, 118]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: nrfA\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by @TF$ and are not responsive to NarP as observed here for @Regulated$ , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "23\n",
      "12\n",
      "Start y end del regulador: [79, 83]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [136, 140]\n",
      "Regulado: cydD\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for @Regulated$ , while nitrate induction of the fdnGHI operon is regulated predominantly by @TF$ but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "23\n",
      "36\n",
      "Start y end del regulador: [218, 222]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [136, 140]\n",
      "Regulado: cydD\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by @TF$ but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "30\n",
      "36\n",
      "Start y end del regulador: [218, 222]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: fdnGHI\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by @TF$ and are not responsive to NarP as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "30\n",
      "12\n",
      "Start y end del regulador: [79, 83]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: fdnGHI\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to @TF$ ( 33 , 39 ) .\n",
      "30\n",
      "44\n",
      "Start y end del regulador: [257, 261]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: fdnGHI\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to @TF$ as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "30\n",
      "18\n",
      "Start y end del regulador: [110, 114]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: fdnGHI\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes @Regulated$ are regulated exclusively by @TF$ and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "7\n",
      "12\n",
      "Start y end del regulador: [79, 83]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [42, 49]\n",
      "Regulado: narGHJI\n",
      "For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) . For example , the nitrate reductase genes @Regulated$ are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by @TF$ but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "7\n",
      "36\n",
      "Start y end del regulador: [218, 222]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [42, 49]\n",
      "Regulado: narGHJI\n",
      "A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the manX â€“ lacZ fusion , while a plasmid in which Mlc was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) . A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the @Regulated$ â€“ lacZ fusion , while a plasmid in which @TF$ was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .\n",
      "29\n",
      "39\n",
      "Start y end del regulador: [188, 191]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: manX\n",
      "A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the manX â€“ lacZ fusion , while a plasmid in which Mlc was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) . A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the @Regulated$ â€“ lacZ fusion , while a plasmid in which Mlc was under the control of the lac promoter , pTZ ( @TF$ ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .\n",
      "29\n",
      "51\n",
      "Start y end del regulador: [242, 245]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: manX\n",
      "We investigated whether DnaA protein and RNA polymerase can coexist at the dnaA promoter by gel - shift and footprinting analyses and whether a direct protein - protein interaction between DnaA protein and RNA polymerase is the mechanism of inhibition of transcription . We investigated whether @TF$ protein and RNA polymerase can coexist at the @Regulated$ promoter by gel - shift and footprinting analyses and whether a direct protein - protein interaction between DnaA protein and RNA polymerase is the mechanism of inhibition of transcription .\n",
      "12\n",
      "3\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: dnaA\n",
      "We investigated whether DnaA protein and RNA polymerase can coexist at the dnaA promoter by gel - shift and footprinting analyses and whether a direct protein - protein interaction between DnaA protein and RNA polymerase is the mechanism of inhibition of transcription . We investigated whether DnaA protein and RNA polymerase can coexist at the @Regulated$ promoter by gel - shift and footprinting analyses and whether a direct protein - protein interaction between @TF$ protein and RNA polymerase is the mechanism of inhibition of transcription .\n",
      "12\n",
      "30\n",
      "Start y end del regulador: [189, 193]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: dnaA\n",
      "In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent dmsA promoter in vitro ( data not shown ) . In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the @TF$ - dependent @Regulated$ promoter in vitro ( data not shown ) .\n",
      "27\n",
      "24\n",
      "Start y end del regulador: [130, 133]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [146, 150]\n",
      "Regulado: dmsA\n",
      "In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent dmsA promoter in vitro ( data not shown ) . In a control experiment , the activity of the @TF$ ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent @Regulated$ promoter in vitro ( data not shown ) .\n",
      "27\n",
      "9\n",
      "Start y end del regulador: [46, 49]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [146, 150]\n",
      "Regulado: dmsA\n",
      "Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( ArgP ) and employed an RNase protection assay to show that nrdA transcription is ArgP regulated in vivo . Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( ArgP ) and employed an RNase protection assay to show that @Regulated$ transcription is @TF$ regulated in vivo .\n",
      "30\n",
      "33\n",
      "Start y end del regulador: [166, 170]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [144, 148]\n",
      "Regulado: nrdA\n",
      "Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( ArgP ) and employed an RNase protection assay to show that nrdA transcription is ArgP regulated in vivo . Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( @TF$ ) and employed an RNase protection assay to show that @Regulated$ transcription is ArgP regulated in vivo .\n",
      "30\n",
      "19\n",
      "Start y end del regulador: [85, 89]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [144, 148]\n",
      "Regulado: nrdA\n",
      "These results suggest that the DNA region between 90 and 128 is important for activation at rhaSR and that putative CRP sites 2 , 3 , and / or 4 may be functional CRP sites . These results suggest that the DNA region between 90 and 128 is important for activation at @Regulated$ and that putative @TF$ sites 2 , 3 , and / or 4 may be functional CRP sites .\n",
      "16\n",
      "20\n",
      "Start y end del regulador: [116, 119]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [92, 97]\n",
      "Regulado: rhaSR\n",
      "These results suggest that the DNA region between 90 and 128 is important for activation at rhaSR and that putative CRP sites 2 , 3 , and / or 4 may be functional CRP sites . These results suggest that the DNA region between 90 and 128 is important for activation at @Regulated$ and that putative CRP sites 2 , 3 , and / or 4 may be functional @TF$ sites .\n",
      "16\n",
      "33\n",
      "Start y end del regulador: [163, 166]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [92, 97]\n",
      "Regulado: rhaSR\n",
      "The binding of two dimers of IciA protein to the IciA I site , which is located 50 to 63 nucleotides from the transcription start site of dnaA promoter 1P , is required to stimulate RNA polymerase binding to the 1P promoter . The binding of two dimers of IciA protein to the @TF$ I site , which is located 50 to 63 nucleotides from the transcription start site of @Regulated$ promoter 1P , is required to stimulate RNA polymerase binding to the 1P promoter .\n",
      "27\n",
      "10\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [138, 142]\n",
      "Regulado: dnaA\n",
      "The binding of two dimers of IciA protein to the IciA I site , which is located 50 to 63 nucleotides from the transcription start site of dnaA promoter 1P , is required to stimulate RNA polymerase binding to the 1P promoter . The binding of two dimers of @TF$ protein to the IciA I site , which is located 50 to 63 nucleotides from the transcription start site of @Regulated$ promoter 1P , is required to stimulate RNA polymerase binding to the 1P promoter .\n",
      "27\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [138, 142]\n",
      "Regulado: dnaA\n",
      "It is possible that NarP and Fnr may activate napF transcription synergistically whereas Fnr and NarL cannot , as discussed below . It is possible that NarP and @TF$ may activate @Regulated$ transcription synergistically whereas Fnr and NarL cannot , as discussed below .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [29, 32]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [46, 50]\n",
      "Regulado: napF\n",
      "It is possible that NarP and Fnr may activate napF transcription synergistically whereas Fnr and NarL cannot , as discussed below . It is possible that NarP and Fnr may activate @Regulated$ transcription synergistically whereas @TF$ and NarL cannot , as discussed below .\n",
      "9\n",
      "13\n",
      "Start y end del regulador: [89, 92]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [46, 50]\n",
      "Regulado: napF\n",
      "This suggests that CRP site 3 is responsible for at least part of the CRP activation of rhaSR expression . This suggests that CRP site 3 is responsible for at least part of the @TF$ activation of @Regulated$ expression .\n",
      "17\n",
      "14\n",
      "Start y end del regulador: [70, 73]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [88, 93]\n",
      "Regulado: rhaSR\n",
      "This suggests that CRP site 3 is responsible for at least part of the CRP activation of rhaSR expression . This suggests that @TF$ site 3 is responsible for at least part of the CRP activation of @Regulated$ expression .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [88, 93]\n",
      "Regulado: rhaSR\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known @TF$ - regulated genes , i . e . , araB , @Regulated$ , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "18\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [76, 80]\n",
      "Regulado: araA\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known AraC - regulated genes , i . e . , araB , @Regulated$ , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "18\n",
      "52\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [76, 80]\n",
      "Regulado: araA\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known @TF$ - regulated genes , i . e . , @Regulated$ , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "16\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: araB\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known AraC - regulated genes , i . e . , @Regulated$ , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "16\n",
      "52\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: araB\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known @TF$ - regulated genes , i . e . , araB , araA , @Regulated$ , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "20\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [83, 87]\n",
      "Regulado: araD\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known AraC - regulated genes , i . e . , araB , araA , @Regulated$ , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "20\n",
      "52\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [83, 87]\n",
      "Regulado: araD\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known @TF$ - regulated genes , i . e . , araB , araA , araD , @Regulated$ , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "22\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [90, 94]\n",
      "Regulado: araE\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , @Regulated$ , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "22\n",
      "52\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [90, 94]\n",
      "Regulado: araE\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known @TF$ - regulated genes , i . e . , araB , araA , araD , araE , @Regulated$ , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "24\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: araF\n",
      "As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip . As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , @Regulated$ , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "24\n",
      "52\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: araF\n",
      "Note that in a control experiment , the activity of the MBP - NarL protein was confirmed by its ability to activate transcription of the NarL - dependent fdnG promoter in vitro ( data not shown ) . Note that in a control experiment , the activity of the MBP - NarL protein was confirmed by its ability to activate transcription of the @TF$ - dependent @Regulated$ promoter in vitro ( data not shown ) .\n",
      "28\n",
      "25\n",
      "Start y end del regulador: [137, 141]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: fdnG\n",
      "Note that in a control experiment , the activity of the MBP - NarL protein was confirmed by its ability to activate transcription of the NarL - dependent fdnG promoter in vitro ( data not shown ) . Note that in a control experiment , the activity of the MBP - @TF$ protein was confirmed by its ability to activate transcription of the NarL - dependent @Regulated$ promoter in vitro ( data not shown ) .\n",
      "28\n",
      "13\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: fdnG\n",
      "Model for MelR - dependent repression of the melR promoter . In the absence of melibiose , MelR subunits occupy Site R and Site 2 forming a repression loop . Model for @TF$ - dependent repression of the @Regulated$ promoter . In the absence of melibiose , MelR subunits occupy Site R and Site 2 forming a repression loop .\n",
      "8\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: melR\n",
      "Model for MelR - dependent repression of the melR promoter . In the absence of melibiose , MelR subunits occupy Site R and Site 2 forming a repression loop . Model for MelR - dependent repression of the @Regulated$ promoter . In the absence of melibiose , @TF$ subunits occupy Site R and Site 2 forming a repression loop .\n",
      "8\n",
      "17\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: melR\n",
      "Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) . Repression by NagC and ChbR in vivo A plasmid overexpressing @TF$ produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .\n",
      "17\n",
      "10\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: chbB\n",
      "Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) . Repression by NagC and @TF$ in vivo A plasmid overexpressing ChbR produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing @TF$ produces less than twofold repression ( Table 5 ) .\n",
      "17\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: chbB\n",
      "Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) . Repression by @TF$ and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .\n",
      "17\n",
      "2\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: chbB\n",
      "Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) . Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing @TF$ produces less than twofold repression ( Table 5 ) .\n",
      "17\n",
      "37\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: chbB\n",
      "Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site . Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow @TF$ to to activate @Regulated$ by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "41\n",
      "37\n",
      "Start y end del regulador: [179, 183]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [199, 203]\n",
      "Regulado: glmU\n",
      "Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site . Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate @Regulated$ by itself ; this could be because @TF$ occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "41\n",
      "49\n",
      "Start y end del regulador: [238, 242]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [199, 203]\n",
      "Regulado: glmU\n",
      "However , the inactivation of DcuS ( IMW553 ) led to a decrease of citC - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and DcuS . However , the inactivation of @TF$ ( IMW553 ) led to a decrease of @Regulated$ - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and DcuS .\n",
      "14\n",
      "5\n",
      "Start y end del regulador: [30, 34]\n",
      "Regulador: DcuS\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: citC\n",
      "However , the inactivation of DcuS ( IMW553 ) led to a decrease of citC - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and DcuS . However , the inactivation of DcuS ( IMW553 ) led to a decrease of @Regulated$ - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and @TF$ .\n",
      "14\n",
      "33\n",
      "Start y end del regulador: [162, 166]\n",
      "Regulador: DcuS\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: citC\n",
      "Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ nagBE â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction . Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ @Regulated$ â€“ lacZ fusion , whose expression is repressed by @TF$ binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction .\n",
      "58\n",
      "68\n",
      "Start y end del regulador: [357, 361]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [300, 305]\n",
      "Regulado: nagBE\n",
      "Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ nagBE â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction . Binding of @TF$ and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ @Regulated$ â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction .\n",
      "58\n",
      "2\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [300, 305]\n",
      "Regulado: nagBE\n",
      "Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ nagBE â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction . Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than @TF$ , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ @Regulated$ â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction .\n",
      "58\n",
      "21\n",
      "Start y end del regulador: [111, 115]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [300, 305]\n",
      "Regulado: nagBE\n",
      "Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ nagBE â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction . Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of @TF$ in vivo . ThreenagE â€“ lacZ fusions were tested : the standard â€˜ loop - forming â€™ @Regulated$ â€“ lacZ fusion , whose expression is repressed by NagC binding co - operatively to the nagE and nagB operators ; and two single operator nagE â€“ lacZ fusions , missing the nagB promoter / operator , but carrying the same nagE â€“ lacZ junction .\n",
      "58\n",
      "40\n",
      "Start y end del regulador: [208, 212]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [300, 305]\n",
      "Regulado: nagBE\n",
      "The impact of DcuR on citC expression might be due to a transcriptional activation of citAB by DcuR , as discussed below . The impact of DcuR on citC expression might be due to a transcriptional activation of @Regulated$ by @TF$ , as discussed below .\n",
      "15\n",
      "17\n",
      "Start y end del regulador: [95, 99]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [86, 91]\n",
      "Regulado: citAB\n",
      "The impact of DcuR on citC expression might be due to a transcriptional activation of citAB by DcuR , as discussed below . The impact of @TF$ on citC expression might be due to a transcriptional activation of @Regulated$ by DcuR , as discussed below .\n",
      "15\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [86, 91]\n",
      "Regulado: citAB\n",
      "The results of this analysis confirm this assertion , and show additionally that NagC is also better able to We previously presented evidence suggesting that the D2 region , and hence NanR and NagC , could activate fimB expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) . The results of this analysis confirm this assertion , and show additionally that @TF$ is also better able to We previously presented evidence suggesting that the D2 region , and hence NanR and NagC , could activate @Regulated$ expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) .\n",
      "37\n",
      "13\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [215, 219]\n",
      "Regulado: fimB\n",
      "The results of this analysis confirm this assertion , and show additionally that NagC is also better able to We previously presented evidence suggesting that the D2 region , and hence NanR and NagC , could activate fimB expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) . The results of this analysis confirm this assertion , and show additionally that NagC is also better able to We previously presented evidence suggesting that the D2 region , and hence NanR and @TF$ , could activate @Regulated$ expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) .\n",
      "37\n",
      "33\n",
      "Start y end del regulador: [193, 197]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [215, 219]\n",
      "Regulado: fimB\n",
      "Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control . Effect of excess @TF$ and NagC on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: chbB\n",
      "Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control . Effect of excess ChbR and NagC on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( @TF$ ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "10\n",
      "47\n",
      "Start y end del regulador: [174, 178]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: chbB\n",
      "Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control . Effect of excess ChbR and @TF$ on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: chbB\n",
      "Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control . Effect of excess ChbR and NagC on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( @TF$ ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "10\n",
      "21\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: chbB\n",
      "Measurements of b - galactosidase activities in the resulting transformants indicated that none of the single base changes in MelR binding site 2 significantly affected MelR - dependent activation of the JK22 melAB promoter ( data not shown ) . Measurements of b - galactosidase activities in the resulting transformants indicated that none of the single base changes in MelR binding site 2 significantly affected @TF$ - dependent activation of the JK22 @Regulated$ promoter ( data not shown ) .\n",
      "32\n",
      "25\n",
      "Start y end del regulador: [169, 173]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [209, 214]\n",
      "Regulado: melAB\n",
      "Measurements of b - galactosidase activities in the resulting transformants indicated that none of the single base changes in MelR binding site 2 significantly affected MelR - dependent activation of the JK22 melAB promoter ( data not shown ) . Measurements of b - galactosidase activities in the resulting transformants indicated that none of the single base changes in @TF$ binding site 2 significantly affected MelR - dependent activation of the JK22 @Regulated$ promoter ( data not shown ) .\n",
      "32\n",
      "19\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [209, 214]\n",
      "Regulado: melAB\n",
      "This indicates that ( i ) the 70 NarL heptamer site is essential for NarL to function as an activator of nrfA expression and that ( ii ) other sites are used for NarL to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) . This indicates that ( i ) the 70 NarL heptamer site is essential for @TF$ to function as an activator of @Regulated$ expression and that ( ii ) other sites are used for NarL to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) .\n",
      "21\n",
      "14\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: nrfA\n",
      "This indicates that ( i ) the 70 NarL heptamer site is essential for NarL to function as an activator of nrfA expression and that ( ii ) other sites are used for NarL to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) . This indicates that ( i ) the 70 @TF$ heptamer site is essential for NarL to function as an activator of @Regulated$ expression and that ( ii ) other sites are used for NarL to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) .\n",
      "21\n",
      "8\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: nrfA\n",
      "This indicates that ( i ) the 70 NarL heptamer site is essential for NarL to function as an activator of nrfA expression and that ( ii ) other sites are used for NarL to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) . This indicates that ( i ) the 70 NarL heptamer site is essential for NarL to function as an activator of @Regulated$ expression and that ( ii ) other sites are used for @TF$ to repress expression independently of the 70 site , in contrast to prior interpretations ( see Discussion ) .\n",
      "21\n",
      "33\n",
      "Start y end del regulador: [162, 166]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: nrfA\n",
      "Thus , the melR promoter is not efficiently repressed by MelR and MelR is overexpressed . Thus , the @Regulated$ promoter is not efficiently repressed by MelR and @TF$ is overexpressed .\n",
      "3\n",
      "12\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: melR\n",
      "Thus , the melR promoter is not efficiently repressed by MelR and MelR is overexpressed . Thus , the @Regulated$ promoter is not efficiently repressed by @TF$ and MelR is overexpressed .\n",
      "3\n",
      "10\n",
      "Start y end del regulador: [57, 61]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: melR\n",
      "In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 . In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of @TF$ - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "29\n",
      "24\n",
      "Start y end del regulador: [128, 131]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: napF\n",
      "In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 . In this study we confirmed the location of the @TF$ - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "29\n",
      "9\n",
      "Start y end del regulador: [47, 50]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: napF\n",
      "In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 . In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of @Regulated$ expression is distinct from that of promoters with the @TF$ - binding site at position 41.5 .\n",
      "29\n",
      "39\n",
      "Start y end del regulador: [218, 221]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: napF\n",
      "To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) . To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or @TF$ produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "46\n",
      "40\n",
      "Start y end del regulador: [221, 224]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [264, 269]\n",
      "Regulado: nagBE\n",
      "To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) . To verify that @TF$ does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "46\n",
      "3\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [264, 269]\n",
      "Regulado: nagBE\n",
      "To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) . To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either @TF$ or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "46\n",
      "20\n",
      "Start y end del regulador: [119, 122]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [264, 269]\n",
      "Regulado: nagBE\n",
      "To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) . To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either @TF$ or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "46\n",
      "38\n",
      "Start y end del regulador: [213, 217]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [264, 269]\n",
      "Regulado: nagBE\n",
      "To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) . To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or @TF$ was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "46\n",
      "22\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [264, 269]\n",
      "Regulado: nagBE\n",
      "We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for zwf . We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that @TF$ is a class I activator ( 16 ) of mar as it is for @Regulated$ .\n",
      "71\n",
      "56\n",
      "Start y end del regulador: [270, 274]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [325, 328]\n",
      "Regulado: zwf\n",
      "We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for zwf . We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by @TF$ requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for @Regulated$ .\n",
      "71\n",
      "33\n",
      "Start y end del regulador: [153, 157]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [325, 328]\n",
      "Regulado: zwf\n",
      "The preferential repression by NagC at the looped nagBE â€“ lacZ fusion , compared with the nagE â€“ lacZ fusion with a single operator site , emphasises that regulation by NagC is normally dependent upon DNA loop formation involving the higher - affinity nagB operator . The preferential repression by @TF$ at the looped @Regulated$ â€“ lacZ fusion , compared with the nagE â€“ lacZ fusion with a single operator site , emphasises that regulation by NagC is normally dependent upon DNA loop formation involving the higher - affinity nagB operator .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [31, 35]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [50, 55]\n",
      "Regulado: nagBE\n",
      "The preferential repression by NagC at the looped nagBE â€“ lacZ fusion , compared with the nagE â€“ lacZ fusion with a single operator site , emphasises that regulation by NagC is normally dependent upon DNA loop formation involving the higher - affinity nagB operator . The preferential repression by NagC at the looped @Regulated$ â€“ lacZ fusion , compared with the nagE â€“ lacZ fusion with a single operator site , emphasises that regulation by @TF$ is normally dependent upon DNA loop formation involving the higher - affinity nagB operator .\n",
      "8\n",
      "30\n",
      "Start y end del regulador: [173, 177]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [50, 55]\n",
      "Regulado: nagBE\n",
      "Taken together , our results suggest that CRP site 3 is the major site required for CRP activation of rhaSR expression . Taken together , our results suggest that CRP site 3 is the major site required for @TF$ activation of @Regulated$ expression .\n",
      "19\n",
      "16\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [102, 107]\n",
      "Regulado: rhaSR\n",
      "Taken together , our results suggest that CRP site 3 is the major site required for CRP activation of rhaSR expression . Taken together , our results suggest that @TF$ site 3 is the major site required for CRP activation of @Regulated$ expression .\n",
      "19\n",
      "7\n",
      "Start y end del regulador: [42, 45]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [102, 107]\n",
      "Regulado: rhaSR\n",
      "Since we now had evidence that CRP was a direct activator of rhaSR expression , we wished to determine whether AR1 and / or AR2 of CRP were necessary for this activation . Since we now had evidence that @TF$ was a direct activator of @Regulated$ expression , we wished to determine whether AR1 and / or AR2 of CRP were necessary for this activation .\n",
      "12\n",
      "6\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [61, 66]\n",
      "Regulado: rhaSR\n",
      "Since we now had evidence that CRP was a direct activator of rhaSR expression , we wished to determine whether AR1 and / or AR2 of CRP were necessary for this activation . Since we now had evidence that CRP was a direct activator of @Regulated$ expression , we wished to determine whether AR1 and / or AR2 of @TF$ were necessary for this activation .\n",
      "12\n",
      "26\n",
      "Start y end del regulador: [131, 134]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [61, 66]\n",
      "Regulado: rhaSR\n",
      "First , expression of the inner membrane transporter chbCBA is activated directly by the ChbR protein , which allows entry of Chb - 6P and generation of the specific ChbR - inducing signal ( by action of ChbG enzyme ; Verma and Mahadevan , 2012 ) . First , expression of the inner membrane transporter @Regulated$ is activated directly by the @TF$ protein , which allows entry of Chb - 6P and generation of the specific ChbR - inducing signal ( by action of ChbG enzyme ; Verma and Mahadevan , 2012 ) .\n",
      "8\n",
      "14\n",
      "Start y end del regulador: [89, 93]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [53, 59]\n",
      "Regulado: chbCBA\n",
      "First , expression of the inner membrane transporter chbCBA is activated directly by the ChbR protein , which allows entry of Chb - 6P and generation of the specific ChbR - inducing signal ( by action of ChbG enzyme ; Verma and Mahadevan , 2012 ) . First , expression of the inner membrane transporter @Regulated$ is activated directly by the ChbR protein , which allows entry of Chb - 6P and generation of the specific @TF$ - inducing signal ( by action of ChbG enzyme ; Verma and Mahadevan , 2012 ) .\n",
      "8\n",
      "29\n",
      "Start y end del regulador: [166, 170]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [53, 59]\n",
      "Regulado: chbCBA\n",
      "Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the melR promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) . Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for @TF$ - dependent repression of the @Regulated$ promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .\n",
      "29\n",
      "23\n",
      "Start y end del regulador: [118, 122]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: melR\n",
      "Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the melR promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) . Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between @TF$ site 1 Other determinants for MelR - dependent repression of the @Regulated$ promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .\n",
      "29\n",
      "17\n",
      "Start y end del regulador: [83, 87]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: melR\n",
      "Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the melR promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) . Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the @Regulated$ promoter To investigate the effects of altering the spacing between @TF$ - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .\n",
      "29\n",
      "40\n",
      "Start y end del regulador: [226, 230]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: melR\n",
      "The expression of ptsG is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to Mlc repression . The expression of @Regulated$ is controlled by @TF$ binding to two operators upstream of two promoters , both of which are subject to Mlc repression .\n",
      "3\n",
      "7\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: ptsG\n",
      "The expression of ptsG is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to Mlc repression . The expression of @Regulated$ is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to @TF$ repression .\n",
      "3\n",
      "23\n",
      "Start y end del regulador: [126, 129]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: ptsG\n",
      "The expression of ptsG is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to Mlc repression . The expression of @Regulated$ is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to @TF$ repression .\n",
      "3\n",
      "23\n",
      "Start y end del regulador: [126, 129]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: ptsG\n",
      "The expression of ptsG is controlled by Mlc binding to two operators upstream of two promoters , both of which are subject to Mlc repression . The expression of @Regulated$ is controlled by @TF$ binding to two operators upstream of two promoters , both of which are subject to Mlc repression .\n",
      "3\n",
      "7\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: ptsG\n",
      "The cooperative binding of Lrp to the argO control region and the footprinting data ( Figs 3 and 4 ) suggest the binding of at least three Lrp dimers to regularly spaced binding sites ( Fig . 1 ) and wrapping of the operator DNA . The cooperative binding of @TF$ to the @Regulated$ control region and the footprinting data ( Figs 3 and 4 ) suggest the binding of at least three Lrp dimers to regularly spaced binding sites ( Fig . 1 ) and wrapping of the operator DNA .\n",
      "7\n",
      "4\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [38, 42]\n",
      "Regulado: argO\n",
      "The cooperative binding of Lrp to the argO control region and the footprinting data ( Figs 3 and 4 ) suggest the binding of at least three Lrp dimers to regularly spaced binding sites ( Fig . 1 ) and wrapping of the operator DNA . The cooperative binding of Lrp to the @Regulated$ control region and the footprinting data ( Figs 3 and 4 ) suggest the binding of at least three @TF$ dimers to regularly spaced binding sites ( Fig . 1 ) and wrapping of the operator DNA .\n",
      "7\n",
      "27\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [38, 42]\n",
      "Regulado: argO\n",
      "Some operons such as malEFG are under the transcriptional activation by the response regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP . Some operons such as @Regulated$ are under the transcriptional activation by the response regulators in Group 1 as well as by @TF$ , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP .\n",
      "4\n",
      "21\n",
      "Start y end del regulador: [121, 124]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [21, 27]\n",
      "Regulado: malEFG\n",
      "Some operons such as malEFG are under the transcriptional activation by the response regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP . Some operons such as @Regulated$ are under the transcriptional activation by the response regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - @TF$ .\n",
      "4\n",
      "46\n",
      "Start y end del regulador: [263, 266]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [21, 27]\n",
      "Regulado: malEFG\n",
      "This was indeed suggested by DÃ©tiollaz et al . ( 49 ) on the basis that IHF sites replacing CAP sites allowed activation of the malT promoter , albeit less than with CAP . This was indeed suggested by DÃ©tiollaz et al . ( 49 ) on the basis that IHF sites replacing @TF$ sites allowed activation of the @Regulated$ promoter , albeit less than with CAP .\n",
      "25\n",
      "19\n",
      "Start y end del regulador: [93, 96]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [129, 133]\n",
      "Regulado: malT\n",
      "This was indeed suggested by DÃ©tiollaz et al . ( 49 ) on the basis that IHF sites replacing CAP sites allowed activation of the malT promoter , albeit less than with CAP . This was indeed suggested by DÃ©tiollaz et al . ( 49 ) on the basis that IHF sites replacing CAP sites allowed activation of the @Regulated$ promoter , albeit less than with @TF$ .\n",
      "25\n",
      "32\n",
      "Start y end del regulador: [167, 170]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [129, 133]\n",
      "Regulado: malT\n",
      "For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the fdnGHI operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) . For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the @Regulated$ operon is controlled principally by @TF$ , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) .\n",
      "29\n",
      "35\n",
      "Start y end del regulador: [188, 192]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [145, 151]\n",
      "Regulado: fdnGHI\n",
      "For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the fdnGHI operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) . For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by @TF$ , while induction of the @Regulated$ operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) .\n",
      "29\n",
      "23\n",
      "Start y end del regulador: [115, 119]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [145, 151]\n",
      "Regulado: fdnGHI\n",
      "It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS . It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and @Regulated$ operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by @TF$ .\n",
      "51\n",
      "67\n",
      "Start y end del regulador: [320, 324]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [232, 236]\n",
      "Regulado: galS\n",
      "It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS . It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for @TF$ than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and @Regulated$ operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .\n",
      "51\n",
      "22\n",
      "Start y end del regulador: [115, 119]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [232, 236]\n",
      "Regulado: galS\n",
      "It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS . It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the @Regulated$ and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by @TF$ .\n",
      "49\n",
      "67\n",
      "Start y end del regulador: [320, 324]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: mglB\n",
      "It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS . It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for @TF$ than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the @Regulated$ and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .\n",
      "49\n",
      "22\n",
      "Start y end del regulador: [115, 119]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: mglB\n",
      "In practice , considerable similarity between the regulatory protein and the regulated protein is only seen in the regulatory protein RbsR and a periplasmic component RbsB of HAT , which is encoded by the operon rbsDABCK under the repression by RbsR . In practice , considerable similarity between the regulatory protein and the regulated protein is only seen in the regulatory protein RbsR and a periplasmic component RbsB of HAT , which is encoded by the operon @Regulated$ under the repression by @TF$ .\n",
      "35\n",
      "40\n",
      "Start y end del regulador: [245, 249]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [212, 220]\n",
      "Regulado: rbsDABCK\n",
      "In practice , considerable similarity between the regulatory protein and the regulated protein is only seen in the regulatory protein RbsR and a periplasmic component RbsB of HAT , which is encoded by the operon rbsDABCK under the repression by RbsR . In practice , considerable similarity between the regulatory protein and the regulated protein is only seen in the regulatory protein @TF$ and a periplasmic component RbsB of HAT , which is encoded by the operon @Regulated$ under the repression by RbsR .\n",
      "35\n",
      "20\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [212, 220]\n",
      "Regulado: rbsDABCK\n",
      "Finally , at the ( rhaB - lacZ ) 110 fusion , which contains the CRP site required for full rhaBAD activation as well as the full RhaS binding site , there was a twofold contribution to activation by CRP when in combination with RhaS - CTD and a fivefold contribution to activation4990 WICKSTRUM ET AL . J . Finally , at the ( rhaB - lacZ ) 110 fusion , which contains the @TF$ site required for full @Regulated$ activation as well as the full RhaS binding site , there was a twofold contribution to activation by CRP when in combination with RhaS - CTD and a fivefold contribution to activation4990 WICKSTRUM ET AL . J .\n",
      "20\n",
      "15\n",
      "Start y end del regulador: [65, 68]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [92, 98]\n",
      "Regulado: rhaBAD\n",
      "Finally , at the ( rhaB - lacZ ) 110 fusion , which contains the CRP site required for full rhaBAD activation as well as the full RhaS binding site , there was a twofold contribution to activation by CRP when in combination with RhaS - CTD and a fivefold contribution to activation4990 WICKSTRUM ET AL . J . Finally , at the ( rhaB - lacZ ) 110 fusion , which contains the CRP site required for full @Regulated$ activation as well as the full RhaS binding site , there was a twofold contribution to activation by @TF$ when in combination with RhaS - CTD and a fivefold contribution to activation4990 WICKSTRUM ET AL . J .\n",
      "20\n",
      "39\n",
      "Start y end del regulador: [200, 203]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [92, 98]\n",
      "Regulado: rhaBAD\n",
      "Melville and Gunsalus ( 1996 ) reported that the hemA gene , which is also repressed by FNR , contains only one half - site of FNR consensus sequence . Melville and Gunsalus ( 1996 ) reported that the @Regulated$ gene , which is also repressed by @TF$ , contains only one half - site of FNR consensus sequence .\n",
      "9\n",
      "17\n",
      "Start y end del regulador: [88, 91]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: hemA\n",
      "Melville and Gunsalus ( 1996 ) reported that the hemA gene , which is also repressed by FNR , contains only one half - site of FNR consensus sequence . Melville and Gunsalus ( 1996 ) reported that the @Regulated$ gene , which is also repressed by FNR , contains only one half - site of @TF$ consensus sequence .\n",
      "9\n",
      "26\n",
      "Start y end del regulador: [127, 130]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: hemA\n",
      "We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) . We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by @TF$ and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .\n",
      "42\n",
      "45\n",
      "Start y end del regulador: [219, 223]\n",
      "Regulador: GlcC\n",
      "Start y end del regulado: [200, 208]\n",
      "Regulado: glcDEFGB\n",
      "We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) . We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for @TF$ ( Pellicer et al . , 1999 ) .\n",
      "42\n",
      "78\n",
      "Start y end del regulador: [373, 377]\n",
      "Regulador: GlcC\n",
      "Start y end del regulado: [200, 208]\n",
      "Regulado: glcDEFGB\n",
      "We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) . We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by GlcC and @TF$ is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .\n",
      "42\n",
      "47\n",
      "Start y end del regulador: [228, 231]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [200, 208]\n",
      "Regulado: glcDEFGB\n",
      "We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) . We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by GlcC and IHF is thought to occur in this way , with @TF$ bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .\n",
      "42\n",
      "57\n",
      "Start y end del regulador: [271, 274]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [200, 208]\n",
      "Regulado: glcDEFGB\n",
      "Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for RhaR activation at rhaSR . Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for @TF$ activation at @Regulated$ .\n",
      "36\n",
      "33\n",
      "Start y end del regulador: [183, 187]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [202, 207]\n",
      "Regulado: rhaSR\n",
      "Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for RhaR activation at rhaSR . Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between @TF$ D276 and 70 R599 might be required for RhaR activation at @Regulated$ .\n",
      "36\n",
      "24\n",
      "Start y end del regulador: [139, 143]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [202, 207]\n",
      "Regulado: rhaSR\n",
      "NarL - mediated repression of the frdA promoter is achieved by NarL binding over a large region centred near the transcription start site and including the FNR site ( Li et al . , 1994 ) .  @TF$ - mediated repression of the @Regulated$ promoter is achieved by NarL binding over a large region centred near the transcription start site and including the FNR site ( Li et al . , 1994 ) .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: frdA\n",
      "NarL - mediated repression of the frdA promoter is achieved by NarL binding over a large region centred near the transcription start site and including the FNR site ( Li et al . , 1994 ) . NarL - mediated repression of the @Regulated$ promoter is achieved by @TF$ binding over a large region centred near the transcription start site and including the FNR site ( Li et al . , 1994 ) .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [63, 67]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: frdA\n",
      "Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) . Similarly , at the @TF$ - activated NarL - repressed @Regulated$ promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "10\n",
      "4\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dmsA\n",
      "Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) . Similarly , at the FNR - activated NarL - repressed @Regulated$ promoter NarL protects a large region that includes the sites for both @TF$ and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "10\n",
      "23\n",
      "Start y end del regulador: [128, 131]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dmsA\n",
      "Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) . Similarly , at the FNR - activated @TF$ - repressed @Regulated$ promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "10\n",
      "7\n",
      "Start y end del regulador: [35, 39]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dmsA\n",
      "Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) . Similarly , at the FNR - activated NarL - repressed @Regulated$ promoter @TF$ protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "10\n",
      "12\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dmsA\n",
      "However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein . However , narG and @Regulated$ operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "4\n",
      "11\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: frdA\n",
      "However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein . However , narG and @Regulated$ operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the @TF$ binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "4\n",
      "25\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: frdA\n",
      "However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein . However , @Regulated$ and frdA operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "2\n",
      "11\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [10, 14]\n",
      "Regulado: narG\n",
      "However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein . However , @Regulated$ and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the @TF$ binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "2\n",
      "25\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [10, 14]\n",
      "Regulado: narG\n",
      "For example , with AraC at the araBAD promoter , a combination of genetics and biochemistry has shown that the binding orientation is such that the C - terminally located HTH motif is adjacent to the promoter - 35 element ( Niland et al . , 1996 ) , and that , during activation , AraC binds to two adjacent sites as a direct repeat ( Reeder and Schleif , 1993 ) . For example , with @TF$ at the @Regulated$ promoter , a combination of genetics and biochemistry has shown that the binding orientation is such that the C - terminally located HTH motif is adjacent to the promoter - 35 element ( Niland et al . , 1996 ) , and that , during activation , AraC binds to two adjacent sites as a direct repeat ( Reeder and Schleif , 1993 ) .\n",
      "7\n",
      "4\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [31, 37]\n",
      "Regulado: araBAD\n",
      "For example , with AraC at the araBAD promoter , a combination of genetics and biochemistry has shown that the binding orientation is such that the C - terminally located HTH motif is adjacent to the promoter - 35 element ( Niland et al . , 1996 ) , and that , during activation , AraC binds to two adjacent sites as a direct repeat ( Reeder and Schleif , 1993 ) . For example , with AraC at the @Regulated$ promoter , a combination of genetics and biochemistry has shown that the binding orientation is such that the C - terminally located HTH motif is adjacent to the promoter - 35 element ( Niland et al . , 1996 ) , and that , during activation , @TF$ binds to two adjacent sites as a direct repeat ( Reeder and Schleif , 1993 ) .\n",
      "7\n",
      "55\n",
      "Start y end del regulador: [281, 285]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [31, 37]\n",
      "Regulado: araBAD\n",
      "Nevertheless , in contrast to the ydhY â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) . Nevertheless , in contrast to the ydhY â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the @Regulated$ promoter possesses a @TF$ - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) .\n",
      "40\n",
      "44\n",
      "Start y end del regulador: [234, 238]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [206, 212]\n",
      "Regulado: fdnGHI\n",
      "Nevertheless , in contrast to the ydhY â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) . Nevertheless , in contrast to the ydhY â€“ T promoter , where @TF$ - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the @Regulated$ promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) .\n",
      "40\n",
      "12\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [206, 212]\n",
      "Regulado: fdnGHI\n",
      "It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) . It is unlikely that @TF$ binding to site 4 contributes directly to an increase in @Regulated$ expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .\n",
      "15\n",
      "4\n",
      "Start y end del regulador: [20, 23]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [81, 86]\n",
      "Regulado: rhaSR\n",
      "It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) . It is unlikely that CRP binding to site 4 contributes directly to an increase in @Regulated$ expression , since transcription activation by @TF$ requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .\n",
      "15\n",
      "22\n",
      "Start y end del regulador: [134, 137]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [81, 86]\n",
      "Regulado: rhaSR\n",
      "Nevertheless , in contrast to the ydhY â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) . Nevertheless , in contrast to the @Regulated$ â€“ T promoter , where @TF$ - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) .\n",
      "6\n",
      "12\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: ydhY\n",
      "Nevertheless , in contrast to the ydhY â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a NarP - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) . Nevertheless , in contrast to the @Regulated$ â€“ T promoter , where NarP - mediated repression requires a 7 - 2 - 7 site centred at + 10.5 , occupation of which will probably inhibit RNA polymerase binding , the fdnGHI promoter possesses a @TF$ - binding 7 - 2 - 7 site centred at 2104.5 that is thought to antagonize activation by NarL ( Wang &amp; Gunsalus , 2003 ) .\n",
      "6\n",
      "44\n",
      "Start y end del regulador: [234, 238]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: ydhY\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of @Regulated$ - 46.5 operon expression by the @TF$ protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .\n",
      "3\n",
      "10\n",
      "Start y end del regulador: [59, 63]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [23, 26]\n",
      "Regulado: aeg\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of @Regulated$ - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the @TF$ protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .\n",
      "3\n",
      "23\n",
      "Start y end del regulador: [137, 141]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [23, 26]\n",
      "Regulado: aeg\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the @TF$ protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .\n",
      "32\n",
      "23\n",
      "Start y end del regulador: [137, 141]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [196, 200]\n",
      "Regulado: dmsA\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of aeg - 46.5 operon expression by the @TF$ protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .\n",
      "32\n",
      "10\n",
      "Start y end del regulador: [59, 63]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [196, 200]\n",
      "Regulado: dmsA\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the @TF$ protein negatively regulates expression of the @Regulated$ , dmsA and nrfA operons in response to nitrate availability .\n",
      "30\n",
      "23\n",
      "Start y end del regulador: [137, 141]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: frdA\n",
      "Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability . Negative regulation of aeg - 46.5 operon expression by the @TF$ protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the @Regulated$ , dmsA and nrfA operons in response to nitrate availability .\n",
      "30\n",
      "10\n",
      "Start y end del regulador: [59, 63]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: frdA\n",
      "aeg - 46.5 operon expression is also negatively regulated by the NarL protein but in this case NarL does not appear to act as a classical repressor .  @Regulated$ - 46.5 operon expression is also negatively regulated by the @TF$ protein but in this case NarL does not appear to act as a classical repressor .\n",
      "0\n",
      "11\n",
      "Start y end del regulador: [65, 69]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [0, 3]\n",
      "Regulado: aeg\n",
      "aeg - 46.5 operon expression is also negatively regulated by the NarL protein but in this case NarL does not appear to act as a classical repressor .  @Regulated$ - 46.5 operon expression is also negatively regulated by the NarL protein but in this case @TF$ does not appear to act as a classical repressor .\n",
      "0\n",
      "17\n",
      "Start y end del regulador: [95, 99]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [0, 3]\n",
      "Regulado: aeg\n",
      "We previously suggested that a MelR dimer simultaneously occupies site R and site 2 , creating a repression loop and the effects of different insertions and deletions on MelR - dependent repression of the melR promoter ( Figure 6 ) are consistent with this ( 17 ) . We previously suggested that a MelR dimer simultaneously occupies site R and site 2 , creating a repression loop and the effects of different insertions and deletions on @TF$ - dependent repression of the @Regulated$ promoter ( Figure 6 ) are consistent with this ( 17 ) .\n",
      "34\n",
      "28\n",
      "Start y end del regulador: [170, 174]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: melR\n",
      "We previously suggested that a MelR dimer simultaneously occupies site R and site 2 , creating a repression loop and the effects of different insertions and deletions on MelR - dependent repression of the melR promoter ( Figure 6 ) are consistent with this ( 17 ) . We previously suggested that a @TF$ dimer simultaneously occupies site R and site 2 , creating a repression loop and the effects of different insertions and deletions on MelR - dependent repression of the @Regulated$ promoter ( Figure 6 ) are consistent with this ( 17 ) .\n",
      "34\n",
      "5\n",
      "Start y end del regulador: [31, 35]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: melR\n",
      "The NarL and NarP proteins compete for the 44.5 binding site ; binding of NarP further induces aeg - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression . The NarL and NarP proteins compete for the 44.5 binding site ; binding of @TF$ further induces @Regulated$ - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .\n",
      "17\n",
      "14\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [95, 98]\n",
      "Regulado: aeg\n",
      "The NarL and NarP proteins compete for the 44.5 binding site ; binding of NarP further induces aeg - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression . The NarL and @TF$ proteins compete for the 44.5 binding site ; binding of NarP further induces @Regulated$ - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [13, 17]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [95, 98]\n",
      "Regulado: aeg\n",
      "Compared with other operons regulated by NagC , nagE - BACD and glmUS , it is very likely that the repression complex involves an interaction between NagC bound to its two sites , NagC1 and NagC2 , so that a DNA loop is formed . Compared with other operons regulated by @TF$ , nagE - BACD and @Regulated$ , it is very likely that the repression complex involves an interaction between NagC bound to its two sites , NagC1 and NagC2 , so that a DNA loop is formed .\n",
      "12\n",
      "6\n",
      "Start y end del regulador: [41, 45]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [64, 69]\n",
      "Regulado: glmUS\n",
      "Compared with other operons regulated by NagC , nagE - BACD and glmUS , it is very likely that the repression complex involves an interaction between NagC bound to its two sites , NagC1 and NagC2 , so that a DNA loop is formed . Compared with other operons regulated by NagC , nagE - BACD and @Regulated$ , it is very likely that the repression complex involves an interaction between @TF$ bound to its two sites , NagC1 and NagC2 , so that a DNA loop is formed .\n",
      "12\n",
      "26\n",
      "Start y end del regulador: [150, 154]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [64, 69]\n",
      "Regulado: glmUS\n",
      "However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC . However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated @Regulated$ operator site has a higher affinity for @TF$ than NagC .\n",
      "21\n",
      "29\n",
      "Start y end del regulador: [177, 180]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: nagE\n",
      "However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC . However the much stronger repression of this fusion by overproduced @TF$ , compared with overproduced NagC , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than NagC .\n",
      "21\n",
      "10\n",
      "Start y end del regulador: [68, 71]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: nagE\n",
      "However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC . However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than @TF$ .\n",
      "21\n",
      "31\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: nagE\n",
      "However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC . However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced @TF$ , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than NagC .\n",
      "21\n",
      "15\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: nagE\n",
      "Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation . Since only the @TF$ protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .\n",
      "7\n",
      "3\n",
      "Start y end del regulador: [15, 19]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [41, 44]\n",
      "Regulado: aeg\n",
      "Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation . Since only the NarP protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of @TF$ activation .\n",
      "7\n",
      "37\n",
      "Start y end del regulador: [213, 217]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [41, 44]\n",
      "Regulado: aeg\n",
      "The threefold contribution to activation by CRP in this experiment is very similar to the twofold contribution found in the above in vivo experiment ( Table 3 ) , confirming that RhaS - CTD is sufficient to allow at least partial CRP activation of rhaBAD expression . The threefold contribution to activation by CRP in this experiment is very similar to the twofold contribution found in the above in vivo experiment ( Table 3 ) , confirming that RhaS - CTD is sufficient to allow at least partial @TF$ activation of @Regulated$ expression .\n",
      "44\n",
      "41\n",
      "Start y end del regulador: [230, 233]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [248, 254]\n",
      "Regulado: rhaBAD\n",
      "The threefold contribution to activation by CRP in this experiment is very similar to the twofold contribution found in the above in vivo experiment ( Table 3 ) , confirming that RhaS - CTD is sufficient to allow at least partial CRP activation of rhaBAD expression . The threefold contribution to activation by @TF$ in this experiment is very similar to the twofold contribution found in the above in vivo experiment ( Table 3 ) , confirming that RhaS - CTD is sufficient to allow at least partial CRP activation of @Regulated$ expression .\n",
      "44\n",
      "6\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [248, 254]\n",
      "Regulado: rhaBAD\n",
      "The glmUS operon , which is regulated positively by NagC , also relies upon co - operative binding of NagC to two sites ( 2 ) . The @Regulated$ operon , which is regulated positively by @TF$ , also relies upon co - operative binding of NagC to two sites ( 2 ) .\n",
      "1\n",
      "9\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [4, 9]\n",
      "Regulado: glmUS\n",
      "The glmUS operon , which is regulated positively by NagC , also relies upon co - operative binding of NagC to two sites ( 2 ) . The @Regulated$ operon , which is regulated positively by NagC , also relies upon co - operative binding of @TF$ to two sites ( 2 ) .\n",
      "1\n",
      "19\n",
      "Start y end del regulador: [102, 106]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [4, 9]\n",
      "Regulado: glmUS\n",
      "Clearly there is a component ( 13 - to 20 - fold ) to the activation by CRP at rhaSR that is independent of both - CTD and RhaR and hence does not function by the same mechanism used at simple class I CRP - dependent promoters , nor does it function through cooperative binding with RhaR . Clearly there is a component ( 13 - to 20 - fold ) to the activation by @TF$ at @Regulated$ that is independent of both - CTD and RhaR and hence does not function by the same mechanism used at simple class I CRP - dependent promoters , nor does it function through cooperative binding with RhaR .\n",
      "19\n",
      "17\n",
      "Start y end del regulador: [72, 75]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [79, 84]\n",
      "Regulado: rhaSR\n",
      "Clearly there is a component ( 13 - to 20 - fold ) to the activation by CRP at rhaSR that is independent of both - CTD and RhaR and hence does not function by the same mechanism used at simple class I CRP - dependent promoters , nor does it function through cooperative binding with RhaR . Clearly there is a component ( 13 - to 20 - fold ) to the activation by CRP at @Regulated$ that is independent of both - CTD and RhaR and hence does not function by the same mechanism used at simple class I @TF$ - dependent promoters , nor does it function through cooperative binding with RhaR .\n",
      "19\n",
      "43\n",
      "Start y end del regulador: [201, 204]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [79, 84]\n",
      "Regulado: rhaSR\n",
      "This component of CRP activation at rhaSR could account for the majority of the CRP activation in a wild - type context and may involve a mechanism , such as DNA bending , that can act from a distance . This component of @TF$ activation at @Regulated$ could account for the majority of the CRP activation in a wild - type context and may involve a mechanism , such as DNA bending , that can act from a distance .\n",
      "6\n",
      "3\n",
      "Start y end del regulador: [18, 21]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [36, 41]\n",
      "Regulado: rhaSR\n",
      "This component of CRP activation at rhaSR could account for the majority of the CRP activation in a wild - type context and may involve a mechanism , such as DNA bending , that can act from a distance . This component of CRP activation at @Regulated$ could account for the majority of the @TF$ activation in a wild - type context and may involve a mechanism , such as DNA bending , that can act from a distance .\n",
      "6\n",
      "14\n",
      "Start y end del regulador: [80, 83]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [36, 41]\n",
      "Regulado: rhaSR\n",
      "Moreover , a common inverted repeat sequence coincided with the defined here NsrR recognition motif was shown to be involved in NsrR - mediated repression of the ytfE gene . Moreover , a common inverted repeat sequence coincided with the defined here NsrR recognition motif was shown to be involved in @TF$ - mediated repression of the @Regulated$ gene .\n",
      "27\n",
      "21\n",
      "Start y end del regulador: [128, 132]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [162, 166]\n",
      "Regulado: ytfE\n",
      "Moreover , a common inverted repeat sequence coincided with the defined here NsrR recognition motif was shown to be involved in NsrR - mediated repression of the ytfE gene . Moreover , a common inverted repeat sequence coincided with the defined here @TF$ recognition motif was shown to be involved in NsrR - mediated repression of the @Regulated$ gene .\n",
      "27\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [162, 166]\n",
      "Regulado: ytfE\n",
      "The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) . The @TF$ - binding site for @Regulated$ extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "6\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [28, 32]\n",
      "Regulado: argO\n",
      "The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) . The ArgP - binding site for @Regulated$ extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the @TF$ - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "6\n",
      "22\n",
      "Start y end del regulador: [118, 122]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [28, 32]\n",
      "Regulado: argO\n",
      "The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) . The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the @TF$ - binding site for @Regulated$ extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "27\n",
      "22\n",
      "Start y end del regulador: [118, 122]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: lysP\n",
      "The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) . The @TF$ - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for @Regulated$ extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "27\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: lysP\n",
      "The simultaneous binding of repressor and activator type proteins to nag DNA implies two corollaries : ( 1 ) NagC does not repress by competition with CAP and ( 2 ) CAP does not activate by displacing NagC . The simultaneous binding of repressor and activator type proteins to @Regulated$ DNA implies two corollaries : ( 1 ) @TF$ does not repress by competition with CAP and ( 2 ) CAP does not activate by displacing NagC .\n",
      "10\n",
      "19\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [69, 72]\n",
      "Regulado: nag\n",
      "The simultaneous binding of repressor and activator type proteins to nag DNA implies two corollaries : ( 1 ) NagC does not repress by competition with CAP and ( 2 ) CAP does not activate by displacing NagC . The simultaneous binding of repressor and activator type proteins to @Regulated$ DNA implies two corollaries : ( 1 ) NagC does not repress by competition with CAP and ( 2 ) CAP does not activate by displacing @TF$ .\n",
      "10\n",
      "37\n",
      "Start y end del regulador: [201, 205]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [69, 72]\n",
      "Regulado: nag\n",
      "Considering the in vivo and in vitro data presented in this work , the mechanism by which ArgP regulates argO transcription ( 24 ) , and the known capacity of Lrp to alter the shape of the DNA by inducing bending and wrapping ( 57 ) , a model for lysP regulation in which binding of Lrp to the lysP control region may favor and / or stabilize the ArgP - RNA polymerase - DNA complexes or introduce DNA conformational changes is conceivable . Considering the in vivo and in vitro data presented in this work , the mechanism by which @TF$ regulates @Regulated$ transcription ( 24 ) , and the known capacity of Lrp to alter the shape of the DNA by inducing bending and wrapping ( 57 ) , a model for lysP regulation in which binding of Lrp to the lysP control region may favor and / or stabilize the ArgP - RNA polymerase - DNA complexes or introduce DNA conformational changes is conceivable .\n",
      "19\n",
      "17\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: argO\n",
      "Considering the in vivo and in vitro data presented in this work , the mechanism by which ArgP regulates argO transcription ( 24 ) , and the known capacity of Lrp to alter the shape of the DNA by inducing bending and wrapping ( 57 ) , a model for lysP regulation in which binding of Lrp to the lysP control region may favor and / or stabilize the ArgP - RNA polymerase - DNA complexes or introduce DNA conformational changes is conceivable . Considering the in vivo and in vitro data presented in this work , the mechanism by which ArgP regulates @Regulated$ transcription ( 24 ) , and the known capacity of Lrp to alter the shape of the DNA by inducing bending and wrapping ( 57 ) , a model for lysP regulation in which binding of Lrp to the lysP control region may favor and / or stabilize the @TF$ - RNA polymerase - DNA complexes or introduce DNA conformational changes is conceivable .\n",
      "19\n",
      "69\n",
      "Start y end del regulador: [347, 351]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [105, 109]\n",
      "Regulado: argO\n",
      "( 2000 ) FNR dependent activation of the class II dmsA and narG promoters of Escherichia coli requires FNR - activating regions 1 and 3 . ( 2000 ) @TF$ dependent activation of the class II @Regulated$ and narG promoters of Escherichia coli requires FNR - activating regions 1 and 3 .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [9, 12]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: dmsA\n",
      "( 2000 ) FNR dependent activation of the class II dmsA and narG promoters of Escherichia coli requires FNR - activating regions 1 and 3 . ( 2000 ) FNR dependent activation of the class II @Regulated$ and narG promoters of Escherichia coli requires @TF$ - activating regions 1 and 3 .\n",
      "10\n",
      "18\n",
      "Start y end del regulador: [103, 106]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: dmsA\n",
      "The Escherichia coli Ada protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the alkA promoter . The Escherichia coli Ada protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of @TF$ activation at the @Regulated$ promoter .\n",
      "32\n",
      "28\n",
      "Start y end del regulador: [182, 185]\n",
      "Regulador: Ada\n",
      "Start y end del regulado: [204, 208]\n",
      "Regulado: alkA\n",
      "The Escherichia coli Ada protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the alkA promoter . The Escherichia coli @TF$ protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the @Regulated$ promoter .\n",
      "32\n",
      "3\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: Ada\n",
      "Start y end del regulado: [204, 208]\n",
      "Regulado: alkA\n",
      "Evidence for multiple OmpR - binding sites in the upstream activation sequence of the ompC promoter in Escherichia coli : a single OmpR - binding site is capable of activating the promoter . Evidence for multiple @TF$ - binding sites in the upstream activation sequence of the @Regulated$ promoter in Escherichia coli : a single OmpR - binding site is capable of activating the promoter .\n",
      "14\n",
      "3\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: ompC\n",
      "Evidence for multiple OmpR - binding sites in the upstream activation sequence of the ompC promoter in Escherichia coli : a single OmpR - binding site is capable of activating the promoter . Evidence for multiple OmpR - binding sites in the upstream activation sequence of the @Regulated$ promoter in Escherichia coli : a single @TF$ - binding site is capable of activating the promoter .\n",
      "14\n",
      "22\n",
      "Start y end del regulador: [131, 135]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: ompC\n",
      "CAT fusion analysis revealed that the operon has a promoter for gntR and another for gntKU , and that the gntR gene is constitutively expressed , while that of gntKU is regulated positively by the cAMP - CRP complex and negatively by GntR . CAT fusion analysis revealed that the operon has a promoter for gntR and another for gntKU , and that the gntR gene is constitutively expressed , while that of @Regulated$ is regulated positively by the cAMP - CRP complex and negatively by @TF$ .\n",
      "29\n",
      "42\n",
      "Start y end del regulador: [234, 238]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [160, 165]\n",
      "Regulado: gntKU\n",
      "CAT fusion analysis revealed that the operon has a promoter for gntR and another for gntKU , and that the gntR gene is constitutively expressed , while that of gntKU is regulated positively by the cAMP - CRP complex and negatively by GntR . CAT fusion analysis revealed that the operon has a promoter for gntR and another for @Regulated$ , and that the gntR gene is constitutively expressed , while that of gntKU is regulated positively by the cAMP - CRP complex and negatively by @TF$ .\n",
      "15\n",
      "42\n",
      "Start y end del regulador: [234, 238]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [85, 90]\n",
      "Regulado: gntKU\n",
      "In a gntR deletion mutant , the expression of a chromosomal gntT : : lacZ fusion is both high and constitutive , confirming that GntR is the negative regulator of gntT . In a gntR deletion mutant , the expression of a chromosomal gntT : : lacZ fusion is both high and constitutive , confirming that @TF$ is the negative regulator of @Regulated$ .\n",
      "30\n",
      "24\n",
      "Start y end del regulador: [129, 133]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [163, 167]\n",
      "Regulado: gntT\n",
      "In a gntR deletion mutant , the expression of a chromosomal gntT : : lacZ fusion is both high and constitutive , confirming that GntR is the negative regulator of gntT . In a gntR deletion mutant , the expression of a chromosomal @Regulated$ : : lacZ fusion is both high and constitutive , confirming that @TF$ is the negative regulator of gntT .\n",
      "11\n",
      "24\n",
      "Start y end del regulador: [129, 133]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [60, 64]\n",
      "Regulado: gntT\n",
      "This results in the repression of transcription , suggesting that the extent of oligomerization of DnaA proteins over two dnaA promoters contributes to the autoregulation of expression of the dnaA gene . This results in the repression of transcription , suggesting that the extent of oligomerization of @TF$ proteins over two dnaA promoters contributes to the autoregulation of expression of the @Regulated$ gene .\n",
      "29\n",
      "15\n",
      "Start y end del regulador: [99, 103]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [192, 196]\n",
      "Regulado: dnaA\n",
      "This results in the repression of transcription , suggesting that the extent of oligomerization of DnaA proteins over two dnaA promoters contributes to the autoregulation of expression of the dnaA gene . This results in the repression of transcription , suggesting that the extent of oligomerization of @TF$ proteins over two @Regulated$ promoters contributes to the autoregulation of expression of the dnaA gene .\n",
      "19\n",
      "15\n",
      "Start y end del regulador: [99, 103]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [122, 126]\n",
      "Regulado: dnaA\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by @TF$ , SoxS and Rob of two @Regulated$ promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [30, 34]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [57, 61]\n",
      "Regulado: tolC\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by @TF$ , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for @Regulated$ in Escherichia coli .\n",
      "22\n",
      "3\n",
      "Start y end del regulador: [30, 34]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [134, 138]\n",
      "Regulado: tolC\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by MarA , SoxS and @TF$ of two @Regulated$ promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "10\n",
      "7\n",
      "Start y end del regulador: [46, 49]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [57, 61]\n",
      "Regulado: tolC\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by MarA , SoxS and @TF$ of two tolC promoters using one binding site : a complex promoter configuration for @Regulated$ in Escherichia coli .\n",
      "22\n",
      "7\n",
      "Start y end del regulador: [46, 49]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [134, 138]\n",
      "Regulado: tolC\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by MarA , @TF$ and Rob of two @Regulated$ promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [57, 61]\n",
      "Regulado: tolC\n",
      "Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli . Transcriptional activation by MarA , @TF$ and Rob of two tolC promoters using one binding site : a complex promoter configuration for @Regulated$ in Escherichia coli .\n",
      "22\n",
      "5\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [134, 138]\n",
      "Regulado: tolC\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "15\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: fumC\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not @Regulated$ or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "89\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [394, 398]\n",
      "Regulado: fumC\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and @Regulated$ DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "99\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [429, 433]\n",
      "Regulado: fumC\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and @Regulated$ DNA as a monomer .\n",
      "109\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [465, 469]\n",
      "Regulado: fumC\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , @Regulated$ , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [74, 78]\n",
      "Regulado: micF\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or @Regulated$ , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "91\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [402, 406]\n",
      "Regulado: micF\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of @Regulated$ , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [55, 58]\n",
      "Regulado: zwf\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate @Regulated$ transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "40\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [170, 173]\n",
      "Regulado: zwf\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of @Regulated$ but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "86\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [382, 385]\n",
      "Regulado: zwf\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends @Regulated$ and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "97\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [421, 424]\n",
      "Regulado: zwf\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds @Regulated$ and fumC DNA as a monomer .\n",
      "107\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [457, 460]\n",
      "Regulado: zwf\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the NanR , @TF$ and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate @Regulated$ expression .\n",
      "24\n",
      "6\n",
      "Start y end del regulador: [29, 32]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: fimB\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the NanR , @TF$ and NagC binding sites closer to the @Regulated$ promoter enhances the ability of the regulators to activate fimB expression .\n",
      "14\n",
      "6\n",
      "Start y end del regulador: [29, 32]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: fimB\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the NanR , IHF and @TF$ binding sites closer to the fimB promoter enhances the ability of the regulators to activate @Regulated$ expression .\n",
      "24\n",
      "8\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: fimB\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the NanR , IHF and @TF$ binding sites closer to the @Regulated$ promoter enhances the ability of the regulators to activate fimB expression .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: fimB\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the @TF$ , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate @Regulated$ expression .\n",
      "24\n",
      "4\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: fimB\n",
      "However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression . However , placing the @TF$ , IHF and NagC binding sites closer to the @Regulated$ promoter enhances the ability of the regulators to activate fimB expression .\n",
      "14\n",
      "4\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: fimB\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "17\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: fumC\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not @Regulated$ or micF .\n",
      "72\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [317, 321]\n",
      "Regulado: not \n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: fumC\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not @Regulated$ or micF .\n",
      "72\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [317, 321]\n",
      "Regulado: not \n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , @Regulated$ , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "19\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: micF\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or @Regulated$ .\n",
      "74\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [325, 329]\n",
      "Regulado:  or \n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , @Regulated$ , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "19\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: micF\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or @Regulated$ .\n",
      "74\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [325, 329]\n",
      "Regulado:  or \n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of @Regulated$ , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [61, 64]\n",
      "Regulado: zwf\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate @Regulated$ transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "38\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [158, 161]\n",
      "Regulado: e z\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of @Regulated$ , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of @Regulated$ but not fumC or micF .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [61, 64]\n",
      "Regulado: zwf\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of @Regulated$ , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "13\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [61, 64]\n",
      "Regulado: zwf\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate @Regulated$ transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "38\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [158, 161]\n",
      "Regulado: e z\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of @Regulated$ but not fumC or micF .\n",
      "69\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [305, 308]\n",
      "Regulado:  of\n",
      "As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by NsrR highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress . As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , @Regulated$ , and ytfE are the three transcripts most tightly regulated by @TF$ highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress .\n",
      "30\n",
      "42\n",
      "Start y end del regulador: [226, 230]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: hmpA\n",
      "As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by NsrR highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress . As @Regulated$ and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by @TF$ highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress .\n",
      "1\n",
      "42\n",
      "Start y end del regulador: [226, 230]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [3, 7]\n",
      "Regulado: hmpA\n",
      "As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by NsrR highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress . As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and @Regulated$ are the three transcripts most tightly regulated by @TF$ highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress .\n",
      "33\n",
      "42\n",
      "Start y end del regulador: [226, 230]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [169, 173]\n",
      "Regulado: ytfE\n",
      "As hmpA and ytfE encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by NsrR highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress . As hmpA and @Regulated$ encode a nitric oxide reductase and a mechanism to repair iron - sulfur centers damaged by nitric oxide , the demonstration that hcp - hcr , hmpA , and ytfE are the three transcripts most tightly regulated by @TF$ highlights the possibility that the hybrid cluster protein , HCP , might also be part of a defense mechanism against reactive nitrogen stress .\n",
      "3\n",
      "42\n",
      "Start y end del regulador: [226, 230]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [12, 16]\n",
      "Regulado: ytfE\n",
      "Among the two dnaA promoters , transcription from the promoter 1P was specifically enhanced by the in vivo overproduction of IciA protein or using in vitro transcription assays of the dnaA gene . Among the two dnaA promoters , transcription from the promoter 1P was specifically enhanced by the in vivo overproduction of @TF$ protein or using in vitro transcription assays of the @Regulated$ gene .\n",
      "30\n",
      "20\n",
      "Start y end del regulador: [125, 129]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [184, 188]\n",
      "Regulado: dnaA\n",
      "Among the two dnaA promoters , transcription from the promoter 1P was specifically enhanced by the in vivo overproduction of IciA protein or using in vitro transcription assays of the dnaA gene . Among the two @Regulated$ promoters , transcription from the promoter 1P was specifically enhanced by the in vivo overproduction of @TF$ protein or using in vitro transcription assays of the dnaA gene .\n",
      "3\n",
      "20\n",
      "Start y end del regulador: [125, 129]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: dnaA\n",
      "Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter . Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; @TF$ acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the @Regulated$ promoter .\n",
      "49\n",
      "21\n",
      "Start y end del regulador: [93, 96]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [212, 217]\n",
      "Regulado: melAB\n",
      "Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter . Optimal expression of the @Regulated$ promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; @TF$ acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter .\n",
      "4\n",
      "21\n",
      "Start y end del regulador: [93, 96]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [26, 31]\n",
      "Regulado: melAB\n",
      "Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter . Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between @TF$ bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the @Regulated$ promoter .\n",
      "49\n",
      "29\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [212, 217]\n",
      "Regulado: melAB\n",
      "Optimal expression of the melAB promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between MelR bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter . Optimal expression of the @Regulated$ promoter requires Sites 1 0 , Site 1 , Site 2 and Site 2 0 ; CRP acts as a ` bridge ' between @TF$ bound at Sites 1 0 and 1 and at Sites 2 and 2 0 , increasing expression from the melAB promoter .\n",
      "4\n",
      "29\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [26, 31]\n",
      "Regulado: melAB\n",
      "Based on data from in vitro studies , Celis suggested that wild - type ArgP both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) . Based on data from in vitro studies , Celis suggested that wild - type @TF$ both represses its own transcription and activates that of @Regulated$ ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) .\n",
      "24\n",
      "14\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: argK\n",
      "Based on data from in vitro studies , Celis suggested that wild - type ArgP both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) . Based on data from in vitro studies , Celis suggested that wild - type @TF$ both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including @Regulated$ ( 12 ) .\n",
      "77\n",
      "14\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [410, 414]\n",
      "Regulado: argK\n",
      "In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) . In contrast , the repression of both soxS and @Regulated$ by @TF$ , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) .\n",
      "9\n",
      "11\n",
      "Start y end del regulador: [53, 57]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [46, 49]\n",
      "Regulado: rob\n",
      "In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) . In contrast , the repression of both soxS and rob by @TF$ , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and @Regulated$ promoter regions ( 15 , 16 ) .\n",
      "33\n",
      "11\n",
      "Start y end del regulador: [53, 57]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [146, 149]\n",
      "Regulado: rob\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "1\n",
      "19\n",
      "Start y end del regulador: [98, 101]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [5, 9]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for @Regulated$ were experimentally defined ( 26 ) .\n",
      "50\n",
      "19\n",
      "Start y end del regulador: [98, 101]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: gntT\n",
      "RhaR regulates transcription of rhaSR by binding promoter DNA spanning 32 to 82 relative to the rhaSR transcription start site .  @TF$ regulates transcription of @Regulated$ by binding promoter DNA spanning 32 to 82 relative to the rhaSR transcription start site .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [32, 37]\n",
      "Regulado: rhaSR\n",
      "RhaR regulates transcription of rhaSR by binding promoter DNA spanning 32 to 82 relative to the rhaSR transcription start site .  @TF$ regulates transcription of rhaSR by binding promoter DNA spanning 32 to 82 relative to the @Regulated$ transcription start site .\n",
      "16\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [96, 101]\n",
      "Regulado: rhaSR\n",
      "A single base substitution in the left half - site when combined with the Fnr consensus sequence at the right half - site exhibited a 138 - fold activation of dmsA - lacZ expression ( Figure 1 , ?JA450 ) , an induction equivalent to the wild - type dmsA promoter that has the symmetrical deviation in the right half - site ( ca . 114 - fold ) . A single base substitution in the left half - site when combined with the @TF$ consensus sequence at the right half - site exhibited a 138 - fold activation of @Regulated$ - lacZ expression ( Figure 1 , ?JA450 ) , an induction equivalent to the wild - type dmsA promoter that has the symmetrical deviation in the right half - site ( ca . 114 - fold ) .\n",
      "30\n",
      "14\n",
      "Start y end del regulador: [74, 77]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [159, 163]\n",
      "Regulado: dmsA\n",
      "A single base substitution in the left half - site when combined with the Fnr consensus sequence at the right half - site exhibited a 138 - fold activation of dmsA - lacZ expression ( Figure 1 , ?JA450 ) , an induction equivalent to the wild - type dmsA promoter that has the symmetrical deviation in the right half - site ( ca . 114 - fold ) . A single base substitution in the left half - site when combined with the @TF$ consensus sequence at the right half - site exhibited a 138 - fold activation of dmsA - lacZ expression ( Figure 1 , ?JA450 ) , an induction equivalent to the wild - type @Regulated$ promoter that has the symmetrical deviation in the right half - site ( ca . 114 - fold ) .\n",
      "49\n",
      "14\n",
      "Start y end del regulador: [74, 77]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [249, 253]\n",
      "Regulado: dmsA\n",
      "Subsequent to rhaSR expression , RhaS binds DNA upstream of rhaBAD at 32 to 81 relative to the transcription start site to increase rhaBAD expression by approximately 1,000 - fold ( to 10 Miller units in single copy ) . Subsequent to rhaSR expression , @TF$ binds DNA upstream of @Regulated$ at 32 to 81 relative to the transcription start site to increase rhaBAD expression by approximately 1,000 - fold ( to 10 Miller units in single copy ) .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [60, 66]\n",
      "Regulado: rhaBAD\n",
      "Subsequent to rhaSR expression , RhaS binds DNA upstream of rhaBAD at 32 to 81 relative to the transcription start site to increase rhaBAD expression by approximately 1,000 - fold ( to 10 Miller units in single copy ) . Subsequent to rhaSR expression , @TF$ binds DNA upstream of rhaBAD at 32 to 81 relative to the transcription start site to increase @Regulated$ expression by approximately 1,000 - fold ( to 10 Miller units in single copy ) .\n",
      "23\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [132, 138]\n",
      "Regulado: rhaBAD\n",
      "We show in this study that strains with null mutations in either argP or yggA exhibit abnormally increased sensitivities to CAN ( Can ss ) and that ArgP is a transcriptional regulator of yggA that mediates the latter ÔÇÖ s induction by Arg . We show in this study that strains with null mutations in either argP or yggA exhibit abnormally increased sensitivities to CAN ( Can ss ) and that @TF$ is a transcriptional regulator of @Regulated$ that mediates the latter ÔÇÖ s induction by Arg .\n",
      "33\n",
      "27\n",
      "Start y end del regulador: [148, 152]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [187, 191]\n",
      "Regulado: yggA\n",
      "We show in this study that strains with null mutations in either argP or yggA exhibit abnormally increased sensitivities to CAN ( Can ss ) and that ArgP is a transcriptional regulator of yggA that mediates the latter ÔÇÖ s induction by Arg . We show in this study that strains with null mutations in either argP or @Regulated$ exhibit abnormally increased sensitivities to CAN ( Can ss ) and that @TF$ is a transcriptional regulator of yggA that mediates the latter ÔÇÖ s induction by Arg .\n",
      "14\n",
      "27\n",
      "Start y end del regulador: [148, 152]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: yggA\n",
      "To evaluate the role of the two MarR binding sites in the repression of marRAB operon transcription ( Fig . 1A ; Cohen et al . , 1993a ; Martin and Rosner , 1995 ) , fusions of the marRAB regulatory region to lacZ were constructed and integrated in the chromosome as single - copy ?RS45 lysogens ( Simons et al . , 1987 ) . To evaluate the role of the two @TF$ binding sites in the repression of @Regulated$ operon transcription ( Fig . 1A ; Cohen et al . , 1993a ; Martin and Rosner , 1995 ) , fusions of the marRAB regulatory region to lacZ were constructed and integrated in the chromosome as single - copy ?RS45 lysogens ( Simons et al . , 1987 ) .\n",
      "14\n",
      "7\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [72, 78]\n",
      "Regulado: marRAB\n",
      "To evaluate the role of the two MarR binding sites in the repression of marRAB operon transcription ( Fig . 1A ; Cohen et al . , 1993a ; Martin and Rosner , 1995 ) , fusions of the marRAB regulatory region to lacZ were constructed and integrated in the chromosome as single - copy ?RS45 lysogens ( Simons et al . , 1987 ) . To evaluate the role of the two @TF$ binding sites in the repression of marRAB operon transcription ( Fig . 1A ; Cohen et al . , 1993a ; Martin and Rosner , 1995 ) , fusions of the @Regulated$ regulatory region to lacZ were constructed and integrated in the chromosome as single - copy ?RS45 lysogens ( Simons et al . , 1987 ) .\n",
      "39\n",
      "7\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [181, 187]\n",
      "Regulado: marRAB\n",
      "We show here that MarA directly mediates upregulation of nfnB expression through a cis - recognition marbox element in the nfnB promoter . We show here that @TF$ directly mediates upregulation of @Regulated$ expression through a cis - recognition marbox element in the nfnB promoter .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [57, 61]\n",
      "Regulado: nfnB\n",
      "We show here that MarA directly mediates upregulation of nfnB expression through a cis - recognition marbox element in the nfnB promoter . We show here that @TF$ directly mediates upregulation of nfnB expression through a cis - recognition marbox element in the @Regulated$ promoter .\n",
      "20\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [123, 127]\n",
      "Regulado: nfnB\n",
      "In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) . In addition , the repression of @Regulated$ translation by @TF$ induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .\n",
      "6\n",
      "9\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [32, 36]\n",
      "Regulado: chiP\n",
      "In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) . In addition , the repression of chiP translation by @TF$ induces premature Rho - dependent transcription termination within the @Regulated$ gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .\n",
      "19\n",
      "9\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: chiP\n",
      "In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) . In addition , the repression of chiP translation by @TF$ induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of @Regulated$ and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .\n",
      "41\n",
      "9\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [269, 273]\n",
      "Regulado: chiP\n",
      "ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .  @TF$ - mediated regulation of @Regulated$ transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [30, 34]\n",
      "Regulado: argO\n",
      "ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .  @TF$ - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the @Regulated$ region from 293 to + 109 ( Fig . 1C ) .\n",
      "34\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [200, 204]\n",
      "Regulado: argO\n",
      "Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of argO ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either ArgP binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) . Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of argO ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either @TF$ binding to @Regulated$ in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .\n",
      "61\n",
      "58\n",
      "Start y end del regulador: [291, 295]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [307, 311]\n",
      "Regulado: argO\n",
      "Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of argO ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either ArgP binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) . Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of @Regulated$ ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either @TF$ binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .\n",
      "17\n",
      "58\n",
      "Start y end del regulador: [291, 295]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [87, 91]\n",
      "Regulado: argO\n",
      "The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) . The strong upregulation of @Regulated$ by the plasmid specified @TF$ observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "4\n",
      "9\n",
      "Start y end del regulador: [61, 69]\n",
      "Regulador: observed\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: nfnB\n",
      "The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) . The strong upregulation of nfnB by the plasmidspecified @TF$ observed in this study contrasts with the reported weaker upregulation of NfnB activity and @Regulated$ expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "23\n",
      "8\n",
      "Start y end del regulador: [56, 60]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: nfnB\n",
      "The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) . The strong upregulation of @Regulated$ by the plasmid specified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by @TF$ ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "4\n",
      "27\n",
      "Start y end del regulador: [177, 178]\n",
      "Regulador: (\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: nfnB\n",
      "The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) . The strong upregulation of nfnB by the plasmid specified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and @Regulated$ expression by @TF$ ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "24\n",
      "27\n",
      "Start y end del regulador: [177, 178]\n",
      "Regulador: (\n",
      "Start y end del regulado: [158, 168]\n",
      "Regulado: expression\n",
      "Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) . Presumably , uninduced levels of @TF$ and / or SoxS mediate a low - level induction of the @Regulated$ gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "18\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: inaA\n",
      "Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) . Presumably , uninduced levels of @TF$ and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of @Regulated$ expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "26\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: inaA\n",
      "Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) . Presumably , uninduced levels of MarA and / or @TF$ mediate a low - level induction of the @Regulated$ gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "18\n",
      "9\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: inaA\n",
      "Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) . Presumably , uninduced levels of MarA and / or @TF$ mediate a low - level induction of the inaA gene ; the lower basal level of @Regulated$ expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "26\n",
      "9\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: inaA\n",
      "The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters . The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the @Regulated$ and micF promoters .\n",
      "54\n",
      "48\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [262, 266]\n",
      "Regulado: fumC\n",
      "The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters . The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the @Regulated$ , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the fumC and micF promoters .\n",
      "19\n",
      "48\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [89, 93]\n",
      "Regulado: fumC\n",
      "The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters . The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the fumC and @Regulated$ promoters .\n",
      "56\n",
      "48\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [271, 275]\n",
      "Regulado: micF\n",
      "The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters . The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , @Regulated$ , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the fumC and micF promoters .\n",
      "21\n",
      "48\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [96, 100]\n",
      "Regulado: micF\n",
      "However , cells that also contain pJW15 , encoding melR , score clearly as Lac minus , due to the MelR - dependent repression of the melR promoter ( i . e . they give pale pink ┬▒ white colonies ) . However , cells that also contain pJW15 , encoding melR , score clearly as Lac minus , due to the @TF$ - dependent repression of the @Regulated$ promoter ( i . e . they give pale pink ┬▒ white colonies ) .\n",
      "26\n",
      "20\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: melR\n",
      "However , cells that also contain pJW15 , encoding melR , score clearly as Lac minus , due to the MelR - dependent repression of the melR promoter ( i . e . they give pale pink ┬▒ white colonies ) . However , cells that also contain pJW15 , encoding @Regulated$ , score clearly as Lac minus , due to the @TF$ - dependent repression of the melR promoter ( i . e . they give pale pink ┬▒ white colonies ) .\n",
      "9\n",
      "20\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: melR\n",
      "Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion . Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , @TF$ and GalS repression at @Regulated$ 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "46\n",
      "41\n",
      "Start y end del regulador: [211, 215]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [239, 243]\n",
      "Regulado: galP\n",
      "Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion . Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , @TF$ and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed @Regulated$ promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "57\n",
      "41\n",
      "Start y end del regulador: [211, 215]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [307, 311]\n",
      "Regulado: galP\n",
      "Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion . Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and @TF$ repression at @Regulated$ 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "46\n",
      "43\n",
      "Start y end del regulador: [220, 224]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [239, 243]\n",
      "Regulado: galP\n",
      "Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion . Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and @TF$ repression at galP 149 corresponds to the strength of the galactose - derepressed @Regulated$ promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "57\n",
      "43\n",
      "Start y end del regulador: [220, 224]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [307, 311]\n",
      "Regulado: galP\n",
      "To investigate the role of GntR in repression of the gntT gene , the expression of a single - copy chromosomal gntT : : lacZ fusion was measured in a gntR mutant . To investigate the role of @TF$ in repression of the @Regulated$ gene , the expression of a single - copy chromosomal gntT : : lacZ fusion was measured in a gntR mutant .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: gntT\n",
      "To investigate the role of GntR in repression of the gntT gene , the expression of a single - copy chromosomal gntT : : lacZ fusion was measured in a gntR mutant . To investigate the role of @TF$ in repression of the gntT gene , the expression of a single - copy chromosomal @Regulated$ : : lacZ fusion was measured in a gntR mutant .\n",
      "21\n",
      "5\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: gntT\n",
      "To measure MelR - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured . To measure @TF$ - dependent repression of @Regulated$ promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .\n",
      "7\n",
      "2\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: melR\n",
      "To measure MelR - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured . To measure @TF$ - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 @Regulated$ lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .\n",
      "28\n",
      "2\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [169, 173]\n",
      "Regulado: melR\n",
      "To measure MelR - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured . To measure @TF$ - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding @Regulated$ , or empty control vector , pJWmelR and b - galactosidase activities were measured .\n",
      "36\n",
      "2\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: melR\n",
      "Altogether , these data indicate that ArgP is able to activate transcription from the dapB promoter , provided that some sequences upstream of 81 are present , and is necessary for lysine to repress dapB expression . Altogether , these data indicate that @TF$ is able to activate transcription from the @Regulated$ promoter , provided that some sequences upstream of 81 are present , and is necessary for lysine to repress dapB expression .\n",
      "14\n",
      "6\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: dapB\n",
      "Altogether , these data indicate that ArgP is able to activate transcription from the dapB promoter , provided that some sequences upstream of 81 are present , and is necessary for lysine to repress dapB expression . Altogether , these data indicate that @TF$ is able to activate transcription from the dapB promoter , provided that some sequences upstream of 81 are present , and is necessary for lysine to repress @Regulated$ expression .\n",
      "34\n",
      "6\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [199, 203]\n",
      "Regulado: dapB\n",
      "Optimal MelR - dependent repression of the melR promoter depends on upstream sequences Transcription initiation of the melR promoter is dependent on the binding of CRP to a 22 bp DNA site centered at position 241.5 , but sequences further upstream are not essential ( Webster et al . , 1988 ) . Optimal @TF$ - dependent repression of the @Regulated$ promoter depends on upstream sequences Transcription initiation of the melR promoter is dependent on the binding of CRP to a 22 bp DNA site centered at position 241.5 , but sequences further upstream are not essential ( Webster et al . , 1988 ) .\n",
      "7\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: melR\n",
      "Optimal MelR - dependent repression of the melR promoter depends on upstream sequences Transcription initiation of the melR promoter is dependent on the binding of CRP to a 22 bp DNA site centered at position 241.5 , but sequences further upstream are not essential ( Webster et al . , 1988 ) . Optimal @TF$ - dependent repression of the melR promoter depends on upstream sequences Transcription initiation of the @Regulated$ promoter is dependent on the binding of CRP to a 22 bp DNA site centered at position 241.5 , but sequences further upstream are not essential ( Webster et al . , 1988 ) .\n",
      "17\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [119, 123]\n",
      "Regulado: melR\n",
      "In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated . In vitro binding of @TF$ to the micF promoter 6,22 , and transcriptional activation of @Regulated$ in vitro 9 have been demonstrated .\n",
      "15\n",
      "4\n",
      "Start y end del regulador: [20, 23]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: micF\n",
      "In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated . In vitro binding of @TF$ to the @Regulated$ promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .\n",
      "7\n",
      "4\n",
      "Start y end del regulador: [20, 23]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [31, 35]\n",
      "Regulado: micF\n",
      "The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR . The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of @Regulated$ by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both @TF$ and NanR .\n",
      "45\n",
      "68\n",
      "Start y end del regulador: [355, 358]\n",
      "Regulador: and\n",
      "Start y end del regulado: [235, 237]\n",
      "Regulado: by\n",
      "The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR . The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of @Regulated$ expression , as would be expected if it prevents activation by both @TF$ and NanR .\n",
      "55\n",
      "68\n",
      "Start y end del regulador: [355, 358]\n",
      "Regulador: and\n",
      "Start y end del regulado: [282, 292]\n",
      "Regulado: expression\n",
      "The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR . The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of @Regulated$ by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and @TF$ .\n",
      "45\n",
      "70\n",
      "Start y end del regulador: [364, 365]\n",
      "Regulador: .\n",
      "Start y end del regulado: [235, 237]\n",
      "Regulado: by\n",
      "The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR . The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of @Regulated$ expression , as would be expected if it prevents activation by both NagC and @TF$ .\n",
      "55\n",
      "70\n",
      "Start y end del regulador: [364, 365]\n",
      "Regulador: .\n",
      "Start y end del regulado: [282, 292]\n",
      "Regulado: expression\n",
      "During the course of our studies on CRP activation of the rhaBAD operon , we compared expression from various rhaBAD promoter fusions ( Fig . 1 ) in crp and crp strain backgrounds . During the course of our studies on @TF$ activation of the @Regulated$ operon , we compared expression from various rhaBAD promoter fusions ( Fig . 1 ) in crp and crp strain backgrounds .\n",
      "11\n",
      "7\n",
      "Start y end del regulador: [36, 39]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [58, 64]\n",
      "Regulado: rhaBAD\n",
      "During the course of our studies on CRP activation of the rhaBAD operon , we compared expression from various rhaBAD promoter fusions ( Fig . 1 ) in crp and crp strain backgrounds . During the course of our studies on @TF$ activation of the rhaBAD operon , we compared expression from various @Regulated$ promoter fusions ( Fig . 1 ) in crp and crp strain backgrounds .\n",
      "19\n",
      "7\n",
      "Start y end del regulador: [36, 39]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [110, 116]\n",
      "Regulado: rhaBAD\n",
      "Repression of the melR promoter carried on different KK81 fragments . The figure shows the degree of MelR - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 . Repression of the  melR promoter carried on different KK81 fragments . The figure shows the degree of @TF$ - dependent repression of the @Regulated$  promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .\n",
      "23\n",
      "17\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [136, 140]\n",
      "Regulado: melR\n",
      "Repression of the melR promoter carried on different KK81 fragments . The figure shows the degree of MelR - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 . Repression of the @Regulated$  promoter carried on different KK81 fragments . The figure shows the degree of @TF$ - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .\n",
      "3\n",
      "17\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: melR\n",
      "Also , both half - sites of the Fnr recognition sequence at dmsA are required for Fnrdependent expression and are similar in their ability to activate dmsA transcription . Also , both half - sites of the @TF$ recognition sequence at dmsA are required for Fnr dependent expression  and are similar in their ability to activate @Regulated$ transcription .\n",
      "27\n",
      "8\n",
      "Start y end del regulador: [32, 35]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [156, 169]\n",
      "Regulado: transcription\n",
      "Also , both half - sites of the Fnr recognition sequence at dmsA are required for Fnrdependent expression and are similar in their ability to activate dmsA transcription . Also , both half - sites of the @TF$ recognition sequence at @Regulated$ are required for Fnr dependent expression and are similar in their ability to activate dmsA transcription .\n",
      "12\n",
      "8\n",
      "Start y end del regulador: [32, 35]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [60, 64]\n",
      "Regulado: dmsA\n",
      "For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) . For instance , Rob has been shown to bind and activate the @Regulated$ promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and @TF$ ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "12\n",
      "34\n",
      "Start y end del regulador: [179, 183]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [59, 62]\n",
      "Regulado: zwf\n",
      "For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) . For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell @Regulated$ regulation cannot be activated by Rob , although the gene responds to SoxS and @TF$ ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "19\n",
      "34\n",
      "Start y end del regulador: [179, 183]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [96, 99]\n",
      "Regulado: zwf\n",
      "For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) . For instance , Rob has been shown to bind and activate the @Regulated$ promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to @TF$ and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "12\n",
      "32\n",
      "Start y end del regulador: [170, 174]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [59, 62]\n",
      "Regulado: zwf\n",
      "For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) . For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell @Regulated$ regulation cannot be activated by Rob , although the gene responds to @TF$ and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "19\n",
      "32\n",
      "Start y end del regulador: [170, 174]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [96, 99]\n",
      "Regulado: zwf\n",
      "The resulting recombinant plasmids were used to monitor MelR - dependent activation at the melAB promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis . The resulting recombinant plasmids were used to monitor @TF$ - dependent activation at the @Regulated$ promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [56, 60]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [91, 96]\n",
      "Regulado: melAB\n",
      "The resulting recombinant plasmids were used to monitor MelR - dependent activation at the melAB promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis . The resulting recombinant plasmids were used to monitor @TF$ - dependent activation at the melAB promoter ; i . e . transcripts initiating at the @Regulated$ promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .\n",
      "25\n",
      "8\n",
      "Start y end del regulador: [56, 60]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [146, 151]\n",
      "Regulado: melAB\n",
      "To evaluate whether ArgP was involved in the transcriptional regulation of lysP , a nonfunctional argP allele ( argP : : Cam r ) was transduced into strain MG - LR carrying the chromosomal P lysP : : lacZ fusion . To evaluate whether @TF$ was involved in the transcriptional regulation of @Regulated$ , a nonfunctional argP allele ( argP : : Cam r ) was transduced into strain MG - LR carrying the chromosomal P lysP : : lacZ fusion .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: lysP\n",
      "To evaluate whether ArgP was involved in the transcriptional regulation of lysP , a nonfunctional argP allele ( argP : : Cam r ) was transduced into strain MG - LR carrying the chromosomal P lysP : : lacZ fusion . To evaluate whether @TF$ was involved in the transcriptional regulation of lysP , a nonfunctional argP allele ( argP : : Cam r ) was transduced into strain MG - LR carrying the chromosomal P @Regulated$ : : lacZ fusion .\n",
      "35\n",
      "3\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [191, 195]\n",
      "Regulado: lysP\n",
      "Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate . Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated @Regulated$ and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "13\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: gatA\n",
      "Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate . Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated @Regulated$ , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "36\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [200, 204]\n",
      "Regulado: gatA\n",
      "Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate . Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , @Regulated$ , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "51\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [277, 281]\n",
      "Regulado: gatA\n",
      "Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate . Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and @Regulated$ transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "15\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [79, 83]\n",
      "Regulado: gatC\n",
      "Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate . Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , @Regulated$ and gatD upon treating cells with salicylate .\n",
      "55\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [291, 295]\n",
      "Regulado: gatC\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both Lrp and @TF$ stimulate @Regulated$ expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions .\n",
      "6\n",
      "4\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: argO\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both Lrp and @TF$ stimulate argO expression and bind to overlapping targets in the @Regulated$ control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions .\n",
      "15\n",
      "4\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [89, 93]\n",
      "Regulado: argO\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both Lrp and @TF$ stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on @Regulated$ expression , in response to environmental conditions .\n",
      "34\n",
      "4\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [220, 224]\n",
      "Regulado: argO\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both @TF$ and ArgP stimulate @Regulated$ expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions .\n",
      "6\n",
      "2\n",
      "Start y end del regulador: [11, 14]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: argO\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both @TF$ and ArgP stimulate argO expression and bind to overlapping targets in the @Regulated$ control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions .\n",
      "15\n",
      "2\n",
      "Start y end del regulador: [11, 14]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [89, 93]\n",
      "Regulado: argO\n",
      "Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on argO expression , in response to environmental conditions . Since both @TF$ and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional activators on @Regulated$ expression , in response to environmental conditions .\n",
      "34\n",
      "2\n",
      "Start y end del regulador: [11, 14]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [220, 224]\n",
      "Regulado: argO\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both @Regulated$ promoters 1P and 2P were repressed by @TF$ protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "8\n",
      "16\n",
      "Start y end del regulador: [94, 98]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: dnaA\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by @TF$ protein , IciA protein specifically activated transcription from @Regulated$ promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "25\n",
      "16\n",
      "Start y end del regulador: [94, 98]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [164, 168]\n",
      "Regulado: dnaA\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by @TF$ protein , IciA protein specifically activated transcription from dnaA promoter 1P , while @Regulated$ promoter 2P was still repressed ( 25 ) .\n",
      "30\n",
      "16\n",
      "Start y end del regulador: [94, 98]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: dnaA\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , @TF$ protein specifically activated transcription from @Regulated$ promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "25\n",
      "19\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [164, 168]\n",
      "Regulado: dnaA\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both @Regulated$ promoters 1P and 2P were repressed by DnaA protein , @TF$ protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "8\n",
      "19\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: dnaA\n",
      "Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) . Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , @TF$ protein specifically activated transcription from dnaA promoter 1P , while @Regulated$ promoter 2P was still repressed ( 25 ) .\n",
      "30\n",
      "19\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: dnaA\n",
      "While NarL was still able to activate and to repress nrfA gene expression , the pattern differed from the wild - type nrfA - lacZ reporter fusion ( Fig . 6B ) . While @TF$ was still able to activate and to repress @Regulated$ gene expression , the pattern differed from the wild - type nrfA - lacZ reporter fusion ( Fig . 6B ) .\n",
      "10\n",
      "1\n",
      "Start y end del regulador: [6, 10]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: nrfA\n",
      "While NarL was still able to activate and to repress nrfA gene expression , the pattern differed from the wild - type nrfA - lacZ reporter fusion ( Fig . 6B ) . While @TF$ was still able to activate and to repress nrfA gene expression , the pattern differed from the wild - type @Regulated$ - lacZ reporter fusion ( Fig . 6B ) .\n",
      "22\n",
      "1\n",
      "Start y end del regulador: [6, 10]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [118, 122]\n",
      "Regulado: nrfA\n",
      "Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE . Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at @Regulated$ and maximally 20 % activation at nagE .\n",
      "49\n",
      "9\n",
      "Start y end del regulador: [51, 54]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [264, 268]\n",
      "Regulado: nagB\n",
      "Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE . Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and @Regulated$ to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE .\n",
      "37\n",
      "9\n",
      "Start y end del regulador: [51, 54]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [211, 215]\n",
      "Regulado: nagB\n",
      "Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE . Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at @Regulated$ .\n",
      "56\n",
      "9\n",
      "Start y end del regulador: [51, 54]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [302, 306]\n",
      "Regulado: nagE\n",
      "Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE . Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of @Regulated$ and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE .\n",
      "35\n",
      "9\n",
      "Start y end del regulador: [51, 54]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [202, 206]\n",
      "Regulado: nagE\n",
      "Since the same level of activation is seen with the truncated single operator site nagE ÔÇô lacZ fusion it implies that no DNA important for CAP stimulation of nagE ÔÇô lacZ expression is situated beyond ÔÇô 100 . Since the same level of activation is seen with the truncated single operator site @Regulated$ ÔÇô lacZ fusion it implies that no DNA important for @TF$ stimulation of nagE ÔÇô lacZ expression is situated beyond ÔÇô 100 .\n",
      "14\n",
      "25\n",
      "Start y end del regulador: [141, 144]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [83, 87]\n",
      "Regulado: nagE\n",
      "Since the same level of activation is seen with the truncated single operator site nagE ÔÇô lacZ fusion it implies that no DNA important for CAP stimulation of nagE ÔÇô lacZ expression is situated beyond ÔÇô 100 . Since the same level of activation is seen with the truncated single operator site nagE ÔÇô lacZ fusion it implies that no DNA important for @TF$ stimulation of @Regulated$ ÔÇô lacZ expression is situated beyond ÔÇô 100 .\n",
      "28\n",
      "25\n",
      "Start y end del regulador: [141, 144]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: nagE\n",
      "The treB treC operon is negatively regulated by TreR , whose gene treR is located upstream of treB but is not part of the operon . The @Regulated$ treC operon is negatively regulated by @TF$ , whose gene treR is located upstream of treB but is not part of the operon .\n",
      "1\n",
      "8\n",
      "Start y end del regulador: [48, 52]\n",
      "Regulador: TreR\n",
      "Start y end del regulado: [4, 8]\n",
      "Regulado: treB\n",
      "The treB treC operon is negatively regulated by TreR , whose gene treR is located upstream of treB but is not part of the operon . The treB treC operon is negatively regulated by @TF$ , whose gene treR is located upstream of @Regulated$ but is not part of the operon .\n",
      "17\n",
      "8\n",
      "Start y end del regulador: [48, 52]\n",
      "Regulador: TreR\n",
      "Start y end del regulado: [94, 98]\n",
      "Regulado: treB\n",
      "This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR . This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that @Regulated$ transcription is regulated by @TF$ .\n",
      "30\n",
      "35\n",
      "Start y end del regulador: [207, 211]\n",
      "Regulador: OxyR\n",
      "Start y end del regulado: [173, 176]\n",
      "Regulado: hcp\n",
      "This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR . This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an @Regulated$ mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by @TF$ .\n",
      "16\n",
      "35\n",
      "Start y end del regulador: [207, 211]\n",
      "Regulador: OxyR\n",
      "Start y end del regulado: [96, 99]\n",
      "Regulado: hcp\n",
      "This observation not only allows us to understand how a modest mutation in O NC2 can affect fimB expression whereas the D3 mutation does not , but it also supports our prior assertion that NanR activates fimB expression without NagC ( Sohanpal et al . , 2004 ) . This observation not only allows us to understand how a modest mutation in O NC2 can affect fimB expression whereas the D3 mutation does not , but it also supports our prior assertion that @TF$ activates @Regulated$ expression without NagC ( Sohanpal et al . , 2004 ) .\n",
      "36\n",
      "34\n",
      "Start y end del regulador: [189, 193]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [204, 208]\n",
      "Regulado: fimB\n",
      "This observation not only allows us to understand how a modest mutation in O NC2 can affect fimB expression whereas the D3 mutation does not , but it also supports our prior assertion that NanR activates fimB expression without NagC ( Sohanpal et al . , 2004 ) . This observation not only allows us to understand how a modest mutation in O NC2 can affect @Regulated$ expression whereas the D3 mutation does not , but it also supports our prior assertion that @TF$ activates fimB expression without NagC ( Sohanpal et al . , 2004 ) .\n",
      "17\n",
      "34\n",
      "Start y end del regulador: [189, 193]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: fimB\n",
      "The results described above also suggest that when O NR is placed closer to the fimB promoter , NanR is better able to activate fimB expression , and further work supports this idea . The results described above also suggest that when O NR is placed closer to the fimB promoter , @TF$ is better able to activate @Regulated$ expression , and further work supports this idea .\n",
      "24\n",
      "18\n",
      "Start y end del regulador: [96, 100]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [128, 132]\n",
      "Regulado: fimB\n",
      "The results described above also suggest that when O NR is placed closer to the fimB promoter , NanR is better able to activate fimB expression , and further work supports this idea . The results described above also suggest that when O NR is placed closer to the @Regulated$ promoter , @TF$ is better able to activate fimB expression , and further work supports this idea .\n",
      "15\n",
      "18\n",
      "Start y end del regulador: [96, 100]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: fimB\n",
      "This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression . This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates @Regulated$ expression .\n",
      "36\n",
      "29\n",
      "Start y end del regulador: [147, 151]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [186, 190]\n",
      "Regulado: gntV\n",
      "This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression . This and the finding that expression of @Regulated$ - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates gntV expression .\n",
      "7\n",
      "29\n",
      "Start y end del regulador: [147, 151]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: gntV\n",
      "So far , our results demonstrate that ArgP directly binds to the lysP promoter / control region and that the T - N 11 - A sequence located close to the RNA polymerase binding site is important for binding and crucial for lysP transcription . So far , our results demonstrate that @TF$ directly binds to the lysP promoter / control region and that the T - N 11 - A sequence located close to the RNA polymerase binding site is important for binding and crucial for @Regulated$ transcription .\n",
      "42\n",
      "7\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [221, 225]\n",
      "Regulado: lysP\n",
      "So far , our results demonstrate that ArgP directly binds to the lysP promoter / control region and that the T - N 11 - A sequence located close to the RNA polymerase binding site is important for binding and crucial for lysP transcription . So far , our results demonstrate that @TF$ directly binds to the @Regulated$ promoter / control region and that the T - N 11 - A sequence located close to the RNA polymerase binding site is important for binding and crucial for lysP transcription .\n",
      "12\n",
      "7\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: lysP\n",
      "Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli melAB promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by MelR . Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli @Regulated$ promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by @TF$ .\n",
      "19\n",
      "40\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [98, 103]\n",
      "Regulado: melAB\n",
      "Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli melAB promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by MelR . Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli melAB promoter 219 which transcription activation at the @Regulated$ promoter is absolutely dependent on the occupation of Site 2 0 by @TF$ .\n",
      "27\n",
      "40\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [155, 160]\n",
      "Regulado: melAB\n",
      "Although there are many moderate to weak matches to the consensus CRP - binding site sequence in the rhaSR - rhaBAD intergenic region , three of these , sites 2 through 4 , seemed most likely to directly influence rhaSR activation . Although there are many moderate to weak matches to the consensus @TF$ - binding site sequence in the rhaSR - rhaBAD intergenic region , three of these , sites 2 through 4 , seemed most likely to directly influence @Regulated$ activation .\n",
      "39\n",
      "11\n",
      "Start y end del regulador: [66, 69]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [214, 219]\n",
      "Regulado: rhaSR\n",
      "Although there are many moderate to weak matches to the consensus CRP - binding site sequence in the rhaSR - rhaBAD intergenic region , three of these , sites 2 through 4 , seemed most likely to directly influence rhaSR activation . Although there are many moderate to weak matches to the consensus @TF$ - binding site sequence in the @Regulated$ - rhaBAD intergenic region , three of these , sites 2 through 4 , seemed most likely to directly influence rhaSR activation .\n",
      "18\n",
      "11\n",
      "Start y end del regulador: [66, 69]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [101, 106]\n",
      "Regulado: rhaSR\n",
      "The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase . The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that @TF$ - mediated inhibition of FNR recruitment is a key component of the regulation of the @Regulated$ ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "46\n",
      "30\n",
      "Start y end del regulador: [156, 160]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [246, 250]\n",
      "Regulado: ydhY\n",
      "The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase . The relative degrees of repression of the @Regulated$ ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that @TF$ - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "7\n",
      "30\n",
      "Start y end del regulador: [156, 160]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: ydhY\n",
      "The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase . The relative degrees of repression of the @Regulated$ ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that @TF$ - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "7\n",
      "53\n",
      "Start y end del regulador: [277, 281]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: ydhY\n",
      "The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase . The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the @Regulated$ ÔÇô T promoter , and that @TF$ - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "46\n",
      "53\n",
      "Start y end del regulador: [277, 281]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [246, 250]\n",
      "Regulado: ydhY\n",
      "The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) . The presence of FNR and @TF$ consensus sequences upstream of @Regulated$ is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [24, 27]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [60, 64]\n",
      "Regulado: caiF\n",
      "The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) . The presence of FNR and @TF$ consensus sequences upstream of caiF is consistent with the activation of transcription of a @Regulated$ - lacZ fusion by these two proteins ( Table 3 ) .\n",
      "20\n",
      "5\n",
      "Start y end del regulador: [24, 27]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: caiF\n",
      "The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) . The presence of @TF$ and CRP consensus sequences upstream of @Regulated$ is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [16, 19]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [60, 64]\n",
      "Regulado: caiF\n",
      "The presence of FNR and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a caiF - lacZ fusion by these two proteins ( Table 3 ) . The presence of @TF$ and CRP consensus sequences upstream of caiF is consistent with the activation of transcription of a @Regulated$ - lacZ fusion by these two proteins ( Table 3 ) .\n",
      "20\n",
      "3\n",
      "Start y end del regulador: [16, 19]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: caiF\n",
      "Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 . Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the @Regulated$ pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .\n",
      "14\n",
      "7\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [102, 107]\n",
      "Regulado: melAB\n",
      "Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 . Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli @Regulated$ promoter 345 Fig . 10 .\n",
      "37\n",
      "7\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [219, 224]\n",
      "Regulado: melAB\n",
      "CsrA appears not to affect eda transcript stability but appears to activate eda , since eda transcription was 1.6 - to 1.8 - fold lower in a csrA background .  @TF$ appears not to affect eda transcript stability but appears to activate @Regulated$ , since eda transcription was 1.6 - to 1.8 - fold lower in a csrA background .\n",
      "12\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: CsrA\n",
      "Start y end del regulado: [76, 79]\n",
      "Regulado: eda\n",
      "CsrA appears not to affect eda transcript stability but appears to activate eda , since eda transcription was 1.6 - to 1.8 - fold lower in a csrA background .  @TF$ appears not to affect @Regulated$ transcript stability but appears to activate eda , since eda transcription was 1.6 - to 1.8 - fold lower in a csrA background .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: CsrA\n",
      "Start y end del regulado: [27, 30]\n",
      "Regulado: eda\n",
      "CsrA appears not to affect eda transcript stability but appears to activate eda , since eda transcription was 1.6 - to 1.8 - fold lower in a csrA background .  @TF$ appears not to affect eda transcript stability but appears to activate eda , since @Regulated$ transcription was 1.6 - to 1.8 - fold lower in a csrA background .\n",
      "15\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: CsrA\n",
      "Start y end del regulado: [88, 91]\n",
      "Regulado: eda\n",
      "Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) . Expression of both nagE and @Regulated$ was shown to be stimulated by @TF$ in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .\n",
      "5\n",
      "12\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [28, 32]\n",
      "Regulado: nagB\n",
      "Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) . Expression of both nagE and nagB was shown to be stimulated by @TF$ in viva , although nagE was much more dependent than @Regulated$ ( Plumbridge , 1990 ) .\n",
      "23\n",
      "12\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [120, 124]\n",
      "Regulado: nagB\n",
      "Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) . Expression of both @Regulated$ and nagB was shown to be stimulated by @TF$ in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .\n",
      "3\n",
      "12\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: nagE\n",
      "Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) . Expression of both nagE and nagB was shown to be stimulated by @TF$ in viva , although @Regulated$ was much more dependent than nagB ( Plumbridge , 1990 ) .\n",
      "17\n",
      "12\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: nagE\n",
      "However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that Lrp stimulates expression of cadBA . However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that @TF$ stimulates expression of @Regulated$ .\n",
      "37\n",
      "33\n",
      "Start y end del regulador: [161, 164]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [190, 195]\n",
      "Regulado: cadBA\n",
      "However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that Lrp stimulates expression of cadBA . However , under inducing conditions , the P @Regulated$ activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that @TF$ stimulates expression of cadBA .\n",
      "8\n",
      "33\n",
      "Start y end del regulador: [161, 164]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [44, 49]\n",
      "Regulado: cadBA\n",
      "The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P . The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the @Regulated$ promoter 1P .\n",
      "22\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [120, 124]\n",
      "Regulado: dnaA\n",
      "The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P . The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P .\n",
      "10\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: dnaA\n",
      "( 1997 ) The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P . ( 1997 ) The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase at to the @Regulated$ promoter 1P .\n",
      "26\n",
      "9\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: dnaA\n",
      "( 1997 ) The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P . ( 1997 ) The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P .\n",
      "13\n",
      "9\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [58, 62]\n",
      "Regulado: dnaA\n",
      "The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P\n",
      "10\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: dnaA\n",
      "The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the @Regulated$ promoter 1P\n",
      "22\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [120, 124]\n",
      "Regulado: dnaA\n",
      "Transcription of the dnaA gene from the promoter 1P has been shown to be activated in vitro and in vivo by the binding of IciA protein to two sites on the dnaA promoter region Transcription of the @Regulated$ gene from the promoter 1P has been shown to be activated in vitro and in vivo by the binding of @TF$ protein to two sites on the dnaA promoter region\n",
      "3\n",
      "24\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [21, 25]\n",
      "Regulado: dnaA\n",
      "Transcription of the dnaA gene from the promoter 1P has been shown to be activated in vitro and in vivo by the binding of IciA protein to two sites on the dnaA promoter region Transcription of the dnaA gene from the promoter 1P has been shown to be activated in vitro and in vivo by the binding of @TF$ protein to two sites on the @Regulated$ promoter region\n",
      "31\n",
      "24\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [155, 159]\n",
      "Regulado: dnaA\n",
      "Occlusion of RNA Polymerase by Oligomerization of DnaA Protein over the dnaA Promoter of Escherichia coli Occlusion of RNA Polymerase by Oligomerization of @TF$ Protein over the @Regulated$ Promoter of Escherichia coli\n",
      "11\n",
      "7\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: dnaA\n",
      " DnaA protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two dnaA promoters , 1P and 2P .  @TF$ protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two @Regulated$ promoters , 1P and 2P .\n",
      "24\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador:  Dna\n",
      "Start y end del regulado: [152, 156]\n",
      "Regulado:  dna\n",
      "Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a CRP binding site positioned at 71 upstream of the gntT transcription start site . Transcription of @Regulated$ is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex to a CRP binding site positioned at 71 upstream of the gntT transcription start site .\n",
      "2\n",
      "19\n",
      "Start y end del regulador: [99, 102]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [17, 21]\n",
      "Regulado: gntT\n",
      "Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a CRP binding site positioned at 71 upstream of the gntT transcription start site . Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex to a CRP binding site positioned at 71 upstream of the @Regulated$ transcription start site .\n",
      "33\n",
      "19\n",
      "Start y end del regulador: [99, 102]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [168, 172]\n",
      "Regulado: gntT\n",
      "Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a CRP binding site positioned at 71 upstream of the gntT transcription start site . Transcription of @Regulated$ is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a @TF$ binding site positioned at 71 upstream of the gntT transcription start site .\n",
      "2\n",
      "24\n",
      "Start y end del regulador: [118, 121]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [17, 21]\n",
      "Regulado: gntT\n",
      "Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a CRP binding site positioned at 71 upstream of the gntT transcription start site . Transcription of gntT is activated by binding of the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex to a @TF$ binding site positioned at 71 upstream of the @Regulated$ transcription start site .\n",
      "33\n",
      "24\n",
      "Start y end del regulador: [118, 121]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [168, 172]\n",
      "Regulado: gntT\n",
      " MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .  @TF$ carrying each of the single substitutions is less able to repress the @Regulated$ promoter, while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter.\n",
      "13\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador:  Mel\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado:  mel\n",
      " MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .  @TF$ carrying each of the single substitutions is less able to repress the melR promoter, while MelR carrying some combinations of substitutions is completely unable to repress the @Regulated$ promoter.\n",
      "28\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador:  Mel\n",
      "Start y end del regulado: [178, 181]\n",
      "Regulado:  th\n",
      " MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .  MelR carrying each of the single substitutions is less able to repress the melR promoter, while @TF$ carrying some combinations of substitutions is completely unable to repress the @Regulated$ promoter.\n",
      "28\n",
      "16\n",
      "Start y end del regulador: [91, 96]\n",
      "Regulador:  whil\n",
      "Start y end del regulado: [178, 181]\n",
      "Regulado:  th\n",
      " MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .  MelR carrying each of the single substitutions is less able to repress the @Regulated$ promoter, while @TF$ carrying some combinations of substitutions is completely unable to repress the melR promoter.\n",
      "13\n",
      "16\n",
      "Start y end del regulador: [91, 96]\n",
      "Regulador:  whil\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado:  mel\n",
      "Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD . Given that cyclic AMP receptor protein ( @TF$ ) , the second activator required for full @Regulated$ expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD .\n",
      "16\n",
      "7\n",
      "Start y end del regulador: [41, 44]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [88, 94]\n",
      "Regulado: rhaBAD\n",
      "Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD . Given that cyclic AMP receptor protein ( @TF$ ) , the second activator required for full rhaBAD expression , cannot activate @Regulated$ expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD .\n",
      "21\n",
      "7\n",
      "Start y end del regulador: [41, 44]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [124, 130]\n",
      "Regulado: rhaBAD\n",
      "Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD . Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full @Regulated$ expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether @TF$ could activate transcription in combination with RhaS - CTD .\n",
      "16\n",
      "35\n",
      "Start y end del regulador: [196, 199]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [88, 94]\n",
      "Regulado: rhaBAD\n",
      "Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD . Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate @Regulated$ expression in a rhaS strain , it was of interest to test whether @TF$ could activate transcription in combination with RhaS - CTD .\n",
      "21\n",
      "35\n",
      "Start y end del regulador: [196, 199]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [124, 130]\n",
      "Regulado: rhaBAD\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that @TF$ overexpression activates expression of fumC , @Regulated$ , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "12\n",
      "5\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: sodA\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that @TF$ overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , @Regulated$ , nfo , and zwf promoters ( 3 ) .\n",
      "41\n",
      "5\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: sodA\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified @TF$ forms DNA - protein complexes with DNA fragments containing the micF , @Regulated$ , nfo , and zwf promoters ( 3 ) .\n",
      "41\n",
      "28\n",
      "Start y end del regulador: [148, 151]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: sodA\n",
      "Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) . Moreover , they found that Rob overexpression activates expression of fumC , @Regulated$ , and inaA ( unknown function ) , but not zwf , and that purified @TF$ forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "12\n",
      "28\n",
      "Start y end del regulador: [148, 151]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: sodA\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by @TF$ ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "1\n",
      "25\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [5, 9]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by @TF$ ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for @Regulated$ were experimentally defined ( 26 ) .\n",
      "50\n",
      "25\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "1\n",
      "28\n",
      "Start y end del regulador: [141, 145]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [5, 9]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for @Regulated$ were experimentally defined ( 26 ) .\n",
      "50\n",
      "28\n",
      "Start y end del regulador: [141, 145]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two @TF$ - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "1\n",
      "45\n",
      "Start y end del regulador: [213, 217]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [5, 9]\n",
      "Regulado: gntT\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two @TF$ - binding elements for @Regulated$ were experimentally defined ( 26 ) .\n",
      "50\n",
      "45\n",
      "Start y end del regulador: [213, 217]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: gntT\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results ArgP - mediated regulation of E . coli argO Reconstitution of @TF$ - mediated regulation of @Regulated$ transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "17\n",
      "12\n",
      "Start y end del regulador: [70, 74]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results @TF$ - mediated regulation of E . coli @Regulated$ Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "9\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results @TF$ - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of @Regulated$ transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "17\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results @TF$ - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of @Regulated$ transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "24\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results @TF$ - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of @Regulated$ transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "36\n",
      "1\n",
      "Start y end del regulador: [8, 12]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [222, 226]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results ArgP - mediated regulation of E . coli @Regulated$ Reconstitution of @TF$ - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "9\n",
      "12\n",
      "Start y end del regulador: [70, 74]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results ArgP - mediated regulation of E . coli argO Reconstitution of @TF$ - mediated regulation of argO transcription in vitro Toward reconstitution of @Regulated$ transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "24\n",
      "12\n",
      "Start y end del regulador: [70, 74]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: argO\n",
      "Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) . Results ArgP - mediated regulation of E . coli argO Reconstitution of @TF$ - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of @Regulated$ transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "36\n",
      "12\n",
      "Start y end del regulador: [70, 74]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [222, 226]\n",
      "Regulado: argO\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the @Regulated$ is dependent on @TF$ and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .\n",
      "9\n",
      "13\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 54]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the hcp is dependent on @TF$ and NarP Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .\n",
      "20\n",
      "13\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [110, 113]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the @Regulated$ is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , @TF$ and NarP ( Figure 1 ) .\n",
      "9\n",
      "45\n",
      "Start y end del regulador: [256, 260]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 54]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , @TF$ and NarP ( Figure 1 ) .\n",
      "20\n",
      "45\n",
      "Start y end del regulador: [256, 260]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [110, 113]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the @Regulated$ is dependent on NarL and @TF$ Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [80, 84]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [51, 54]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and @TF$ Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .\n",
      "20\n",
      "15\n",
      "Start y end del regulador: [80, 84]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [110, 113]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the @Regulated$ is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and @TF$ ( Figure 1 ) .\n",
      "9\n",
      "47\n",
      "Start y end del regulador: [265, 269]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [51, 54]\n",
      "Regulado: hcp\n",
      "Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) . Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and @TF$ ( Figure 1 ) .\n",
      "20\n",
      "47\n",
      "Start y end del regulador: [265, 269]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [110, 113]\n",
      "Regulado: hcp\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the IciA I site , was located upstream @TF$ protein binds two sites in the @Regulated$ promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) .\n",
      "19\n",
      "12\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: dnaA\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the @TF$ I site , was located upstream IciA protein binds two sites in the @Regulated$ promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) .\n",
      "19\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: dnaA\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the @TF$ I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the @Regulated$ promoter region ( Fig . 3 ) .\n",
      "39\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: dnaA\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the IciA I site , was located upstream @TF$ protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the @Regulated$ promoter region ( Fig . 3 ) .\n",
      "39\n",
      "12\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: dnaA\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the IciA I site , was located upstream IciA protein binds two sites in the @Regulated$ promoter region DNase I - protection experiments were performed to identify binding sites of the @TF$ protein in the dnaA promoter region ( Fig . 3 ) .\n",
      "19\n",
      "35\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: dnaA\n",
      "One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) . One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the @TF$ protein in the @Regulated$ promoter region ( Fig . 3 ) .\n",
      "39\n",
      "35\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [219, 223]\n",
      "Regulado: dnaA\n",
      "More recently , ArgP was shown to bind to an approximately 60 - bp - long sequence partially overlapping the argO promoter More recently , @TF$ was shown to bind to an approximately 60 - bp - long sequence partially overlapping the @Regulated$ promoter\n",
      "20\n",
      "3\n",
      "Start y end del regulador: [16, 20]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [109, 113]\n",
      "Regulado: argO\n",
      "As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter . As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how @TF$ represses @Regulated$ at the same time as it activates the nrf promoter .\n",
      "22\n",
      "20\n",
      "Start y end del regulador: [95, 98]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [109, 114]\n",
      "Regulado: acsP1\n",
      "As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter . As the DNA binding site for @TF$ overlaps the @Regulated$ - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [28, 31]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [45, 50]\n",
      "Regulado: acsP1\n",
      "As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter . As the DNA binding site for @TF$ overlaps the acsP1 - 35 element , it is easy to understand how FNR represses @Regulated$ at the same time as it activates the nrf promoter .\n",
      "22\n",
      "6\n",
      "Start y end del regulador: [28, 31]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [109, 114]\n",
      "Regulado: acsP1\n",
      "As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter . As the DNA binding site for FNR overlaps the @Regulated$ - 35 element , it is easy to understand how @TF$ represses acsP1 at the same time as it activates the nrf promoter .\n",
      "9\n",
      "20\n",
      "Start y end del regulador: [95, 98]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [45, 50]\n",
      "Regulado: acsP1\n",
      " Figure 1 NarL regulates the E . coli yeaR promoter ( A ) This Figure shows the E . coli K - 12 yeaR promoter sequence from positions 64 to + 86 with respect to the transcription start site ( + 1 ) ( 12 ) . Figure 1 @TF$ regulates the E . coli @Regulated$ promoter ( A ) This Figure shows the E . coli K - 12 yeaR promoter sequence from positions 64 to + 86 with respect to the transcription start site ( + 1 ) ( 12 ) .\n",
      "8\n",
      "2\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador:  Nar\n",
      "Start y end del regulado: [37, 41]\n",
      "Regulado:  yea\n",
      "Figure 1 NarL regulates the E . coli yeaR promoter ( A ) This Figure shows the E . coli K - 12 yeaR promoter sequence from positions 64 to + 86 with respect to the transcription start site ( + 1 ) ( 12 ) . Figure 1 @TF$ regulates the E . coli yeaR promoter ( A ) This Figure shows the E . coli K - 12 @Regulated$ promoter sequence from positions 64 to + 86 with respect to the transcription start site ( + 1 ) ( 12 ) .\n",
      "23\n",
      "2\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [95, 99]\n",
      "Regulado: yeaR\n",
      "The ArgP protein enhances the expression of the argK gene The @TF$ protein enhances the expression of the @Regulated$ gene\n",
      "8\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: argK\n",
      "To test the biological activity of the ArgP protein on the arginine transport system , the amounts of mRNA synthesized by a 502 bp KpnI - EcoRV DNA fragment containing the N terminus and the control region of the argK gene was investigated . To test the biological activity of the @TF$ protein on the arginine transport system , the amounts of mRNA synthesized by a 502 bp KpnI - EcoRV DNA fragment containing the N terminus and the control region of the @Regulated$ gene was investigated .\n",
      "39\n",
      "7\n",
      "Start y end del regulador: [39, 43]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [213, 217]\n",
      "Regulado: argK\n",
      "RESULTS Oligomerization of DnaA Protein to the dnaA Promoter â€” Sequence - specific DNA binding of DnaA protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box . RESULTS Oligomerization of DnaA Protein to the dnaA Promoter â€” Sequence - specific DNA binding of @TF$ protein to the @Regulated$ promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box .\n",
      "20\n",
      "16\n",
      "Start y end del regulador: [100, 104]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [120, 124]\n",
      "Regulado: dnaA\n",
      "RESULTS Oligomerization of DnaA Protein to the dnaA Promoter â€” Sequence - specific DNA binding of DnaA protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box . RESULTS Oligomerization of @TF$ Protein to the @Regulated$ Promoter â€” Sequence - specific DNA binding of DnaA protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box .\n",
      "7\n",
      "3\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: dnaA\n",
      "RESULTS Oligomerization of DnaA Protein to the dnaA Promoter â€” Sequence - specific DNA binding of DnaA protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box . RESULTS Oligomerization of @TF$ Protein to the dnaA Promoter â€” Sequence - specific DNA binding of DnaA protein to the @Regulated$ promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box .\n",
      "20\n",
      "3\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [120, 124]\n",
      "Regulado: dnaA\n",
      "RESULTS Oligomerization of DnaA Protein to the dnaA Promoter â€” Sequence - specific DNA binding of DnaA protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box . RESULTS Oligomerization of DnaA Protein to the @Regulated$ Promoter â€” Sequence - specific DNA binding of @TF$ protein to the dnaA promoter region ( Fig . 1 ) was studied in detail using a combined gel - shift and chemical footprinting assay to determine the extent of DNA binding to the region flanking the consensus DNA - binding site , the DnaA box .\n",
      "7\n",
      "16\n",
      "Start y end del regulador: [100, 104]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: dnaA\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of @Regulated$ in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .\n",
      "67\n",
      "73\n",
      "Start y end del regulador: [352, 356]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [314, 318]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , @Regulated$ , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential @TF$ binding site upstream of rimK is that identified for ybjC .\n",
      "16\n",
      "85\n",
      "Start y end del regulador: [409, 413]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [81, 85]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and @Regulated$ ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential @TF$ binding site upstream of rimK is that identified for ybjC .\n",
      "41\n",
      "85\n",
      "Start y end del regulador: [409, 413]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [198, 202]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of @Regulated$ in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential @TF$ binding site upstream of rimK is that identified for ybjC .\n",
      "67\n",
      "85\n",
      "Start y end del regulador: [409, 413]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [314, 318]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential @TF$ binding site upstream of @Regulated$ is that identified for ybjC .\n",
      "90\n",
      "85\n",
      "Start y end del regulador: [409, 413]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [439, 443]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , @Regulated$ , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .\n",
      "16\n",
      "73\n",
      "Start y end del regulador: [352, 356]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [81, 85]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and @Regulated$ ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .\n",
      "41\n",
      "73\n",
      "Start y end del regulador: [352, 356]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [198, 202]\n",
      "Regulado: rimK\n",
      "Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC . Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of @Regulated$ is that identified for ybjC .\n",
      "90\n",
      "73\n",
      "Start y end del regulador: [352, 356]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [439, 443]\n",
      "Regulado: rimK\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in @Regulated$ expression due to a lack of cAMP / @TF$ activation ( 5 ; see also below ) .\n",
      "66\n",
      "75\n",
      "Start y end del regulador: [360, 363]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [320, 324]\n",
      "Regulado: nagE\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the @TF$ + 6 mutation on @Regulated$ expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "12\n",
      "7\n",
      "Start y end del regulador: [34, 37]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [54, 58]\n",
      "Regulado: nagE\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the @TF$ + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in @Regulated$ expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "38\n",
      "7\n",
      "Start y end del regulador: [34, 37]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [187, 191]\n",
      "Regulado: nagE\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the @TF$ + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in @Regulated$ expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "66\n",
      "7\n",
      "Start y end del regulador: [34, 37]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [320, 324]\n",
      "Regulado: nagE\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the CAP + 6 mutation on @Regulated$ expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / @TF$ activation ( 5 ; see also below ) .\n",
      "12\n",
      "75\n",
      "Start y end del regulador: [360, 363]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [54, 58]\n",
      "Regulado: nagE\n",
      "There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) . There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in @Regulated$ expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / @TF$ activation ( 5 ; see also below ) .\n",
      "38\n",
      "75\n",
      "Start y end del regulador: [360, 363]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [187, 191]\n",
      "Regulado: nagE\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the @TF$ I site and how this binding activates transcription from the @Regulated$ promoter 1P .\n",
      "43\n",
      "32\n",
      "Start y end del regulador: [216, 220]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [282, 286]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing @TF$ protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "12\n",
      "7\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing @TF$ protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the @Regulated$ promoter 1P .\n",
      "43\n",
      "7\n",
      "Start y end del regulador: [74, 78]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [282, 286]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "12\n",
      "26\n",
      "Start y end del regulador: [181, 185]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the @Regulated$ promoter 1P .\n",
      "43\n",
      "26\n",
      "Start y end del regulador: [181, 185]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [282, 286]\n",
      "Regulado: dnaA\n",
      "DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P . DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the @TF$ I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "12\n",
      "32\n",
      "Start y end del regulador: [216, 220]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [100, 104]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of @TF$ protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "13\n",
      "21\n",
      "Start y end del regulador: [140, 144]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of @TF$ protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the @Regulated$ promoters .\n",
      "53\n",
      "21\n",
      "Start y end del regulador: [140, 144]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [310, 314]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of DnaA protein to regions flanking the @TF$ box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "13\n",
      "27\n",
      "Start y end del regulador: [177, 181]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the @TF$ box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the @Regulated$ promoters .\n",
      "53\n",
      "27\n",
      "Start y end del regulador: [177, 181]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [310, 314]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two @TF$ monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "13\n",
      "37\n",
      "Start y end del regulador: [221, 225]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two @TF$ monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the @Regulated$ promoters .\n",
      "53\n",
      "37\n",
      "Start y end del regulador: [221, 225]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [310, 314]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the @TF$ box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "13\n",
      "41\n",
      "Start y end del regulador: [242, 246]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: dnaA\n",
      "These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters . These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the @TF$ box is not sufficient for occlusion of RNA polymerase from the @Regulated$ promoters .\n",
      "53\n",
      "41\n",
      "Start y end del regulador: [242, 246]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [310, 314]\n",
      "Regulado: dnaA\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or @TF$ to activate transcription of @Regulated$ in vitro ( 18 , 26 ) .\n",
      "48\n",
      "43\n",
      "Start y end del regulador: [228, 232]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [262, 268]\n",
      "Regulado: marRAB\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of @Regulated$ which binds @TF$ and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "31\n",
      "34\n",
      "Start y end del regulador: [180, 184]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [161, 167]\n",
      "Regulado: marRAB\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds @TF$ and enables either purified MalE - SoxS or MarA to activate transcription of @Regulated$ in vitro ( 18 , 26 ) .\n",
      "48\n",
      "34\n",
      "Start y end del regulador: [180, 184]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [262, 268]\n",
      "Regulado: marRAB\n",
      "Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) . Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of @Regulated$ which binds MarA and enables either purified MalE - SoxS or @TF$ to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "31\n",
      "43\n",
      "Start y end del regulador: [228, 232]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [161, 167]\n",
      "Regulado: marRAB\n",
      "Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR . Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of @TF$ that are important in transcription activation at the @Regulated$ promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "25\n",
      "16\n",
      "Start y end del regulador: [112, 115]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [170, 175]\n",
      "Regulado: melAB\n",
      "Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR . Substitutions within @TF$ that affect transcription activation at the @Regulated$ promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "9\n",
      "2\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [69, 74]\n",
      "Regulado: melAB\n",
      "Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR . Substitutions within @TF$ that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the @Regulated$ promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "25\n",
      "2\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [170, 175]\n",
      "Regulado: melAB\n",
      "Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR . Substitutions within CRP that affect transcription activation at the @Regulated$ promoter To identify the residues of @TF$ that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "9\n",
      "16\n",
      "Start y end del regulador: [112, 115]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [69, 74]\n",
      "Regulado: melAB\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for @TF$ in the @Regulated$ control region .\n",
      "42\n",
      "39\n",
      "Start y end del regulador: [261, 264]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [272, 276]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for @TF$ in the @Regulated$ control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "14\n",
      "11\n",
      "Start y end del regulador: [80, 83]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for @TF$ in the argO control region The formation of multiple Lrp â€“ @Regulated$ operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "23\n",
      "11\n",
      "Start y end del regulador: [80, 83]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for @TF$ in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the @Regulated$ control region .\n",
      "42\n",
      "11\n",
      "Start y end del regulador: [80, 83]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [272, 276]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the @Regulated$ control region The formation of multiple @TF$ â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "14\n",
      "21\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple @TF$ â€“ @Regulated$ operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "23\n",
      "21\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple @TF$ â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the @Regulated$ control region .\n",
      "42\n",
      "21\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [272, 276]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the @Regulated$ control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for @TF$ in the argO control region .\n",
      "14\n",
      "39\n",
      "Start y end del regulador: [261, 264]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: argO\n",
      "In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region . In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ @Regulated$ operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for @TF$ in the argO control region .\n",
      "23\n",
      "39\n",
      "Start y end del regulador: [261, 264]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [145, 149]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for @Regulated$ , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "6\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with @Regulated$ ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "62\n",
      "14\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [291, 295]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for @Regulated$ , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by @TF$ , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "6\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by @TF$  , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with @Regulated$ ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "62\n",
      "38\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [291, 295]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for @Regulated$ , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "6\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive . Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with @Regulated$ ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "62\n",
      "82\n",
      "Start y end del regulador: [390, 394]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [291, 295]\n",
      "Regulado: argO\n",
      "The results described above indicated that ArgP binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression . The results described above indicated that @TF$ binds the @Regulated$ regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [58, 62]\n",
      "Regulado: lysP\n",
      "The results described above indicated that ArgP binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression . The results described above indicated that @TF$ binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of @Regulated$ expression .\n",
      "46\n",
      "6\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [279, 283]\n",
      "Regulado: lysP\n",
      "The results described above indicated that ArgP binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression . The results described above indicated that ArgP binds the @Regulated$ regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such @TF$ binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression .\n",
      "9\n",
      "27\n",
      "Start y end del regulador: [169, 173]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [58, 62]\n",
      "Regulado: lysP\n",
      "The results described above indicated that ArgP binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such ArgP binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of lysP expression . The results described above indicated that ArgP binds the lysP regulatory region to mediate its nearly 35 - fold transcriptional activation and that the absence of such @TF$ binding either in argP mutants or upon Lys supplementation in argP strains results in very low levels of @Regulated$ expression .\n",
      "46\n",
      "27\n",
      "Start y end del regulador: [169, 173]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [279, 283]\n",
      "Regulado: lysP\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and @Regulated$ are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "43\n",
      "51\n",
      "Start y end del regulador: [240, 243]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [200, 204]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for @Regulated$ transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "12\n",
      "84\n",
      "Start y end del regulador: [392, 395]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [56, 62]\n",
      "Regulado: araBAD\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that @Regulated$ and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "41\n",
      "84\n",
      "Start y end del regulador: [392, 395]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [189, 195]\n",
      "Regulado: araBAD\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for @Regulated$ transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "12\n",
      "51\n",
      "Start y end del regulador: [240, 243]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [56, 62]\n",
      "Regulado: araBAD\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and @Regulated$ are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "43\n",
      "51\n",
      "Start y end del regulador: [240, 243]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [200, 204]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for @Regulated$ transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "27\n",
      "51\n",
      "Start y end del regulador: [240, 243]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [125, 129]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to @Regulated$ protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "77\n",
      "51\n",
      "Start y end del regulador: [240, 243]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [353, 357]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for @Regulated$ transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "27\n",
      "84\n",
      "Start y end del regulador: [392, 395]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [125, 129]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and @Regulated$ are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "43\n",
      "84\n",
      "Start y end del regulador: [392, 395]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [200, 204]\n",
      "Regulado: araC\n",
      "Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc . Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to @Regulated$ protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "77\n",
      "84\n",
      "Start y end del regulador: [392, 395]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [353, 357]\n",
      "Regulado: araC\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .  @TF$ represses the @Regulated$ promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "3\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [18, 21]\n",
      "Regulado: ogt\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .  @TF$  represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the @Regulated$ promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "19\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [103, 106]\n",
      "Regulado: ogt\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .  @TF$ represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the @Regulated$ : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "25\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [136, 139]\n",
      "Regulado: ogt\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 . Fis represses the @Regulated$ promoter by displacing upstream - bound NarL To investigate a role of @TF$ at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "3\n",
      "16\n",
      "Start y end del regulador: [92, 95]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [18, 21]\n",
      "Regulado: ogt\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 . Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of @TF$ at the @Regulated$ promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "19\n",
      "16\n",
      "Start y end del regulador: [92, 95]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [103, 106]\n",
      "Regulado: ogt\n",
      "Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 . Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of @TF$ at the ogt promoter , expression of the @Regulated$ : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "25\n",
      "16\n",
      "Start y end del regulador: [92, 95]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [136, 139]\n",
      "Regulado: ogt\n",
      "In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed CRP carrying the TI158 substitution that improves CRP - dependent activation of the melAB promoter in vivo . In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed CRP carrying the TI158 substitution that improves @TF$ - dependent activation of the @Regulated$ promoter in vivo .\n",
      "38\n",
      "32\n",
      "Start y end del regulador: [187, 190]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [221, 226]\n",
      "Regulado: melAB\n",
      "In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed CRP carrying the TI158 substitution that improves CRP - dependent activation of the melAB promoter in vivo . In order to investigate further the role of residues in and around AR1 in transcription activation at the @Regulated$ promoter , we puri Â® ed @TF$ carrying the TI158 substitution that improves CRP - dependent activation of the melAB promoter in vivo .\n",
      "18\n",
      "25\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [106, 111]\n",
      "Regulado: melAB\n",
      "In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed CRP carrying the TI158 substitution that improves CRP - dependent activation of the melAB promoter in vivo . In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed @TF$ carrying the TI158 substitution that improves CRP - dependent activation of the @Regulated$ promoter in vivo .\n",
      "38\n",
      "25\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [221, 226]\n",
      "Regulado: melAB\n",
      "In order to investigate further the role of residues in and around AR1 in transcription activation at the melAB promoter , we puri Â® ed CRP carrying the TI158 substitution that improves CRP - dependent activation of the melAB promoter in vivo . In order to investigate further the role of residues in and around AR1 in transcription activation at the @Regulated$ promoter , we puri Â® ed CRP carrying the TI158 substitution that improves @TF$ - dependent activation of the melAB promoter in vivo .\n",
      "18\n",
      "32\n",
      "Start y end del regulador: [187, 190]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [106, 111]\n",
      "Regulado: melAB\n",
      "Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters . Our results indicate that the repression of transcription from the two @Regulated$ promoters ( 1P and 2P ) by @TF$ protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "11\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [71, 75]\n",
      "Regulado: dnaA\n",
      "Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters . Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by @TF$ protein is promoted by the oligomerization of DnaA protein over @Regulated$ promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "30\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [172, 176]\n",
      "Regulado: dnaA\n",
      "Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters . Our results indicate that the repression of transcription from the two @Regulated$ promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of @TF$ protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "11\n",
      "27\n",
      "Start y end del regulador: [154, 158]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [71, 75]\n",
      "Regulado: dnaA\n",
      "Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters . Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of @TF$ protein over @Regulated$ promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "30\n",
      "27\n",
      "Start y end del regulador: [154, 158]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [172, 176]\n",
      "Regulado: dnaA\n",
      "The role of different MelR sites in activation of the melAB promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the melAB transcript start Fig . 6 . The role of different @TF$ sites in activation of the @Regulated$ promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the melAB transcript start Fig . 6 .\n",
      "10\n",
      "4\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [54, 59]\n",
      "Regulado: melAB\n",
      "The role of different MelR sites in activation of the melAB promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the melAB transcript start Fig . 6 . The role of different @TF$ sites in activation of the melAB promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the @Regulated$ transcript start Fig . 6 .\n",
      "26\n",
      "4\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [162, 167]\n",
      "Regulado: melAB\n",
      "The role of different MelR sites in activation of the melAB promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the melAB transcript start Fig . 6 . The role of different MelR sites in activation of the @Regulated$ promoter The above footprinting experiments suggest that five @TF$ molecules can bind upstream of the melAB transcript start Fig . 6 .\n",
      "10\n",
      "19\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [54, 59]\n",
      "Regulado: melAB\n",
      "The role of different MelR sites in activation of the melAB promoter The above footprinting experiments suggest that five MelR molecules can bind upstream of the melAB transcript start Fig . 6 . The role of different MelR sites in activation of the melAB promoter The above footprinting experiments suggest that five @TF$ molecules can bind upstream of the @Regulated$ transcript start Fig . 6 .\n",
      "26\n",
      "19\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [162, 167]\n",
      "Regulado: melAB\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) @TF$ is intrinsically a more potent activator for P @Regulated$ as compared with Lrp .\n",
      "53\n",
      "44\n",
      "Start y end del regulador: [211, 215]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [263, 267]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "15\n",
      "12\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P @Regulated$ as compared with Lrp .\n",
      "53\n",
      "12\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [263, 267]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and ArgP stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) @TF$ is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "15\n",
      "44\n",
      "Start y end del regulador: [211, 215]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both @TF$ and ArgP stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "15\n",
      "10\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both @TF$ and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P @Regulated$ as compared with Lrp .\n",
      "53\n",
      "10\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [263, 267]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and ArgP stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with @TF$ .\n",
      "15\n",
      "57\n",
      "Start y end del regulador: [285, 288]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: argO\n",
      "The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp . The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P @Regulated$ as compared with @TF$ .\n",
      "53\n",
      "57\n",
      "Start y end del regulador: [285, 288]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [263, 267]\n",
      "Regulado: argO\n",
      "In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) . In the case of @Regulated$ , @TF$ mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "4\n",
      "6\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: argO\n",
      "In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) . In the case of argO , @TF$ mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "44\n",
      "6\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [207, 211]\n",
      "Regulado: argO\n",
      "In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) . In the case of @Regulated$ , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by @TF$ of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "4\n",
      "47\n",
      "Start y end del regulador: [224, 228]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: argO\n",
      "In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) . In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by @TF$ of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "44\n",
      "47\n",
      "Start y end del regulador: [224, 228]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [207, 211]\n",
      "Regulado: argO\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that @TF$ plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "35\n",
      "25\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [186, 190]\n",
      "Regulado: dcuB\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of @Regulated$ were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that @TF$ plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "8\n",
      "25\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dcuB\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that @TF$ plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of @Regulated$ transcription .\n",
      "62\n",
      "25\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [333, 337]\n",
      "Regulado: dcuB\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of @Regulated$ were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that @TF$ is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "8\n",
      "48\n",
      "Start y end del regulador: [246, 250]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: dcuB\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that @TF$ is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "35\n",
      "48\n",
      "Start y end del regulador: [246, 250]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [186, 190]\n",
      "Regulado: dcuB\n",
      "Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription . Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that @TF$ is not responsible for the FNR - independent mechanism of anaerobic activation of @Regulated$ transcription .\n",
      "62\n",
      "48\n",
      "Start y end del regulador: [246, 250]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [333, 337]\n",
      "Regulado: dcuB\n",
      "When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) . When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , @TF$ inhibited @Regulated$ transcription ( see Fig . 6B ) .\n",
      "26\n",
      "24\n",
      "Start y end del regulador: [104, 108]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [119, 122]\n",
      "Regulado: rob\n",
      "When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) . When RNAP was added to the @Regulated$ promoter at 0 Â° C prior to @TF$ addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) .\n",
      "6\n",
      "14\n",
      "Start y end del regulador: [59, 63]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [27, 30]\n",
      "Regulado: rob\n",
      "When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) . When RNAP was added to the rob promoter at 0 Â° C prior to @TF$ addition and then transfer to 37 Â° C , MarA inhibited @Regulated$ transcription ( see Fig . 6B ) .\n",
      "26\n",
      "14\n",
      "Start y end del regulador: [59, 63]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [119, 122]\n",
      "Regulado: rob\n",
      "When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) . When RNAP was added to the @Regulated$ promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , @TF$ inhibited rob transcription ( see Fig . 6B ) .\n",
      "6\n",
      "24\n",
      "Start y end del regulador: [104, 108]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [27, 30]\n",
      "Regulado: rob\n",
      "To determine whether ArgP directly regulates lysP transcription , we tested binding of ArgP to the lysP promoter / control region . To determine whether @TF$ directly regulates @Regulated$ transcription , we tested binding of ArgP to the lysP promoter / control region .\n",
      "6\n",
      "3\n",
      "Start y end del regulador: [21, 25]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: lysP\n",
      "To determine whether ArgP directly regulates lysP transcription , we tested binding of ArgP to the lysP promoter / control region . To determine whether @TF$ directly regulates lysP transcription , we tested binding of ArgP to the @Regulated$ promoter / control region .\n",
      "16\n",
      "3\n",
      "Start y end del regulador: [21, 25]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [99, 103]\n",
      "Regulado: lysP\n",
      "To determine whether ArgP directly regulates lysP transcription , we tested binding of ArgP to the lysP promoter / control region . To determine whether ArgP directly regulates @Regulated$ transcription , we tested binding of @TF$ to the lysP promoter / control region .\n",
      "6\n",
      "13\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: lysP\n",
      "To determine whether ArgP directly regulates lysP transcription , we tested binding of ArgP to the lysP promoter / control region . To determine whether ArgP directly regulates lysP transcription , we tested binding of @TF$ to the @Regulated$ promoter / control region .\n",
      "16\n",
      "13\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [99, 103]\n",
      "Regulado: lysP\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that @TF$ was required for activation of @Regulated$ expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that @TF$ was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the @Regulated$ promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "27\n",
      "8\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that @TF$ was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to @Regulated$ promoter DNA had not been obtained .\n",
      "38\n",
      "8\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of @Regulated$ expression ( 40 ) suggested that @TF$ binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "14\n",
      "21\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that @TF$ binds to DNA at the @Regulated$ promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "27\n",
      "21\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that @TF$ binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to @Regulated$ promoter DNA had not been obtained .\n",
      "38\n",
      "21\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of @Regulated$ expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of @TF$ protein binding to rhaT promoter DNA had not been obtained .\n",
      "14\n",
      "34\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the @Regulated$ promoter , but direct evidence of @TF$ protein binding to rhaT promoter DNA had not been obtained .\n",
      "27\n",
      "34\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [160, 164]\n",
      "Regulado: rhaT\n",
      "DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained . DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of @TF$ protein binding to @Regulated$ promoter DNA had not been obtained .\n",
      "38\n",
      "34\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [223, 227]\n",
      "Regulado: rhaT\n",
      "Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli . Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) @TF$ is a transcriptional activator of @Regulated$ , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "25\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [144, 148]\n",
      "Regulado: yggA\n",
      "Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli . Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) @TF$ is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) @Regulated$ encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "48\n",
      "19\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [242, 246]\n",
      "Regulado: yggA\n",
      "Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli . Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of @Regulated$ , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "25\n",
      "30\n",
      "Start y end del regulador: [158, 162]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [144, 148]\n",
      "Regulado: yggA\n",
      "Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli . Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) @Regulated$ encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "48\n",
      "30\n",
      "Start y end del regulador: [158, 162]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [242, 246]\n",
      "Regulado: yggA\n",
      "LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 )   @TF$ also mediates Lys repression ( of @Regulated$ ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) \n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: LysR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: lysA\n",
      "LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 )   @TF$ also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for @Regulated$ ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) \n",
      "31\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: LysR\n",
      "Start y end del regulado: [166, 170]\n",
      "Regulado: lysA\n",
      "LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 )  LysR also mediates Lys repression ( of @Regulated$ ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to @TF$ converts it into its activator conformation ( 30 , 36 ) \n",
      "7\n",
      "41\n",
      "Start y end del regulador: [210, 214]\n",
      "Regulador: LysR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: lysA\n",
      "LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 )  LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for @Regulated$ ) and that the latter â€™ s binding to @TF$ converts it into its activator conformation ( 30 , 36 ) \n",
      "31\n",
      "41\n",
      "Start y end del regulador: [210, 214]\n",
      "Regulador: LysR\n",
      "Start y end del regulado: [166, 170]\n",
      "Regulado: lysA\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that @Regulated$ is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "11\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected @Regulated$ runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "38\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [197, 201]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for @Regulated$ ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "90\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [457, 461]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the @Regulated$ coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "102\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [505, 509]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein @Regulated$ is annotated ygfD ) .\n",
      "125\n",
      "19\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [630, 634]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that @Regulated$ is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "11\n",
      "76\n",
      "Start y end del regulador: [392, 396]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected @Regulated$ runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "38\n",
      "76\n",
      "Start y end del regulador: [392, 396]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [197, 201]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for @Regulated$ ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "90\n",
      "76\n",
      "Start y end del regulador: [392, 396]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [457, 461]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the @Regulated$ coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "102\n",
      "76\n",
      "Start y end del regulador: [392, 396]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [505, 509]\n",
      "Regulado: argK\n",
      "Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) . Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein @Regulated$ is annotated ygfD ) .\n",
      "125\n",
      "76\n",
      "Start y end del regulador: [392, 396]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [630, 634]\n",
      "Regulado: argK\n",
      "Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter . Role of @TF$ in activation of the @Regulated$ promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter .\n",
      "7\n",
      "2\n",
      "Start y end del regulador: [8, 11]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [33, 38]\n",
      "Regulado: melAB\n",
      "Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter . Role of @TF$ in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the @Regulated$ promoter .\n",
      "25\n",
      "2\n",
      "Start y end del regulador: [8, 11]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [144, 149]\n",
      "Regulado: melAB\n",
      "Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter . Role of CRP in activation of the @Regulated$ promoter The results from the in vitro studies reported above suggest that @TF$ interacts directly at the melAB promoter .\n",
      "7\n",
      "20\n",
      "Start y end del regulador: [114, 117]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [33, 38]\n",
      "Regulado: melAB\n",
      "Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter . Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that @TF$ interacts directly at the @Regulated$ promoter .\n",
      "25\n",
      "20\n",
      "Start y end del regulador: [114, 117]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [144, 149]\n",
      "Regulado: melAB\n",
      "Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold . Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed @TF$ at @Regulated$ , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold .\n",
      "27\n",
      "25\n",
      "Start y end del regulador: [155, 159]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [163, 168]\n",
      "Regulado: rhaSR\n",
      "Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold . Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed @TF$ at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating @Regulated$ by approximately 30 - fold .\n",
      "53\n",
      "25\n",
      "Start y end del regulador: [155, 159]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [310, 315]\n",
      "Regulado: rhaSR\n",
      "Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold . Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - @TF$ - CTD activating @Regulated$ by approximately 30 - fold .\n",
      "53\n",
      "49\n",
      "Start y end del regulador: [288, 292]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [310, 315]\n",
      "Regulado: rhaSR\n",
      "Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold . Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at @Regulated$ , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - @TF$ - CTD activating rhaSR by approximately 30 - fold .\n",
      "27\n",
      "49\n",
      "Start y end del regulador: [288, 292]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [163, 168]\n",
      "Regulado: rhaSR\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , @TF$ binds the @Regulated$ regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region .\n",
      "6\n",
      "3\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: lysP\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , @TF$ binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses @Regulated$ in vivo by engendering the loss of ArgP binding to the lysP operator region .\n",
      "42\n",
      "3\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [195, 199]\n",
      "Regulado: lysP\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , @TF$ binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the @Regulated$ operator region .\n",
      "54\n",
      "3\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [255, 259]\n",
      "Regulado: lysP\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , ArgP binds the @Regulated$ regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of @TF$ binding to the lysP operator region .\n",
      "6\n",
      "50\n",
      "Start y end del regulador: [235, 239]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: lysP\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses @Regulated$ in vivo by engendering the loss of @TF$ binding to the lysP operator region .\n",
      "42\n",
      "50\n",
      "Start y end del regulador: [235, 239]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [195, 199]\n",
      "Regulado: lysP\n",
      "In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region . In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of @TF$ binding to the @Regulated$ operator region .\n",
      "54\n",
      "50\n",
      "Start y end del regulador: [235, 239]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [255, 259]\n",
      "Regulado: lysP\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the melR promoter Our results show that @TF$ - dependent repression of the @Regulated$ promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "20\n",
      "14\n",
      "Start y end del regulador: [76, 80]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of @TF$ binding at the @Regulated$ promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of @TF$ binding at the melR promoter Our results show that MelR - dependent repression of the @Regulated$ promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "20\n",
      "4\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the @Regulated$ promoter Our results show that @TF$ - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "8\n",
      "14\n",
      "Start y end del regulador: [76, 80]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the @Regulated$ promoter Our results show that MelR - dependent repression of the melR promoter is contingent on @TF$ binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "8\n",
      "25\n",
      "Start y end del regulador: [142, 146]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the @Regulated$ promoter is contingent on @TF$ binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "20\n",
      "25\n",
      "Start y end del regulador: [142, 146]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the @Regulated$ promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by @TF$ binding to upstream sites 2 , 1 and 1 0 .\n",
      "8\n",
      "35\n",
      "Start y end del regulador: [193, 197]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: melR\n",
      "In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 . In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the @Regulated$ promoter is contingent on MelR binding to site R and is somehow modulated by @TF$ binding to upstream sites 2 , 1 and 1 0 .\n",
      "20\n",
      "35\n",
      "Start y end del regulador: [193, 197]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: melR\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of @TF$ in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of @Regulated$ is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "30\n",
      "4\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: nagE\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of CAP in repression of @Regulated$ â€“ B is likely to be purely architectural , it is clear that a major part of the @TF$ activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "8\n",
      "27\n",
      "Start y end del regulador: [129, 132]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: nagE\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the @TF$ activation of @Regulated$ is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "30\n",
      "27\n",
      "Start y end del regulador: [129, 132]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: nagE\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of CAP in repression of @Regulated$ â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between @TF$ and RNA polymerase .\n",
      "8\n",
      "39\n",
      "Start y end del regulador: [199, 202]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: nagE\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of @Regulated$ is due to protein â€“ protein contacts between @TF$ and RNA polymerase .\n",
      "30\n",
      "39\n",
      "Start y end del regulador: [199, 202]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: nagE\n",
      "Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase . Although the role of @TF$ in repression of @Regulated$ â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: nagE\n",
      "The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression . The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that @TF$ , but not NarP , represses @Regulated$ expression .\n",
      "42\n",
      "35\n",
      "Start y end del regulador: [195, 199]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [227, 231]\n",
      "Regulado: dcuB\n",
      "The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression . The organization of the putative @TF$ sites in the @Regulated$ operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression .\n",
      "9\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: dcuB\n",
      "The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression . The organization of the putative @TF$ sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses @Regulated$ expression .\n",
      "42\n",
      "5\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [227, 231]\n",
      "Regulado: dcuB\n",
      "The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression . The organization of the putative NarL sites in the @Regulated$ operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that @TF$ , but not NarP , represses dcuB expression .\n",
      "9\n",
      "35\n",
      "Start y end del regulador: [195, 199]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: dcuB\n",
      "Molecular Microbiology ( 2009 ) 74 ( 6 ) , 1513 – 1526 doi : 10.1111 / j . 1365 - 2958.2009.06950 . x First published online 19 November 2009 Competitive activation of the Escherichia coli argO gene coding for an arginine exporter by the transcriptional regulators Lrp and ArgPmmi _ 6950 1513 .. 1526 Eveline Peeters , Phu Nguyen Le Minh , Maria Foulquié - Moreno and Daniel Charlier * Erfelijkheidsleer en Microbiologie , Vrije Universiteit Brussel ( VUB ) , Pleinlaan 2 , B - 1050 Brussels , Belgium . Molecular Microbiology ( 2009 ) 74 ( 6 ) , 1513 – 1526 doi : 10.1111 / j . 1365 - 2958.2009.06950 . x First published online 19 November 2009 Competitive activation of the Escherichia coli @Regulated$ gene coding for an arginine exporter by the transcriptional regulators @TF$ and ArgPmmi _ 6950 1513 .. 1526 Eveline Peeters , Phu Nguyen Le Minh , Maria Foulquié - Moreno and Daniel Charlier * Erfelijkheidsleer en Microbiologie , Vrije Universiteit Brussel ( VUB ) , Pleinlaan 2 , B - 1050 Brussels , Belgium .\n",
      "36\n",
      "47\n",
      "Start y end del regulador: [265, 268]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: argO\n",
      "Molecular Microbiology ( 1996 ) 19 ( 2 ) , 389 – 396 Transcriptional activation of the dnaA gene encoding the initiator for oriC replication by IciA protein , an inhibitor of in vitro oriC replication in Escherichia coli Yong Sun Lee , 1 , 2 Hakjung Kim 1 and Deog Su Hwang 1 * 1 The Institute for Molecular Biology and Genetics , and 2 Department of Microbiology , Seoul National University , Seoul 151 - 742 , Korea . Molecular Microbiology ( 1996 ) 19 ( 2 ) , 389 – 396 Transcriptional activation of the @Regulated$ gene encoding the initiator for oriC replication by @TF$ protein , an inhibitor of in vitro oriC replication in Escherichia coli Yong Sun Lee , 1 , 2 Hakjung Kim 1 and Deog Su Hwang 1 * 1 The Institute for Molecular Biology and Genetics , and 2 Department of Microbiology , Seoul National University , Seoul 151 - 742 , Korea .\n",
      "17\n",
      "26\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [87, 91]\n",
      "Regulado: dnaA\n",
      "Molecular Microbiology ( 2000 ) 36 ( 1 ) , 211 ± 222 Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein Tamara A . Molecular Microbiology ( 2000 ) 36 ( 1 ) , 211 ± 222 Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein Tamara A .\n",
      "19\n",
      "25\n",
      "Start y end del regulador: [131, 135]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [102, 107]\n",
      "Regulado: melAB\n",
      "Molecular Microbiology ( 2000 ) 36 ( 1 ) , 223 ± 229 Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop Joseph T . Molecular Microbiology ( 2000 ) 36 ( 1 ) , 223 ± 229 Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop Joseph T .\n",
      "18\n",
      "21\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [88, 92]\n",
      "Regulado: melR\n",
      "36 , No . 8 2667 – 2676 doi : 10.1093 / nar / gkn119 Autoregulation of the Escherichia coli melR promoter : repression involves four molecules of MelR Shivanthi Samarasinghe 1 , Mohamed Samir El - Robh 1 , David C . 36 , No . 8 2667 – 2676 doi : 10.1093 / nar / gkn119 Autoregulation of the Escherichia coli @Regulated$ promoter : repression involves four molecules of @TF$ Shivanthi Samarasinghe 1 , Mohamed Samir El - Robh 1 , David C .\n",
      "20\n",
      "28\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: melR\n",
      "174 , No . 15 Localization of Upstream Sequence Elements Required for Nitrate and Anaerobic Induction of fdn ( Formate Dehydrogenase - N ) Operon Expression in Escherichia coli K - 12 JING LI ' AND VALLEY STEWART ' 2 * Sections of Microbiology ' and Genetics &amp; Development , 2 Cornell University , Ithaca , New York 14853 - 8101 Received 25 March 1992 / Accepted 19 May 1992 Two transcriptional activators , the FNR and NARL proteins , are required for induction of the fdnGHI operon , encoding Escherichia coli formate dehydrogenase - N . 174 , No . 15 Localization of Upstream Sequence Elements Required for Nitrate and Anaerobic Induction of fdn ( Formate Dehydrogenase - N ) Operon Expression in Escherichia coli K - 12 JING LI ' AND VALLEY STEWART ' 2 * Sections of Microbiology ' and Genetics &amp; Development , 2 Cornell University , Ithaca , New York 14853 - 8101 Received 25 March 1992 / Accepted 19 May 1992 Two transcriptional activators , the @TF$ and NARL proteins , are required for induction of the @Regulated$ operon , encoding Escherichia coli formate dehydrogenase - N .\n",
      "86\n",
      "75\n",
      "Start y end del regulador: [416, 419]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [474, 480]\n",
      "Regulado: fdnGHI\n",
      "Primer extension and lacZ - operon fusion analyses revealed that gntT has one strong and two weak promoters , all of which are regulated positively by cAMP - CRP and negatively by GntR . Primer extension and lacZ - operon fusion analyses revealed that @Regulated$ has one strong and two weak promoters , all of which are regulated positively by cAMP - CRP and negatively by @TF$ .\n",
      "10\n",
      "32\n",
      "Start y end del regulador: [180, 184]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: gntT\n",
      "NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose . NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while @TF$ represses three PTS operons , ptsG , @Regulated$ and ptsHIcrr , involved in the uptake of glucose .\n",
      "22\n",
      "14\n",
      "Start y end del regulador: [88, 91]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [129, 135]\n",
      "Regulado: manXYZ\n",
      "NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose . @TF$ represses @Regulated$ , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .\n",
      "2\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: nagE\n",
      "NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose . NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while @TF$ represses three PTS operons , @Regulated$ , manXYZ and ptsHIcrr , involved in the uptake of glucose .\n",
      "20\n",
      "14\n",
      "Start y end del regulador: [88, 91]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [122, 126]\n",
      "Regulado: ptsG\n",
      "NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose . NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while @TF$ represses three PTS operons , ptsG , manXYZ and @Regulated$ , involved in the uptake of glucose .\n",
      "24\n",
      "14\n",
      "Start y end del regulador: [88, 91]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [140, 148]\n",
      "Regulado: ptsHIcrr\n",
      "( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and Fis - dependent repression controls expression from the Escherichia coli yeaR and ogt promoters Derrick J . ( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and @TF$ - dependent repression controls expression from the Escherichia coli yeaR and @Regulated$ promoters Derrick J .\n",
      "39\n",
      "27\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [219, 222]\n",
      "Regulado: ogt\n",
      "( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and Fis - dependent repression controls expression from the Escherichia coli yeaR and ogt promoters Derrick J . ( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between @TF$ - dependent activation and Fis - dependent repression controls expression from the Escherichia coli yeaR and @Regulated$ promoters Derrick J .\n",
      "39\n",
      "22\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [219, 222]\n",
      "Regulado: ogt\n",
      "( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and Fis - dependent repression controls expression from the Escherichia coli yeaR and ogt promoters Derrick J . ( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and @TF$ - dependent repression controls expression from the Escherichia coli @Regulated$ and ogt promoters Derrick J .\n",
      "37\n",
      "27\n",
      "Start y end del regulador: [137, 140]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [210, 214]\n",
      "Regulado: yeaR\n",
      "( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between NarL - dependent activation and Fis - dependent repression controls expression from the Escherichia coli yeaR and ogt promoters Derrick J . ( 2009 ) 420 , 249 – 257 ( Printed in Great Britain ) doi : 10.1042 / BJ20090183 249 Competition between @TF$ - dependent activation and Fis - dependent repression controls expression from the Escherichia coli @Regulated$ and ogt promoters Derrick J .\n",
      "37\n",
      "22\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [210, 214]\n",
      "Regulado: yeaR\n",
      "Activation by Lrp interferes with the previously demonstrated activation of the argO promoter by ArgP . Activation by Lrp interferes with the previously demonstrated activation of the @Regulated$ promoter by @TF$ .\n",
      "11\n",
      "14\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: argO\n",
      "Activation by Lrp interferes with the previously demonstrated activation of the argO promoter by ArgP . Activation by @TF$ interferes with the previously demonstrated activation of the @Regulated$ promoter by ArgP .\n",
      "11\n",
      "2\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: argO\n",
      "Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate . Summary Expression of the narK gene of Escherichia coli , like the @Regulated$ operon , is positively regulated by two frans - acting factors : @TF$ , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .\n",
      "12\n",
      "25\n",
      "Start y end del regulador: [140, 143]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [67, 74]\n",
      "Regulado: narGHJI\n",
      "Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate . Summary Expression of the @Regulated$ gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and @TF$ , which is activated by the presence of nitrate .\n",
      "4\n",
      "35\n",
      "Start y end del regulador: [195, 199]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: narK\n",
      "Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , AraC , the activator - repressor that regulates the araBAD operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another . Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , @TF$ , the activator - repressor that regulates the @Regulated$ operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another .\n",
      "31\n",
      "22\n",
      "Start y end del regulador: [132, 136]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [184, 190]\n",
      "Regulado: araBAD\n",
      "JOHNSON AND ROBERT F . SCHLEIF * Department of Biology , Johns Hopkins University , Baltimore , Maryland 21218 Received 20 September 1999 / Accepted 12 January 2000 Full activation of transcription of the araFGH promoter , p FGH , requires both the catabolite activator protein ( CAP ) and AraC protein . JOHNSON AND ROBERT F . SCHLEIF * Department of Biology , Johns Hopkins University , Baltimore , Maryland 21218 Received 20 September 1999 / Accepted 12 January 2000 Full activation of transcription of the @Regulated$ promoter , p FGH , requires both the catabolite activator protein ( CAP ) and @TF$ protein .\n",
      "34\n",
      "50\n",
      "Start y end del regulador: [290, 294]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [205, 211]\n",
      "Regulado: araFGH\n",
      "JOHNSON AND ROBERT F . SCHLEIF * Department of Biology , Johns Hopkins University , Baltimore , Maryland 21218 Received 20 September 1999 / Accepted 12 January 2000 Full activation of transcription of the araFGH promoter , p FGH , requires both the catabolite activator protein ( CAP ) and AraC protein . JOHNSON AND ROBERT F . SCHLEIF * Department of Biology , Johns Hopkins University , Baltimore , Maryland 21218 Received 20 September 1999 / Accepted 12 January 2000 Full activation of transcription of the @Regulated$ promoter , p FGH , requires both the catabolite activator protein ( @TF$ ) and AraC protein .\n",
      "34\n",
      "47\n",
      "Start y end del regulador: [280, 283]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [205, 211]\n",
      "Regulado: araFGH\n",
      "GntR represses the GntI genes gntKU and gntT , whereas GntH was previously suggested to be an activator for the GntII genes gntV and idnDO - gntWH . @TF$ represses the GntI genes @Regulated$ and gntT , whereas GntH was previously suggested to be an activator for the GntII genes gntV and idnDO - gntWH .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [30, 35]\n",
      "Regulado: gntKU\n",
      "GntR represses the GntI genes gntKU and gntT , whereas GntH was previously suggested to be an activator for the GntII genes gntV and idnDO - gntWH . @TF$ represses the GntI genes gntKU and @Regulated$ , whereas GntH was previously suggested to be an activator for the GntII genes gntV and idnDO - gntWH .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: gntT\n",
      "Optimal repression requires MelR binding to a site that overlaps the melR transcription start point and to upstream sites . Optimal repression requires @TF$ binding to a site that overlaps the @Regulated$ transcription start point and to upstream sites .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [28, 32]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: melR\n",
      "GntR represses the GntI genes gntKU and gntT , whereas GntH was previously suggested to be an activator for the GntII genes gntV and idnDO - gntWH . GntR represses the GntI genes gntKU and gntT , whereas @TF$ was previously suggested to be an activator for the GntII genes @Regulated$ and idnDO - gntWH .\n",
      "22\n",
      "10\n",
      "Start y end del regulador: [55, 59]\n",
      "Regulador: GntH\n",
      "Start y end del regulado: [124, 128]\n",
      "Regulado: gntV\n",
      "476 - 480 , January 1989 Biochemistry In vivo DNA loops in araCBAD : Size limits and helical repeat ( supercoiling / twist / periodicity / linking number deficit / looping energetics ) DONG - HEE LEE AND ROBERT F . SCHLEIF Biochemistry Department , Brandeis University , Waltham , MA 02254 Communicated by Alexander Rich , October 14 , 1988 ABSTRACT Formation of a DNA loop by AraC proteins bound at the aral and araO2 sites , whose center - to - center distance is 211 base pairs , is necessary for repression of the araBAD promoter , PBAD , of Escherichia coli . 476 - 480 , January 1989 Biochemistry In vivo DNA loops in araCBAD : Size limits and helical repeat ( supercoiling / twist / periodicity / linking number deficit / looping energetics ) DONG - HEE LEE AND ROBERT F . SCHLEIF Biochemistry Department , Brandeis University , Waltham , MA 02254 Communicated by Alexander Rich , October 14 , 1988 ABSTRACT Formation of a DNA loop by @TF$ proteins bound at the aral and araO2 sites , whose center - to - center distance is 211 base pairs , is necessary for repression of the @Regulated$ promoter , PBAD , of Escherichia coli .\n",
      "96\n",
      "68\n",
      "Start y end del regulador: [377, 381]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [518, 524]\n",
      "Regulado: araBAD\n",
      "A melAB promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both MelR - binding sites . A @Regulated$ promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both @TF$ - binding sites .\n",
      "1\n",
      "24\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [2, 7]\n",
      "Regulado: melAB\n",
      "These findings suggest that IciA protein may regulate the initiation of chromosomal DNA replication at oriC by controlling expression of the dnaA gene , as well as by inhibiting the initiation of chromosomal DNA replication at oriC . These findings suggest that @TF$ protein may regulate the initiation of chromosomal DNA replication at oriC by controlling expression of the @Regulated$ gene , as well as by inhibiting the initiation of chromosomal DNA replication at oriC .\n",
      "21\n",
      "4\n",
      "Start y end del regulador: [28, 32]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: dnaA\n",
      "We have identi ® ed residues within activating region 1 ( AR1 ) of CRP that are important in transcription activation of the melAB promoter . We have identi ® ed residues within activating region 1 ( AR1 ) of @TF$ that are important in transcription activation of the @Regulated$ promoter .\n",
      "23\n",
      "14\n",
      "Start y end del regulador: [67, 70]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [125, 130]\n",
      "Regulado: melAB\n",
      "MelR stimulates transcription initiation at the melAB promoter by binding to four sites centered at positions 120.5 , 100.5 , 62.5 , and 42.5 upstream of the transcript start point . @TF$ stimulates transcription initiation at the @Regulated$ promoter by binding to four sites centered at positions 120.5 , 100.5 , 62.5 , and 42.5 upstream of the transcript start point .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [48, 53]\n",
      "Regulado: melAB\n",
      "Summary Expression from the Escherichia coli nrfA promoter ( pnrfA ) is activated by both the FNR protein ( an anaerobically triggered transcription activator ) and the NarL or NarP proteins ( transcription activators triggered by nitrite and nitrate ) . Summary Expression from the Escherichia coli @Regulated$ promoter ( pnrfA ) is activated by both the FNR protein ( an anaerobically triggered transcription activator ) and the @TF$ or NarP proteins ( transcription activators triggered by nitrite and nitrate ) .\n",
      "6\n",
      "27\n",
      "Start y end del regulador: [169, 173]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: nrfA\n",
      "Summary Expression from the Escherichia coli nrfA promoter ( pnrfA ) is activated by both the FNR protein ( an anaerobically triggered transcription activator ) and the NarL or NarP proteins ( transcription activators triggered by nitrite and nitrate ) . Summary Expression from the Escherichia coli @Regulated$ promoter ( pnrfA ) is activated by both the FNR protein ( an anaerobically triggered transcription activator ) and the NarL or @TF$ proteins ( transcription activators triggered by nitrite and nitrate ) .\n",
      "6\n",
      "29\n",
      "Start y end del regulador: [177, 181]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: nrfA\n",
      "RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT . RhaR activates transcription of rhaSR , and @TF$ activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , @Regulated$ , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "22\n",
      "7\n",
      "Start y end del regulador: [44, 48]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [137, 143]\n",
      "Regulado: rhaBAD\n",
      "RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT . @TF$ activates transcription of @Regulated$ , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [32, 37]\n",
      "Regulado: rhaSR\n",
      "RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT . RhaR activates transcription of rhaSR , and @TF$ activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , @Regulated$ .\n",
      "38\n",
      "7\n",
      "Start y end del regulador: [44, 48]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [218, 222]\n",
      "Regulado: rhaT\n",
      "In this study , we show that the global transcription factor FNR is a class II activator at the ynfEFGHI promoter and that NarL represses activation by binding to a single target that overlaps the promoter 10 element . In this study , we show that the global transcription factor FNR is a class II activator at the @Regulated$ promoter and that @TF$ represses activation by binding to a single target that overlaps the promoter 10 element .\n",
      "19\n",
      "23\n",
      "Start y end del regulador: [123, 127]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [96, 104]\n",
      "Regulado: ynfEFGHI\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to @Regulated$ , dapB , and gdhA ) .\n",
      "55\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [274, 278]\n",
      "Regulado: argO\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and @Regulated$ ( in addition to argO , dapB , and gdhA ) .\n",
      "50\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [253, 256]\n",
      "Regulado: asd\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , @Regulated$ , and gdhA ) .\n",
      "57\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [281, 285]\n",
      "Regulado: dapB\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , @Regulated$ , and asd ( in addition to argO , dapB , and gdhA ) .\n",
      "47\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [242, 246]\n",
      "Regulado: dapD\n",
      "In contrast , IHF does not enhance the ability of NanR to activate fimB expression in the wild - type background . In contrast , IHF does not enhance the ability of @TF$ to activate @Regulated$ expression in the wild - type background .\n",
      "13\n",
      "10\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: fimB\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and @Regulated$ ) .\n",
      "60\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [292, 296]\n",
      "Regulado: gdhA\n",
      "As for the hdeA and purA promoters , which are repressed by MarA , the rob marbox is also in the “ backward ” orientation . As for the @Regulated$ and purA promoters , which are repressed by @TF$ , the rob marbox is also in the “ backward ” orientation .\n",
      "3\n",
      "12\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: hdeA\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , @Regulated$ , dapD , and asd ( in addition to argO , dapB , and gdhA ) .\n",
      "45\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [235, 239]\n",
      "Regulado: lysA\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , @Regulated$ , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) .\n",
      "43\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [228, 232]\n",
      "Regulado: lysC\n",
      "In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) . In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including @Regulated$ , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) .\n",
      "41\n",
      "34\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [221, 225]\n",
      "Regulado: lysP\n",
      "Plasmids overproducing either the NagC protein or the Mlc protein repress the expression of manX , but the effect of the Mlc protein is stronger . Plasmids overproducing either the @TF$ protein or the Mlc protein repress the expression of @Regulated$ , but the effect of the Mlc protein is stronger .\n",
      "14\n",
      "4\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: manX\n",
      "As for the hdeA and purA promoters , which are repressed by MarA , the rob marbox is also in the “ backward ” orientation . As for the hdeA and @Regulated$ promoters , which are repressed by @TF$ , the rob marbox is also in the “ backward ” orientation .\n",
      "5\n",
      "12\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [20, 24]\n",
      "Regulado: purA\n",
      "nanC expression is also activated by the regulators cyclic AMP - catabolite activator protein , OmpR , and CpxR . @Regulated$ expression is also activated by the regulators cyclic AMP - catabolite activator protein , OmpR , and @TF$ .\n",
      "0\n",
      "18\n",
      "Start y end del regulador: [107, 111]\n",
      "Regulador: CpxR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: nanC\n",
      "nanC expression is also activated by the regulators cyclic AMP - catabolite activator protein , OmpR , and CpxR . @Regulated$ expression is also activated by the regulators cyclic AMP - catabolite activator protein , @TF$ , and CpxR .\n",
      "0\n",
      "15\n",
      "Start y end del regulador: [96, 100]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: nanC\n",
      "Combined , the effector modulated activation of argO transcription by ArgP and Lrp must ensure an adapted and fine - tuned synthesis of the transporter in response to environmental conditions . Combined , the effector modulated activation of @Regulated$ transcription by @TF$ and Lrp must ensure an adapted and fine - tuned synthesis of the transporter in response to environmental conditions .\n",
      "7\n",
      "10\n",
      "Start y end del regulador: [70, 74]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: argO\n",
      "Combined , the effector modulated activation of argO transcription by ArgP and Lrp must ensure an adapted and fine - tuned synthesis of the transporter in response to environmental conditions . Combined , the effector modulated activation of @Regulated$ transcription by ArgP and @TF$ must ensure an adapted and fine - tuned synthesis of the transporter in response to environmental conditions .\n",
      "7\n",
      "12\n",
      "Start y end del regulador: [79, 82]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: argO\n",
      "Purified His 6 - tagged ArgP protein binds with an apparent K d of 35 nM to the dapB promoter in a gel retardation assay , provided that sequences up to 103 are present . Purified His 6 - tagged @TF$ protein binds with an apparent K d of 35 nM to the @Regulated$ promoter in a gel retardation assay , provided that sequences up to 103 are present .\n",
      "18\n",
      "5\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: dapB\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , @Regulated$ , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "13\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [61, 64]\n",
      "Regulado: fpr\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , @Regulated$ , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "19\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [81, 84]\n",
      "Regulado: nfo\n",
      "Although most NarLdependent promoters are also co - dependent on a second transcription factor , FNR protein , two targets , the yeaR and ogt promoters , are activated by NarL alone with no involvement of FNR . Although most NarLdependent promoters are also co - dependent on a second transcription factor , FNR protein , two targets , the yeaR and @Regulated$ promoters , are activated by @TF$ alone with no involvement of FNR .\n",
      "24\n",
      "30\n",
      "Start y end del regulador: [171, 175]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [138, 141]\n",
      "Regulado: ogt\n",
      "We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer . We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and @Regulated$ , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "22\n",
      "3\n",
      "Start y end del regulador: [14, 17]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [91, 95]\n",
      "Regulado: sodA\n",
      "Although most NarLdependent promoters are also co - dependent on a second transcription factor , FNR protein , two targets , the yeaR and ogt promoters , are activated by NarL alone with no involvement of FNR . Although most NarLdependent promoters are also co - dependent on a second transcription factor , FNR protein , two targets , the @Regulated$ and ogt promoters , are activated by @TF$ alone with no involvement of FNR .\n",
      "22\n",
      "30\n",
      "Start y end del regulador: [171, 175]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [129, 133]\n",
      "Regulado: yeaR\n",
      "The gntR gene is constitutively expressed , gluconate permease , and a thermoresistant glu - whereas the gntKU genes are positively regulated by the cAMP – CRP complex and negatively by GntR , and gntU expression is further repressed by attenuation , * Corresponding author . The gntR gene is constitutively expressed , gluconate permease , and a thermoresistant glu - whereas the @Regulated$ genes are positively regulated by the cAMP – CRP complex and negatively by @TF$ , and gntU expression is further repressed by attenuation , * Corresponding author .\n",
      "17\n",
      "31\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [105, 110]\n",
      "Regulado: gntKU\n",
      "Northern blotting analysis indicated that RbsR represses the purHD operon for de novo synthesis of purine nucleotide but activates the add and udk genes involved in the salvage pathway of purine nucleotide synthesis . Northern blotting analysis indicated that @TF$ represses the purHD operon for de novo synthesis of purine nucleotide but activates the @Regulated$ and udk genes involved in the salvage pathway of purine nucleotide synthesis .\n",
      "20\n",
      "5\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [135, 138]\n",
      "Regulado: add\n",
      "Northern blotting analysis indicated that RbsR represses the purHD operon for de novo synthesis of purine nucleotide but activates the add and udk genes involved in the salvage pathway of purine nucleotide synthesis . Northern blotting analysis indicated that @TF$ represses the purHD operon for de novo synthesis of purine nucleotide but activates the add and @Regulated$ genes involved in the salvage pathway of purine nucleotide synthesis .\n",
      "22\n",
      "5\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [143, 146]\n",
      "Regulado: udk\n",
      "S1 mapping located transcriptional start points and showed that RhaR , and possibly RhaS , positively regulate transcription from thcb structural gene promoters as well as transcription from their own promoter . S1 mapping located transcriptional start points and showed that @TF$ , and possibly RhaS , positively regulate transcription from @Regulated$ structural gene promoters as well as transcription from their own promoter .\n",
      "19\n",
      "9\n",
      "Start y end del regulador: [64, 68]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [130, 134]\n",
      "Regulado: thcb\n",
      "S1 mapping located transcriptional start points and showed that RhaR , and possibly RhaS , positively regulate transcription from thcb structural gene promoters as well as transcription from their own promoter . S1 mapping located transcriptional start points and showed that RhaR , and possibly @TF$ , positively regulate transcription from @Regulated$ structural gene promoters as well as transcription from their own promoter .\n",
      "19\n",
      "13\n",
      "Start y end del regulador: [84, 88]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [130, 134]\n",
      "Regulado: thcb\n",
      "These results fit with a model in which ArgP contributes to enhanced transcription of dapB when lysine becomes limiting . * Corresponding author . These results fit with a model in which @TF$ contributes to enhanced transcription of @Regulated$ when lysine becomes limiting . * Corresponding author .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: dapB\n",
      "We found that purified MarA was sufficient in vitro to repress transcription of both purA and hdeA . We found that purified @TF$ was sufficient in vitro to repress transcription of both purA and @Regulated$ .\n",
      "16\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [94, 98]\n",
      "Regulado: hdeA\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , @Regulated$ , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "15\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [67, 70]\n",
      "Regulado: fpr\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , @Regulated$ , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "15\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [67, 70]\n",
      "Regulado: fpr\n",
      "Using in vitro assays and mutagenesis of promoter fusions in whole cells , we identified the cis regulatory element involved in MarA upregulation of the oxygeninsensitive nitroreductase nfnB gene . Using in vitro assays and mutagenesis of promoter fusions in whole cells , we identified the cis regulatory element involved in @TF$ upregulation of the oxygeninsensitive nitroreductase @Regulated$ gene .\n",
      "27\n",
      "21\n",
      "Start y end del regulador: [128, 132]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [186, 190]\n",
      "Regulado: nfnB\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , @Regulated$ , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "21\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [87, 90]\n",
      "Regulado: nfo\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , @Regulated$ , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "21\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [87, 90]\n",
      "Regulado: nfo\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and @Regulated$ ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "24\n",
      "5\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: sodA\n",
      "Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF . Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and @Regulated$ ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "24\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: sodA\n",
      "This loop helps prevent AraC protein from entering its inducing state and helps hold the uninduced , or basal level , of expression of the araBAD genes at a low level . This loop helps prevent @TF$ protein from entering its inducing state and helps hold the uninduced , or basal level , of expression of the @Regulated$ genes at a low level .\n",
      "25\n",
      "4\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [139, 145]\n",
      "Regulado: araBAD\n",
      "NagC repression is critical in maintaining chiP mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA . NagC repression is critical in maintaining @Regulated$ mRNA levels low enough , relative to @TF$ , to allow full silencing by this sRNA .\n",
      "6\n",
      "14\n",
      "Start y end del regulador: [85, 89]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: chiP\n",
      "NagC repression is critical in maintaining chiP mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA . @TF$ repression is critical in maintaining @Regulated$ mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: chiP\n",
      "These studies also show that the cyclic AMP receptor protein ( CRP ) interacts with the melAB promoter and increases MelR - dependent transcription activation . These studies also show that the cyclic AMP receptor protein ( CRP ) interacts with the @Regulated$ promoter and increases @TF$ - dependent transcription activation .\n",
      "16\n",
      "20\n",
      "Start y end del regulador: [117, 121]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [88, 93]\n",
      "Regulado: melAB\n",
      "In addition , it was discovered that RpoS activates adhE transcription at both start sites . In addition , it was discovered that @TF$ activates @Regulated$ transcription at both start sites .\n",
      "9\n",
      "7\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: RpoS\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: adhE\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , @Regulated$ , ybaO , mdlA , yfhD , and ybiS ) .\n",
      "26\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [157, 161]\n",
      "Regulado: aslB\n",
      "Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA . Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( inaA , @Regulated$ , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "17\n",
      "2\n",
      "Start y end del regulador: [12, 15]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: fumC\n",
      "Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA . Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( @Regulated$ , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "15\n",
      "2\n",
      "Start y end del regulador: [12, 15]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [85, 89]\n",
      "Regulado: inaA\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( @Regulated$ , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) .\n",
      "22\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [143, 147]\n",
      "Regulado: inaA\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , @Regulated$ , aslB , ybaO , mdlA , yfhD , and ybiS ) .\n",
      "24\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [150, 154]\n",
      "Regulado: marR\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , ybaO , @Regulated$ , yfhD , and ybiS ) .\n",
      "30\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [171, 175]\n",
      "Regulado: mdlA\n",
      "DNase I footprinting studies identified Fisbinding sites that overlap the yeaR promoter NarL site at position 43.5 , and the ogt promoter NarL site at position 78.5 , and suggest that Fis represses both promoters by displacing NarL . DNase I footprinting studies identified Fisbinding sites that overlap the yeaR promoter NarL site at position 43.5 , and the @Regulated$ promoter NarL site at position 78.5 , and suggest that @TF$ represses both promoters by displacing NarL .\n",
      "20\n",
      "31\n",
      "Start y end del regulador: [184, 187]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [125, 128]\n",
      "Regulado: ogt\n",
      "Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA . Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( inaA , fumC , @Regulated$ ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "19\n",
      "2\n",
      "Start y end del regulador: [12, 15]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [99, 103]\n",
      "Regulado: sodA\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , @Regulated$ , mdlA , yfhD , and ybiS ) .\n",
      "28\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [164, 168]\n",
      "Regulado: ybaO\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and @Regulated$ ) .\n",
      "35\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [189, 193]\n",
      "Regulado: ybiS\n",
      "DNase I footprinting studies identified Fisbinding sites that overlap the yeaR promoter NarL site at position 43.5 , and the ogt promoter NarL site at position 78.5 , and suggest that Fis represses both promoters by displacing NarL . DNase I footprinting studies identified Fisbinding sites that overlap the @Regulated$ promoter NarL site at position 43.5 , and the ogt promoter NarL site at position 78.5 , and suggest that @TF$ represses both promoters by displacing NarL .\n",
      "10\n",
      "31\n",
      "Start y end del regulador: [184, 187]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [74, 78]\n",
      "Regulado: yeaR\n",
      "We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) . We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , @Regulated$ , and ybiS ) .\n",
      "32\n",
      "15\n",
      "Start y end del regulador: [107, 110]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [178, 182]\n",
      "Regulado: yfhD\n",
      "These results argue that different conformational states of MelR are responsible for activation of the melAB promoter and repression of the melR promoter . These results argue that different conformational states of @TF$ are responsible for activation of the @Regulated$ promoter and repression of the melR promoter .\n",
      "15\n",
      "8\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [103, 108]\n",
      "Regulado: melAB\n",
      "These results argue that different conformational states of MelR are responsible for activation of the melAB promoter and repression of the melR promoter . These results argue that different conformational states of @TF$ are responsible for activation of the melAB promoter and repression of the @Regulated$ promoter .\n",
      "21\n",
      "8\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: melR\n",
      "These results establish the unconventional napF operon control region architecture , in which the major promoter P1 is activated by the Fnr protein bound to a site centered at 64.5 with respect to the transcription initiation site , working in conjunction with the phospho - NarP protein bound to a site centered at 44.5 . These results establish the unconventional @Regulated$ operon control region architecture , in which the major promoter P1 is activated by the @TF$ protein bound to a site centered at 64.5 with respect to the transcription initiation site , working in conjunction with the phospho - NarP protein bound to a site centered at 44.5 .\n",
      "5\n",
      "21\n",
      "Start y end del regulador: [136, 139]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: napF\n",
      "One gene , galT , was repressed by Rob . One gene , @Regulated$ , was repressed by @TF$ .\n",
      "3\n",
      "8\n",
      "Start y end del regulador: [35, 38]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: galT\n",
      "Studies of the araBAD promoter , p BAD , show that to activate transcription the AraC protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) . Studies of the @Regulated$ promoter , p BAD , show that to activate transcription the @TF$ protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) .\n",
      "3\n",
      "15\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [15, 21]\n",
      "Regulado: araBAD\n",
      "The expression of araC is repressed by its own product and stimulated by the CAP system ( 5 ) . The expression of @Regulated$ is repressed by its own product and stimulated by the @TF$ system ( 5 ) .\n",
      "3\n",
      "14\n",
      "Start y end del regulador: [77, 80]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: araC\n",
      "In order to simplify our analysis , we exploited an engineered derivative of the melAB promoter in which MelR binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation . In order to simplify our analysis , we exploited an engineered derivative of the @Regulated$ promoter in which @TF$ binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation .\n",
      "14\n",
      "18\n",
      "Start y end del regulador: [105, 109]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [81, 86]\n",
      "Regulado: melAB\n",
      "MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose metabolism ( 7 ) . MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , @TF$ , regulates expression of the @Regulated$ operon that encodes products essential for melibiose metabolism ( 7 ) .\n",
      "41\n",
      "35\n",
      "Start y end del regulador: [183, 186]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [217, 222]\n",
      "Regulado: melAB\n",
      "This suggests that Fnr and NarP may act synergistically to activate napF operon expression . This suggests that @TF$ and NarP may act synergistically to activate @Regulated$ operon expression .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [68, 72]\n",
      "Regulado: napF\n",
      "This suggests that Fnr and NarP may act synergistically to activate napF operon expression . This suggests that Fnr and @TF$ may act synergistically to activate @Regulated$ operon expression .\n",
      "11\n",
      "5\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [68, 72]\n",
      "Regulado: napF\n",
      "The most intensely examined operon is araBAD , where AraC is required for repression and activation of BAD transcription ( reviewed by Schleif , 1992 ) . The most intensely examined operon is @Regulated$ , where @TF$ is required for repression and activation of BAD transcription ( reviewed by Schleif , 1992 ) .\n",
      "6\n",
      "9\n",
      "Start y end del regulador: [53, 57]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [38, 44]\n",
      "Regulado: araBAD\n",
      "The most intensely examined operon is araBAD , where AraC is required for repression and activation of BAD transcription ( reviewed by Schleif , 1992 ) . The most intensely examined operon is @Regulated$ , where @TF$ is required for repression and activation of BAD transcription ( reviewed by Schleif , 1992 ) .\n",
      "6\n",
      "9\n",
      "Start y end del regulador: [53, 57]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [38, 44]\n",
      "Regulado: araBAD\n",
      "Thus , transcription of the marRAB operon is autorepressed by MarR and autoactivated by MarA at a site that also can be activated by SoxS . Thus , transcription of the @Regulated$ operon is autorepressed by MarR and autoactivated by @TF$ at a site that also can be activated by SoxS .\n",
      "5\n",
      "14\n",
      "Start y end del regulador: [88, 92]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [28, 34]\n",
      "Regulado: marRAB\n",
      "Thus , transcription of the marRAB operon is autorepressed by MarR and autoactivated by MarA at a site that also can be activated by SoxS . Thus , transcription of the @Regulated$ operon is autorepressed by @TF$ and autoactivated by MarA at a site that also can be activated by SoxS .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [28, 34]\n",
      "Regulado: marRAB\n",
      "Expression of the operon , consisting of marR , marA , and marB , is normally repressed by MarR but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) . Expression of the operon , consisting of marR , @Regulated$ , and marB , is normally repressed by @TF$ but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .\n",
      "9\n",
      "18\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: marA\n",
      "Expression of the operon , consisting of marR , marA , and marB , is normally repressed by MarR but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) . Expression of the operon , consisting of marR , marA , and @Regulated$ , is normally repressed by @TF$ but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .\n",
      "12\n",
      "18\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [59, 63]\n",
      "Regulado: marB\n",
      "MelR activates expression of the melA and melB genes , encoding an a - galactosidase and a melibiose permease , respectively . @TF$ activates expression of the @Regulated$ and melB genes , encoding an a - galactosidase and a melibiose permease , respectively .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [33, 37]\n",
      "Regulado: melA\n",
      "MelR activates expression of the melA and melB genes , encoding an a - galactosidase and a melibiose permease , respectively . @TF$ activates expression of the melA and @Regulated$ genes , encoding an a - galactosidase and a melibiose permease , respectively .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: melB\n",
      "Apparently , only the NarP protein is competent to activate transcription of the aeg - 46.5 operon when bound to the 44.5 region . Apparently , only the @TF$ protein is competent to activate transcription of the @Regulated$ - 46.5 operon when bound to the 44.5 region .\n",
      "13\n",
      "4\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [81, 84]\n",
      "Regulado: aeg\n",
      "CRP binding upstream of the E . coli malK promoter repositions three molecules of MalT on the DNA such that they are able to activate transcription ( Richet et al . , 1991 ) . @TF$ binding upstream of the E . coli @Regulated$ promoter repositions three molecules of MalT on the DNA such that they are able to activate transcription ( Richet et al . , 1991 ) .\n",
      "8\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [37, 41]\n",
      "Regulado: malK\n",
      "CRP binding upstream of the E . coli malK promoter repositions three molecules of MalT on the DNA such that they are able to activate transcription ( Richet et al . , 1991 ) . CRP binding upstream of the E . coli @Regulated$ promoter repositions three molecules of @TF$ on the DNA such that they are able to activate transcription ( Richet et al . , 1991 ) .\n",
      "8\n",
      "14\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MalT\n",
      "Start y end del regulado: [37, 41]\n",
      "Regulado: malK\n",
      "MarA activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) . @TF$ activates the expression of diverse genes , including @Regulated$ , micF , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) .\n",
      "9\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [59, 64]\n",
      "Regulado: acrAB\n",
      "MarA activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) . @TF$ activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , slp , and @Regulated$ , which endow cells with resistance ( 3 ) .\n",
      "27\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [110, 114]\n",
      "Regulado: inaA\n",
      "MarA activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) . @TF$ activates the expression of diverse genes , including acrAB , @Regulated$ , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) .\n",
      "11\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: micF\n",
      "MarA activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , slp , and inaA , which endow cells with resistance ( 3 ) . @TF$ activates the expression of diverse genes , including acrAB , micF , mlr - 1 , - 2 , and - 3 , @Regulated$ , and inaA , which endow cells with resistance ( 3 ) .\n",
      "24\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [100, 103]\n",
      "Regulado: slp\n",
      "This work is a first step in studying the mechanism ( s ) by which AraC regulates transcription at the araE promoter , p E , and at p FGH . This work is a first step in studying the mechanism ( s ) by which @TF$ regulates transcription at the @Regulated$ promoter , p E , and at p FGH .\n",
      "20\n",
      "15\n",
      "Start y end del regulador: [67, 71]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [103, 107]\n",
      "Regulado: araE\n",
      "FNR can only activate transcription at the E . coli nirB promoter when the FIS and IHF repressors are removed by the binding of NarL or NarP ( Browning et al . , 2000 ) . @TF$ can only activate transcription at the E . coli @Regulated$ promoter when the FIS and IHF repressors are removed by the binding of NarL or NarP ( Browning et al . , 2000 ) .\n",
      "10\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: nirB\n",
      "In the second part of the present study , we selected another E . coli promoter , the ogt promoter , that transcriptome analysis [ 11 ] had predicted to be activated by NarL alone , and we have used biochemical and genetic analysis to compare its expression and organization with the yeaR promoter . In the second part of the present study , we selected another E . coli promoter , the @Regulated$ promoter , that transcriptome analysis [ 11 ] had predicted to be activated by @TF$ alone , and we have used biochemical and genetic analysis to compare its expression and organization with the yeaR promoter .\n",
      "18\n",
      "33\n",
      "Start y end del regulador: [169, 173]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [86, 89]\n",
      "Regulado: ogt\n",
      "RhaR activates transcription of the operon that encodes the two activator proteins , rhaSR ( 37 , 38 ) . @TF$ activates transcription of the operon that encodes the two activator proteins , @Regulated$ ( 37 , 38 ) .\n",
      "13\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [85, 90]\n",
      "Regulado: rhaSR\n",
      "The three activators may have different quantitative effects on particular promoters , for example SoxS activates fpr to a much greater extent than does MarA . The three activators may have different quantitative effects on particular promoters , for example SoxS activates @Regulated$ to a much greater extent than does @TF$ .\n",
      "16\n",
      "24\n",
      "Start y end del regulador: [153, 157]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [114, 117]\n",
      "Regulado: fpr\n",
      "The three activators may have different quantitative effects on particular promoters , for example SoxS activates fpr to a much greater extent than does MarA . The three activators may have different quantitative effects on particular promoters , for example @TF$ activates @Regulated$ to a much greater extent than does MarA .\n",
      "16\n",
      "14\n",
      "Start y end del regulador: [99, 103]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [114, 117]\n",
      "Regulado: fpr\n",
      "To determine whether the repression by MarA was direct or occurred by an indirect action , we focused on two genes , purA and hdeA . To determine whether the repression by @TF$ was direct or occurred by an indirect action , we focused on two genes , purA and @Regulated$ .\n",
      "24\n",
      "6\n",
      "Start y end del regulador: [39, 43]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [126, 130]\n",
      "Regulado: hdeA\n",
      "To determine whether the repression by MarA was direct or occurred by an indirect action , we focused on two genes , purA and hdeA . To determine whether the repression by @TF$ was direct or occurred by an indirect action , we focused on two genes , @Regulated$ and hdeA .\n",
      "22\n",
      "6\n",
      "Start y end del regulador: [39, 43]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: purA\n",
      "The expression of xylF is negatively regulated by Fis and RpoS at the transcriptional level ( 50 ) , and a putative regulator , xylR , was found downstream of xylFGH . The expression of @Regulated$ is negatively regulated by @TF$ and RpoS at the transcriptional level ( 50 ) , and a putative regulator , xylR , was found downstream of xylFGH .\n",
      "3\n",
      "8\n",
      "Start y end del regulador: [50, 53]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: xylF\n",
      "The expression of xylF is negatively regulated by Fis and RpoS at the transcriptional level ( 50 ) , and a putative regulator , xylR , was found downstream of xylFGH . The expression of @Regulated$ is negatively regulated by Fis and @TF$ at the transcriptional level ( 50 ) , and a putative regulator , xylR , was found downstream of xylFGH .\n",
      "3\n",
      "10\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: RpoS\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: xylF\n",
      "In the intact cell , transcription of purA was decreased in cells constitutively producing MarA ( 4 ) . In the intact cell , transcription of @Regulated$ was decreased in cells constitutively producing @TF$ ( 4 ) .\n",
      "7\n",
      "14\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [38, 42]\n",
      "Regulado: purA\n",
      "The melR promoter and its repression by MelR . The @Regulated$ promoter and its repression by @TF$ .\n",
      "1\n",
      "7\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [4, 8]\n",
      "Regulado: melR\n",
      "The rbsDACBK operon is the only target of regulation so far identified to be under the control of RbsR ( Lopilato et al . , 1984 ; Mauzy &amp; Hermodson , 1992a ; Laikova et al . , 2001 ) . The @Regulated$ operon is the only target of regulation so far identified to be under the control of @TF$ ( Lopilato et al . , 1984 ; Mauzy &amp; Hermodson , 1992a ; Laikova et al . , 2001 ) .\n",
      "1\n",
      "18\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: RbsR\n",
      "Start y end del regulado: [4, 12]\n",
      "Regulado: rbsDACBK\n",
      "This is likely due to the greater extent of activation by SoxS of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and sodA . This is likely due to the greater extent of activation by @TF$ of promoters involved in superoxide defense , e . g . , @Regulated$ , fpr , zwf , fumC , and sodA .\n",
      "24\n",
      "11\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [119, 123]\n",
      "Regulado: acnA\n",
      "This is likely due to the greater extent of activation by SoxS of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and sodA . This is likely due to the greater extent of activation by @TF$ of promoters involved in superoxide defense , e . g . , acnA , @Regulated$ , zwf , fumC , and sodA .\n",
      "26\n",
      "11\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [126, 129]\n",
      "Regulado: fpr\n",
      "This is likely due to the greater extent of activation by SoxS of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and sodA . This is likely due to the greater extent of activation by @TF$ of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , @Regulated$ , and sodA .\n",
      "30\n",
      "11\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [138, 142]\n",
      "Regulado: fumC\n",
      "Expression of hdeA was repressed by induced expression of MarA ( see Ref . 5 , Supplemental Material ) . Expression of @Regulated$ was repressed by induced expression of @TF$ ( see Ref . 5 , Supplemental Material ) .\n",
      "2\n",
      "9\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: hdeA\n",
      "Recent experiments on the mlc - regulated genes showed that although NagC binding to Mlc operators could be detected in vitro , there was no or little regulation in vivo ( 6,17 , unpublished results ) . Recent experiments on the @Regulated$ - regulated genes showed that although @TF$ binding to Mlc operators could be detected in vitro , there was no or little regulation in vivo ( 6,17 , unpublished results ) .\n",
      "4\n",
      "11\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [26, 29]\n",
      "Regulado: mlc\n",
      "This is likely due to the greater extent of activation by SoxS of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and sodA . This is likely due to the greater extent of activation by @TF$ of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and @Regulated$ .\n",
      "33\n",
      "11\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [149, 153]\n",
      "Regulado: sodA\n",
      "This is likely due to the greater extent of activation by SoxS of promoters involved in superoxide defense , e . g . , acnA , fpr , zwf , fumC , and sodA . This is likely due to the greater extent of activation by @TF$ of promoters involved in superoxide defense , e . g . , acnA , fpr , @Regulated$ , fumC , and sodA .\n",
      "28\n",
      "11\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [132, 135]\n",
      "Regulado: zwf\n",
      "To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and NarL we have extended our studies to the c / s - acting regions required for expression of the narK gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) . To explore the possible mechanisms involved in the integrated transcriptional activation by @TF$ and NarL we have extended our studies to the c / s - acting regions required for expression of the @Regulated$ gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .\n",
      "33\n",
      "12\n",
      "Start y end del regulador: [92, 95]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [195, 199]\n",
      "Regulado: narK\n",
      "To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and NarL we have extended our studies to the c / s - acting regions required for expression of the narK gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) . To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and @TF$ we have extended our studies to the c / s - acting regions required for expression of the @Regulated$ gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .\n",
      "33\n",
      "14\n",
      "Start y end del regulador: [100, 104]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [195, 199]\n",
      "Regulado: narK\n",
      "This results in the formation of a repression loop and the repression of the araC promoter by AraC ( reviewed by Schleif , 1996 ) . This results in the formation of a repression loop and the repression of the @Regulated$ promoter by @TF$ ( reviewed by Schleif , 1996 ) .\n",
      "14\n",
      "17\n",
      "Start y end del regulador: [94, 98]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: araC\n",
      "Based on data from in vitro studies , Celis suggested that wild - type ArgP both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) . Based on data from in vitro studies , Celis suggested that wild - type @TF$ both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the @Regulated$ d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) .\n",
      "38\n",
      "14\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [208, 212]\n",
      "Regulado: argP\n",
      "The fourth promoter predicted to be regulated by NsrR was P hcp , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR . The fourth promoter predicted to be regulated by @TF$ was P @Regulated$ , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR .\n",
      "11\n",
      "8\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [60, 63]\n",
      "Regulado: hcp\n",
      "This suggested that the increased melAB expression in the W3133 - 2 mutant could be due to better binding of MelR to site 2 . This suggested that the increased @Regulated$ expression in the W3133 - 2 mutant could be due to better binding of @TF$ to site 2 .\n",
      "5\n",
      "20\n",
      "Start y end del regulador: [109, 113]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [34, 39]\n",
      "Regulado: melAB\n",
      "3259 This shows that the NarL protein represses aeg - 46 . 3259 This shows that the @TF$ protein represses @Regulated$ - 46 .\n",
      "8\n",
      "5\n",
      "Start y end del regulador: [25, 29]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [48, 51]\n",
      "Regulado: aeg\n",
      "The active ( oxidized or nitrosylated ) form of SoxR activates transcription of the soxS gene up to 100 - fold . The active ( oxidized or nitrosylated ) form of @TF$ activates transcription of the @Regulated$ gene up to 100 - fold .\n",
      "14\n",
      "9\n",
      "Start y end del regulador: [48, 52]\n",
      "Regulador: SoxR\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: soxS\n",
      "In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) . In contrast , the repression of both @Regulated$ and rob by SoxS , @TF$ , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) .\n",
      "7\n",
      "13\n",
      "Start y end del regulador: [60, 63]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [37, 41]\n",
      "Regulado: soxS\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with @Regulated$ and tolC by the @TF$ , Rob , and SoxS transcription factors .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [39, 44]\n",
      "Regulado: acrAB\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with @Regulated$ and tolC by the MarA , @TF$ , and SoxS transcription factors .\n",
      "6\n",
      "13\n",
      "Start y end del regulador: [68, 71]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [39, 44]\n",
      "Regulado: acrAB\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with @Regulated$ and tolC by the MarA , Rob , and @TF$ transcription factors .\n",
      "6\n",
      "16\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [39, 44]\n",
      "Regulado: acrAB\n",
      "In vitro transcription of the dnaA gene was inhibited by DnaA protein ( 15 ) . In vitro transcription of the @Regulated$ gene was inhibited by @TF$ protein ( 15 ) .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [57, 61]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [30, 34]\n",
      "Regulado: dnaA\n",
      "Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) . Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "3\n",
      "19\n",
      "Start y end del regulador: [98, 101]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [14, 19]\n",
      "Regulado: gntKU\n",
      "Additional activation of the marRAB operon is achieved by the binding of Fis to a site just upstream of the marbox at bp - 92 to - 73 ( Martin and Rosner , 1997 ) . Additional activation of the @Regulated$ operon is achieved by the binding of @TF$ to a site just upstream of the marbox at bp - 92 to - 73 ( Martin and Rosner , 1997 ) .\n",
      "4\n",
      "12\n",
      "Start y end del regulador: [73, 76]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [29, 35]\n",
      "Regulado: marRAB\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of @Regulated$ is coregulated with acrAB and tolC by the @TF$ , Rob , and SoxS transcription factors .\n",
      "2\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: acrZ\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of @Regulated$ is coregulated with acrAB and tolC by the MarA , @TF$ , and SoxS transcription factors .\n",
      "2\n",
      "13\n",
      "Start y end del regulador: [68, 71]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: acrZ\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of @Regulated$ is coregulated with acrAB and tolC by the MarA , Rob , and @TF$ transcription factors .\n",
      "2\n",
      "16\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: acrZ\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with acrAB and @Regulated$ by the @TF$ , Rob , and SoxS transcription factors .\n",
      "8\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: tolC\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with acrAB and @Regulated$ by the MarA , @TF$ , and SoxS transcription factors .\n",
      "8\n",
      "13\n",
      "Start y end del regulador: [68, 71]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: tolC\n",
      "Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors . Expression of acrZ is coregulated with acrAB and @Regulated$ by the MarA , Rob , and @TF$ transcription factors .\n",
      "8\n",
      "16\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: tolC\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and @Regulated$ operons is controlled by both the NarL and @TF$ proteins .\n",
      "36\n",
      "45\n",
      "Start y end del regulador: [238, 242]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [190, 194]\n",
      "Regulado: fdnG\n",
      "MelR is essential for induction of the melAB operon that is responsible for melibiose metabolism . @TF$ is essential for induction of the @Regulated$ operon that is responsible for melibiose metabolism .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [39, 44]\n",
      "Regulado: melAB\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and @TF$ proteins .\n",
      "24\n",
      "45\n",
      "Start y end del regulador: [238, 242]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [126, 130]\n",
      "Regulado: nirB\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the @Regulated$ and nrfABCDEFG operons , @TF$ protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "2\n",
      "7\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [8, 14]\n",
      "Regulado: nirBDC\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the @Regulated$ and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - @TF$ or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "2\n",
      "22\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [8, 14]\n",
      "Regulado: nirBDC\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the @Regulated$ and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - @TF$ protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "2\n",
      "25\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [8, 14]\n",
      "Regulado: nirBDC\n",
      "Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins . Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , @Regulated$ and fdnG operons is controlled by both the NarL and @TF$ proteins .\n",
      "34\n",
      "45\n",
      "Start y end del regulador: [238, 242]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: nrfA\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the nirBDC and @Regulated$ operons , @TF$ protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "4\n",
      "7\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [19, 29]\n",
      "Regulado: nrfABCDEFG\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the nirBDC and @Regulated$ operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - @TF$ or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "4\n",
      "22\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [19, 29]\n",
      "Regulado: nrfABCDEFG\n",
      "For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) . For the nirBDC and @Regulated$ operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - @TF$ protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "4\n",
      "25\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [19, 29]\n",
      "Regulado: nrfABCDEFG\n",
      "An isorepressor , GalS , has also been identified as © 2008 The Authors Journal compilation © 2008 Blackwell Publishing LtdNagC , GalR and GalS repression at galP 147 Fig . 1 . An isorepressor , GalS , has also been identified as © 2008 The Authors Journal compilation © 2008 Blackwell Publishing LtdNagC , @TF$ and GalS repression at @Regulated$ 147 Fig . 1 .\n",
      "27\n",
      "22\n",
      "Start y end del regulador: [130, 134]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: galP\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , @Regulated$ ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "122\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [587, 592]\n",
      "Regulado: acrAB\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , @Regulated$ ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "62\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [307, 313]\n",
      "Regulado: marRAB\n",
      "We show here that MarA directly represses both purA and hdeA . We show here that @TF$ directly represses both purA and @Regulated$ .\n",
      "10\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [56, 60]\n",
      "Regulado: hdeA\n",
      "We show here that MarA directly represses both purA and hdeA . We show here that @TF$ directly represses both @Regulated$ and hdeA .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: purA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , @Regulated$ ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "110\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [532, 536]\n",
      "Regulado: acnA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , @Regulated$ and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "46\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: fldA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and @Regulated$ ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "48\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [250, 254]\n",
      "Regulado: fldB\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , @Regulated$ ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "14\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [80, 83]\n",
      "Regulado: fpr\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , @Regulated$ ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "104\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [510, 514]\n",
      "Regulado: fumC\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , @Regulated$ ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "92\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [457, 460]\n",
      "Regulado: fur\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , @Regulated$ ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "40\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [213, 217]\n",
      "Regulado: inaA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , @Regulated$ ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "22\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [124, 128]\n",
      "Regulado: micF\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , @Regulated$ ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "56\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [281, 284]\n",
      "Regulado: nfo\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , @Regulated$ ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "72\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [362, 366]\n",
      "Regulado: nfsA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , @Regulated$ ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "33\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: ribA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and @Regulated$ ( a modifier of ribosomal protein S6 ) .\n",
      "130\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [620, 624]\n",
      "Regulado: rimK\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are @Regulated$ ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "7\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [37, 41]\n",
      "Regulado: sodA\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , @Regulated$ ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "115\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [553, 557]\n",
      "Regulado: tolC\n",
      "The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) . The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , @Regulated$ ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "82\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [409, 412]\n",
      "Regulado: zwf\n",
      "Interestingly , AraC - dependent transcription initiation at the araBAD promoter is increased by the cyclic AMP receptor protein ( CRP ) , a well - characterized global regulator , controlled by cyclic AMP ( cAMP ) , that activates the expression of a number of genes in response to a variety of stresses , including glucose starvation ( reviewed by Kolb et al . , 1993 ) . Interestingly , @TF$ - dependent transcription initiation at the @Regulated$ promoter is increased by the cyclic AMP receptor protein ( CRP ) , a well - characterized global regulator , controlled by cyclic AMP ( cAMP ) , that activates the expression of a number of genes in response to a variety of stresses , including glucose starvation ( reviewed by Kolb et al . , 1993 ) .\n",
      "9\n",
      "2\n",
      "Start y end del regulador: [16, 20]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [65, 71]\n",
      "Regulado: araBAD\n",
      "The Ada protein has two activation domains , one of which is an AraC / XylS family domain which is required to activate transcription of the alkA operon ( 33 ) . The @TF$ protein has two activation domains , one of which is an AraC / XylS family domain which is required to activate transcription of the @Regulated$ operon ( 33 ) .\n",
      "26\n",
      "1\n",
      "Start y end del regulador: [4, 7]\n",
      "Regulador: Ada\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: alkA\n",
      "These binding sites are located upstream of the Fnr - binding site in the control regions of the NarL - activated narG , fdnG and narK operons . These binding sites are located upstream of the Fnr - binding site in the control regions of the @TF$ - activated narG , @Regulated$ and narK operons .\n",
      "23\n",
      "18\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: fdnG\n",
      "These binding sites are located upstream of the Fnr - binding site in the control regions of the NarL - activated narG , fdnG and narK operons . These binding sites are located upstream of the Fnr - binding site in the control regions of the @TF$ - activated narG , fdnG and @Regulated$ operons .\n",
      "25\n",
      "18\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [130, 134]\n",
      "Regulado: narK\n",
      "In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation . In the first point , such a case has been reported in the @Regulated$ ( M ) operon concerning galactose utilization , which is regulated by @TF$ and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .\n",
      "13\n",
      "26\n",
      "Start y end del regulador: [135, 139]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [58, 64]\n",
      "Regulado: galETK\n",
      "In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation . In the first point , such a case has been reported in the @Regulated$ ( M ) operon concerning galactose utilization , which is regulated by GalR and @TF$ [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .\n",
      "13\n",
      "28\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [58, 64]\n",
      "Regulado: galETK\n",
      "For example , MarA binds tightly to the marRAB promoter in vitro yet stimulates transcription by only ~ 3 - fold ; no significant binding to inaA can be demonstrated by gel shift experiments , yet transcription is stimulated &gt; 5 - fold . For example , @TF$ binds tightly to the @Regulated$ promoter in vitro yet stimulates transcription by only ~ 3 - fold ; no significant binding to inaA can be demonstrated by gel shift experiments , yet transcription is stimulated &gt; 5 - fold .\n",
      "8\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [40, 46]\n",
      "Regulado: marRAB\n",
      "At the nirB promoter ( pnirB ) , it has been shown that IHF and Fis drive the promoter DNA into an inhibitory architecture , which represses FNR - dependent transcription . At the @Regulated$ promoter ( pnirB ) , it has been shown that IHF and @TF$ drive the promoter DNA into an inhibitory architecture , which represses FNR - dependent transcription .\n",
      "2\n",
      "15\n",
      "Start y end del regulador: [64, 67]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [7, 11]\n",
      "Regulado: nirB\n",
      "For example , MarA binds tightly to the marRAB promoter in vitro yet stimulates transcription by only ~ 3 - fold ; no significant binding to inaA can be demonstrated by gel shift experiments , yet transcription is stimulated &gt; 5 - fold . For example , @TF$ binds tightly to the marRAB promoter in vitro yet stimulates transcription by only ~ 3 - fold ; no significant binding to @Regulated$ can be demonstrated by gel shift experiments , yet transcription is stimulated &gt; 5 - fold .\n",
      "26\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: inaA\n",
      "In the second point , there is also an example that AraC positively and negatively regulates the araBAD operon , the mechanism of which might be similar to some extent to that of the GntII regulation . In the second point , there is also an example that @TF$ positively and negatively regulates the @Regulated$ operon , the mechanism of which might be similar to some extent to that of the GntII regulation .\n",
      "17\n",
      "11\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [97, 103]\n",
      "Regulado: araBAD\n",
      "In the second point , there is also an example that AraC positively and negatively regulates the araBAD operon , the mechanism of which might be similar to some extent to that of the GntII regulation . In the second point , there is also an example that @TF$ positively and negatively regulates the @Regulated$ operon , the mechanism of which might be similar to some extent to that of the GntII regulation .\n",
      "17\n",
      "11\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [97, 103]\n",
      "Regulado: araBAD\n",
      "Thus , the left and right half - sites are equivalent in their ability to confer Fnr - dependent activation of dmsA gene expression by RNA polymerase . Thus , the left and right half - sites are equivalent in their ability to confer @TF$ - dependent activation of @Regulated$ gene expression by RNA polymerase .\n",
      "21\n",
      "16\n",
      "Start y end del regulador: [81, 84]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [111, 115]\n",
      "Regulado: dmsA\n",
      "The results in this report also demonstrate that gntT is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which was found be essential for full induction . The results in this report also demonstrate that @Regulated$ is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex , which was found be essential for full induction .\n",
      "8\n",
      "23\n",
      "Start y end del regulador: [120, 123]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: gntT\n",
      "This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for Fnr - dependent repression at the cydA P1 promoter [ 15 ] . This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for @TF$ - dependent repression at the @Regulated$ P1 promoter [ 15 ] .\n",
      "26\n",
      "20\n",
      "Start y end del regulador: [103, 106]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [137, 141]\n",
      "Regulado: cydA\n",
      "Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) . Mutational analysis of the @Regulated$ and nrfA operon control regions has identified cis - acting regions that are required for both @TF$ and NarP - dependent induction ( Tyson et al . , 1994 ) .\n",
      "4\n",
      "21\n",
      "Start y end del regulador: [127, 131]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: nirB\n",
      "Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) . Mutational analysis of the @Regulated$ and nrfA operon control regions has identified cis - acting regions that are required for both NarL and @TF$ - dependent induction ( Tyson et al . , 1994 ) .\n",
      "4\n",
      "23\n",
      "Start y end del regulador: [136, 140]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: nirB\n",
      "Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) . Mutational analysis of the nirB and @Regulated$ operon control regions has identified cis - acting regions that are required for both @TF$ and NarP - dependent induction ( Tyson et al . , 1994 ) .\n",
      "6\n",
      "21\n",
      "Start y end del regulador: [127, 131]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [36, 40]\n",
      "Regulado: nrfA\n",
      "Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) . Mutational analysis of the nirB and @Regulated$ operon control regions has identified cis - acting regions that are required for both NarL and @TF$ - dependent induction ( Tyson et al . , 1994 ) .\n",
      "6\n",
      "23\n",
      "Start y end del regulador: [136, 140]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [36, 40]\n",
      "Regulado: nrfA\n",
      "RhaS activates transcription of the rhaBAD and rhaT operons by binding as a dimer to sites that overlap the 35 hexamers of the promoters by 4 bp and extend upstream to 81 and 82 , respectively ( see Fig . 1 for the rhaBAD promoter ) ( 10 , 45 ) . @TF$ activates transcription of the rhaBAD and @Regulated$ operons by binding as a dimer to sites that overlap the 35 hexamers of the promoters by 4 bp and extend upstream to 81 and 82 , respectively ( see Fig . 1 for the rhaBAD promoter ) ( 10 , 45 ) .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: rhaT\n",
      "Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) . Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by @TF$ and CRP ( Stoner &amp; Schleif , 1983 ) .\n",
      "2\n",
      "20\n",
      "Start y end del regulador: [125, 129]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: araE\n",
      "Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) . Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and @TF$ ( Stoner &amp; Schleif , 1983 ) .\n",
      "2\n",
      "22\n",
      "Start y end del regulador: [134, 137]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: araE\n",
      "Hence , in this work , we used in vivo and in vitro methods to investigate the determinants needed for MelR - dependent repression of the melR promoter . Hence , in this work , we used in vivo and in vitro methods to investigate the determinants needed for @TF$ - dependent repression of the @Regulated$ promoter .\n",
      "26\n",
      "20\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [138, 142]\n",
      "Regulado: melR\n",
      "Similarly , RhaR activates transcription of the rhaSR operon by binding as a dimer to a site that overlaps the RNAP binding site by 4 bp and extends upstream to 82 ( Fig . 1 ) ( 43 ) . Similarly , @TF$ activates transcription of the @Regulated$ operon by binding as a dimer to a site that overlaps the RNAP binding site by 4 bp and extends upstream to 82 ( Fig . 1 ) ( 43 ) .\n",
      "7\n",
      "2\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [48, 53]\n",
      "Regulado: rhaSR\n",
      "Besides , MarA activates fumC , fpr , nfo and other genes possibly involved in response to the oxidative stress . Besides , @TF$ activates fumC , @Regulated$ , nfo and other genes possibly involved in response to the oxidative stress .\n",
      "6\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [32, 35]\n",
      "Regulado: fpr\n",
      "Besides , MarA activates fumC , fpr , nfo and other genes possibly involved in response to the oxidative stress . Besides , @TF$ activates @Regulated$ , fpr , nfo and other genes possibly involved in response to the oxidative stress .\n",
      "4\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [25, 29]\n",
      "Regulado: fumC\n",
      "Besides , MarA activates fumC , fpr , nfo and other genes possibly involved in response to the oxidative stress . Besides , @TF$ activates fumC , fpr , @Regulated$ and other genes possibly involved in response to the oxidative stress .\n",
      "8\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [38, 41]\n",
      "Regulado: nfo\n",
      "AraC - dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) . @TF$ - dependent transcription initiation at the @Regulated$ promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [49, 55]\n",
      "Regulado: araBAD\n",
      "AraC - dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) . AraC - dependent transcription initiation at the @Regulated$ promoter is increased by @TF$ , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .\n",
      "7\n",
      "12\n",
      "Start y end del regulador: [81, 84]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [49, 55]\n",
      "Regulado: araBAD\n",
      "To explore the origin of the indirect effect , we tested whether CRP was involved in regulation of rhaSR expression . To explore the origin of the indirect effect , we tested whether @TF$ was involved in regulation of @Regulated$ expression .\n",
      "18\n",
      "12\n",
      "Start y end del regulador: [65, 68]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [99, 104]\n",
      "Regulado: rhaSR\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , @Regulated$ , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "29\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: fumC\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , @Regulated$ , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "29\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: fumC\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and @Regulated$ , suggesting broadly overlapping activities of these three regulators .\n",
      "36\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [171, 175]\n",
      "Regulado: inaA\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and @Regulated$ , suggesting broadly overlapping activities of these three regulators .\n",
      "36\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [171, 175]\n",
      "Regulado: inaA\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , @Regulated$ , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "31\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: micF\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , @Regulated$ , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "31\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [154, 158]\n",
      "Regulado: micF\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including @Regulated$ , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "27\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [140, 144]\n",
      "Regulado: sodA\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , @Regulated$ , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "33\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [161, 164]\n",
      "Regulado: zwf\n",
      "The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators . The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , @Regulated$ , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "33\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [161, 164]\n",
      "Regulado: zwf\n",
      "Figure 4 shows autoradiographs from electromobility shift assays designed to investigate how NarL represses FNR - dependent activation of the ynfE promoter . Figure 4 shows autoradiographs from electromobility shift assays designed to investigate how @TF$ represses FNR - dependent activation of the @Regulated$ promoter .\n",
      "20\n",
      "12\n",
      "Start y end del regulador: [93, 97]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: ynfE\n",
      "MelRdependent activation of the melAB promoter requires the binding of MelR subunits to four 18 bp sites and the binding of a single CRP dimer to a low - affinity target sequence ( Belyaeva et al . , 2000 ; Wade et al . , 2001 ; see Fig . 1A ) . MelRdependent activation of the @Regulated$ promoter requires the binding of MelR subunits to four 18 bp sites and the binding of a single @TF$ dimer to a low - affinity target sequence ( Belyaeva et al . , 2000 ; Wade et al . , 2001 ; see Fig . 1A ) .\n",
      "4\n",
      "23\n",
      "Start y end del regulador: [133, 136]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [32, 37]\n",
      "Regulado: melAB\n",
      "MelRdependent activation of the melAB promoter requires the binding of MelR subunits to four 18 bp sites and the binding of a single CRP dimer to a low - affinity target sequence ( Belyaeva et al . , 2000 ; Wade et al . , 2001 ; see Fig . 1A ) . MelRdependent activation of the @Regulated$ promoter requires the binding of @TF$ subunits to four 18 bp sites and the binding of a single CRP dimer to a low - affinity target sequence ( Belyaeva et al . , 2000 ; Wade et al . , 2001 ; see Fig . 1A ) .\n",
      "4\n",
      "10\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [32, 37]\n",
      "Regulado: melAB\n",
      "Thus , the dmsA promoter exhibits a preference for - 41.5 target sites like other Fnr - regulated class II promoters [ 16 ] . Thus , the @Regulated$ promoter exhibits a preference for - 41.5 target sites like other @TF$ - regulated class II promoters [ 16 ] .\n",
      "3\n",
      "15\n",
      "Start y end del regulador: [82, 85]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: dmsA\n",
      "While the soxS gene is under the redox - regulated , positive control of SoxR , marA is under negative control by MarR , a repressor whose DNA - binding activity is regulated by the binding of small molecules with toxic effects ( 1 , 31 , 39 ) . While the @Regulated$ gene is under the redox - regulated , positive control of @TF$ , marA is under negative control by MarR , a repressor whose DNA - binding activity is regulated by the binding of small molecules with toxic effects ( 1 , 31 , 39 ) .\n",
      "2\n",
      "14\n",
      "Start y end del regulador: [73, 77]\n",
      "Regulador: SoxR\n",
      "Start y end del regulado: [10, 14]\n",
      "Regulado: soxS\n",
      "Transcription of nfnB is greatly increased by induced or constitutive chromosomal expression of MarA but is unaltered by constitutive expression of chromosomal SoxS and Rob . Transcription of @Regulated$ is greatly increased by induced or constitutive chromosomal expression of @TF$ but is unaltered by constitutive expression of chromosomal SoxS and Rob .\n",
      "2\n",
      "13\n",
      "Start y end del regulador: [96, 100]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [17, 21]\n",
      "Regulado: nfnB\n",
      "Our results indicate that yggA encodes an ArgP - regulated Arg exporter in E . coli . Our results indicate that @Regulated$ encodes an @TF$ - regulated Arg exporter in E . coli .\n",
      "4\n",
      "7\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: yggA\n",
      "As the chb transcript accumulates , ChiX pairing with the intercistronic chbBC mRNA elicits the degradation of the sRNA resulting in the progressive relief of chiP repression ( Figueroa - Bossi et al . , 2009 ; Overgaard et al . , 2009 ) . As the chb transcript accumulates , @TF$ pairing with the intercistronic chbBC mRNA elicits the degradation of the sRNA resulting in the progressive relief of @Regulated$ repression ( Figueroa - Bossi et al . , 2009 ; Overgaard et al . , 2009 ) .\n",
      "25\n",
      "6\n",
      "Start y end del regulador: [36, 40]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [159, 163]\n",
      "Regulado: chiP\n",
      "We show in this work that yjhA is indeed regulated by NanR and several other regulators and that it encodes a Neu5Ac - specific channel . We show in this work that @Regulated$ is indeed regulated by @TF$ and several other regulators and that it encodes a Neu5Ac - specific channel .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [54, 58]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: yjhA\n",
      "Expression of the E . coli argO gene is activated by ArgP , also known as IciA , which is a transcriptional activator of the LysR type transcriptional regulator family ( Nandineni and Gowrishankar , 2004 ; Laishram and Gowrishankar , 2007 ) . Expression of the E . coli @Regulated$ gene is activated by @TF$ , also known as IciA , which is a transcriptional activator of the LysR type transcriptional regulator family ( Nandineni and Gowrishankar , 2004 ; Laishram and Gowrishankar , 2007 ) .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [53, 57]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: argO\n",
      "In subsequent work ( Williams et al . , 1994 ) , we were able to prove that MelR binding to both these sites ( called Site 1 and Site 2 ; see Fig . 1A ) is necessary for melibiose - dependent activation of the melAB promoter in vivo . In subsequent work ( Williams et al . , 1994 ) , we were able to prove that @TF$ binding to both these sites ( called Site 1 and Site 2 ; see Fig . 1A ) is necessary for melibiose - dependent activation of the @Regulated$ promoter in vivo .\n",
      "46\n",
      "18\n",
      "Start y end del regulador: [76, 80]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [210, 215]\n",
      "Regulado: melAB\n",
      "The CRP site required for full activation at rhaSR is located upstream but not adjacent to the RhaR binding site and is centered at 111.5 ( Fig . 1 ) ( 17 ) . The @TF$ site required for full activation at @Regulated$ is located upstream but not adjacent to the RhaR binding site and is centered at 111.5 ( Fig . 1 ) ( 17 ) .\n",
      "8\n",
      "1\n",
      "Start y end del regulador: [4, 7]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [45, 50]\n",
      "Regulado: rhaSR\n",
      "The effect of cis mutations on Fnr - dependent activation of dmsA - lacZ expression is shown in the lower right portion of the figure . ß - galactosidase activity was measured from the cells containing the indicated fusion inserted in single copy at the att site . The effect of cis mutations on @TF$ - dependent activation of @Regulated$ - lacZ expression is shown in the lower right portion of the figure . ß - galactosidase activity was measured from the cells containing the indicated fusion inserted in single copy at the att site .\n",
      "11\n",
      "6\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [61, 65]\n",
      "Regulado: dmsA\n",
      "IciA protein binds to two sites in the dnaA promoter region . @TF$ protein binds to two sites in the @Regulated$ promoter region .\n",
      "8\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: dnaA\n",
      "Two regions of enhanced XylR binding in the presence of xylose were revealed in the xylAB / xylFGH intergenic space by DMS footprinting [ 4 ] . Two regions of enhanced @TF$ binding in the presence of xylose were revealed in the @Regulated$ / xylFGH intergenic space by DMS footprinting [ 4 ] .\n",
      "15\n",
      "4\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: XylR\n",
      "Start y end del regulado: [84, 89]\n",
      "Regulado: xylAB\n",
      "Two regions of enhanced XylR binding in the presence of xylose were revealed in the xylAB / xylFGH intergenic space by DMS footprinting [ 4 ] . Two regions of enhanced @TF$ binding in the presence of xylose were revealed in the xylAB / @Regulated$ intergenic space by DMS footprinting [ 4 ] .\n",
      "17\n",
      "4\n",
      "Start y end del regulador: [24, 28]\n",
      "Regulador: XylR\n",
      "Start y end del regulado: [92, 98]\n",
      "Regulado: xylFGH\n",
      "This demonstrated that either NarL or NarP is required to activate transcription from the hcp promoter . This demonstrated that either @TF$ or NarP is required to activate transcription from the @Regulated$ promoter .\n",
      "14\n",
      "4\n",
      "Start y end del regulador: [30, 34]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [90, 93]\n",
      "Regulado: hcp\n",
      "This demonstrated that either NarL or NarP is required to activate transcription from the hcp promoter . This demonstrated that either NarL or @TF$ is required to activate transcription from the @Regulated$ promoter .\n",
      "14\n",
      "6\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [90, 93]\n",
      "Regulado: hcp\n",
      "The phospho - DcuR protein is known to activate the transcription of three operons : the dcuB - fumB operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the dctA gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) . The phospho - @TF$ protein is known to activate the transcription of three operons : the dcuB - fumB operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the @Regulated$ gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) .\n",
      "44\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [224, 228]\n",
      "Regulado: dctA\n",
      "The phospho - DcuR protein is known to activate the transcription of three operons : the dcuB - fumB operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the dctA gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) . The phospho - @TF$ protein is known to activate the transcription of three operons : the @Regulated$ - fumB operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the dctA gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) .\n",
      "16\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [89, 93]\n",
      "Regulado: dcuB\n",
      "The phospho - DcuR protein is known to activate the transcription of three operons : the dcuB - fumB operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the dctA gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) . The phospho - @TF$ protein is known to activate the transcription of three operons : the dcuB - @Regulated$ operon encoding the DcuB carrier and fumarase B , the frd - ABCD operon encoding fumarate reductase ( Fig . 1C ) , and the dctA gene encoding the aerobic C 4 - dicarboxylate transporter ( 2 , 17 , 18 , 67 ) .\n",
      "18\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [96, 100]\n",
      "Regulado: fumB\n",
      "This is particularly true in the case of nagB , but activation of nagE requires in addition protein – protein interactions between CAP and the RNA polymerase machinery . This is particularly true in the case of nagB , but activation of @Regulated$ requires in addition protein – protein interactions between @TF$ and the RNA polymerase machinery .\n",
      "13\n",
      "22\n",
      "Start y end del regulador: [131, 134]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [66, 70]\n",
      "Regulado: nagE\n",
      "Concluding remarks The hcp gene has been shown to be regulated by FNR , NarL and NarP proteins . Concluding remarks The @Regulated$ gene has been shown to be regulated by @TF$ , NarL and NarP proteins .\n",
      "3\n",
      "12\n",
      "Start y end del regulador: [66, 69]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [23, 26]\n",
      "Regulado: hcp\n",
      "Concluding remarks The hcp gene has been shown to be regulated by FNR , NarL and NarP proteins . Concluding remarks The @Regulated$ gene has been shown to be regulated by FNR , @TF$ and NarP proteins .\n",
      "3\n",
      "14\n",
      "Start y end del regulador: [72, 76]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [23, 26]\n",
      "Regulado: hcp\n",
      "Concluding remarks The hcp gene has been shown to be regulated by FNR , NarL and NarP proteins . Concluding remarks The @Regulated$ gene has been shown to be regulated by FNR , NarL and @TF$ proteins .\n",
      "3\n",
      "16\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [23, 26]\n",
      "Regulado: hcp\n",
      "The inability of NarL to activate napF operon expression led us to hypothesize that NarL is deficient in the mechanism by which NarP activates transcription from the 44.5 binding site ( 9 ) . The inability of NarL to activate @Regulated$ operon expression led us to hypothesize that NarL is deficient in the mechanism by which @TF$ activates transcription from the 44.5 binding site ( 9 ) .\n",
      "6\n",
      "22\n",
      "Start y end del regulador: [128, 132]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: napF\n",
      "Lrp was also found to be involved in the regulation of cadBA expression . @TF$ was also found to be involved in the regulation of @Regulated$ expression .\n",
      "11\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [55, 60]\n",
      "Regulado: cadBA\n",
      "We also extended the previous studies by Bhende and Egan ( 4 ) of RhaS - mediated transcription activation at rhaBAD . We also extended the previous studies by Bhende and Egan ( 4 ) of @TF$ - mediated transcription activation at @Regulated$ .\n",
      "20\n",
      "14\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [110, 116]\n",
      "Regulado: rhaBAD\n",
      "The results show that the dcuB gene is expressed exclusively under anaerobic conditions in a manner that is FNR dependent and that it is repressed by NarL in the presence of nitrate and is subject to CRP - mediated catabolite repression . The results show that the @Regulated$ gene is expressed exclusively under anaerobic conditions in a manner that is FNR dependent and that it is repressed by NarL in the presence of nitrate and is subject to @TF$ - mediated catabolite repression .\n",
      "5\n",
      "36\n",
      "Start y end del regulador: [200, 203]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [26, 30]\n",
      "Regulado: dcuB\n",
      "The horizontal shaded boxes denote 22 bp DNA sites for CRP : one site centred at position 2195.5 is responsible for the activation of the melR promoter , while the other , centred at position 281.5 , is discussed in this paper . The horizontal shaded boxes denote 22 bp DNA sites for @TF$ : one site centred at position 2195.5 is responsible for the activation of the @Regulated$ promoter , while the other , centred at position 281.5 , is discussed in this paper .\n",
      "25\n",
      "10\n",
      "Start y end del regulador: [55, 58]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [138, 142]\n",
      "Regulado: melR\n",
      "The results demonstrate that stimulation of the inaA promoter by MarA and SoxS in vivo requires only sequences from - 51 to + 3 ( data not shown ) . The results demonstrate that stimulation of the @Regulated$ promoter by @TF$ and SoxS in vivo requires only sequences from - 51 to + 3 ( data not shown ) .\n",
      "7\n",
      "10\n",
      "Start y end del regulador: [65, 69]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: inaA\n",
      "Since the results above suggested that the argO regulatory region upstream of 89 is not required for ArgP binding , we tested fragments with increasing length of downstream sequence in the EMSA experiments . Since the results above suggested that the @Regulated$ regulatory region upstream of 89 is not required for @TF$ binding , we tested fragments with increasing length of downstream sequence in the EMSA experiments .\n",
      "7\n",
      "17\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: argO\n",
      "These indicate that DnaA and IciA proteins independently bind to their respective binding sites on the dnaA # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y . These indicate that @TF$ and IciA proteins independently bind to their respective binding sites on the @Regulated$ # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .\n",
      "16\n",
      "3\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [103, 107]\n",
      "Regulado: dnaA\n",
      "These indicate that DnaA and IciA proteins independently bind to their respective binding sites on the dnaA # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y . These indicate that DnaA and @TF$ proteins independently bind to their respective binding sites on the @Regulated$ # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .\n",
      "16\n",
      "5\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [103, 107]\n",
      "Regulado: dnaA\n",
      "Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ± 222Activation at the E . coli melAB promoter 213 in vitro and that this activation is dependent on MelR and melibiose . Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ± 222Activation at the E . coli @Regulated$ promoter 213 in vitro and that this activation is dependent on @TF$ and melibiose .\n",
      "19\n",
      "31\n",
      "Start y end del regulador: [166, 170]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [97, 102]\n",
      "Regulado: melAB\n",
      "mdaA ( also known as nfsA ) but not ybjC had previously been shown to be regulated by SoxS although no binding site had been identified ( Liochev et al . , 1999 ) . @Regulated$ ( also known as nfsA ) but not ybjC had previously been shown to be regulated by @TF$ although no binding site had been identified ( Liochev et al . , 1999 ) .\n",
      "0\n",
      "18\n",
      "Start y end del regulador: [86, 90]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: mdaA\n",
      "Instead , only R603A ( 73 % ) of 70 reduced Rob - dependent activation of the micF promoter . Instead , only R603A ( 73 % ) of 70 reduced @TF$ - dependent activation of the @Regulated$ promoter .\n",
      "17\n",
      "11\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: micF\n",
      "The shaded boxes denote 22 bp DNA sites for CRP : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter . The shaded boxes denote 22 bp DNA sites for @TF$ : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the @Regulated$ promoter .\n",
      "41\n",
      "9\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [222, 227]\n",
      "Regulado: melAB\n",
      "The shaded boxes denote 22 bp DNA sites for CRP : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter . The shaded boxes denote 22 bp DNA sites for @TF$ : one site centered at position 241.5 is responsible for the activation of the @Regulated$ promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .\n",
      "24\n",
      "9\n",
      "Start y end del regulador: [44, 47]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [127, 131]\n",
      "Regulado: melR\n",
      "Many Cel + strains carry mutations in nagC The NagC repressor that regulates the nag operon involved in N - acetylglucosamine metabolism was also shown to regulate the chb operon ( Plumbridge and Pellegrini , 2004 ) . Many Cel + strains carry mutations in nagC The @TF$ repressor that regulates the @Regulated$ operon involved in N - acetylglucosamine metabolism was also shown to regulate the chb operon ( Plumbridge and Pellegrini , 2004 ) .\n",
      "14\n",
      "9\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [81, 84]\n",
      "Regulado: nag\n",
      "These results show that constitutive and inducible expression of the chromosomal MarA was able to upregulate nfnB expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal SoxS and Rob proteins . These results show that constitutive and inducible expression of the chromosomal @TF$ was able to upregulate @Regulated$ expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal SoxS and Rob proteins .\n",
      "16\n",
      "11\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [109, 113]\n",
      "Regulado: nfnB\n",
      "These results show that constitutive and inducible expression of the chromosomal MarA was able to upregulate nfnB expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal SoxS and Rob proteins . These results show that constitutive and inducible expression of the chromosomal MarA was able to upregulate @Regulated$ expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal SoxS and @TF$ proteins .\n",
      "16\n",
      "34\n",
      "Start y end del regulador: [229, 232]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [109, 113]\n",
      "Regulado: nfnB\n",
      "These results show that constitutive and inducible expression of the chromosomal MarA was able to upregulate nfnB expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal SoxS and Rob proteins . These results show that constitutive and inducible expression of the chromosomal MarA was able to upregulate @Regulated$ expression in contrast with the lack of activation mediated by constitutive expression of the chromosomal @TF$ and Rob proteins .\n",
      "16\n",
      "32\n",
      "Start y end del regulador: [220, 224]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [109, 113]\n",
      "Regulado: nfnB\n",
      "Further studies showed that the melR promoter is a typical Class II CRPdependent promoter , activated by a single dimer of CRP , that binds to a 22 bp DNA site centred at position 241.5 with respect to the transcription start point ( reviewed by Busby and Ebright , 1997 ) . Further studies showed that the @Regulated$ promoter is a typical Class II CRPdependent promoter , activated by a single dimer of @TF$ , that binds to a 22 bp DNA site centred at position 241.5 with respect to the transcription start point ( reviewed by Busby and Ebright , 1997 ) .\n",
      "5\n",
      "21\n",
      "Start y end del regulador: [123, 126]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [32, 36]\n",
      "Regulado: melR\n",
      "Overproduction of IciA protein enhances in vivo transcription from dnaA promoter 1P The recombinant plasmid pIC1 contains the iciA coding region under control of the inducible araB promoter of vector pING1 ( Hwang et al . , 1992 ; Johnston et al . , 1985 ) . Overproduction of @TF$ protein enhances in vivo transcription from @Regulated$ promoter 1P The recombinant plasmid pIC1 contains the iciA coding region under control of the inducible araB promoter of vector pING1 ( Hwang et al . , 1992 ; Johnston et al . , 1985 ) .\n",
      "9\n",
      "2\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: dnaA\n",
      "With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer . With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters @Regulated$ and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "91\n",
      "81\n",
      "Start y end del regulador: [368, 372]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [437, 441]\n",
      "Regulado: fumC\n",
      "With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer . With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters fumC and @Regulated$ , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "93\n",
      "81\n",
      "Start y end del regulador: [368, 372]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [446, 450]\n",
      "Regulado: micF\n",
      "These observations indicate that the region that is essential for normal Fnr - dependent induction of aeg - 46.5 operon expression is located downstream of position 85 . These observations indicate that the region that is essential for normal @TF$ - dependent induction of @Regulated$ - 46.5 operon expression is located downstream of position 85 .\n",
      "16\n",
      "11\n",
      "Start y end del regulador: [73, 76]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [102, 105]\n",
      "Regulado: aeg\n",
      "At the time that these experiments were done , the genes known to be ArgP regulated in vivo included the following ( gene functions are indicated in parentheses ) : argO ( Arg export ) , dapB ( dihydrodipicolinate reductase in the diaminopimelate - Lys biosynthetic pathway ) , and K . pneumoniae gdhA ( glutamate dehydrogenase for NH 4 assimilation and glutamate biosynthesis ) . At the time that these experiments were done , the genes known to be @TF$ regulated in vivo included the following ( gene functions are indicated in parentheses ) : @Regulated$ ( Arg export ) , dapB ( dihydrodipicolinate reductase in the diaminopimelate - Lys biosynthetic pathway ) , and K . pneumoniae gdhA ( glutamate dehydrogenase for NH 4 assimilation and glutamate biosynthesis ) .\n",
      "30\n",
      "14\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [165, 169]\n",
      "Regulado: argO\n",
      "At the time that these experiments were done , the genes known to be ArgP regulated in vivo included the following ( gene functions are indicated in parentheses ) : argO ( Arg export ) , dapB ( dihydrodipicolinate reductase in the diaminopimelate - Lys biosynthetic pathway ) , and K . pneumoniae gdhA ( glutamate dehydrogenase for NH 4 assimilation and glutamate biosynthesis ) . At the time that these experiments were done , the genes known to be @TF$ regulated in vivo included the following ( gene functions are indicated in parentheses ) : argO ( Arg export ) , @Regulated$ ( dihydrodipicolinate reductase in the diaminopimelate - Lys biosynthetic pathway ) , and K . pneumoniae gdhA ( glutamate dehydrogenase for NH 4 assimilation and glutamate biosynthesis ) .\n",
      "36\n",
      "14\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [187, 191]\n",
      "Regulado: dapB\n",
      "Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the MarA activation of nfnB . Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the @TF$ activation of @Regulated$ .\n",
      "64\n",
      "61\n",
      "Start y end del regulador: [322, 326]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [341, 345]\n",
      "Regulado: nfnB\n",
      "Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA . Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( @Regulated$ , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .\n",
      "13\n",
      "6\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: EvgA\n",
      "Start y end del regulado: [94, 99]\n",
      "Regulado: acrAB\n",
      "Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA . Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( acrAB , emrKY , @Regulated$ and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .\n",
      "17\n",
      "6\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: EvgA\n",
      "Start y end del regulado: [110, 114]\n",
      "Regulado: mdfA\n",
      "Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA . Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and @Regulated$ ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .\n",
      "19\n",
      "6\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: EvgA\n",
      "Start y end del regulado: [119, 124]\n",
      "Regulado: yhiUV\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of zwf , @Regulated$ , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "11\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [68, 71]\n",
      "Regulado: fpr\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , @Regulated$ , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "13\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [74, 78]\n",
      "Regulado: fumC\n",
      "Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced Rob dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and inaA ) ( Fig . 2a – c ) . Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced @TF$ dependent transcription activation of the three class II promoters ( i . e . , @Regulated$ , micF , and inaA ) ( Fig . 2a – c ) .\n",
      "58\n",
      "42\n",
      "Start y end del regulador: [225, 228]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [308, 312]\n",
      "Regulado: fumC\n",
      "Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced Rob dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and inaA ) ( Fig . 2a – c ) . Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced @TF$ dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and @Regulated$ ) ( Fig . 2a – c ) .\n",
      "63\n",
      "42\n",
      "Start y end del regulador: [225, 228]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [326, 330]\n",
      "Regulado: inaA\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , @Regulated$ , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "15\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [81, 85]\n",
      "Regulado: micF\n",
      "Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced Rob dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and inaA ) ( Fig . 2a – c ) . Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced @TF$ dependent transcription activation of the three class II promoters ( i . e . , fumC , @Regulated$ , and inaA ) ( Fig . 2a – c ) .\n",
      "60\n",
      "42\n",
      "Start y end del regulador: [225, 228]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [315, 319]\n",
      "Regulado: micF\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , @Regulated$ , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "17\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [88, 91]\n",
      "Regulado: nfo\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and @Regulated$ , promoters known to be activated by MalE - SoxS and MarA .\n",
      "20\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [98, 102]\n",
      "Regulado: sodA\n",
      "Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA . Sufficiency of @TF$ for in vitro transcriptional activation of @Regulated$ , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "9\n",
      "2\n",
      "Start y end del regulador: [15, 18]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [62, 65]\n",
      "Regulado: zwf\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , @Regulated$ , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "23\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [129, 132]\n",
      "Regulado: fpr\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , @Regulated$ , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "23\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [129, 132]\n",
      "Regulado: fpr\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , @Regulated$ , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "25\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: fumC\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , @Regulated$ , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "25\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [135, 139]\n",
      "Regulado: fumC\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , @Regulated$ , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "27\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: micF\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , @Regulated$ , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "27\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: micF\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , @Regulated$ , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "29\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [149, 152]\n",
      "Regulado: nfo\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , @Regulated$ , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "29\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [149, 152]\n",
      "Regulado: nfo\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and @Regulated$ promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "32\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [159, 163]\n",
      "Regulado: sodA\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and @Regulated$ promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "32\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [159, 163]\n",
      "Regulado: sodA\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the @Regulated$ , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "21\n",
      "10\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [123, 126]\n",
      "Regulado: zwf\n",
      "Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein . Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the @Regulated$ , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "21\n",
      "8\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [123, 126]\n",
      "Regulado: zwf\n",
      "As expected , the RH180 substitution had little effect on CRP dependent activation of the wild - type melAB promoter . As expected , the RH180 substitution had little effect on @TF$ dependent activation of the wild - type @Regulated$ promoter .\n",
      "18\n",
      "10\n",
      "Start y end del regulador: [58, 61]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [102, 107]\n",
      "Regulado: melAB\n",
      "The first gene of the emrRAB operon encodes transcription factor EmrR that represses transcription of this operon . The first gene of the @Regulated$ operon encodes transcription factor @TF$ that represses transcription of this operon .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [65, 69]\n",
      "Regulador: EmrR\n",
      "Start y end del regulado: [22, 28]\n",
      "Regulado: emrRAB\n",
      "Autoradiographs of EMSA analyses of Lrp binding to the argO control region without and with 7 mM L - leucine in the binding buffer respectively . Autoradiographs of EMSA analyses of @TF$ binding to the @Regulated$ control region without and with 7 mM L - leucine in the binding buffer respectively .\n",
      "9\n",
      "5\n",
      "Start y end del regulador: [36, 39]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [55, 59]\n",
      "Regulado: argO\n",
      "Thus , the dmsA promoter is repressed by NarL binding to multiple sites spread across 80 bp covering the entire promoter ( 1 , 5 ) . Thus , the @Regulated$ promoter is repressed by @TF$ binding to multiple sites spread across 80 bp covering the entire promoter ( 1 , 5 ) .\n",
      "3\n",
      "8\n",
      "Start y end del regulador: [41, 45]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: dmsA\n",
      "In the same experiment we also examined the expression of hdeA , which had been observed in another macroarray study to be repressed upon MarA induction in cells ( 5 ) . In the same experiment we also examined the expression of @Regulated$ , which had been observed in another macroarray study to be repressed upon @TF$ induction in cells ( 5 ) .\n",
      "10\n",
      "24\n",
      "Start y end del regulador: [138, 142]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [58, 62]\n",
      "Regulado: hdeA\n",
      "Taken together , these results support a model in which O NC2 enhances the binding of NagC to O NC1 , with the two sites functioning jointly to control fimB expression . Taken together , these results support a model in which O NC2 enhances the binding of @TF$ to O NC1 , with the two sites functioning jointly to control @Regulated$ expression .\n",
      "29\n",
      "16\n",
      "Start y end del regulador: [86, 90]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [152, 156]\n",
      "Regulado: fimB\n",
      "The ArgP protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( argP ) gene ( ThoÈny et al . , 1991 ) . The @TF$ protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( @Regulated$ ) gene ( ThoÈny et al . , 1991 ) .\n",
      "35\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [185, 189]\n",
      "Regulado: argP\n",
      "Scanning - replacement mutagenesis of the region separating O NC1 from O NC2 and the effect of IHF on fimB expression Although the results described above show that O NC1 and O NC2 function together , the distance between the centre of the two operators ( 212 bp ) is longer than for any other NagC - regulated operon . Scanning - replacement mutagenesis of the region separating O NC1 from O NC2 and the effect of IHF on @Regulated$ expression Although the results described above show that O NC1 and O NC2 function together , the distance between the centre of the two operators ( 212 bp ) is longer than for any other @TF$ - regulated operon .\n",
      "19\n",
      "55\n",
      "Start y end del regulador: [294, 298]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [102, 106]\n",
      "Regulado: fimB\n",
      "We reasoned that MelR - dependent repression of the melR promoter must be less efficient in these mutants . We reasoned that @TF$ - dependent repression of the @Regulated$ promoter must be less efficient in these mutants .\n",
      "9\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: melR\n",
      "The expression of both fusions was increased by growth on galactose and to a lesser extent by growth on GlcNAc , supporting the hypothesis that NagC is involved in repression of galP ( Table 2 ) . The expression of both fusions was increased by growth on galactose and to a lesser extent by growth on GlcNAc , supporting the hypothesis that @TF$ is involved in repression of @Regulated$ ( Table 2 ) .\n",
      "31\n",
      "25\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [178, 182]\n",
      "Regulado: galP\n",
      "Our results suggest that repression by NarL at the ynfE promoter may be unusual and might be due to locking of the FNR - RNAP - promoterVOL . Our results suggest that repression by @TF$ at the @Regulated$ promoter may be unusual and might be due to locking of the FNR - RNAP - promoterVOL .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [39, 43]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: ynfE\n",
      "Coincidence of the sites for SoxS - and Rob - mediated activation of zwf transcription in vitro . Coincidence of the sites for SoxS - and @TF$ - mediated activation of @Regulated$ transcription in vitro .\n",
      "13\n",
      "8\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [69, 72]\n",
      "Regulado: zwf\n",
      "Coincidence of the sites for SoxS - and Rob - mediated activation of zwf transcription in vitro . Coincidence of the sites for @TF$ - and Rob - mediated activation of @Regulated$ transcription in vitro .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [69, 72]\n",
      "Regulado: zwf\n",
      "This resembles the known co - regulation of acrAB and tolC by the transcriptional factor MarA . This resembles the known co - regulation of @Regulated$ and tolC by the transcriptional factor @TF$ .\n",
      "8\n",
      "15\n",
      "Start y end del regulador: [89, 93]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [44, 49]\n",
      "Regulado: acrAB\n",
      "ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate catabolism . @TF$ regulates expression of the uxaCA , uxaB and @Regulated$ operons involved in galacturonate catabolism .\n",
      "9\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ExuR\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: exuT\n",
      "ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate catabolism . @TF$ regulates expression of the uxaCA , @Regulated$ and exuT operons involved in galacturonate catabolism .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ExuR\n",
      "Start y end del regulado: [41, 45]\n",
      "Regulado: uxaB\n",
      "ExuR regulates expression of the uxaCA , uxaB and exuT operons involved in galacturonate catabolism . @TF$ regulates expression of the @Regulated$ , uxaB and exuT operons involved in galacturonate catabolism .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ExuR\n",
      "Start y end del regulado: [33, 38]\n",
      "Regulado: uxaCA\n",
      "This resembles the known co - regulation of acrAB and tolC by the transcriptional factor MarA . This resembles the known co - regulation of acrAB and @Regulated$ by the transcriptional factor @TF$ .\n",
      "10\n",
      "15\n",
      "Start y end del regulador: [89, 93]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [54, 58]\n",
      "Regulado: tolC\n",
      "They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA . They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and @TF$ .\n",
      "3\n",
      "23\n",
      "Start y end del regulador: [111, 115]\n",
      "Regulador: ArcA\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: acnA\n",
      "They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA . They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , @TF$ and ArcA .\n",
      "3\n",
      "21\n",
      "Start y end del regulador: [103, 106]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: acnA\n",
      "They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA . They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by SoxS , @TF$ , CRP and ArcA .\n",
      "3\n",
      "19\n",
      "Start y end del regulador: [97, 100]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: acnA\n",
      "They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA . They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by @TF$ , FNR , CRP and ArcA .\n",
      "3\n",
      "17\n",
      "Start y end del regulador: [90, 94]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: acnA\n",
      "According to previous results ( 16 ) , IciA protein was able to activate in vitro transcription from dnaA promoter 1P in the presence of limiting amounts of RNA polymerase . According to previous results ( 16 ) , @TF$ protein was able to activate in vitro transcription from @Regulated$ promoter 1P in the presence of limiting amounts of RNA polymerase .\n",
      "18\n",
      "8\n",
      "Start y end del regulador: [39, 43]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [101, 105]\n",
      "Regulado: dnaA\n",
      "This protein , engineered to facilitate purification and hereafter referred to as MarA , had been shown previously to activate nfnB transcription in vitro ( 20 ) . This protein , engineered to facilitate purification and hereafter referred to as @TF$ , had been shown previously to activate @Regulated$ transcription in vitro ( 20 ) .\n",
      "20\n",
      "12\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [127, 131]\n",
      "Regulado: nfnB\n",
      "Recently , we demonstrated that activation of in vitro zwf transcription by MalE - SoxS ( 18 ) , and by MarA as well ( 19 ) , requires this same DNA binding element and that this soxbox / marbox can bring in vitro gnd transcription under control of these two activators ( 18 , 19 ) . Recently , we demonstrated that activation of in vitro @Regulated$ transcription by MalE - SoxS ( 18 ) , and by @TF$ as well ( 19 ) , requires this same DNA binding element and that this soxbox / marbox can bring in vitro gnd transcription under control of these two activators ( 18 , 19 ) .\n",
      "9\n",
      "21\n",
      "Start y end del regulador: [104, 108]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [55, 58]\n",
      "Regulado: zwf\n",
      "Thus , overproduction of IciA protein specifically increased transcription from the dnaA promoter 1P . Thus , overproduction of @TF$ protein specifically increased transcription from the @Regulated$ promoter 1P .\n",
      "11\n",
      "4\n",
      "Start y end del regulador: [25, 29]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: dnaA\n",
      "The transport of glucuronides and their subsequent interconversion to the glucuronate in E . coli is mediated by the genes of the uidAB operon , which is under negative regulation by UidR and UxuR ( Blanco , 1987 ) . The transport of glucuronides and their subsequent interconversion to the glucuronate in E . coli is mediated by the genes of the @Regulated$ operon , which is under negative regulation by @TF$ and UxuR ( Blanco , 1987 ) .\n",
      "22\n",
      "31\n",
      "Start y end del regulador: [183, 187]\n",
      "Regulador: UidR\n",
      "Start y end del regulado: [130, 135]\n",
      "Regulado: uidAB\n",
      "The transport of glucuronides and their subsequent interconversion to the glucuronate in E . coli is mediated by the genes of the uidAB operon , which is under negative regulation by UidR and UxuR ( Blanco , 1987 ) . The transport of glucuronides and their subsequent interconversion to the glucuronate in E . coli is mediated by the genes of the @Regulated$ operon , which is under negative regulation by UidR and @TF$ ( Blanco , 1987 ) .\n",
      "22\n",
      "33\n",
      "Start y end del regulador: [192, 196]\n",
      "Regulador: UxuR\n",
      "Start y end del regulado: [130, 135]\n",
      "Regulado: uidAB\n",
      "GntR thus seems to function as Inhibition of gluconate uptake by alternative sugars in Hfr G6MD2 a negative regulator of gntT by binding to the putative harboring pGNTTK sequence ( see below ) . @TF$ thus seems to function as Inhibition of gluconate uptake by alternative sugars in Hfr G6MD2 a negative regulator of @Regulated$ by binding to the putative harboring pGNTTK sequence ( see below ) .\n",
      "20\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: gntT\n",
      "20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces SoxS - dependent transcription activation of the fpr and zwf promoters . 20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces @TF$ - dependent transcription activation of the @Regulated$ and zwf promoters .\n",
      "23\n",
      "16\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [136, 139]\n",
      "Regulado: fpr\n",
      "As with MalE - SoxS and MarA ( 18 , 19 ) , Rob stimulated in vitro transcription about threefold from the zwf promoter of templates Z1 , Z2 , Z3 , Z4 , and Z5 ( 9 ) , whose 5 ´ deletion end points reside at positions - 140 , - 86 , - 76 , - 67 , and - 62 , respectively ( Fig . 3A ) but failed to enhance transcription from construct Z6 , whose 5 ´ deletion end point lies at position - 57 ( 9 ) . As with MalE - SoxS and MarA ( 18 , 19 ) , @TF$ stimulated in vitro transcription about threefold from the @Regulated$ promoter of templates Z1 , Z2 , Z3 , Z4 , and Z5 ( 9 ) , whose 5 ´ deletion end points reside at positions - 140 , - 86 , - 76 , - 67 , and - 62 , respectively ( Fig . 3A ) but failed to enhance transcription from construct Z6 , whose 5 ´ deletion end point lies at position - 57 ( 9 ) .\n",
      "22\n",
      "13\n",
      "Start y end del regulador: [43, 46]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [106, 109]\n",
      "Regulado: zwf\n",
      "20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces SoxS - dependent transcription activation of the fpr and zwf promoters . 20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces @TF$ - dependent transcription activation of the fpr and @Regulated$ promoters .\n",
      "25\n",
      "16\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [144, 147]\n",
      "Regulado: zwf\n",
      "To determine whether site II is essential for repression by MarR , marO133 , a mutant deleted for the 20 bp immediately preceding marR ( pos . 1425 - 1444 ) but wild - type for marRAB ( unpublished data ) , was studied . To determine whether site II is essential for repression by @TF$ , marO133 , a mutant deleted for the 20 bp immediately preceding marR ( pos . 1425 - 1444 ) but wild - type for @Regulated$ ( unpublished data ) , was studied .\n",
      "36\n",
      "10\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [177, 183]\n",
      "Regulado: marRAB\n",
      "As shown previously ( Wang and Kaguni , 1987 ) , purified DnaA protein repressed transcription from the dnaA promoters , 1P and 2P ( Fig . 5A ) . As shown previously ( Wang and Kaguni , 1987 ) , purified @TF$ protein repressed transcription from the @Regulated$ promoters , 1P and 2P ( Fig . 5A ) .\n",
      "18\n",
      "12\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: dnaA\n",
      "In the presence of 200 nM MarA , there was a 5 – 6 - fold activation in nfnB transcription , as expected ( Fig . 2 ) . In the presence of 200 nM @TF$ , there was a 5 – 6 - fold activation in @Regulated$ transcription , as expected ( Fig . 2 ) .\n",
      "18\n",
      "6\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nfnB\n",
      "Although DnaA protein , at concentrations of 75 ng and greater , reduced transcription from rpmH promoter 1P , repression was preferential for the dnaA promoters . Although @TF$ protein , at concentrations of 75 ng and greater , reduced transcription from rpmH promoter 1P , repression was preferential for the @Regulated$ promoters .\n",
      "24\n",
      "1\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [147, 151]\n",
      "Regulado: dnaA\n",
      "In a second set of experiments , we used the in vitro transcription assay to investigate whether CRP affected activation at the melAB promoter . In a second set of experiments , we used the in vitro transcription assay to investigate whether @TF$ affected activation at the @Regulated$ promoter .\n",
      "22\n",
      "17\n",
      "Start y end del regulador: [97, 100]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [128, 133]\n",
      "Regulado: melAB\n",
      "Although DnaA protein , at concentrations of 75 ng and greater , reduced transcription from rpmH promoter 1P , repression was preferential for the dnaA promoters . Although @TF$ protein , at concentrations of 75 ng and greater , reduced transcription from @Regulated$ promoter 1P , repression was preferential for the dnaA promoters .\n",
      "15\n",
      "1\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [92, 96]\n",
      "Regulado: rpmH\n",
      "The enhanced binding of RNA polymerase to the dnaA promoter 1P by IciA protein was confirmed using a DNase I protection assay ( Fig . 4C ) . The enhanced binding of RNA polymerase to the @Regulated$ promoter 1P by @TF$ protein was confirmed using a DNase I protection assay ( Fig . 4C ) .\n",
      "8\n",
      "12\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [46, 50]\n",
      "Regulado: dnaA\n",
      "The same concentration and batch of MarA protein , on the other hand , repressed both purA and hdeA ( Fig . 2 ) . The same concentration and batch of @TF$ protein , on the other hand , repressed both purA and @Regulated$ ( Fig . 2 ) .\n",
      "18\n",
      "6\n",
      "Start y end del regulador: [36, 40]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [95, 99]\n",
      "Regulado: hdeA\n",
      "The same concentration and batch of MarA protein , on the other hand , repressed both purA and hdeA ( Fig . 2 ) . The same concentration and batch of @TF$ protein , on the other hand , repressed both @Regulated$ and hdeA ( Fig . 2 ) .\n",
      "16\n",
      "6\n",
      "Start y end del regulador: [36, 40]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: purA\n",
      "The average repression ratio was 0.44 for purA ( see Fig . 5 , WT ) and 0.60 for hdeA ( see Fig . 6 , WT ) relative to controls without MarA . The average repression ratio was 0.44 for purA ( see Fig . 5 , WT ) and 0.60 for @Regulated$ ( see Fig . 6 , WT ) relative to controls without @TF$ .\n",
      "19\n",
      "32\n",
      "Start y end del regulador: [136, 140]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [81, 85]\n",
      "Regulado: hdeA\n",
      "Furthermore , we have demonstrated that Lrp - mediated activation of P argO is also operative , and even more efficient , in an argP deletion mutant ( see below ) . Furthermore , we have demonstrated that @TF$ - mediated activation of P @Regulated$ is also operative , and even more efficient , in an argP deletion mutant ( see below ) .\n",
      "12\n",
      "6\n",
      "Start y end del regulador: [40, 43]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [71, 75]\n",
      "Regulado: argO\n",
      "Whereas # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396Transcriptional activation of the dnaA gene by IciA protein 393 Fig . 6 . Whereas # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396Transcriptional activation of the @Regulated$ gene by @TF$ protein 393 Fig . 6 .\n",
      "18\n",
      "21\n",
      "Start y end del regulador: [125, 129]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [112, 116]\n",
      "Regulado: dnaA\n",
      "These results showed that MarA by itself could directly repress purA and hdeA transcription . These results showed that @TF$ by itself could directly repress purA and @Regulated$ transcription .\n",
      "12\n",
      "4\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: hdeA\n",
      "These results showed that MarA by itself could directly repress purA and hdeA transcription . These results showed that @TF$ by itself could directly repress @Regulated$ and hdeA transcription .\n",
      "10\n",
      "4\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: purA\n",
      "Upon transformation into strain JCP75 , this plasmid led to red colonies on MacConkey lactose agar plates supplemented with lysine , confirming that ArgP is able to enhance dapB transcription . Upon transformation into strain JCP75 , this plasmid led to red colonies on MacConkey lactose agar plates supplemented with lysine , confirming that @TF$ is able to enhance @Regulated$ transcription .\n",
      "28\n",
      "23\n",
      "Start y end del regulador: [149, 153]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [173, 177]\n",
      "Regulado: dapB\n",
      "Recent genetic data show that Rob represses transcription of the galT gene ( M . H . J . B . and B . D . , unpublished results ) . Recent genetic data show that @TF$ represses transcription of the @Regulated$ gene ( M . H . J . B . and B . D . , unpublished results ) .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [30, 33]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: galT\n",
      "The L - leucine concentration was set at 7 mM , corresponding to the concentration of total leucine in cells grown in © 2009 The Authors Journal compilation © 2009 Blackwell Publishing Ltd , Molecular Microbiology , 74 , 1513 – 1526Lrp and ArgP activation of argO transcription 1517 leucine - supplemented medium ( Quay et al . , 1977 ) . The L - leucine concentration was set at 7 mM , corresponding to the concentration of total leucine in cells grown in © 2009 The Authors Journal compilation © 2009 Blackwell Publishing Ltd , Molecular Microbiology , 74 , 1513 – 1526Lrp and @TF$ activation of @Regulated$ transcription 1517 leucine - supplemented medium ( Quay et al . , 1977 ) .\n",
      "46\n",
      "43\n",
      "Start y end del regulador: [240, 244]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [259, 263]\n",
      "Regulado: argO\n",
      "Operons observed to be differentially expressed include not only all four of the transcripts , hmpA , ytfE , ygbA , and hcp - hcr , known or predicted to be NsrR - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) . Operons observed to be differentially expressed include not only all four of the transcripts , @Regulated$ , ytfE , ygbA , and hcp - hcr , known or predicted to be @TF$ - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) .\n",
      "15\n",
      "31\n",
      "Start y end del regulador: [157, 161]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [95, 99]\n",
      "Regulado: hmpA\n",
      "This Fnr ( D154A ) protein was able to activate ( napF - lacZ ) expression in vivo ( data not shown ) . This @TF$ ( D154A ) protein was able to activate ( @Regulated$ - lacZ ) expression in vivo ( data not shown ) .\n",
      "11\n",
      "1\n",
      "Start y end del regulador: [5, 8]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: napF\n",
      "Operons observed to be differentially expressed include not only all four of the transcripts , hmpA , ytfE , ygbA , and hcp - hcr , known or predicted to be NsrR - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) . Operons observed to be differentially expressed include not only all four of the transcripts , hmpA , ytfE , @Regulated$ , and hcp - hcr , known or predicted to be @TF$ - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) .\n",
      "19\n",
      "31\n",
      "Start y end del regulador: [157, 161]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [109, 113]\n",
      "Regulado: ygbA\n",
      "Operons observed to be differentially expressed include not only all four of the transcripts , hmpA , ytfE , ygbA , and hcp - hcr , known or predicted to be NsrR - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) . Operons observed to be differentially expressed include not only all four of the transcripts , hmpA , @Regulated$ , ygbA , and hcp - hcr , known or predicted to be @TF$ - regulated ( 4 , 40 ) , but also other transcripts predicted to be regulated by nitrite or RNS generated from nitrite , for example , nitric oxide ( 11 ) .\n",
      "17\n",
      "31\n",
      "Start y end del regulador: [157, 161]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [102, 106]\n",
      "Regulado: ytfE\n",
      "These observations confirm the importance of the 64.5 sequence in the Fnr - dependent induction of aeg - 46.5 operon expression . These observations confirm the importance of the 64.5 sequence in the @TF$ - dependent induction of @Regulated$ - 46.5 operon expression .\n",
      "16\n",
      "11\n",
      "Start y end del regulador: [70, 73]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [99, 102]\n",
      "Regulado: aeg\n",
      "Because previous studies have shown that the MarA - binding sites of two other repressed promoters ( hdeA and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position . Because previous studies have shown that the @TF$ - binding sites of two other repressed promoters ( @Regulated$ and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .\n",
      "17\n",
      "7\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [101, 105]\n",
      "Regulado: hdeA\n",
      "Because previous studies have shown that the MarA - binding sites of two other repressed promoters ( hdeA and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position . Because previous studies have shown that the @TF$ - binding sites of two other repressed promoters ( hdeA and @Regulated$ ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .\n",
      "19\n",
      "7\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [110, 114]\n",
      "Regulado: purA\n",
      "The three metD genes are regulated by the MetJ repressor . The three @Regulated$ genes are regulated by the @TF$ repressor .\n",
      "2\n",
      "8\n",
      "Start y end del regulador: [42, 46]\n",
      "Regulador: MetJ\n",
      "Start y end del regulado: [10, 14]\n",
      "Regulado: metD\n",
      "( ii ) Mutations within residues in this region in AraC ( Cys204 3 Tyr , Ser208 3 Ala , Arg210 3 Cys , and His212 3 Tyr or Ala ) reduced binding to and decreased transcriptional activation from the P araBAD promoter ( 26 – 28 , 39 , 78 ) . ( ii ) Mutations within residues in this region in @TF$ ( Cys204 3 Tyr , Ser208 3 Ala , Arg210 3 Cys , and His212 3 Tyr or Ala ) reduced binding to and decreased transcriptional activation from the P @Regulated$ promoter ( 26 – 28 , 39 , 78 ) .\n",
      "41\n",
      "10\n",
      "Start y end del regulador: [51, 55]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [200, 206]\n",
      "Regulado: araBAD\n",
      "In the case of the ydbC promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by NsrR . In the case of the @Regulated$ promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by @TF$ .\n",
      "5\n",
      "39\n",
      "Start y end del regulador: [221, 225]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: ydbC\n",
      "Next , we attempted to reproduce MelR - dependent repression of the melR promoter in vitro . Next , we attempted to reproduce @TF$ - dependent repression of the @Regulated$ promoter in vitro .\n",
      "12\n",
      "6\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [68, 72]\n",
      "Regulado: melR\n",
      "nanC expression was predicted to be regulated by the two - component regulatory system CpxR - CpxA by transcriptome analysis ( 16 ) . @Regulated$ expression was predicted to be regulated by the two - component regulatory system @TF$ - CpxA by transcriptome analysis ( 16 ) .\n",
      "0\n",
      "14\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: CpxR\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: nanC\n",
      "The – 111 to – 91 site overlaps sequences identified as binding the anaerobic regulators , ArcA and FNR , and the catabolite regulator CRP which have been shown to regulate the acnA P2 promoter ( Cunningham et al . , 1997 ) . The – 111 to – 91 site overlaps sequences identified as binding the anaerobic regulators , ArcA and FNR , and the catabolite regulator @TF$ which have been shown to regulate the @Regulated$ P2 promoter ( Cunningham et al . , 1997 ) .\n",
      "32\n",
      "24\n",
      "Start y end del regulador: [135, 138]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [177, 181]\n",
      "Regulado: acnA\n",
      "As shown in Fig . 2 , induction of ArgP synthesis was soon followed by a sharp stimulation of dapB lacZ transcription . As shown in Fig . 2 , induction of @TF$ synthesis was soon followed by a sharp stimulation of @Regulated$ lacZ transcription .\n",
      "19\n",
      "9\n",
      "Start y end del regulador: [35, 39]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [94, 98]\n",
      "Regulado: dapB\n",
      "In vivo S1 nuclease - protection assays ( Fig . 4 ) and in vitro run - off transcription assays ( Figs 5 and 6 ) indicate that IciA protein specifically activates transcription from dnaA promoter 1P . In vivo S1 nuclease - protection assays ( Fig . 4 ) and in vitro run - off transcription assays ( Figs 5 and 6 ) indicate that @TF$ protein specifically activates transcription from @Regulated$ promoter 1P .\n",
      "34\n",
      "28\n",
      "Start y end del regulador: [127, 131]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [182, 186]\n",
      "Regulado: dnaA\n",
      "The microarray data for the other promoters that are apparently activated by NsrR must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of ytfE , the 14 - fold repression of hmpA , and the up - to - 15 - fold repression of hcp . The microarray data for the other promoters that are apparently activated by @TF$ must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of ytfE , the 14 - fold repression of hmpA , and the up - to - 15 - fold repression of @Regulated$ .\n",
      "84\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [447, 450]\n",
      "Regulado: hcp\n",
      "The microarray data for the other promoters that are apparently activated by NsrR must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of ytfE , the 14 - fold repression of hmpA , and the up - to - 15 - fold repression of hcp . The microarray data for the other promoters that are apparently activated by @TF$ must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of ytfE , the 14 - fold repression of @Regulated$ , and the up - to - 15 - fold repression of hcp .\n",
      "71\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [398, 402]\n",
      "Regulado: hmpA\n",
      "The figure also shows the measured amount of MelR - dependent repression of the melR promoter on the KK81 fragment carrying the different Site R substitutions . The figure also shows the measured amount of @TF$ - dependent repression of the @Regulated$ promoter on the KK81 fragment carrying the different Site R substitutions .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: melR\n",
      "The microarray data for the other promoters that are apparently activated by NsrR must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of ytfE , the 14 - fold repression of hmpA , and the up - to - 15 - fold repression of hcp . The microarray data for the other promoters that are apparently activated by @TF$ must be evaluated cautiously , however , because few of these transcripts encode proteins known or suspected to be involved in the response to RNS , and most of the differences were much smaller ( typically two - to threefold apparent activation ) than the 30 - fold repression of @Regulated$ , the 14 - fold repression of hmpA , and the up - to - 15 - fold repression of hcp .\n",
      "63\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [363, 367]\n",
      "Regulado: ytfE\n",
      "Three potential Lrp binding sites can be found in the argO control region ( Pot - box 1 , 2 and 3 ; Fig . 1 ) . Three potential @TF$ binding sites can be found in the @Regulated$ control region ( Pot - box 1 , 2 and 3 ; Fig . 1 ) .\n",
      "10\n",
      "2\n",
      "Start y end del regulador: [16, 19]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [54, 58]\n",
      "Regulado: argO\n",
      "The experiment illustrated in Figure 2 shows that MelR has little or no effect on the synthesis of RNA - 1 but represses synthesis of the transcript from the melR promoter . The experiment illustrated in Figure 2 shows that @TF$ has little or no effect on the synthesis of RNA - 1 but represses synthesis of the transcript from the @Regulated$ promoter .\n",
      "29\n",
      "8\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [158, 162]\n",
      "Regulado: melR\n",
      "The transcription of the marA gene is regulated by the repressor MarR ( Alekshun and Levy , 1997 ) . The transcription of the @Regulated$ gene is regulated by the repressor @TF$ ( Alekshun and Levy , 1997 ) .\n",
      "4\n",
      "11\n",
      "Start y end del regulador: [65, 69]\n",
      "Regulador: MarR\n",
      "Start y end del regulado: [25, 29]\n",
      "Regulado: marA\n",
      "The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the melR promoter by MelR differs from the repression found with the longer KK81 fragment . The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the @Regulated$ promoter by @TF$ differs from the repression found with the longer KK81 fragment .\n",
      "19\n",
      "22\n",
      "Start y end del regulador: [114, 118]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: melR\n",
      "In a strain with a wild - type chromosomal marRAB operon , Rob activates the promoter , and therefore the rob + mar + strain produces more MarR repressor than the rob mar + strain . In a strain with a wild - type chromosomal @Regulated$ operon , @TF$ activates the promoter , and therefore the rob + mar + strain produces more MarR repressor than the rob mar + strain .\n",
      "9\n",
      "12\n",
      "Start y end del regulador: [59, 62]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [43, 49]\n",
      "Regulado: marRAB\n",
      "For example , binding of Lrp to the isolated strong binding site 2 of the ilvIH control region occurred in the nM range ( Cui et al . , 1996 ) but , in contrast , binding to site 2 of the pap operon could not be detected by EMSA , although this site was successfully used for the growth of co - crystals ( de los Rios and Perona , 2007 ) . For example , binding of @TF$ to the isolated strong binding site 2 of the @Regulated$ control region occurred in the nM range ( Cui et al . , 1996 ) but , in contrast , binding to site 2 of the pap operon could not be detected by EMSA , although this site was successfully used for the growth of co - crystals ( de los Rios and Perona , 2007 ) .\n",
      "15\n",
      "5\n",
      "Start y end del regulador: [25, 28]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [74, 79]\n",
      "Regulado: ilvIH\n",
      "ArgP binds a site upstream of the dapB promoter . @TF$ binds a site upstream of the @Regulated$ promoter .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: dapB\n",
      "Nystrom showed that induction of the RelA protein results in high expression of eda ( 47 ) . Nystrom showed that induction of the @TF$ protein results in high expression of @Regulated$ ( 47 ) .\n",
      "13\n",
      "6\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: RelA\n",
      "Start y end del regulado: [80, 83]\n",
      "Regulado: eda\n",
      "In summary , these results demonstrate that gntT is positively regulated by CRP . In summary , these results demonstrate that @Regulated$ is positively regulated by @TF$ .\n",
      "7\n",
      "12\n",
      "Start y end del regulador: [76, 79]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [44, 48]\n",
      "Regulado: gntT\n",
      "We reasoned that some part of the melAB promoter might be responsible for the high level of MelR dependent repression of the melR promoter seen with the KK81 fragment . We reasoned that some part of the melAB promoter might be responsible for the high level of @TF$ dependent repression of the @Regulated$ promoter seen with the KK81 fragment .\n",
      "22\n",
      "17\n",
      "Start y end del regulador: [92, 96]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [125, 129]\n",
      "Regulado: melR\n",
      "2 When the two dnaA promoters were repressed by DnaA protein , IciA protein was able to restore transcription from the 1P promoter with little effect on transcription from the 2P promoter ( 25 ) . 2 When the two @Regulated$ promoters were repressed by @TF$ protein , IciA protein was able to restore transcription from the 1P promoter with little effect on transcription from the 2P promoter ( 25 ) .\n",
      "4\n",
      "9\n",
      "Start y end del regulador: [48, 52]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: dnaA\n",
      "2 When the two dnaA promoters were repressed by DnaA protein , IciA protein was able to restore transcription from the 1P promoter with little effect on transcription from the 2P promoter ( 25 ) . 2 When the two @Regulated$ promoters were repressed by DnaA protein , @TF$ protein was able to restore transcription from the 1P promoter with little effect on transcription from the 2P promoter ( 25 ) .\n",
      "4\n",
      "12\n",
      "Start y end del regulador: [63, 67]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: dnaA\n",
      "The results of the present study extend these observations and show that the MetR protein also stimulates the in vitro expression of metH and that both the MetE and MetH proteins synthesized in vitro are enzymatically active . The results of the present study extend these observations and show that the @TF$ protein also stimulates the in vitro expression of @Regulated$ and that both the MetE and MetH proteins synthesized in vitro are enzymatically active .\n",
      "22\n",
      "13\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [133, 137]\n",
      "Regulado: metH\n",
      "Thus , stimulation of b1452 was dependent on MarA and a region ( – 131 to – 111 ) identified as a potential binding site for the activators . map and mdaB . Thus , stimulation of @Regulated$ was dependent on @TF$ and a region ( – 131 to – 111 ) identified as a potential binding site for the activators . map and mdaB .\n",
      "4\n",
      "8\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [22, 27]\n",
      "Regulado: b1452\n",
      "The mechanism of IciA protein stimulation of transcription from dnaA promoter 1P in the presence of inhibitory amounts of DnaA protein was examined . The mechanism of IciA protein stimulation of transcription from @Regulated$ promoter 1P in the presence of inhibitory amounts of @TF$ protein was examined .\n",
      "9\n",
      "19\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: dnaA\n",
      "The mechanism of IciA protein stimulation of transcription from dnaA promoter 1P in the presence of inhibitory amounts of DnaA protein was examined . The mechanism of @TF$ protein stimulation of transcription from @Regulated$ promoter 1P in the presence of inhibitory amounts of DnaA protein was examined .\n",
      "9\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: dnaA\n",
      "Results in Fig . 5 show that MelR - dependent repression of the melR promoter is little altered by the changes in Site 1 ( KK81S3 ) or Site 1 0 ( KK81S4 ) . Results in Fig . 5 show that @TF$ - dependent repression of the @Regulated$ promoter is little altered by the changes in Site 1 ( KK81S3 ) or Site 1 0 ( KK81S4 ) .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: melR\n",
      "MelR - dependent repression of the melR promoter carried by each derivative was then measured , as above . @TF$ - dependent repression of the @Regulated$ promoter carried by each derivative was then measured , as above .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [35, 39]\n",
      "Regulado: melR\n",
      "ArgP and cadBA regulation . @TF$ and @Regulated$ regulation .\n",
      "2\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [9, 14]\n",
      "Regulado: cadBA\n",
      "RESULTS CRP both directly and indirectly activates rhaBAD . RESULTS @TF$ both directly and indirectly activates @Regulated$ .\n",
      "7\n",
      "1\n",
      "Start y end del regulador: [8, 11]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [51, 57]\n",
      "Regulado: rhaBAD\n",
      "Hwang we have shown that IciA protein activates transcription from the promoter 1P of the dnaA gene . Hwang we have shown that @TF$ protein activates transcription from the promoter 1P of the @Regulated$ gene .\n",
      "15\n",
      "5\n",
      "Start y end del regulador: [25, 29]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [90, 94]\n",
      "Regulado: dnaA\n",
      "Thus , when MelR - binding Site 2 is altered in the KK81 fragment , the pattern of MelRdependent repression of the melR promoter is similar to that seen with the shorter KK101 fragment ( Fig . 4 ) . Thus , when @TF$ - binding Site 2 is altered in the KK81 fragment , the pattern of MelRdependent repression of the @Regulated$ promoter is similar to that seen with the shorter KK101 fragment ( Fig . 4 ) .\n",
      "22\n",
      "3\n",
      "Start y end del regulador: [12, 16]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [115, 119]\n",
      "Regulado: melR\n",
      "On the other hand , activation of the P araB promoter was delayed when glucose was present , consistent with the regulation of this promoter also by CRP ( 6 ) . On the other hand , activation of the P @Regulated$ promoter was delayed when glucose was present , consistent with the regulation of this promoter also by @TF$ ( 6 ) .\n",
      "9\n",
      "27\n",
      "Start y end del regulador: [149, 152]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: araB\n",
      "This suggests that IHF binding to ibs has a small inhibitory effect on the ability of NanR to activate fimB expression . This suggests that IHF binding to ibs has a small inhibitory effect on the ability of @TF$ to activate @Regulated$ expression .\n",
      "19\n",
      "16\n",
      "Start y end del regulador: [86, 90]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [103, 107]\n",
      "Regulado: fimB\n",
      "Since our findings indicated that ArgP is needed for activation of lysP transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant . Since our findings indicated that @TF$ is needed for activation of @Regulated$ transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .\n",
      "11\n",
      "5\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: lysP\n",
      "Thus , it seems that Fnr can regulate adhE expression from the downstream transcriptional start site only in the physical absence of the upstream region . Thus , it seems that @TF$ can regulate @Regulated$ expression from the downstream transcriptional start site only in the physical absence of the upstream region .\n",
      "8\n",
      "5\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [38, 42]\n",
      "Regulado: adhE\n",
      "The results suggest that the NarP protein specifically interacts with the 44.5 region in order to activate aeg - 46.5 operon expression . The results suggest that the @TF$ protein specifically interacts with the 44.5 region in order to activate @Regulated$ - 46.5 operon expression .\n",
      "17\n",
      "5\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [107, 110]\n",
      "Regulado: aeg\n",
      "An important observation was that purified MarA was able to activate transcription in vitro of nfnB . An important observation was that purified @TF$ was able to activate transcription in vitro of @Regulated$ .\n",
      "15\n",
      "6\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [95, 99]\n",
      "Regulado: nfnB\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( nfo , rimK , tolC , @Regulated$ , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed .\n",
      "17\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: fldB\n",
      "These results together indicated that marbox 3 , and particularly its RE2 , was critical for repression of hdeA by MarA . These results together indicated that marbox 3 , and particularly its RE2 , was critical for repression of @Regulated$ by @TF$ .\n",
      "18\n",
      "20\n",
      "Start y end del regulador: [115, 119]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [107, 111]\n",
      "Regulado: hdeA\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( nfo , rimK , tolC , fldB , and @Regulated$ ) were activated with values below threshold while one ( ribA ) appeared slightly repressed .\n",
      "20\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [86, 90]\n",
      "Regulado: mdaA\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( @Regulated$ , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed .\n",
      "11\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [55, 58]\n",
      "Regulado: nfo\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( @Regulated$ ) appeared slightly repressed .\n",
      "31\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [148, 152]\n",
      "Regulado: ribA\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( nfo , @Regulated$ , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed .\n",
      "13\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [61, 65]\n",
      "Regulado: rimK\n",
      "Of the remaining seven SoxS - activated genes , five ( nfo , rimK , tolC , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed . Of the remaining seven @TF$ - activated genes , five ( nfo , rimK , @Regulated$ , fldB , and mdaA ) were activated with values below threshold while one ( ribA ) appeared slightly repressed .\n",
      "15\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [68, 72]\n",
      "Regulado: tolC\n",
      "Taking these results together with those presented above , it is concluded that the cooperative interaction of NagC bound to O NC1 and O NC2 , with IHF bound to the site identified in this work , forms a nucleoprotein complex that activates fimB expression . Taking these results together with those presented above , it is concluded that the cooperative interaction of NagC bound to O NC1 and O NC2 , with @TF$ bound to the site identified in this work , forms a nucleoprotein complex that activates @Regulated$ expression .\n",
      "43\n",
      "27\n",
      "Start y end del regulador: [148, 151]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: fimB\n",
      "Taking these results together with those presented above , it is concluded that the cooperative interaction of NagC bound to O NC1 and O NC2 , with IHF bound to the site identified in this work , forms a nucleoprotein complex that activates fimB expression . Taking these results together with those presented above , it is concluded that the cooperative interaction of @TF$ bound to O NC1 and O NC2 , with IHF bound to the site identified in this work , forms a nucleoprotein complex that activates @Regulated$ expression .\n",
      "43\n",
      "17\n",
      "Start y end del regulador: [111, 115]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [241, 245]\n",
      "Regulado: fimB\n",
      "Analysis of the lysP promoter revealed a conserved T - N 11 - A motif , a typical binding site of LysR - type transcriptional regu - VOL . Analysis of the @Regulated$ promoter revealed a conserved T - N 11 - A motif , a typical binding site of @TF$ - type transcriptional regu - VOL .\n",
      "3\n",
      "21\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: LysR\n",
      "Start y end del regulado: [16, 20]\n",
      "Regulado: lysP\n",
      "The identification of NagC as an activator of fimB expression indicated that the expression of the recombinase would be inhibited by GlcNAc . The identification of @TF$ as an activator of @Regulated$ expression indicated that the expression of the recombinase would be inhibited by GlcNAc .\n",
      "8\n",
      "3\n",
      "Start y end del regulador: [22, 26]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [46, 50]\n",
      "Regulado: fimB\n",
      "Thus , we have investigated the involvement of aCTD in CRP - dependent activation at the melAB promoter by using a preparation of E . coli RNAP , reconstituted with truncated a subunits that lack the C - terminal 73 residues . Thus , we have investigated the involvement of aCTD in @TF$ - dependent activation at the @Regulated$ promoter by using a preparation of E . coli RNAP , reconstituted with truncated a subunits that lack the C - terminal 73 residues .\n",
      "16\n",
      "10\n",
      "Start y end del regulador: [55, 58]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [89, 94]\n",
      "Regulado: melAB\n",
      "RNA polymerase binding to the region containing dnaA promoter 1P ( lane 3 ) was inhibited by DnaA protein ( lane 4 ) . RNA polymerase binding to the region containing @Regulated$ promoter 1P ( lane 3 ) was inhibited by @TF$ protein ( lane 4 ) .\n",
      "7\n",
      "17\n",
      "Start y end del regulador: [93, 97]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: dnaA\n",
      "Unlike most other promoters , yhbW seemed to require additional sequence upstream of the – 76 to – 57 site for maximum activation by MarA , SoxS and Rob . Unlike most other promoters , @Regulated$ seemed to require additional sequence upstream of the – 76 to – 57 site for maximum activation by @TF$ , SoxS and Rob .\n",
      "5\n",
      "24\n",
      "Start y end del regulador: [133, 137]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [30, 34]\n",
      "Regulado: yhbW\n",
      "Thus , NanR and NagC appear to activate fimB expression independently . Thus , NanR and @TF$ appear to activate @Regulated$ expression independently .\n",
      "8\n",
      "4\n",
      "Start y end del regulador: [16, 20]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: fimB\n",
      "Thus , NanR and NagC appear to activate fimB expression independently . Thus , @TF$ and NagC appear to activate @Regulated$ expression independently .\n",
      "8\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: fimB\n",
      "Accordingly we reasoned that a loss of fimB expression as a result of decreased NagC activity in the D3 background might be compensated for by enhanced activation by NanR for some reason . Accordingly we reasoned that a loss of @Regulated$ expression as a result of decreased NagC activity in the D3 background might be compensated for by enhanced activation by @TF$ for some reason .\n",
      "7\n",
      "28\n",
      "Start y end del regulador: [166, 170]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: fimB\n",
      "However , the situation is not so simple , and , in the absence of melibiose , efficient repression requires DNA sequences upstream of the melR promoter ; in particular , MelR binding Site 2 is required . However , the situation is not so simple , and , in the absence of melibiose , efficient repression requires DNA sequences upstream of the @Regulated$ promoter ; in particular , @TF$ binding Site 2 is required .\n",
      "25\n",
      "31\n",
      "Start y end del regulador: [171, 175]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [139, 143]\n",
      "Regulado: melR\n",
      "However , CRP is able to activate transcription from the melAB promoter with RNAP containing both fulllength and truncated a subunits . However , @TF$ is able to activate transcription from the @Regulated$ promoter with RNAP containing both fulllength and truncated a subunits .\n",
      "10\n",
      "2\n",
      "Start y end del regulador: [10, 13]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [57, 62]\n",
      "Regulado: melAB\n",
      "It should be mentioned that Plamann and Stauffer ( 35 ) noted that a deletion in this area resulted in a loss of MetR activation of the in vivo synthesis of a metE - lac fusion protein . It should be mentioned that Plamann and Stauffer ( 35 ) noted that a deletion in this area resulted in a loss of @TF$ activation of the in vivo synthesis of a @Regulated$ - lac fusion protein .\n",
      "32\n",
      "23\n",
      "Start y end del regulador: [113, 117]\n",
      "Regulador: MetR\n",
      "Start y end del regulado: [159, 163]\n",
      "Regulado: metE\n",
      "The mutation in nagO1 behaved as expected : expression was higher in glycerol , giving a © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and GalS repression at galP 151 value comparable to the nagC mutation and expression was strongly increased by growth on galactose . The mutation in nagO1 behaved as expected : expression was higher in glycerol , giving a © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , @TF$ and GalS repression at @Regulated$ 151 value comparable to the nagC mutation and expression was strongly increased by growth on galactose .\n",
      "42\n",
      "37\n",
      "Start y end del regulador: [208, 212]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [236, 240]\n",
      "Regulado: galP\n",
      "The mutation in nagO1 behaved as expected : expression was higher in glycerol , giving a © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and GalS repression at galP 151 value comparable to the nagC mutation and expression was strongly increased by growth on galactose . The mutation in nagO1 behaved as expected : expression was higher in glycerol , giving a © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and @TF$ repression at @Regulated$ 151 value comparable to the nagC mutation and expression was strongly increased by growth on galactose .\n",
      "42\n",
      "39\n",
      "Start y end del regulador: [217, 221]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [236, 240]\n",
      "Regulado: galP\n",
      "Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how MelR - dependent repression of the melR promoter is modulated by the elements at the upstream melAB promoter . Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how @TF$ - dependent repression of the @Regulated$ promoter is modulated by the elements at the upstream melAB promoter .\n",
      "27\n",
      "21\n",
      "Start y end del regulador: [107, 111]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: melR\n",
      "RhaR residues D276 , E284 , and D285 are important for rhaSR transcription activation . @TF$ residues D276 , E284 , and D285 are important for @Regulated$ transcription activation .\n",
      "11\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [55, 60]\n",
      "Regulado: rhaSR\n",
      "We hypothesize that CRP binding to some or all of these sites may influence transcription activation at rhaSR . We hypothesize that @TF$ binding to some or all of these sites may influence transcription activation at @Regulated$ .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [20, 23]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [104, 109]\n",
      "Regulado: rhaSR\n",
      "IciA protein stimulates RNA polymerase binding to dnaA promoter 1P in the presence of inhibitory amounts of DnaA protein . @TF$ protein stimulates RNA polymerase binding to @Regulated$ promoter 1P in the presence of inhibitory amounts of DnaA protein .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: dnaA\n",
      "An alternative explanation is that repression occurs by the ' roadblock ' mechanism described for PurR repression of the purB promoter ( He and Zalkin , 1992 ) . An alternative explanation is that repression occurs by the ' roadblock ' mechanism described for @TF$ repression of the @Regulated$ promoter ( He and Zalkin , 1992 ) .\n",
      "19\n",
      "15\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: PurR\n",
      "Start y end del regulado: [121, 125]\n",
      "Regulado: purB\n",
      "Schematic representation of regulation of araBAD and araC promoters by CAP , P1 , and P2 . negative control of Pc by P1 and P2 . Schematic representation of regulation of @Regulated$ and araC promoters by @TF$ , P1 , and P2 . negative control of Pc by P1 and P2 .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [71, 74]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [42, 48]\n",
      "Regulado: araBAD\n",
      "Schematic representation of regulation of araBAD and araC promoters by CAP , P1 , and P2 . negative control of Pc by P1 and P2 . Schematic representation of regulation of araBAD and @Regulated$ promoters by @TF$ , P1 , and P2 . negative control of Pc by P1 and P2 .\n",
      "7\n",
      "10\n",
      "Start y end del regulador: [71, 74]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: araC\n",
      "Translation of the chiP mRNA is repressed by base - pairing with the ChiX sRNA . Translation of the @Regulated$ mRNA is repressed by base - pairing with the @TF$ sRNA .\n",
      "3\n",
      "13\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: chiP\n",
      "These data clearly indicate that the hcp promoter is activated by FNR and repressed by NsrR . These data clearly indicate that the @Regulated$ promoter is activated by FNR and repressed by @TF$ .\n",
      "6\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [37, 40]\n",
      "Regulado: hcp\n",
      "In summary , the results clearly demonstrate that both operators are required for repression of gntT by GntR . In summary , the results clearly demonstrate that both operators are required for repression of @Regulated$ by @TF$ .\n",
      "15\n",
      "17\n",
      "Start y end del regulador: [104, 108]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [96, 100]\n",
      "Regulado: gntT\n",
      "Finally , either NarL or NarP was able to activate nrfA - lacZ expression independently of the other , while only NarL could repress expression . Finally , either NarL or @TF$ was able to activate @Regulated$ - lacZ expression independently of the other , while only NarL could repress expression .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [25, 29]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: nrfA\n",
      "In the case of the ytfE promoter , repression by NsrR is the dominant regulatory mechanism , though there is also some evidence for anaerobic repression by FNR , as reported previously ( 25 ) . In the case of the @Regulated$ promoter , repression by @TF$ is the dominant regulatory mechanism , though there is also some evidence for anaerobic repression by FNR , as reported previously ( 25 ) .\n",
      "5\n",
      "10\n",
      "Start y end del regulador: [49, 53]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [19, 23]\n",
      "Regulado: ytfE\n",
      "It is interesting to compare the characteristics of NagCmediated repression of nag degradative genes with the NagC - mediated activation of the glm genes . It is interesting to compare the characteristics of NagCmediated repression of @Regulated$ degradative genes with the @TF$ - mediated activation of the glm genes .\n",
      "11\n",
      "16\n",
      "Start y end del regulador: [110, 114]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [79, 82]\n",
      "Regulado: nag\n",
      "These results show that the NarP protein activates F ( aeg - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the NarL protein antagonizes this activation . These results show that the NarP protein activates F ( @Regulated$ - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the @TF$ protein antagonizes this activation .\n",
      "10\n",
      "26\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [55, 58]\n",
      "Regulado: aeg\n",
      "To confirm the role of ArgP in the regulation of lysP transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) . To confirm the role of @TF$ in the regulation of @Regulated$ transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) .\n",
      "10\n",
      "5\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [49, 53]\n",
      "Regulado: lysP\n",
      "At low to intermediate levels of nitrate ( 2 to 6 mM ) NarP appeared to antagonize the ability of NarL to activate fdnG operon expression . At low to intermediate levels of nitrate ( 2 to 6 mM ) NarP appeared to antagonize the ability of @TF$ to activate @Regulated$ operon expression .\n",
      "23\n",
      "20\n",
      "Start y end del regulador: [98, 102]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [115, 119]\n",
      "Regulado: fdnG\n",
      "Both inaA and marRAB are known Rob - regulated genes ( 5 , 21 ) . Both @Regulated$ and marRAB are known @TF$ - regulated genes ( 5 , 21 ) .\n",
      "1\n",
      "6\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [5, 9]\n",
      "Regulado: inaA\n",
      "Both inaA and marRAB are known Rob - regulated genes ( 5 , 21 ) . Both inaA and @Regulated$ are known @TF$ - regulated genes ( 5 , 21 ) .\n",
      "3\n",
      "6\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [14, 20]\n",
      "Regulado: marRAB\n",
      "Results in Figure 3C show that Val273 MelR gives a small enhancement in repression of the melR promoter carrying the 13T substitution in site R , B . Results in Figure 3C show that Val273 @TF$ gives a small enhancement in repression of the @Regulated$ promoter carrying the 13T substitution in site R , B .\n",
      "16\n",
      "7\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [90, 94]\n",
      "Regulado: melR\n",
      "NsrR : sixth protein to regulate transcription factor at P nrfA . @TF$ : sixth protein to regulate transcription factor at P @Regulated$ .\n",
      "10\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [59, 63]\n",
      "Regulado: nrfA\n",
      "First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] . First , repression of nanC and @Regulated$ divergent transcripts by @TF$ has been observed both in vivo and in vitro [ 32 ] .\n",
      "6\n",
      "10\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [31, 35]\n",
      "Regulado: fimB\n",
      "First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] . First , repression of @Regulated$ and fimB divergent transcripts by @TF$ has been observed both in vivo and in vitro [ 32 ] .\n",
      "4\n",
      "10\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [22, 26]\n",
      "Regulado: nanC\n",
      "In its description as IciA , ArgP has been reported to regulate dnaA and nrdA transcription ( 20 , 27 – 29 ) . In its description as IciA , @TF$ has been reported to regulate @Regulated$ and nrdA transcription ( 20 , 27 – 29 ) .\n",
      "12\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: dnaA\n",
      "It thus seems likely that the ability of IHF alone to activate fimB expression is enhanced in the D3 mutant background . It thus seems likely that the ability of @TF$ alone to activate @Regulated$ expression is enhanced in the D3 mutant background .\n",
      "12\n",
      "8\n",
      "Start y end del regulador: [41, 44]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [63, 67]\n",
      "Regulado: fimB\n",
      "In its description as IciA , ArgP has been reported to regulate dnaA and nrdA transcription ( 20 , 27 – 29 ) . In its description as IciA , @TF$ has been reported to regulate dnaA and @Regulated$ transcription ( 20 , 27 – 29 ) .\n",
      "14\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: nrdA\n",
      "Therefore , arafl is essential for both repression and activation and acts as a switch to allow both the repressor and the activator forms of the AraC protein to control the araBAD promoter . Therefore , arafl is essential for both repression and activation and acts as a switch to allow both the repressor and the activator forms of the @TF$ protein to control the @Regulated$ promoter .\n",
      "31\n",
      "26\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: araBAD\n",
      "Therefore , arafl is essential for both repression and activation and acts as a switch to allow both the repressor and the activator forms of the AraC protein to control the araBAD promoter . Therefore , arafl is essential for both repression and activation and acts as a switch to allow both the repressor and the activator forms of the @TF$ protein to control the @Regulated$ promoter .\n",
      "31\n",
      "26\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [174, 180]\n",
      "Regulado: araBAD\n",
      "However , in the presence of MarA the rob transcript levels decreased over time , but as expected the levels of the gnd transcript ( the control ) remained unaffected ( Fig . 6C ) . However , in the presence of @TF$ the @Regulated$ transcript levels decreased over time , but as expected the levels of the gnd transcript ( the control ) remained unaffected ( Fig . 6C ) .\n",
      "8\n",
      "6\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [38, 41]\n",
      "Regulado: rob\n",
      "Thus MarA is able to repress the rob promoter after the preformed open complex clears the promoter . Thus @TF$ is able to repress the @Regulated$ promoter after the preformed open complex clears the promoter .\n",
      "7\n",
      "1\n",
      "Start y end del regulador: [5, 9]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [33, 36]\n",
      "Regulado: rob\n",
      "The latter step is critical for the initiation of the entire induction cascade , as the chbBCARFG operon itself is repressed by NagC ( Plumbridge and Pellegrini , 2004 ) . The latter step is critical for the initiation of the entire induction cascade , as the @Regulated$ operon itself is repressed by @TF$ ( Plumbridge and Pellegrini , 2004 ) .\n",
      "16\n",
      "22\n",
      "Start y end del regulador: [128, 132]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [88, 97]\n",
      "Regulado: chbBCARFG\n",
      "Thus the D3 region is not apparently required for NanR to activate fimB expression in the wildtype background , and it could contain specific sequences that inhibit this . Thus the D3 region is not apparently required for @TF$ to activate @Regulated$ expression in the wildtype background , and it could contain specific sequences that inhibit this .\n",
      "12\n",
      "9\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [67, 71]\n",
      "Regulado: fimB\n",
      "Thus , binding of MetJ to this area could serve to simultaneously repress both metR and metE expression . Thus , binding of @TF$ to this area could serve to simultaneously repress both metR and @Regulated$ expression .\n",
      "16\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MetJ\n",
      "Start y end del regulado: [88, 92]\n",
      "Regulado: metE\n",
      "Thus , binding of MetJ to this area could serve to simultaneously repress both metR and metE expression . Thus , binding of @TF$ to this area could serve to simultaneously repress both @Regulated$ and metE expression .\n",
      "14\n",
      "4\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: MetJ\n",
      "Start y end del regulado: [79, 83]\n",
      "Regulado: metR\n",
      "A common phenotype associated with transcriptional activation of micF by Rob , SoxS and MarA is multiple antibiotic resistance . A common phenotype associated with transcriptional activation of @Regulated$ by Rob , SoxS and @TF$ is multiple antibiotic resistance .\n",
      "8\n",
      "14\n",
      "Start y end del regulador: [88, 92]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: micF\n",
      "A common phenotype associated with transcriptional activation of micF by Rob , SoxS and MarA is multiple antibiotic resistance . A common phenotype associated with transcriptional activation of @Regulated$ by @TF$ , SoxS and MarA is multiple antibiotic resistance .\n",
      "8\n",
      "10\n",
      "Start y end del regulador: [73, 76]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: micF\n",
      "A common phenotype associated with transcriptional activation of micF by Rob , SoxS and MarA is multiple antibiotic resistance . A common phenotype associated with transcriptional activation of @Regulated$ by Rob , @TF$ and MarA is multiple antibiotic resistance .\n",
      "8\n",
      "12\n",
      "Start y end del regulador: [79, 83]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [65, 69]\n",
      "Regulado: micF\n",
      "MelR and CRP interact at the melAB promoter to activate transcription , but this interaction can be exploited to repress transcription ( Figure 3 ) . @TF$ and CRP interact at the @Regulated$ promoter to activate transcription , but this interaction can be exploited to repress transcription ( Figure 3 ) .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [29, 34]\n",
      "Regulado: melAB\n",
      "Deletion of crp ( Table 5 ) resulted in a level of expression from each fusion that was similar to the expression from ( rhaS - lacZ ) 90 in the crp strain background , supporting our hypothesis that full rhaSR activation requires CRP . Deletion of crp ( Table 5 ) resulted in a level of expression from each fusion that was similar to the expression from ( rhaS - lacZ ) 90 in the crp strain background , supporting our hypothesis that full @Regulated$ activation requires @TF$ .\n",
      "40\n",
      "43\n",
      "Start y end del regulador: [231, 234]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [205, 210]\n",
      "Regulado: rhaSR\n",
      "Two approaches were used to confirm that NsrR is this sixth factor that regulates P nrfA . Two approaches were used to confirm that @TF$ is this sixth factor that regulates P @Regulated$ .\n",
      "15\n",
      "7\n",
      "Start y end del regulador: [41, 45]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [84, 88]\n",
      "Regulado: nrfA\n",
      "Therefore , the induction of dcuB depends on functional DcuS / DcuR . Therefore , the induction of @Regulated$ depends on functional DcuS / @TF$ .\n",
      "5\n",
      "11\n",
      "Start y end del regulador: [63, 67]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [29, 33]\n",
      "Regulado: dcuB\n",
      "The physiological significance of the repression of purA and hdeA by MarA is not known . The physiological significance of the repression of purA and @Regulated$ by @TF$ is not known .\n",
      "9\n",
      "11\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [61, 65]\n",
      "Regulado: hdeA\n",
      "The physiological significance of the repression of purA and hdeA by MarA is not known . The physiological significance of the repression of @Regulated$ and hdeA by @TF$ is not known .\n",
      "7\n",
      "11\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: purA\n",
      "These latter results , together with the experiments described here showing that MetJ binds to this region of DNA , strongly suggest that the - 8 to + 27 region is involved in metE repression . These latter results , together with the experiments described here showing that @TF$ binds to this region of DNA , strongly suggest that the - 8 to + 27 region is involved in @Regulated$ repression .\n",
      "33\n",
      "12\n",
      "Start y end del regulador: [81, 85]\n",
      "Regulador: MetJ\n",
      "Start y end del regulado: [176, 180]\n",
      "Regulado: metE\n",
      "We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two CRP sites was required for rhaSR activation . We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two @TF$ sites was required for @Regulated$ activation .\n",
      "23\n",
      "18\n",
      "Start y end del regulador: [110, 113]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [137, 142]\n",
      "Regulado: rhaSR\n",
      "These results indicated that ArgP is responsible for the transcriptional activation of lysP in the absence of lysine . These results indicated that @TF$ is responsible for the transcriptional activation of @Regulated$ in the absence of lysine .\n",
      "12\n",
      "4\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [87, 91]\n",
      "Regulado: lysP\n",
      "101 no . 46 16325 yjhA encodes an outer membrane channel that can facilitate Neu 5 Ac uptake ( G . Condemine , personal communication ) , and this gene is also repressed strongly by NanR , an effect which is modulated by NagC binding to the NagC1 site . 101 no . 46 16325 @Regulated$ encodes an outer membrane channel that can facilitate Neu 5 Ac uptake ( G . Condemine , personal communication ) , and this gene is also repressed strongly by @TF$ , an effect which is modulated by NagC binding to the NagC1 site .\n",
      "5\n",
      "35\n",
      "Start y end del regulador: [182, 186]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: yjhA\n",
      "Similarly , at the melAB promoter , activation by CRP appears to be dependent on direct protein ± protein interactions with MelR . Similarly , at the @Regulated$ promoter , activation by @TF$ appears to be dependent on direct protein ± protein interactions with MelR .\n",
      "4\n",
      "9\n",
      "Start y end del regulador: [50, 53]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [19, 24]\n",
      "Regulado: melAB\n",
      "The repression of purA by MarA was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) . The repression of @Regulated$ by @TF$ was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) .\n",
      "3\n",
      "5\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [18, 22]\n",
      "Regulado: purA\n",
      "One of the ara loops , that involving AraC bound at araO2 and araI , was previously shown to generate repression of araBAD ( 5 - 7 ) . One of the ara loops , that involving @TF$ bound at araO2 and araI , was previously shown to generate repression of @Regulated$ ( 5 - 7 ) .\n",
      "22\n",
      "8\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [116, 122]\n",
      "Regulado: araBAD\n",
      "This study demonstrates that MarA can repress the expression of the rob gene both in vivo and in vitro via a marbox sequence located within its promoter . This study demonstrates that @TF$ can repress the expression of the @Regulated$ gene both in vivo and in vitro via a marbox sequence located within its promoter .\n",
      "11\n",
      "4\n",
      "Start y end del regulador: [29, 33]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [68, 71]\n",
      "Regulado: rob\n",
      "Dominant - negative AraC mutants that fail to activate transcription of araBAD were selected , and their interactions with specific bases of the binding site were probed with chemically modified DNA . Dominant - negative @TF$ mutants that fail to activate transcription of @Regulated$ were selected , and their interactions with specific bases of the binding site were probed with chemically modified DNA .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [72, 78]\n",
      "Regulado: araBAD\n",
      "We suppose that this improvement in MelR binding is responsible for the increased melAB promoter activity in the W3133 - 2 mutant , although we cannot prove that it is solely responsible . We suppose that this improvement in @TF$ binding is responsible for the increased @Regulated$ promoter activity in the W3133 - 2 mutant , although we cannot prove that it is solely responsible .\n",
      "13\n",
      "6\n",
      "Start y end del regulador: [36, 40]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [82, 87]\n",
      "Regulado: melAB\n",
      "It is possible that NarP and Fnr may activate napF transcription synergistically whereas Fnr and NarL cannot , as discussed below . It is possible that @TF$ and Fnr may activate @Regulated$ transcription synergistically whereas Fnr and NarL cannot , as discussed below .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [46, 50]\n",
      "Regulado: napF\n",
      "In conclusion , MelR and CRP bind to the melAB promoter co - operatively to adjacent sites in order to activate transcription in a co - dependent manner . In conclusion , MelR and @TF$ bind to the @Regulated$ promoter co - operatively to adjacent sites in order to activate transcription in a co - dependent manner .\n",
      "9\n",
      "5\n",
      "Start y end del regulador: [25, 28]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [41, 46]\n",
      "Regulado: melAB\n",
      "In conclusion , MelR and CRP bind to the melAB promoter co - operatively to adjacent sites in order to activate transcription in a co - dependent manner . In conclusion , @TF$ and CRP bind to the @Regulated$ promoter co - operatively to adjacent sites in order to activate transcription in a co - dependent manner .\n",
      "9\n",
      "3\n",
      "Start y end del regulador: [16, 20]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [41, 46]\n",
      "Regulado: melAB\n",
      "The mefA operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four MetJ dimers , compared with two in the minimal complexes formed on the synthetic sites . The @Regulated$ operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four @TF$ dimers , compared with two in the minimal complexes formed on the synthetic sites .\n",
      "1\n",
      "24\n",
      "Start y end del regulador: [142, 146]\n",
      "Regulador: MetJ\n",
      "Start y end del regulado: [4, 8]\n",
      "Regulado: mefA\n",
      "In a nagC strain , the threefold increase in chbB – lacZ expression compared with the wild type is , as expected , eliminated by the NagC plasmid . In a nagC strain , the threefold increase in @Regulated$ – lacZ expression compared with the wild type is , as expected , eliminated by the @TF$ plasmid .\n",
      "9\n",
      "26\n",
      "Start y end del regulador: [133, 137]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: chbB\n",
      "While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor NagC requires the wild - type segment of D7 DNA in order to be able to activate fimB expression . While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor @TF$ requires the wild - type segment of D7 DNA in order to be able to activate @Regulated$ expression .\n",
      "34\n",
      "17\n",
      "Start y end del regulador: [96, 100]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [176, 180]\n",
      "Regulado: fimB\n",
      "While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor NagC requires the wild - type segment of D7 DNA in order to be able to activate fimB expression . While the basis for this effect is uncertain , it is nevertheless evident that neither @TF$ nor NagC requires the wild - type segment of D7 DNA in order to be able to activate @Regulated$ expression .\n",
      "34\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [176, 180]\n",
      "Regulado: fimB\n",
      "ß - Galactosidase activity Promoter Mutated site minus NO 3 plus NO 3 Fold induction ogt100 93 + 1 439 + 12 4.7 ogt102 NarL I 92 + 5 107 + 6 1.2 ogt104 NarL II 56 + 2 87 + 3 1.6 the yeaR promoter is repressed by the binding of NsrR to a target that overlaps the 10 hexamer element ( Figure 1A ) [ 12,31,32 ] . ß - Galactosidase activity Promoter Mutated site minus NO 3 plus NO 3 Fold induction ogt100 93 + 1 439 + 12 4.7 ogt102 NarL I 92 + 5 107 + 6 1.2 ogt104 NarL II 56 + 2 87 + 3 1.6 the @Regulated$ promoter is repressed by the binding of @TF$ to a target that overlaps the 10 hexamer element ( Figure 1A ) [ 12,31,32 ] .\n",
      "44\n",
      "52\n",
      "Start y end del regulador: [227, 231]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [182, 186]\n",
      "Regulado: yeaR\n",
      "The extent of AraC stimulation of araBAD is at least 25 - fold larger . The extent of @TF$ stimulation of @Regulated$ is at least 25 - fold larger .\n",
      "6\n",
      "3\n",
      "Start y end del regulador: [14, 18]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [34, 40]\n",
      "Regulado: araBAD\n",
      "In vitro activation of the transcription of araBAD operon by AraC activator . In vitro activation of the transcription of @Regulated$ operon by @TF$ activator .\n",
      "7\n",
      "10\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [44, 50]\n",
      "Regulado: araBAD\n",
      "A 3.4 - fold decrease of citC - lacZ induction was observed when DcuR was inactivated ( IMW554 ) . A 3.4 - fold decrease of @Regulated$ - lacZ induction was observed when @TF$ was inactivated ( IMW554 ) .\n",
      "6\n",
      "13\n",
      "Start y end del regulador: [65, 69]\n",
      "Regulador: DcuR\n",
      "Start y end del regulado: [25, 29]\n",
      "Regulado: citC\n",
      "To test this hypothesis further , and to examine the effect of operator - to - promoter distance on NagC activation , the effect of deleting the 274 bp region ( D7 ) described above on fimB expression was also analysed ( Fig . 8 ) . To test this hypothesis further , and to examine the effect of operator - to - promoter distance on @TF$ activation , the effect of deleting the 274 bp region ( D7 ) described above on @Regulated$ expression was also analysed ( Fig . 8 ) .\n",
      "36\n",
      "19\n",
      "Start y end del regulador: [100, 104]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [185, 189]\n",
      "Regulado: fimB\n",
      "So there is no strong repression of chbB – lacZ by plasmid - encoded NagC . So there is no strong repression of @Regulated$ – lacZ by plasmid - encoded @TF$ .\n",
      "7\n",
      "14\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [36, 40]\n",
      "Regulado: chbB\n",
      "The results of this analysis confirm this assertion , and show additionally that NagC is also better able to We previously presented evidence suggesting that the D2 region , and hence NanR and NagC , could activate fimB expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) . The results of this analysis confirm this assertion , and show additionally that NagC is also better able to We previously presented evidence suggesting that the D2 region , and hence @TF$ and NagC , could activate @Regulated$ expression by blocking the inhibitory effect of additional cis - acting element ( s ) positioned closer to nanC ( El - Labany et al . , 2003 ) .\n",
      "37\n",
      "31\n",
      "Start y end del regulador: [184, 188]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [215, 219]\n",
      "Regulado: fimB\n",
      "Results illustrated in Figure 5B show that MelR - dependent repression of the melR promoter is unaffected when CRP binding at this target is prevented . Results illustrated in Figure 5B show that @TF$ - dependent repression of the @Regulated$ promoter is unaffected when CRP binding at this target is prevented .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: melR\n",
      "Thus , maximum activation of the mar regulon promoter inaA depends on the autostimulatory effect of MarA on the mar operon . Thus , maximum activation of the mar regulon promoter @Regulated$ depends on the autostimulatory effect of @TF$ on the mar operon .\n",
      "9\n",
      "16\n",
      "Start y end del regulador: [100, 104]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [54, 58]\n",
      "Regulado: inaA\n",
      "The AraC protein activates and represses the araBAD genes , depending upon the presence of the inducer and cAMP / CAP which change the occupancy of the different sites ( Reeder and Schleif , 1993 and references therein ) . The @TF$ protein activates and represses the @Regulated$ genes , depending upon the presence of the inducer and cAMP / CAP which change the occupancy of the different sites ( Reeder and Schleif , 1993 and references therein ) .\n",
      "7\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [45, 51]\n",
      "Regulado: araBAD\n",
      "Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the galP gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and GalS and including at least 280 bp of DNA . Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the @Regulated$ gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as @TF$ and GalS and including at least 280 bp of DNA .\n",
      "60\n",
      "76\n",
      "Start y end del regulador: [428, 432]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [330, 334]\n",
      "Regulado: galP\n",
      "Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the galP gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and GalS and including at least 280 bp of DNA . Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the @Regulated$ gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and @TF$ and including at least 280 bp of DNA .\n",
      "60\n",
      "78\n",
      "Start y end del regulador: [437, 441]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [330, 334]\n",
      "Regulado: galP\n",
      "Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the galP gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and GalS and including at least 280 bp of DNA . Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the @Regulated$ gene , is mediated by a distinctly different nucleoprotein complex involving @TF$ as well as GalR and GalS and including at least 280 bp of DNA .\n",
      "60\n",
      "72\n",
      "Start y end del regulador: [412, 416]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [330, 334]\n",
      "Regulado: galP\n",
      "During the course of this work , it became apparent that GntR might affect gntT expression by a mechanism that is independent of negative control by operator binding as well as activation by cAMP - CRP . During the course of this work , it became apparent that @TF$ might affect @Regulated$ expression by a mechanism that is independent of negative control by operator binding as well as activation by cAMP - CRP .\n",
      "14\n",
      "11\n",
      "Start y end del regulador: [57, 61]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: gntT\n",
      "This change confers partial CRP independence on the melR promoter ( data not shown ) and also causes a small reduction in MelR - dependent repression . This change confers partial CRP independence on the @Regulated$ promoter ( data not shown ) and also causes a small reduction in @TF$ - dependent repression .\n",
      "8\n",
      "22\n",
      "Start y end del regulador: [122, 126]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: melR\n",
      "Table 1 shows that two amino acid residues of 70 R4 are required for Rob - dependent activation of fumC and that only one is required for activation of micF . Table 1 shows that two amino acid residues of 70 R4 are required for @TF$ - dependent activation of @Regulated$ and that only one is required for activation of micF .\n",
      "19\n",
      "14\n",
      "Start y end del regulador: [69, 72]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [99, 103]\n",
      "Regulado: fumC\n",
      "Table 1 shows that two amino acid residues of 70 R4 are required for Rob - dependent activation of fumC and that only one is required for activation of micF . Table 1 shows that two amino acid residues of 70 R4 are required for @TF$ - dependent activation of fumC and that only one is required for activation of @Regulated$ .\n",
      "29\n",
      "14\n",
      "Start y end del regulador: [69, 72]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [152, 156]\n",
      "Regulado: micF\n",
      "The data from independent duplicate cultures were completely consistent with NsrR repression of P nrfA [ 1,200 and 1,270 nmol of o - nitrophenyl -- D - galactopyranoside hydrolyzed min 1 ( mg bacterial dry mass ) 1 for the pGIT1 transformant compared with 180 and 230 units for the pGIT8 transformant ] . The data from independent duplicate cultures were completely consistent with @TF$ repression of P @Regulated$ [ 1,200 and 1,270 nmol of o - nitrophenyl -- D - galactopyranoside hydrolyzed min 1 ( mg bacterial dry mass ) 1 for the pGIT1 transformant compared with 180 and 230 units for the pGIT8 transformant ] .\n",
      "14\n",
      "10\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: NsrR\n",
      "Start y end del regulado: [98, 102]\n",
      "Regulado: nrfA\n",
      "Results illustrated in Figure 5B show that the MelR - dependent repression of the melR promoter in the TB210 fragment is unchanged in the TB211 and TB222 derivatives . Results illustrated in Figure 5B show that the @TF$ - dependent repression of the @Regulated$ promoter in the TB210 fragment is unchanged in the TB211 and TB222 derivatives .\n",
      "14\n",
      "8\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [82, 86]\n",
      "Regulado: melR\n",
      "Other unknowns are how OmpR activates micF transcription while binding at far upstream sequences , i . e . in the - 156 to - 216 region ( Figure 4 ( a ) ) . Other unknowns are how @TF$ activates @Regulated$ transcription while binding at far upstream sequences , i . e . in the - 156 to - 216 region ( Figure 4 ( a ) ) .\n",
      "6\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [38, 42]\n",
      "Regulado: micF\n",
      "Results are the mean of two independent cultures . © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 443 represses weakly by itself and strongly in the presence of nagC + , implying that part of the repression by ChbR involves enhancing the binding of NagC to its promoter proximal site . Results are the mean of two independent cultures . © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the @Regulated$ promoter 443 represses weakly by itself and strongly in the presence of nagC + , implying that part of the repression by ChbR involves enhancing the binding of @TF$ to its promoter proximal site .\n",
      "25\n",
      "54\n",
      "Start y end del regulador: [307, 311]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [142, 146]\n",
      "Regulado: chbB\n",
      "Discussion We showed in earlier work that fimB expression , and hence the OFF - to - ON phase switching of type 1 fimbriation , is activated by NanR and NagC and hence repressed Fig . 9 . Discussion We showed in earlier work that @Regulated$ expression , and hence the OFF - to - ON phase switching of type 1 fimbriation , is activated by NanR and @TF$ and hence repressed Fig . 9 .\n",
      "7\n",
      "30\n",
      "Start y end del regulador: [153, 157]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: fimB\n",
      "Discussion We showed in earlier work that fimB expression , and hence the OFF - to - ON phase switching of type 1 fimbriation , is activated by NanR and NagC and hence repressed Fig . 9 . Discussion We showed in earlier work that @Regulated$ expression , and hence the OFF - to - ON phase switching of type 1 fimbriation , is activated by @TF$ and NagC and hence repressed Fig . 9 .\n",
      "7\n",
      "28\n",
      "Start y end del regulador: [144, 148]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [42, 46]\n",
      "Regulado: fimB\n",
      "However , the napF control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by NarP being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) . However , the @Regulated$ control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by @TF$ being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) .\n",
      "3\n",
      "25\n",
      "Start y end del regulador: [141, 145]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [14, 18]\n",
      "Regulado: napF\n",
      "However , in vitro experiments show IHF to repress ompC transcription from two of its three promoters . However , in vitro experiments show @TF$ to repress @Regulated$ transcription from two of its three promoters .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [36, 39]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: ompC\n",
      "In the previous regulatory model , the expression of waaP and waaY was regulated by RfaH ( SfrB ) , an antiterminator protein , and this regulation was achieved over the whole transcript from the top promoter in front of waaQ ( 29 ) . In the previous regulatory model , the expression of @Regulated$ and waaY was regulated by @TF$ ( SfrB ) , an antiterminator protein , and this regulation was achieved over the whole transcript from the top promoter in front of waaQ ( 29 ) .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [84, 88]\n",
      "Regulador: RfaH\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: waaP\n",
      "In the previous regulatory model , the expression of waaP and waaY was regulated by RfaH ( SfrB ) , an antiterminator protein , and this regulation was achieved over the whole transcript from the top promoter in front of waaQ ( 29 ) . In the previous regulatory model , the expression of waaP and @Regulated$ was regulated by @TF$ ( SfrB ) , an antiterminator protein , and this regulation was achieved over the whole transcript from the top promoter in front of waaQ ( 29 ) .\n",
      "11\n",
      "15\n",
      "Start y end del regulador: [84, 88]\n",
      "Regulador: RfaH\n",
      "Start y end del regulado: [62, 66]\n",
      "Regulado: waaY\n",
      "Strikingly , all induction is lost with the JK14 fragment , showing that MelR binding to Site 1 and Site 1 0 is essential for activation of the melAB promoter . Strikingly , all induction is lost with the JK14 fragment , showing that @TF$ binding to Site 1 and Site 1 0 is essential for activation of the @Regulated$ promoter .\n",
      "28\n",
      "13\n",
      "Start y end del regulador: [73, 77]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [144, 149]\n",
      "Regulado: melAB\n",
      "NanR and NagC activate fimB expression by binding to operator sequences ( O NR and O NC1 respectively ; Fig . 1 ) situated further upstream of the regulated promoter than is usual in bacteria . NanR and @TF$ activate @Regulated$ expression by binding to operator sequences ( O NR and O NC1 respectively ; Fig . 1 ) situated further upstream of the regulated promoter than is usual in bacteria .\n",
      "4\n",
      "2\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: fimB\n",
      "NanR and NagC activate fimB expression by binding to operator sequences ( O NR and O NC1 respectively ; Fig . 1 ) situated further upstream of the regulated promoter than is usual in bacteria . @TF$ and NagC activate @Regulated$ expression by binding to operator sequences ( O NR and O NC1 respectively ; Fig . 1 ) situated further upstream of the regulated promoter than is usual in bacteria .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: fimB\n",
      "References Belyaeva , T . , Wade , J . , Webster , C . , Howard , V . , Thomas , M . , Hyde , E . , et al . ( 2000 ) Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . References Belyaeva , T . , Wade , J . , Webster , C . , Howard , V . , Thomas , M . , Hyde , E . , et al . ( 2000 ) Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "43\n",
      "49\n",
      "Start y end del regulador: [195, 199]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [166, 171]\n",
      "Regulado: melAB\n",
      "Furthermore , as NagC binding to O NC1 prevents CRP - dependent activation of nanC ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well . Furthermore , as NagC binding to O NC1 prevents @TF$ - dependent activation of @Regulated$ ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .\n",
      "14\n",
      "9\n",
      "Start y end del regulador: [48, 51]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: nanC\n",
      "In the second part of the present study , we investigated the E . coli ogt promoter , which Constantinidou et al . [ 11 ] had also found to be induced by NarL independently of FNR . In the second part of the present study , we investigated the E . coli @Regulated$ promoter , which Constantinidou et al . [ 11 ] had also found to be induced by @TF$ independently of FNR .\n",
      "15\n",
      "33\n",
      "Start y end del regulador: [154, 158]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [71, 74]\n",
      "Regulado: ogt\n",
      "Therefore , the 22 heptamer site is a critical site for mediating NarL repression of nrfA gene expression . Therefore , the 22 heptamer site is a critical site for mediating @TF$ repression of @Regulated$ gene expression .\n",
      "15\n",
      "12\n",
      "Start y end del regulador: [66, 70]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [85, 89]\n",
      "Regulado: nrfA\n",
      "Considered together , these data show that IHF promotes an interaction between NagC bound to O NC1 and O NC2 , forming a stable nucleoprotein complex that activates fimB expression . Considered together , these data show that IHF promotes an interaction between @TF$ bound to O NC1 and O NC2 , forming a stable nucleoprotein complex that activates @Regulated$ expression .\n",
      "28\n",
      "12\n",
      "Start y end del regulador: [79, 83]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [165, 169]\n",
      "Regulado: fimB\n",
      "Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) . Previously , overexpression of @TF$ was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of @Regulated$ , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .\n",
      "23\n",
      "4\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [146, 150]\n",
      "Regulado: fumC\n",
      "Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) . Previously , overexpression of @TF$ was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , @Regulated$ , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .\n",
      "25\n",
      "4\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [153, 157]\n",
      "Regulado: inaA\n",
      "Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) . Previously , overexpression of @TF$ was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and @Regulated$ in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .\n",
      "28\n",
      "4\n",
      "Start y end del regulador: [31, 34]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [164, 168]\n",
      "Regulado: sodA\n",
      "The model proposed by Celis ( 11 , 12 ) to explain the Can r phenotype of an argP d mutant assumed ( i ) that the mutation represents a loss - of - function ( that is , dominant - negative ) allele of argP and ( ii ) that wild - type ArgP is a transcriptional activator of argK , whose product in turn acts to enhance Arg uptake through two different transport systems . The model proposed by Celis ( 11 , 12 ) to explain the Can r phenotype of an argP d mutant assumed ( i ) that the mutation represents a loss - of - function ( that is , dominant - negative ) allele of argP and ( ii ) that wild - type @TF$ is a transcriptional activator of @Regulated$ , whose product in turn acts to enhance Arg uptake through two different transport systems .\n",
      "60\n",
      "54\n",
      "Start y end del regulador: [234, 238]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [273, 277]\n",
      "Regulado: argK\n",
      "As already mentioned , a potential ArgP - binding site , ATG AAGGTGTCTTAT , is centered at position 59 in the lysP promoter / control region ( Fig . 2 ) . As already mentioned , a potential @TF$ - binding site , ATG AAGGTGTCTTAT , is centered at position 59 in the @Regulated$ promoter / control region ( Fig . 2 ) .\n",
      "21\n",
      "6\n",
      "Start y end del regulador: [35, 39]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [110, 114]\n",
      "Regulado: lysP\n",
      "Interestingly , these same three CRP mutants had similar small activation defects at the divergent rhaBAD promoter ( 14 ) . Interestingly , these same three @TF$ mutants had similar small activation defects at the divergent @Regulated$ promoter ( 14 ) .\n",
      "15\n",
      "5\n",
      "Start y end del regulador: [33, 36]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [99, 105]\n",
      "Regulado: rhaBAD\n",
      "Activation of chbBCA expression by ChbR is essential to relieve ChiX repression of chiP . Activation of chbBCA expression by ChbR is essential to relieve @TF$ repression of @Regulated$ .\n",
      "13\n",
      "10\n",
      "Start y end del regulador: [64, 68]\n",
      "Regulador: ChiX\n",
      "Start y end del regulado: [83, 87]\n",
      "Regulado: chiP\n",
      "MelR - dependent repression of the melR promoter in each of the new constructs was measured as above and the results are illustrated in Figure 6B . @TF$ - dependent repression of the @Regulated$ promoter in each of the new constructs was measured as above and the results are illustrated in Figure 6B .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [35, 39]\n",
      "Regulado: melR\n",
      "Thus , not only does IHF stimulate both FimB and FimE recombination , as well as activate fimbrial expression in phase ON cells , but it is now clear that it enhances fimB expression too . Thus , not only does @TF$ stimulate both FimB and FimE recombination , as well as activate fimbrial expression in phase ON cells , but it is now clear that it enhances @Regulated$ expression too .\n",
      "32\n",
      "5\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [167, 171]\n",
      "Regulado: fimB\n",
      "Indeed , the GalS isorepressor was discovered because , in the absence of GalR , there was some residual repression of the galE gene . Indeed , the @TF$ isorepressor was discovered because , in the absence of GalR , there was some residual repression of the @Regulated$ gene .\n",
      "22\n",
      "3\n",
      "Start y end del regulador: [13, 17]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [123, 127]\n",
      "Regulado: galE\n",
      "Coeffectors of ArgP in yggA regulation . Coeffectors of @TF$ in @Regulated$ regulation .\n",
      "4\n",
      "2\n",
      "Start y end del regulador: [15, 19]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [23, 27]\n",
      "Regulado: yggA\n",
      "DISCUSSION The napF operon exhibits a unique pattern of expression and a unique control region architecture in comparison to other characterized E . coli Fnr - and Nar - regulated operons . DISCUSSION The @Regulated$ operon exhibits a unique pattern of expression and a unique control region architecture in comparison to other characterized E . coli @TF$ - and Nar - regulated operons .\n",
      "2\n",
      "24\n",
      "Start y end del regulador: [154, 157]\n",
      "Regulador: Fnr\n",
      "Start y end del regulado: [15, 19]\n",
      "Regulado: napF\n",
      "In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and ybjC were recently shown to be activated by constitutive expression of MarA ( 5 ) . In addition , the PQ - or SoxSresponsive genes gatABD , @Regulated$ , nfnB , and ybjC were recently shown to be activated by constitutive expression of @TF$ ( 5 ) .\n",
      "11\n",
      "27\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [56, 60]\n",
      "Regulado: gltA\n",
      "In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and ybjC were recently shown to be activated by constitutive expression of MarA ( 5 ) . In addition , the PQ - or SoxSresponsive genes gatABD , gltA , @Regulated$ , and ybjC were recently shown to be activated by constitutive expression of @TF$ ( 5 ) .\n",
      "13\n",
      "27\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [63, 67]\n",
      "Regulado: nfnB\n",
      "In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and ybjC were recently shown to be activated by constitutive expression of MarA ( 5 ) . In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and @Regulated$ were recently shown to be activated by constitutive expression of @TF$ ( 5 ) .\n",
      "16\n",
      "27\n",
      "Start y end del regulador: [145, 149]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [74, 78]\n",
      "Regulado: ybjC\n",
      "To study the ArgP - binding site in the lysP promoter / control region in more detail , DNase I footprinting analysis was performed in the absence and presence of lysine ( Fig . 5 ) . To study the @TF$ - binding site in the @Regulated$ promoter / control region in more detail , DNase I footprinting analysis was performed in the absence and presence of lysine ( Fig . 5 ) .\n",
      "9\n",
      "3\n",
      "Start y end del regulador: [13, 17]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: lysP\n",
      "In our previous study ( 17 ) , we reported that repression of the melR promoter by MelR was unaffected by point mutations in either site 1 or site 1 0 . In our previous study ( 17 ) , we reported that repression of the @Regulated$ promoter by @TF$ was unaffected by point mutations in either site 1 or site 1 0 .\n",
      "14\n",
      "17\n",
      "Start y end del regulador: [83, 87]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [66, 70]\n",
      "Regulado: melR\n",
      "The ybjC gene was also activated by expression of SoxS in our experiments ( Table 1 ) . The @Regulated$ gene was also activated by expression of @TF$ in our experiments ( Table 1 ) .\n",
      "1\n",
      "9\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [4, 8]\n",
      "Regulado: ybjC\n",
      "Given that the activation by full - length , chromosomally expressed RhaS at rhaBAD is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold . Given that the activation by full - length , chromosomally expressed @TF$ at @Regulated$ is approximately 33 - fold higher than that of chromosomally expressed RhaR at rhaSR , comparable efficiencies of activation by the CTDs to their full - length counterparts would have resulted in His 6 - RhaR - CTD activating rhaSR by approximately 30 - fold .\n",
      "13\n",
      "11\n",
      "Start y end del regulador: [69, 73]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [77, 83]\n",
      "Regulado: rhaBAD\n",
      "The ArgP - mediated induction of yggA expression is also observed upon the addition of the Arg biosynthetic precursor citrulline ( Fig . 3 ) or ornithine ( data not shown ) , most likely by virtue of its conversion to Arg within the cells . The @TF$ - mediated induction of @Regulated$ expression is also observed upon the addition of the Arg biosynthetic precursor citrulline ( Fig . 3 ) or ornithine ( data not shown ) , most likely by virtue of its conversion to Arg within the cells .\n",
      "6\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [33, 37]\n",
      "Regulado: yggA\n",
      "This suggests that at the complex fucPIK promoter , any interactions between CRP and the - CTD do not contribute significantly to promoter activation . This suggests that at the complex @Regulated$ promoter , any interactions between @TF$ and the - CTD do not contribute significantly to promoter activation .\n",
      "6\n",
      "12\n",
      "Start y end del regulador: [77, 80]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [34, 40]\n",
      "Regulado: fucPIK\n",
      "Results illustrated in Figure 6B show that MelR - dependent repression of the melR promoter in the TB28 fragment is reduced . Results illustrated in Figure 6B show that @TF$ - dependent repression of the @Regulated$ promoter in the TB28 fragment is reduced .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: melR\n",
      "The treB treC operon is negatively regulated by TreR , whose gene treR is located upstream of treB but is not part of the operon . The treB @Regulated$ operon is negatively regulated by @TF$ , whose gene treR is located upstream of treB but is not part of the operon .\n",
      "2\n",
      "8\n",
      "Start y end del regulador: [48, 52]\n",
      "Regulador: TreR\n",
      "Start y end del regulado: [9, 13]\n",
      "Regulado: treC\n",
      "Transient repressor effect of Fis on the growth phase - regulated osmE promoter of Escherichia coli K12 . Transient repressor effect of @TF$ on the growth phase - regulated @Regulated$ promoter of Escherichia coli K12 .\n",
      "11\n",
      "4\n",
      "Start y end del regulador: [30, 33]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [66, 70]\n",
      "Regulado: osmE\n",
      "In contrast , in all three known examples of MarA - repressed genes ( hdeA , purA , and rob ) , the marbox is in the backward orientation and is proximal to or overlapping the RNAP - binding site ( see Fig . 8 ) . In contrast , in all three known examples of @TF$ - repressed genes ( @Regulated$ , purA , and rob ) , the marbox is in the backward orientation and is proximal to or overlapping the RNAP - binding site ( see Fig . 8 ) .\n",
      "14\n",
      "9\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: hdeA\n",
      "In contrast , in all three known examples of MarA - repressed genes ( hdeA , purA , and rob ) , the marbox is in the backward orientation and is proximal to or overlapping the RNAP - binding site ( see Fig . 8 ) . In contrast , in all three known examples of @TF$ - repressed genes ( hdeA , @Regulated$ , and rob ) , the marbox is in the backward orientation and is proximal to or overlapping the RNAP - binding site ( see Fig . 8 ) .\n",
      "16\n",
      "9\n",
      "Start y end del regulador: [45, 49]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: purA\n",
      "The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the NarL protein ( e . g . , the narG , fdnG , nrfA , and nirB promoters ) . The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the @TF$ protein ( e . g . , the narG , @Regulated$ , nrfA , and nirB promoters ) .\n",
      "36\n",
      "25\n",
      "Start y end del regulador: [138, 142]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [174, 178]\n",
      "Regulado: fdnG\n",
      "Only one NagC operator has been identified as important for repression at galP . Only one @TF$ operator has been identified as important for repression at @Regulated$ .\n",
      "12\n",
      "2\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [74, 78]\n",
      "Regulado: galP\n",
      "The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the NarL protein ( e . g . , the narG , fdnG , nrfA , and nirB promoters ) . The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the @TF$ protein ( e . g . , the @Regulated$ , fdnG , nrfA , and nirB promoters ) .\n",
      "34\n",
      "25\n",
      "Start y end del regulador: [138, 142]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [167, 171]\n",
      "Regulado: narG\n",
      "The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the NarL protein ( e . g . , the narG , fdnG , nrfA , and nirB promoters ) . The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the @TF$ protein ( e . g . , the narG , fdnG , @Regulated$ , and nirB promoters ) .\n",
      "38\n",
      "25\n",
      "Start y end del regulador: [138, 142]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: nrfA\n",
      "The napF operon provides the only known example of a promoter activated by NarP but not NarL . The @Regulated$ operon provides the only known example of a promoter activated by @TF$ but not NarL .\n",
      "1\n",
      "13\n",
      "Start y end del regulador: [75, 79]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [4, 8]\n",
      "Regulado: napF\n",
      "Although there is no simple rationale for why the yeaR and ogt promoters are also repressed by Fis , we suggest that rapidly growing cells may opt out of certain stress responses , and we speculate that RNS may be a small risk in these conditions compared with other stresses . Although there is no simple rationale for why the yeaR and @Regulated$ promoters are also repressed by @TF$ , we suggest that rapidly growing cells may opt out of certain stress responses , and we speculate that RNS may be a small risk in these conditions compared with other stresses .\n",
      "11\n",
      "17\n",
      "Start y end del regulador: [95, 98]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [59, 62]\n",
      "Regulado: ogt\n",
      "Although there is no simple rationale for why the yeaR and ogt promoters are also repressed by Fis , we suggest that rapidly growing cells may opt out of certain stress responses , and we speculate that RNS may be a small risk in these conditions compared with other stresses . Although there is no simple rationale for why the @Regulated$ and ogt promoters are also repressed by @TF$ , we suggest that rapidly growing cells may opt out of certain stress responses , and we speculate that RNS may be a small risk in these conditions compared with other stresses .\n",
      "9\n",
      "17\n",
      "Start y end del regulador: [95, 98]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: yeaR\n",
      "The artPIQM operon is , besides being downregulated by Lrp ( Hung et al . , 2002 ) , also repressed by ArgR ( Caldara et al . , 2007 ) . The @Regulated$ operon is , besides being downregulated by Lrp ( Hung et al . , 2002 ) , also repressed by @TF$ ( Caldara et al . , 2007 ) .\n",
      "1\n",
      "22\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: ArgR\n",
      "Start y end del regulado: [4, 11]\n",
      "Regulado: artPIQM\n",
      "The artPIQM operon is , besides being downregulated by Lrp ( Hung et al . , 2002 ) , also repressed by ArgR ( Caldara et al . , 2007 ) . The @Regulated$ operon is , besides being downregulated by @TF$ ( Hung et al . , 2002 ) , also repressed by ArgR ( Caldara et al . , 2007 ) .\n",
      "1\n",
      "9\n",
      "Start y end del regulador: [55, 58]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [4, 11]\n",
      "Regulado: artPIQM\n",
      "Model of AraC induction by L - arabinose at the araBAD promoter and relevant sites at the divergent araC promoter . Model of @TF$ induction by L - arabinose at the @Regulated$ promoter and relevant sites at the divergent araC promoter .\n",
      "10\n",
      "2\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [48, 54]\n",
      "Regulado: araBAD\n",
      "One obvious difference between the synthetic promoter and the napF operon promoter is that transcription from the latter is also activated by the NarP protein bound to a site centered at position 44.5 . One obvious difference between the synthetic promoter and the @Regulated$ operon promoter is that transcription from the latter is also activated by the @TF$ protein bound to a site centered at position 44.5 .\n",
      "9\n",
      "23\n",
      "Start y end del regulador: [146, 150]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [62, 66]\n",
      "Regulado: napF\n",
      "Thus , activation of fimB expression by NanR , as judged by the effect of mutations in nanR , orin its operator sequence ( O NR ) , is enhanced considerably in the D3 deletion mutant . Thus , activation of @Regulated$ expression by @TF$ , as judged by the effect of mutations in nanR , orin its operator sequence ( O NR ) , is enhanced considerably in the D3 deletion mutant .\n",
      "4\n",
      "7\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [21, 25]\n",
      "Regulado: fimB\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , @Regulated$ , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "6\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [34, 40]\n",
      "Regulado: marRAB\n",
      "acrZ is regulated by MarA , Rob , and SoxS . @Regulated$ is regulated by @TF$ , Rob , and SoxS .\n",
      "0\n",
      "4\n",
      "Start y end del regulador: [21, 25]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: acrZ\n",
      "acrZ is regulated by MarA , Rob , and SoxS . @Regulated$ is regulated by MarA , @TF$ , and SoxS .\n",
      "0\n",
      "6\n",
      "Start y end del regulador: [28, 31]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: acrZ\n",
      "acrZ is regulated by MarA , Rob , and SoxS . @Regulated$ is regulated by MarA , Rob , and @TF$ .\n",
      "0\n",
      "9\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [0, 4]\n",
      "Regulado: acrZ\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , @Regulated$ , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "8\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: aslB\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while @Regulated$ expression was repressed .\n",
      "28\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [151, 155]\n",
      "Regulado: galT\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that @Regulated$ , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "4\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [27, 31]\n",
      "Regulado: inaA\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , aslB , ybaO , @Regulated$ , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "12\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [57, 61]\n",
      "Regulado: mdlA\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , aslB , @Regulated$ , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "10\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: ybaO\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and @Regulated$ were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "17\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [75, 79]\n",
      "Regulado: ybiS\n",
      "Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed . Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "14\n",
      "25\n",
      "Start y end del regulador: [139, 142]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [64, 68]\n",
      "Regulado: yfhD\n",
      "Rob contributes significantly to the expression of the marRAB operon , which regulates the intrinsic resistance of E . coli to various antibiotics , bactericidal agents , and organic solvents ( 5 , 10 , 13 ) . @TF$ contributes significantly to the expression of the @Regulated$ operon , which regulates the intrinsic resistance of E . coli to various antibiotics , bactericidal agents , and organic solvents ( 5 , 10 , 13 ) .\n",
      "8\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [55, 61]\n",
      "Regulado: marRAB\n",
      "AraC - dependent transcription regulation has been most studied at the araBAD and araC genes , which are expressed from divergent promoters , paraBAD and paraC , whose transcription start sites are separated by 166 bp . @TF$ - dependent transcription regulation has been most studied at the araBAD and @Regulated$ genes , which are expressed from divergent promoters , paraBAD and paraC , whose transcription start sites are separated by 166 bp .\n",
      "13\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [82, 86]\n",
      "Regulado: araC\n",
      "By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate fimB expression . By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither NanR nor @TF$ requires this segment of DNA in order to activate @Regulated$ expression .\n",
      "36\n",
      "26\n",
      "Start y end del regulador: [126, 130]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: fimB\n",
      "By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate fimB expression . By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither @TF$ nor NagC requires this segment of DNA in order to activate @Regulated$ expression .\n",
      "36\n",
      "24\n",
      "Start y end del regulador: [117, 121]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [181, 185]\n",
      "Regulado: fimB\n",
      "Conclusions In this paper , we have reported an in vitro system for the study of transcription activation of the E . coli melAB promoter by MelR . Conclusions In this paper , we have reported an in vitro system for the study of transcription activation of the E . coli @Regulated$ promoter by @TF$ .\n",
      "23\n",
      "26\n",
      "Start y end del regulador: [140, 144]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [122, 127]\n",
      "Regulado: melAB\n",
      "For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the fdnGHI operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) . For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the @Regulated$ operon is controlled principally by NarL , but @TF$ also effects a degree of control ( observed with a narL mutant ) ( 21 ) .\n",
      "29\n",
      "38\n",
      "Start y end del regulador: [199, 203]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [145, 151]\n",
      "Regulado: fdnGHI\n",
      "The repression of the manXYZ operon by a plasmid expressing NagC is in agreement with the original observations of Vogler and Lengeler ( 1989 ) . The repression of the @Regulated$ operon by a plasmid expressing @TF$ is in agreement with the original observations of Vogler and Lengeler ( 1989 ) .\n",
      "4\n",
      "10\n",
      "Start y end del regulador: [60, 64]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [22, 28]\n",
      "Regulado: manXYZ\n",
      "Among functions that could contribute to cellular resistance to environmental agents , Rob enhances expression of mdlA , which encodes a multiple - drug - resistance - like ATP - binding component of a transport system ( 3 ) , and strongly enhances expression of micF , the antisense RNA that downregulates the outer membrane porin OmpF . Among functions that could contribute to cellular resistance to environmental agents , @TF$ enhances expression of @Regulated$ , which encodes a multiple - drug - resistance - like ATP - binding component of a transport system ( 3 ) , and strongly enhances expression of micF , the antisense RNA that downregulates the outer membrane porin OmpF .\n",
      "16\n",
      "12\n",
      "Start y end del regulador: [87, 90]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [114, 118]\n",
      "Regulado: mdlA\n",
      "Among functions that could contribute to cellular resistance to environmental agents , Rob enhances expression of mdlA , which encodes a multiple - drug - resistance - like ATP - binding component of a transport system ( 3 ) , and strongly enhances expression of micF , the antisense RNA that downregulates the outer membrane porin OmpF . Among functions that could contribute to cellular resistance to environmental agents , @TF$ enhances expression of mdlA , which encodes a multiple - drug - resistance - like ATP - binding component of a transport system ( 3 ) , and strongly enhances expression of @Regulated$ , the antisense RNA that downregulates the outer membrane porin OmpF .\n",
      "45\n",
      "12\n",
      "Start y end del regulador: [87, 90]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [263, 267]\n",
      "Regulado: micF\n",
      "Indeed , the effect of the mutation in OE5 was similar to © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and GalS repression at galP 153 that in the weakly binding OE2 and OE3 sites ( a mutation in OE4 has not yet been tested ) . Indeed , the effect of the mutation in OE5 was similar to © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , @TF$ and GalS repression at @Regulated$ 153 that in the weakly binding OE2 and OE3 sites ( a mutation in OE4 has not yet been tested ) .\n",
      "38\n",
      "33\n",
      "Start y end del regulador: [177, 181]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: galP\n",
      "Indeed , the effect of the mutation in OE5 was similar to © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and GalS repression at galP 153 that in the weakly binding OE2 and OE3 sites ( a mutation in OE4 has not yet been tested ) . Indeed , the effect of the mutation in OE5 was similar to © 2008 The Authors Journal compilation © 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 – 157NagC , GalR and @TF$ repression at @Regulated$ 153 that in the weakly binding OE2 and OE3 sites ( a mutation in OE4 has not yet been tested ) .\n",
      "38\n",
      "35\n",
      "Start y end del regulador: [186, 190]\n",
      "Regulador: GalS\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: galP\n",
      "NagC is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the glmUS operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) . @TF$ is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the @Regulated$ operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) .\n",
      "27\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [149, 154]\n",
      "Regulado: glmUS\n",
      "At the moment we do not know how NagC activates the fimB promoter , but two general scenarios seem most plausible . At the moment we do not know how @TF$ activates the @Regulated$ promoter , but two general scenarios seem most plausible .\n",
      "11\n",
      "8\n",
      "Start y end del regulador: [33, 37]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [52, 56]\n",
      "Regulado: fimB\n",
      "Note that the magnitude of the effect of CRP on activation of the melAB promoter in the in vitro transcription experiment ( Fig . 2B ) is less than might have been expected from the footprinting and in vivo experiments ( Fig . 4 and Table 1 ) . Note that the magnitude of the effect of @TF$ on activation of the @Regulated$ promoter in the in vitro transcription experiment ( Fig . 2B ) is less than might have been expected from the footprinting and in vivo experiments ( Fig . 4 and Table 1 ) .\n",
      "13\n",
      "8\n",
      "Start y end del regulador: [41, 44]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [66, 71]\n",
      "Regulado: melAB\n",
      "Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven . Two other regulators , OmpR and @TF$ , activate @Regulated$ expression , although a direct control has not been proven .\n",
      "9\n",
      "6\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: CpxR\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: nanC\n",
      "Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven . Two other regulators , @TF$ and CpxR , activate @Regulated$ expression , although a direct control has not been proven .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: nanC\n",
      "Previous results reported by Bhende and Egan ( 4 ) identified two amino acid residues in 70 , K593 and R599 , that were important for RhaS mediated transcription activation at rhaBAD and rhaT . Previous results reported by Bhende and Egan ( 4 ) identified two amino acid residues in 70 , K593 and R599 , that were important for @TF$ mediated transcription activation at @Regulated$ and rhaT .\n",
      "31\n",
      "26\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [176, 182]\n",
      "Regulado: rhaBAD\n",
      "Two gels were prepared , with identical culture samples on each gel , and each blot was probed with the primary antibody corresponding to the purified protein samples loaded , as indicated to the right . - S , anti - RhaS antibody ; - R , anti - RhaR antibody . allow CRP to contribute to rhaBAD activation . Two gels were prepared , with identical culture samples on each gel , and each blot was probed with the primary antibody corresponding to the purified protein samples loaded , as indicated to the right . - S , anti - RhaS antibody ; - R , anti - RhaR antibody . allow @TF$ to contribute to @Regulated$ activation .\n",
      "57\n",
      "53\n",
      "Start y end del regulador: [268, 271]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [289, 295]\n",
      "Regulado: rhaBAD\n",
      "Previous results reported by Bhende and Egan ( 4 ) identified two amino acid residues in 70 , K593 and R599 , that were important for RhaS mediated transcription activation at rhaBAD and rhaT . Previous results reported by Bhende and Egan ( 4 ) identified two amino acid residues in 70 , K593 and R599 , that were important for @TF$ mediated transcription activation at rhaBAD and @Regulated$ .\n",
      "33\n",
      "26\n",
      "Start y end del regulador: [134, 138]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [187, 191]\n",
      "Regulado: rhaT\n",
      "It was shown that to activate transcription in P araBAD , the AraC protein binding site must overlap the 35 region of the promoter by 4 bp ( 212 ) . It was shown that to activate transcription in P @Regulated$ , the @TF$ protein binding site must overlap the 35 region of the promoter by 4 bp ( 212 ) .\n",
      "9\n",
      "12\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [49, 55]\n",
      "Regulado: araBAD\n",
      "Several Rob - regulated genes , such as marA , noted above , seem to encode other regulatory proteins . Several @TF$ - regulated genes , such as @Regulated$ , noted above , seem to encode other regulatory proteins .\n",
      "8\n",
      "1\n",
      "Start y end del regulador: [8, 11]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [40, 44]\n",
      "Regulado: marA\n",
      "BACTERIOL . present study , we identified four amino acid residues in 70 , K593 , L595 , R599 , and R608 , which were important for RhaR mediated transcription activation of rhaSR ( Fig . 2B ) . BACTERIOL . present study , we identified four amino acid residues in 70 , K593 , L595 , R599 , and R608 , which were important for @TF$ mediated transcription activation of @Regulated$ ( Fig . 2B ) .\n",
      "32\n",
      "27\n",
      "Start y end del regulador: [132, 136]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [174, 179]\n",
      "Regulado: rhaSR\n",
      "Although these results are consistent with the requirement for CAP and ChbR for chbB activation there is a significant difference between the in vivo and in vitro data . Although these results are consistent with the requirement for @TF$ and ChbR for @Regulated$ activation there is a significant difference between the in vivo and in vitro data .\n",
      "13\n",
      "9\n",
      "Start y end del regulador: [63, 66]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: chbB\n",
      "Although these results are consistent with the requirement for CAP and ChbR for chbB activation there is a significant difference between the in vivo and in vitro data . Although these results are consistent with the requirement for CAP and @TF$ for @Regulated$ activation there is a significant difference between the in vivo and in vitro data .\n",
      "13\n",
      "11\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: chbB\n",
      "( 2000 ) Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . ( 2000 ) Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [58, 63]\n",
      "Regulado: melAB\n",
      "This unusual spacing in relation to the RNA polymerase binding site may reflect a distinct mechanism by which Fnr ( and NarP ) activates aeg - 46.5 operon expression . This unusual spacing in relation to the RNA polymerase binding site may reflect a distinct mechanism by which Fnr ( and @TF$ ) activates @Regulated$ - 46.5 operon expression .\n",
      "24\n",
      "21\n",
      "Start y end del regulador: [120, 124]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [137, 140]\n",
      "Regulado: aeg\n",
      "NarP thus performs a unique role to fine - tune fdnG operon expression by delaying the ability of NarL to activate when nitrate levels are low . @TF$ thus performs a unique role to fine - tune @Regulated$ operon expression by delaying the ability of NarL to activate when nitrate levels are low .\n",
      "10\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: fdnG\n",
      "It is interesting to speculate that this is the element responsible for binding GntR and repression of gntKU expression . It is interesting to speculate that this is the element responsible for binding @TF$ and repression of @Regulated$ expression .\n",
      "17\n",
      "13\n",
      "Start y end del regulador: [80, 84]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [103, 108]\n",
      "Regulado: gntKU\n",
      "At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) . At the same time , it is noteworthy that two of the genes that are most prominently regulated by @TF$ , @Regulated$ and lysP , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) .\n",
      "21\n",
      "19\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: argO\n",
      "At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) . At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , @Regulated$ and lysP , are both also regulated by the leucineresponsive general transcriptional regulator @TF$ ( 37 , 41 ) .\n",
      "21\n",
      "35\n",
      "Start y end del regulador: [203, 206]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [104, 108]\n",
      "Regulado: argO\n",
      "At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) . At the same time , it is noteworthy that two of the genes that are most prominently regulated by @TF$ , argO and @Regulated$ , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) .\n",
      "23\n",
      "19\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [113, 117]\n",
      "Regulado: lysP\n",
      "At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucineresponsive general transcriptional regulator Lrp ( 37 , 41 ) . At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and @Regulated$ , are both also regulated by the leucineresponsive general transcriptional regulator @TF$ ( 37 , 41 ) .\n",
      "23\n",
      "35\n",
      "Start y end del regulador: [203, 206]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [113, 117]\n",
      "Regulado: lysP\n",
      "This scenario is plausible , because ArgP and Lrp competitively activate argO ( 35 ) . This scenario is plausible , because @TF$ and Lrp competitively activate @Regulated$ ( 35 ) .\n",
      "11\n",
      "6\n",
      "Start y end del regulador: [37, 41]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: argO\n",
      "This scenario is plausible , because ArgP and Lrp competitively activate argO ( 35 ) . This scenario is plausible , because ArgP and @TF$ competitively activate @Regulated$ ( 35 ) .\n",
      "11\n",
      "8\n",
      "Start y end del regulador: [46, 49]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: argO\n",
      "In this study , galT was the only gene repressed by Rob . In this study , @Regulated$ was the only gene repressed by @TF$ .\n",
      "4\n",
      "11\n",
      "Start y end del regulador: [52, 55]\n",
      "Regulador: Rob\n",
      "Start y end del regulado: [16, 20]\n",
      "Regulado: galT\n",
      "Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli . Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / @TF$ , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .\n",
      "46\n",
      "17\n",
      "Start y end del regulador: [101, 104]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [265, 274]\n",
      "Regulado: chbBCARFG\n",
      "Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli . Discussion The results described here show that three transcription factors , @TF$ , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .\n",
      "46\n",
      "11\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: ChbR\n",
      "Start y end del regulado: [265, 274]\n",
      "Regulado: chbBCARFG\n",
      "Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli . Discussion The results described here show that three transcription factors , ChbR , @TF$ and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .\n",
      "46\n",
      "13\n",
      "Start y end del regulador: [85, 89]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [265, 274]\n",
      "Regulado: chbBCARFG\n",
      "As at the yeaR promoter , nitrate - dependent induction can be repressed by Fis that binds to a single site overlapping the upstream DNA site for NarL . As at the @Regulated$ promoter , nitrate - dependent induction can be repressed by @TF$ that binds to a single site overlapping the upstream DNA site for NarL .\n",
      "3\n",
      "14\n",
      "Start y end del regulador: [76, 79]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [10, 14]\n",
      "Regulado: yeaR\n",
      "Dong X , Li S , DeMoss J : Upstream sequence elements required for NarL - mediated activation of transcription from the narGHJI promoter of Escherichia coli . Dong X , Li S , DeMoss J : Upstream sequence elements required for @TF$ - mediated activation of transcription from the @Regulated$ promoter of Escherichia coli .\n",
      "22\n",
      "14\n",
      "Start y end del regulador: [67, 71]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [120, 127]\n",
      "Regulado: narGHJI\n",
      "Thus , if the model proposed above is correct , it seems probable that NanR must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate fimB expression in the wild - type background as well . Thus , if the model proposed above is correct , it seems probable that @TF$ must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate @Regulated$ expression in the wild - type background as well .\n",
      "36\n",
      "14\n",
      "Start y end del regulador: [71, 75]\n",
      "Regulador: NanR\n",
      "Start y end del regulado: [197, 201]\n",
      "Regulado: fimB\n",
      "One of the more striking points in each model is that the NarP protein is nonessential ( i . e . , not a major player ) and that it can function as an antagonist of NarL in its ability to activate nirB or nrfA operon expression . One of the more striking points in each model is that the NarP protein is nonessential ( i . e . , not a major player ) and that it can function as an antagonist of @TF$ in its ability to activate @Regulated$ or nrfA operon expression .\n",
      "42\n",
      "36\n",
      "Start y end del regulador: [165, 169]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [197, 201]\n",
      "Regulado: nirB\n",
      "One of the more striking points in each model is that the NarP protein is nonessential ( i . e . , not a major player ) and that it can function as an antagonist of NarL in its ability to activate nirB or nrfA operon expression . One of the more striking points in each model is that the NarP protein is nonessential ( i . e . , not a major player ) and that it can function as an antagonist of @TF$ in its ability to activate nirB or @Regulated$ operon expression .\n",
      "44\n",
      "36\n",
      "Start y end del regulador: [165, 169]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [205, 209]\n",
      "Regulado: nrfA\n",
      "Thus , it appears that NarL and NarP adopt overlapping mechanisms to inhibit ydhY – T expression . Thus , it appears that NarL and @TF$ adopt overlapping mechanisms to inhibit @Regulated$ – T expression .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: ydhY\n",
      "Thus , the dcuSR operon control region contains an authenticated 7 - 2 - 7 heptamer pair for phospho - NarL binding overlapping the likely 35 region of the promoter ( Fig . 2B ) and therefore positioned appropriately to effect the observed repression ( Table 2 ) . Thus , the @Regulated$ operon control region contains an authenticated 7 - 2 - 7 heptamer pair for phospho - @TF$ binding overlapping the likely 35 region of the promoter ( Fig . 2B ) and therefore positioned appropriately to effect the observed repression ( Table 2 ) .\n",
      "3\n",
      "20\n",
      "Start y end del regulador: [103, 107]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [11, 16]\n",
      "Regulado: dcuSR\n",
      "NorR usually regulates cytoplasmic NO reductase norVW and sometimes another membrane - bound NO reductase ( norB ) and the NO dioxygenase hmp . @TF$ usually regulates cytoplasmic NO reductase @Regulated$ and sometimes another membrane - bound NO reductase ( norB ) and the NO dioxygenase hmp .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NorR\n",
      "Start y end del regulado: [48, 53]\n",
      "Regulado: norVW\n",
      "NorR usually regulates cytoplasmic NO reductase norVW and sometimes another membrane - bound NO reductase ( norB ) and the NO dioxygenase hmp . @TF$ usually regulates cytoplasmic NO reductase norVW and sometimes another membrane - bound NO reductase ( norB ) and the NO dioxygenase @Regulated$ .\n",
      "22\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NorR\n",
      "Start y end del regulado: [138, 141]\n",
      "Regulado: hmp\n",
      "The mechanism of transcriptional activation of micF and ompC by OmpR as well as a role of IHF in this activation need to be better understood . The mechanism of transcriptional activation of @Regulated$ and ompC by @TF$ as well as a role of IHF in this activation need to be better understood .\n",
      "6\n",
      "10\n",
      "Start y end del regulador: [64, 68]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: micF\n",
      "NorR usually regulates cytoplasmic NO reductase norVW and sometimes another membrane - bound NO reductase ( norB ) and the NO dioxygenase hmp . @TF$ usually regulates cytoplasmic NO reductase norVW and sometimes another membrane - bound NO reductase ( @Regulated$ ) and the NO dioxygenase hmp .\n",
      "16\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NorR\n",
      "Start y end del regulado: [108, 112]\n",
      "Regulado: norB\n",
      "The mechanism of transcriptional activation of micF and ompC by OmpR as well as a role of IHF in this activation need to be better understood . The mechanism of transcriptional activation of micF and @Regulated$ by @TF$ as well as a role of IHF in this activation need to be better understood .\n",
      "8\n",
      "10\n",
      "Start y end del regulador: [64, 68]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [56, 60]\n",
      "Regulado: ompC\n",
      "DISCUSSION CRP activates rhaSR from at least one newly identified binding site . DISCUSSION @TF$ activates @Regulated$ from at least one newly identified binding site .\n",
      "3\n",
      "1\n",
      "Start y end del regulador: [11, 14]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [25, 30]\n",
      "Regulado: rhaSR\n",
      "Thus , NarL - mediated repression of the ydhY promoter follows the general pattern established by the frdA and dmsA promoters . Thus , @TF$ - mediated repression of the @Regulated$ promoter follows the general pattern established by the frdA and dmsA promoters .\n",
      "8\n",
      "2\n",
      "Start y end del regulador: [7, 11]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [41, 45]\n",
      "Regulado: ydhY\n",
      "Recently , a direct regulatory connection between oxidative stress and iron metabolism was shown by Zheng et al . , who demonstrated the transcriptional activation of fur by SoxS and OxyR ( 49 ) . Recently , a direct regulatory connection between oxidative stress and iron metabolism was shown by Zheng et al . , who demonstrated the transcriptional activation of @Regulated$ by SoxS and @TF$ ( 49 ) .\n",
      "26\n",
      "30\n",
      "Start y end del regulador: [183, 187]\n",
      "Regulador: OxyR\n",
      "Start y end del regulado: [167, 170]\n",
      "Regulado: fur\n",
      "Recently , a direct regulatory connection between oxidative stress and iron metabolism was shown by Zheng et al . , who demonstrated the transcriptional activation of fur by SoxS and OxyR ( 49 ) . Recently , a direct regulatory connection between oxidative stress and iron metabolism was shown by Zheng et al . , who demonstrated the transcriptional activation of @Regulated$ by @TF$ and OxyR ( 49 ) .\n",
      "26\n",
      "28\n",
      "Start y end del regulador: [174, 178]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [167, 170]\n",
      "Regulado: fur\n",
      "The NarL and NarP proteins are each able to activate expression of the nirB and nrfA operons by interaction with common DNA - binding sites . The @TF$ and NarP proteins are each able to activate expression of the @Regulated$ and nrfA operons by interaction with common DNA - binding sites .\n",
      "13\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [71, 75]\n",
      "Regulado: nirB\n",
      "The NarL and NarP proteins are each able to activate expression of the nirB and nrfA operons by interaction with common DNA - binding sites . The NarL and @TF$ proteins are each able to activate expression of the @Regulated$ and nrfA operons by interaction with common DNA - binding sites .\n",
      "13\n",
      "3\n",
      "Start y end del regulador: [13, 17]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [71, 75]\n",
      "Regulado: nirB\n",
      "The NarL and NarP proteins are each able to activate expression of the nirB and nrfA operons by interaction with common DNA - binding sites . The @TF$ and NarP proteins are each able to activate expression of the nirB and @Regulated$ operons by interaction with common DNA - binding sites .\n",
      "15\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: nrfA\n",
      "The NarL and NarP proteins are each able to activate expression of the nirB and nrfA operons by interaction with common DNA - binding sites . The NarL and @TF$ proteins are each able to activate expression of the nirB and @Regulated$ operons by interaction with common DNA - binding sites .\n",
      "15\n",
      "3\n",
      "Start y end del regulador: [13, 17]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [80, 84]\n",
      "Regulado: nrfA\n",
      "The NarL / NarP protein binding sites required for activation of the nirB and nrfA operons are located further upstream ( 69.5 and 74.5 , respectively ) than that of the aeg - 46.5 operon ( 44.5 ) . The @TF$ / NarP protein binding sites required for activation of the @Regulated$ and nrfA operons are located further upstream ( 69.5 and 74.5 , respectively ) than that of the aeg - 46.5 operon ( 44.5 ) .\n",
      "12\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [69, 73]\n",
      "Regulado: nirB\n",
      "The NarL / NarP protein binding sites required for activation of the nirB and nrfA operons are located further upstream ( 69.5 and 74.5 , respectively ) than that of the aeg - 46.5 operon ( 44.5 ) . The NarL / @TF$ protein binding sites required for activation of the nirB and @Regulated$ operons are located further upstream ( 69.5 and 74.5 , respectively ) than that of the aeg - 46.5 operon ( 44.5 ) .\n",
      "14\n",
      "3\n",
      "Start y end del regulador: [11, 15]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: nrfA\n",
      "NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression . NarX and / or NarQ sensors , both @TF$ and NarP are phosphorylated where either response regulator can then activate @Regulated$ gene expression .\n",
      "20\n",
      "8\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: nrfA\n",
      "NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression . NarX and / or NarQ sensors , both NarL and @TF$ are phosphorylated where either response regulator can then activate @Regulated$ gene expression .\n",
      "20\n",
      "10\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [117, 121]\n",
      "Regulado: nrfA\n",
      "Lrp stimulates transcription of lysP by direct binding to its control region . @TF$ stimulates transcription of @Regulated$ by direct binding to its control region .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [32, 36]\n",
      "Regulado: lysP\n",
      "Given the proximity of this binding site to the start site for transcription , these results suggest a model where AraC sterically inhibits the binding of XylR and RNA polymerase to the P xylA promoter , consistent with our experimental results . Given the proximity of this binding site to the start site for transcription , these results suggest a model where @TF$ sterically inhibits the binding of XylR and RNA polymerase to the P @Regulated$ promoter , consistent with our experimental results .\n",
      "33\n",
      "20\n",
      "Start y end del regulador: [115, 119]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [188, 192]\n",
      "Regulado: xylA\n",
      "Our aim in this work was to ascertain if such a simple model applied to MelR - dependent repression of the melR promoter . Our aim in this work was to ascertain if such a simple model applied to @TF$ - dependent repression of the @Regulated$ promoter .\n",
      "21\n",
      "15\n",
      "Start y end del regulador: [72, 76]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [107, 111]\n",
      "Regulado: melR\n",
      "These genes , together with xylAB , are positively regulated by the same protein ( XylR [ Fig . 4 ] ) , which appears to recognize the consensus sequences located in their promoterVOL . These genes , together with @Regulated$ , are positively regulated by the same protein ( @TF$ [ Fig . 4 ] ) , which appears to recognize the consensus sequences located in their promoterVOL .\n",
      "5\n",
      "15\n",
      "Start y end del regulador: [83, 87]\n",
      "Regulador: XylR\n",
      "Start y end del regulado: [28, 33]\n",
      "Regulado: xylAB\n",
      "Model for MelR and CRP binding during activation at the melAB promoter . Model for MelR and @TF$ binding during activation at the @Regulated$ promoter .\n",
      "10\n",
      "4\n",
      "Start y end del regulador: [19, 22]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [56, 61]\n",
      "Regulado: melAB\n",
      "Model for MelR and CRP binding during activation at the melAB promoter . Model for @TF$ and CRP binding during activation at the @Regulated$ promoter .\n",
      "10\n",
      "2\n",
      "Start y end del regulador: [10, 14]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [56, 61]\n",
      "Regulado: melAB\n",
      "Model for transcription activation at the melAB promoter . In the absence of melibiose , MelR subunits ( M ) occupy Site 1 0 , Site 1 and Site 2 and the CRP dimer is recruited to the DNA between Site 1 and Site 2 . Model for transcription activation at the @Regulated$ promoter . In the absence of melibiose , @TF$ subunits ( M ) occupy Site 1 0 , Site 1 and Site 2 and the CRP dimer is recruited to the DNA between Site 1 and Site 2 .\n",
      "6\n",
      "15\n",
      "Start y end del regulador: [89, 93]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [42, 47]\n",
      "Regulado: melAB\n",
      "( B ) Transcription activation by RhaS expressed from the chromosome in SME1048 ( wild - type rhaSR ) . ( B ) Transcription activation by @TF$ expressed from the chromosome in SME1048 ( wild - type @Regulated$ ) .\n",
      "17\n",
      "6\n",
      "Start y end del regulador: [34, 38]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [94, 99]\n",
      "Regulado: rhaSR\n",
      "A revised model for NarL - dependent activation of nirB gene expression is shown in Fig . 9 . A revised model for @TF$ - dependent activation of @Regulated$ gene expression is shown in Fig . 9 .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [51, 55]\n",
      "Regulado: nirB\n",
      "( 2001 ) Systematic mutagenesis of the DNA binding sites for SoxS in the zwf and fpr promoters of Escherichia coli : identifying nucleotides required for DNA binding and transcription activation . ( 2001 ) Systematic mutagenesis of the DNA binding sites for @TF$ in the zwf and @Regulated$ promoters of Escherichia coli : identifying nucleotides required for DNA binding and transcription activation .\n",
      "16\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: SoxS\n",
      "Start y end del regulado: [81, 84]\n",
      "Regulado: fpr\n",
      "CsrA appears to activate eda expression but does so by an unknown mechanism . @TF$ appears to activate @Regulated$ expression but does so by an unknown mechanism .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: CsrA\n",
      "Start y end del regulado: [25, 28]\n",
      "Regulado: eda\n",
      "However , it is possible that the closely related YjgS protein , which is 46 % identical to GntR , is involved in the positive regulation of gntT . However , it is possible that the closely related @TF$ protein , which is 46 % identical to GntR , is involved in the positive regulation of @Regulated$ .\n",
      "27\n",
      "9\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: YjgS\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: gntT\n",
      "It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by Mlc ( 6 – 11 ) . It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and @Regulated$ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by @TF$ ( 6 – 11 ) .\n",
      "24\n",
      "46\n",
      "Start y end del regulador: [246, 249]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [136, 142]\n",
      "Regulado: manXYZ\n",
      "It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by Mlc ( 6 – 11 ) . It should be remembered that @Regulated$ encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by @TF$ ( 6 – 11 ) .\n",
      "5\n",
      "46\n",
      "Start y end del regulador: [246, 249]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [29, 33]\n",
      "Regulado: nagE\n",
      "It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by Mlc ( 6 – 11 ) . It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( @Regulated$ and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by @TF$ ( 6 – 11 ) .\n",
      "22\n",
      "46\n",
      "Start y end del regulador: [246, 249]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [127, 131]\n",
      "Regulado: ptsG\n",
      "It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( ptsHIcrr ) have been shown to be regulated by Mlc ( 6 – 11 ) . It should be remembered that nagE encodes the GlcNAc - specific PTS transporter and the genes for two other PTS transporters ( ptsG and manXYZ ) plus the genes for the central components of the PTS ( @Regulated$ ) have been shown to be regulated by @TF$ ( 6 – 11 ) .\n",
      "37\n",
      "46\n",
      "Start y end del regulador: [246, 249]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [200, 208]\n",
      "Regulado: ptsHIcrr\n",
      "These results suggest that both GntRbinding sequences , R1 and R2 , are responsible for the repression of gntKU expression , although the affinity of GntR or GntH for R1 seems to be different from that for R2 . These results suggest that both GntRbinding sequences , R1 and R2 , are responsible for the repression of @Regulated$ expression , although the affinity of @TF$ or GntH for R1 seems to be different from that for R2 .\n",
      "18\n",
      "25\n",
      "Start y end del regulador: [150, 154]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [106, 111]\n",
      "Regulado: gntKU\n",
      "NagC is required for full repression of the chbB operon and it serves to limit induction when the supply of chitobiose is low . @TF$ is required for full repression of the @Regulated$ operon and it serves to limit induction when the supply of chitobiose is low .\n",
      "8\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [44, 48]\n",
      "Regulado: chbB\n",
      "These results confirm that regulation of the gntT gene by GntR is similar to other negatively regulated systems . These results confirm that regulation of the @Regulated$ gene by @TF$ is similar to other negatively regulated systems .\n",
      "7\n",
      "10\n",
      "Start y end del regulador: [58, 62]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [45, 49]\n",
      "Regulado: gntT\n",
      "Thus , antagonization of NarP - independent activation of aeg - 46.5 operon expression by the NarL protein should be more pronounced in the presence of nitrate than nitrate . Thus , antagonization of @TF$ - independent activation of @Regulated$ - 46.5 operon expression by the NarL protein should be more pronounced in the presence of nitrate than nitrate .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [25, 29]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [58, 61]\n",
      "Regulado: aeg\n",
      "Mechanism of activation by CRP at rhaSR . Mechanism of activation by @TF$ at @Regulated$ .\n",
      "6\n",
      "4\n",
      "Start y end del regulador: [27, 30]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [34, 39]\n",
      "Regulado: rhaSR\n",
      "For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid - associated protein HU . For example , the @Regulated$ operon is regulated by three factors GalR , @TF$ and the nucleoid - associated protein HU .\n",
      "4\n",
      "13\n",
      "Start y end del regulador: [69, 72]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [18, 24]\n",
      "Regulado: galETK\n",
      "For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid - associated protein HU . For example , the @Regulated$ operon is regulated by three factors @TF$ , CAP and the nucleoid - associated protein HU .\n",
      "4\n",
      "11\n",
      "Start y end del regulador: [62, 66]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [18, 24]\n",
      "Regulado: galETK\n",
      "For example , the galETK operon is regulated by three factors GalR , CAP and the nucleoid - associated protein HU . For example , the @Regulated$ operon is regulated by three factors GalR , CAP and the nucleoid - associated protein @TF$ .\n",
      "4\n",
      "20\n",
      "Start y end del regulador: [111, 113]\n",
      "Regulador: HU\n",
      "Start y end del regulado: [18, 24]\n",
      "Regulado: galETK\n",
      "Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "6\n",
      "12\n",
      "Start y end del regulador: [78, 82]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [49, 54]\n",
      "Regulado: melAB\n",
      "Another example is the NarL / NarP - and FNR - stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) . Another example is the NarL / NarP - and FNR - stimulated expression of @Regulated$ and nrf genes , which are repressed by @TF$ and IHF ( Browning et al . , 2002 ) .\n",
      "14\n",
      "23\n",
      "Start y end del regulador: [116, 119]\n",
      "Regulador: Fis\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nirB\n",
      "Another example is the NarL / NarP - and FNR - stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) . Another example is the NarL / NarP - and @TF$ - stimulated expression of @Regulated$ and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) .\n",
      "14\n",
      "9\n",
      "Start y end del regulador: [41, 44]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nirB\n",
      "Another example is the NarL / NarP - and FNR - stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) . Another example is the NarL / NarP - and FNR - stimulated expression of @Regulated$ and nrf genes , which are repressed by Fis and @TF$ ( Browning et al . , 2002 ) .\n",
      "14\n",
      "25\n",
      "Start y end del regulador: [124, 127]\n",
      "Regulador: IHF\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nirB\n",
      "Another example is the NarL / NarP - and FNR - stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) . Another example is the @TF$ / NarP - and FNR - stimulated expression of @Regulated$ and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) .\n",
      "14\n",
      "4\n",
      "Start y end del regulador: [23, 27]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nirB\n",
      "Another example is the NarL / NarP - and FNR - stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) . Another example is the NarL / @TF$ - and FNR - stimulated expression of @Regulated$ and nrf genes , which are repressed by Fis and IHF ( Browning et al . , 2002 ) .\n",
      "14\n",
      "6\n",
      "Start y end del regulador: [30, 34]\n",
      "Regulador: NarP\n",
      "Start y end del regulado: [72, 76]\n",
      "Regulado: nirB\n",
      "Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase sigma subunit . Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase sigma subunit .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [49, 54]\n",
      "Regulado: melAB\n",
      "We favor the idea that the repression of gntT is the result of DNA looping through interaction between the two GntR molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) . We favor the idea that the repression of @Regulated$ is the result of DNA looping through interaction between the two @TF$ molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) .\n",
      "8\n",
      "20\n",
      "Start y end del regulador: [111, 115]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [41, 45]\n",
      "Regulado: gntT\n",
      "A greater extent of derepression results from mutation of the external operator site by comparison to the internal operator site mutants , suggesting that binding of GntR to the external site results in a stronger repression of the gntT gene . A greater extent of derepression results from mutation of the external operator site by comparison to the internal operator site mutants , suggesting that binding of @TF$ to the external site results in a stronger repression of the @Regulated$ gene .\n",
      "38\n",
      "26\n",
      "Start y end del regulador: [166, 170]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [232, 236]\n",
      "Regulado: gntT\n",
      "RhaS activates transcription from rhaBAD , and transcription from the rhaS gene is controlled by RhaR ( 63 , 244 ) . @TF$ activates transcription from @Regulated$ , and transcription from the rhaS gene is controlled by RhaR ( 63 , 244 ) .\n",
      "4\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [34, 40]\n",
      "Regulado: rhaBAD\n",
      "RhaS activates transcription from rhaBAD , and transcription from the rhaS gene is controlled by RhaR ( 63 , 244 ) . RhaS activates transcription from rhaBAD , and transcription from the @Regulated$ gene is controlled by @TF$ ( 63 , 244 ) .\n",
      "10\n",
      "15\n",
      "Start y end del regulador: [97, 101]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [70, 74]\n",
      "Regulado: rhaS\n",
      "The inducer for ribose phosphate isomerase B and the mechanism by which RpiR regulates rpiB gene expression remain to be elucidated . The inducer for ribose phosphate isomerase B and the mechanism by which @TF$ regulates @Regulated$ gene expression remain to be elucidated .\n",
      "14\n",
      "12\n",
      "Start y end del regulador: [72, 76]\n",
      "Regulador: RpiR\n",
      "Start y end del regulado: [87, 91]\n",
      "Regulado: rpiB\n",
      "These results appear to be consistent with those from the experiments with lacZ operon fusions , as shown in Table 4 , and it is likely that the binding of GntR to a single element ( the R1 sequence ) is capable of decreasing the transcription of the gntKU genes . These results appear to be consistent with those from the experiments with lacZ operon fusions , as shown in Table 4 , and it is likely that the binding of @TF$ to a single element ( the R1 sequence ) is capable of decreasing the transcription of the @Regulated$ genes .\n",
      "48\n",
      "30\n",
      "Start y end del regulador: [156, 160]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [251, 256]\n",
      "Regulado: gntKU\n",
      "The NagC - regulated nagB – lacZ fusion has been described previously ( Plumbridge and Kolb , 1998 ) . The @TF$ - regulated @Regulated$ – lacZ fusion has been described previously ( Plumbridge and Kolb , 1998 ) .\n",
      "4\n",
      "1\n",
      "Start y end del regulador: [4, 8]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [21, 25]\n",
      "Regulado: nagB\n",
      "Alternatively , - CTD residues 321 , 322 , and 323 were defective at a rhaBAD promoter fusion that was activated only by the RhaS protein . Alternatively , - CTD residues 321 , 322 , and 323 were defective at a @Regulated$ promoter fusion that was activated only by the @TF$ protein .\n",
      "15\n",
      "24\n",
      "Start y end del regulador: [125, 129]\n",
      "Regulador: RhaS\n",
      "Start y end del regulado: [71, 77]\n",
      "Regulado: rhaBAD\n",
      "( 1987 ) Transcriptional repression of the dnaA gene of Escherichia coli by DnaA protein . ( 1987 ) Transcriptional repression of the @Regulated$ gene of Escherichia coli by @TF$ protein .\n",
      "7\n",
      "13\n",
      "Start y end del regulador: [76, 80]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [43, 47]\n",
      "Regulado: dnaA\n",
      "Together , these results indicate that Lrp upregulates expression of cadBA . Together , these results indicate that @TF$ upregulates expression of @Regulated$ .\n",
      "10\n",
      "6\n",
      "Start y end del regulador: [39, 42]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [69, 74]\n",
      "Regulado: cadBA\n",
      "( 1989 ) DnaA protein regulates transcription of the rpoH gene of Escherichia coli . ( 1989 ) @TF$ protein regulates transcription of the @Regulated$ gene of Escherichia coli .\n",
      "9\n",
      "3\n",
      "Start y end del regulador: [9, 13]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [53, 57]\n",
      "Regulado: rpoH\n",
      "Lysine represses transcription of the Escherichia coli dapB gene by preventing its activation by the ArgP activator . Lysine represses transcription of the Escherichia coli @Regulated$ gene by preventing its activation by the @TF$ activator .\n",
      "7\n",
      "15\n",
      "Start y end del regulador: [101, 105]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [55, 59]\n",
      "Regulado: dapB\n",
      "With the exception of xylA , the last AraC - regulated gene to be identified was araJ , more than 30 years ago ( 27 ) . With the exception of xylA , the last @TF$ - regulated gene to be identified was @Regulated$ , more than 30 years ago ( 27 ) .\n",
      "16\n",
      "8\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [81, 85]\n",
      "Regulado: araJ\n",
      "With the exception of xylA , the last AraC - regulated gene to be identified was araJ , more than 30 years ago ( 27 ) . With the exception of @Regulated$ , the last @TF$ - regulated gene to be identified was araJ , more than 30 years ago ( 27 ) .\n",
      "4\n",
      "8\n",
      "Start y end del regulador: [38, 42]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [22, 26]\n",
      "Regulado: xylA\n",
      "It is important to note that the molecular mechanisms of lysinedependent regulation by ArgP differ between argO and dapB ( 2 , 24 ) . It is important to note that the molecular mechanisms of lysinedependent regulation by @TF$ differ between argO and @Regulated$ ( 2 , 24 ) .\n",
      "18\n",
      "13\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [116, 120]\n",
      "Regulado: dapB\n",
      "( 1995 ) Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . ( 1995 ) Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of @TF$ and FNR when bound in tandem .\n",
      "9\n",
      "18\n",
      "Start y end del regulador: [106, 109]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: ansB\n",
      "( 1995 ) Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . ( 1995 ) Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of CRP and @TF$ when bound in tandem .\n",
      "9\n",
      "20\n",
      "Start y end del regulador: [114, 117]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [48, 52]\n",
      "Regulado: ansB\n",
      "In experiments using native nag DNA , both sites are of DNA looping was reinforced by the strict phasing requirement of the AraC binding sites for repression of the araBAD operon ( Dunn et al . , 1984 ; Lee &amp; Schleif , 1989 ) . In experiments using native nag DNA , both sites are of DNA looping was reinforced by the strict phasing requirement of the @TF$ binding sites for repression of the @Regulated$ operon ( Dunn et al . , 1984 ; Lee &amp; Schleif , 1989 ) .\n",
      "29\n",
      "22\n",
      "Start y end del regulador: [124, 128]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [165, 171]\n",
      "Regulado: araBAD\n",
      "( 2000 ) Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . ( 2000 ) Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [58, 63]\n",
      "Regulado: melAB\n",
      "Upstream sequence elements required for NarL - mediated activation of transcription from the narGHJI promoter of Escherichia coli . Upstream sequence elements required for @TF$ - mediated activation of transcription from the @Regulated$ promoter of Escherichia coli .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [93, 100]\n",
      "Regulado: narGHJI\n",
      "We also identified AraC - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and polB ) . We also identified @TF$ - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and @Regulated$ ) .\n",
      "24\n",
      "3\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [141, 145]\n",
      "Regulado: polB\n",
      "We also identified AraC - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and polB ) . We also identified @TF$ - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( @Regulated$ and polB ) .\n",
      "22\n",
      "3\n",
      "Start y end del regulador: [19, 23]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [132, 136]\n",
      "Regulado: ygeA\n",
      "( 2000 ) Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . ( 2000 ) Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [58, 63]\n",
      "Regulado: melAB\n",
      "In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the CRP in the P rhaBAD promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) . In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the @TF$ in the P @Regulated$ promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) .\n",
      "34\n",
      "30\n",
      "Start y end del regulador: [174, 177]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [187, 193]\n",
      "Regulado: rhaBAD\n",
      "AraC has previously been shown to repress its own transcription by binding to a region overlapping the araC promoter elements ( 32 ) . @TF$ has previously been shown to repress its own transcription by binding to a region overlapping the @Regulated$ promoter elements ( 32 ) .\n",
      "17\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [103, 107]\n",
      "Regulado: araC\n",
      "The repression of the araBAD operon occurs by cooperative interaction between two distal AraC binding sites araOz and urd ( 19 , 39,40 ) . The repression of the @Regulated$ operon occurs by cooperative interaction between two distal @TF$ binding sites araOz and urd ( 19 , 39,40 ) .\n",
      "4\n",
      "13\n",
      "Start y end del regulador: [89, 93]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [22, 28]\n",
      "Regulado: araBAD\n",
      "The location of the AraC binding site upstream of ytfQ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP . The location of the @TF$ binding site upstream of @Regulated$ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP .\n",
      "9\n",
      "4\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [50, 54]\n",
      "Regulado: ytfQ\n",
      "GalR has been shown to regulate ytfQ ( 44 ) ( Fig . S1 in the supplemental material ) . @TF$ has been shown to regulate @Regulated$ ( 44 ) ( Fig . S1 in the supplemental material ) .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: GalR\n",
      "Start y end del regulado: [32, 36]\n",
      "Regulado: ytfQ\n",
      "Lrp - dependent regulation of lysP and cadBA . @TF$ - dependent regulation of lysP and @Regulated$ .\n",
      "7\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [39, 44]\n",
      "Regulado: cadBA\n",
      "Lrp - dependent regulation of lysP and cadBA . @TF$ - dependent regulation of @Regulated$ and cadBA .\n",
      "5\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [30, 34]\n",
      "Regulado: lysP\n",
      "( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop . ( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop .\n",
      "8\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [44, 48]\n",
      "Regulado: melR\n",
      "( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that full repression requires the formation of a repression loop . ( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that full repression requires the formation of a repression loop .\n",
      "8\n",
      "11\n",
      "Start y end del regulador: [61, 65]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [44, 48]\n",
      "Regulado: melR\n",
      "Recently , the implication of Lrp in the regulation of the gene encoding the arginine exporter ( argO ) was reported . Recently , the implication of @TF$ in the regulation of the gene encoding the arginine exporter ( @Regulated$ ) was reported .\n",
      "17\n",
      "5\n",
      "Start y end del regulador: [30, 33]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [97, 101]\n",
      "Regulado: argO\n",
      "( 2000 ) Transcription activation at the Escherichia coli melAB promoter : the role of MelR and the cyclic AMP receptor protein . ( 2000 ) Transcription activation at the Escherichia coli @Regulated$ promoter : the role of @TF$ and the cyclic AMP receptor protein .\n",
      "9\n",
      "15\n",
      "Start y end del regulador: [87, 91]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [58, 63]\n",
      "Regulado: melAB\n",
      "Here , we found that Lrp binds to several sites at the lysP control region and is able to potentiate the transcriptional activation mediated by ArgP when lysine becomes limiting . Here , we found that @TF$ binds to several sites at the @Regulated$ control region and is able to potentiate the transcriptional activation mediated by ArgP when lysine becomes limiting .\n",
      "12\n",
      "5\n",
      "Start y end del regulador: [21, 24]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [55, 59]\n",
      "Regulado: lysP\n",
      "Only one other novel AraC - regulated gene identified in this work , E . coli ytfQ , has a known connection to arabinose metabolism ( 39 ) . Only one other novel @TF$ - regulated gene identified in this work , E . coli @Regulated$ , has a known connection to arabinose metabolism ( 39 ) .\n",
      "16\n",
      "4\n",
      "Start y end del regulador: [21, 25]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [78, 82]\n",
      "Regulado: ytfQ\n",
      "According to the results described here , Lrp participates in the activation of cadBA under inducing conditions . According to the results described here , @TF$ participates in the activation of @Regulated$ under inducing conditions .\n",
      "13\n",
      "7\n",
      "Start y end del regulador: [42, 45]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [80, 85]\n",
      "Regulado: cadBA\n",
      "Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase sigma subunit . Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase sigma subunit .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [49, 54]\n",
      "Regulado: melAB\n",
      "( 2004 ) Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase s subunit . ( 2004 ) Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase s subunit .\n",
      "9\n",
      "14\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [58, 63]\n",
      "Regulado: melAB\n",
      "Upstream sequence elements required for NarL - mediated activation of transcription from the narGHJI promoter of Escherichia coli . Upstream sequence elements required for @TF$ - mediated activation of transcription from the @Regulated$ promoter of Escherichia coli .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [93, 100]\n",
      "Regulado: narGHJI\n",
      "Hence , polB regulation by AraC may be widely conserved . Hence , @Regulated$ regulation by @TF$ may be widely conserved .\n",
      "2\n",
      "5\n",
      "Start y end del regulador: [27, 31]\n",
      "Regulador: AraC\n",
      "Start y end del regulado: [8, 12]\n",
      "Regulado: polB\n",
      "Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene gntT by GntR and the cyclic AMP ( cAMP ) - cAMP receptor protein complex . Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene @Regulated$ by @TF$ and the cyclic AMP ( cAMP ) - cAMP receptor protein complex .\n",
      "12\n",
      "14\n",
      "Start y end del regulador: [104, 108]\n",
      "Regulador: GntR\n",
      "Start y end del regulado: [96, 100]\n",
      "Regulado: gntT\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of @TF$ and FNR when bound in tandem .\n",
      "6\n",
      "15\n",
      "Start y end del regulador: [97, 100]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of CRP and @TF$ when bound in tandem .\n",
      "6\n",
      "17\n",
      "Start y end del regulador: [105, 108]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression . The yeaS ( @Regulated$ ) gene of Escherichia coli encodes an exporter of leucine , and the @TF$ protein regulates its expression .\n",
      "3\n",
      "17\n",
      "Start y end del regulador: [84, 87]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [11, 15]\n",
      "Regulado: leuE\n",
      "Auto - activation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli . Auto - activation of the @Regulated$ multiple antibiotic resistance operon by the @TF$ transcriptional activator in Escherichia coli .\n",
      "5\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [25, 31]\n",
      "Regulado: marRAB\n",
      "Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with the C - terminal domain of the RNA polymerase alpha subunit . Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with the C - terminal domain of the RNA polymerase alpha subunit .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [49, 54]\n",
      "Regulado: melAB\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of @TF$ and FNR when bound in tandem .\n",
      "6\n",
      "15\n",
      "Start y end del regulador: [97, 100]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of CRP and @TF$ when bound in tandem .\n",
      "6\n",
      "17\n",
      "Start y end del regulador: [105, 108]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "Role of multiple CytR binding sites on cooperativity , competition , and induction at the Escherichia coli udp promoter . Role of multiple @TF$ binding sites on cooperativity , competition , and induction at the Escherichia coli @Regulated$ promoter .\n",
      "17\n",
      "3\n",
      "Start y end del regulador: [17, 21]\n",
      "Regulador: CytR\n",
      "Start y end del regulado: [107, 110]\n",
      "Regulado: udp\n",
      "Transcriptional activation of the dnaA gene encoding the initiator for oriC replication by IciA protein , an inhibitor of in vitro oriC replication in Escherichia coli . Transcriptional activation of the @Regulated$ gene encoding the initiator for oriC replication by @TF$ protein , an inhibitor of in vitro oriC replication in Escherichia coli .\n",
      "4\n",
      "13\n",
      "Start y end del regulador: [91, 95]\n",
      "Regulador: IciA\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: dnaA\n",
      "Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase sigma subunit . Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase sigma subunit .\n",
      "6\n",
      "11\n",
      "Start y end del regulador: [82, 86]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [49, 54]\n",
      "Regulado: melAB\n",
      "Transcriptional repression of the dnaA gene of Escherichia coli by DnaA protein . Transcriptional repression of the @Regulated$ gene of Escherichia coli by @TF$ protein .\n",
      "4\n",
      "10\n",
      "Start y end del regulador: [67, 71]\n",
      "Regulador: DnaA\n",
      "Start y end del regulado: [34, 38]\n",
      "Regulado: dnaA\n",
      "Interdependence of activation at rhaSR by cyclic AMP receptor protein , the RNA polymerase alpha subunit C - terminal domain and RhaR . Interdependence of activation at @Regulated$ by cyclic AMP receptor protein , the RNA polymerase alpha subunit C - terminal domain and @TF$ .\n",
      "4\n",
      "21\n",
      "Start y end del regulador: [129, 133]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [33, 38]\n",
      "Regulado: rhaSR\n",
      "( 2001 ) DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N - acetylglucosamine specific transporter in Escherichia coli . ( 2001 ) DNA binding sites for the @TF$ and NagC proteins : regulation of @Regulated$ , encoding the N - acetylglucosamine specific transporter in Escherichia coli .\n",
      "15\n",
      "8\n",
      "Start y end del regulador: [35, 38]\n",
      "Regulador: Mlc\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: nagE\n",
      "( 2001 ) DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N - acetylglucosamine specific transporter in Escherichia coli . ( 2001 ) DNA binding sites for the Mlc and @TF$ proteins : regulation of @Regulated$ , encoding the N - acetylglucosamine specific transporter in Escherichia coli .\n",
      "15\n",
      "10\n",
      "Start y end del regulador: [43, 47]\n",
      "Regulador: NagC\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: nagE\n",
      "Upstream sequence elements required for NarL - mediated activation of transcription from the narGHJI promoter of Escherichia coli . Upstream sequence elements required for @TF$ - mediated activation of transcription from the @Regulated$ promoter of Escherichia coli .\n",
      "13\n",
      "5\n",
      "Start y end del regulador: [40, 44]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [93, 100]\n",
      "Regulado: narGHJI\n",
      "( 1999 ) Lrp binds to two regions in the dadAX promoter region of Escherichia coli to repress and activate transcription directly . ( 1999 ) @TF$ binds to two regions in the @Regulated$ promoter region of Escherichia coli to repress and activate transcription directly .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [9, 12]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [41, 46]\n",
      "Regulado: dadAX\n",
      "( 1999 ) Lrp binds to two regions in the dadAX promoter region of Escherichia coli to repress and activate transcription directly . ( 1999 ) @TF$ binds to two regions in the @Regulated$ promoter region of Escherichia coli to repress and activate transcription directly .\n",
      "10\n",
      "3\n",
      "Start y end del regulador: [9, 12]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [41, 46]\n",
      "Regulado: dadAX\n",
      "Separate contributions of UhpA and CAP to activation of transcription of the uhpT promoter of Escherichia coli . Separate contributions of UhpA and @TF$ to activation of transcription of the @Regulated$ promoter of Escherichia coli .\n",
      "12\n",
      "5\n",
      "Start y end del regulador: [35, 38]\n",
      "Regulador: CAP\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: uhpT\n",
      "Separate contributions of UhpA and CAP to activation of transcription of the uhpT promoter of Escherichia coli . Separate contributions of @TF$ and CAP to activation of transcription of the @Regulated$ promoter of Escherichia coli .\n",
      "12\n",
      "3\n",
      "Start y end del regulador: [26, 30]\n",
      "Regulador: UhpA\n",
      "Start y end del regulado: [77, 81]\n",
      "Regulado: uhpT\n",
      "Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop . Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop .\n",
      "5\n",
      "8\n",
      "Start y end del regulador: [52, 56]\n",
      "Regulador: MelR\n",
      "Start y end del regulado: [35, 39]\n",
      "Regulado: melR\n",
      "DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli deoCABC promoters by the DeoR repressor . DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli @Regulated$ promoters by the @TF$ repressor .\n",
      "19\n",
      "23\n",
      "Start y end del regulador: [170, 174]\n",
      "Regulador: DeoR\n",
      "Start y end del regulado: [145, 152]\n",
      "Regulado: deoCABC\n",
      "CadC - mediated activation of the cadBA promoter in Escherichia coli . @TF$ - mediated activation of the @Regulated$ promoter in Escherichia coli .\n",
      "6\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: CadC\n",
      "Start y end del regulado: [34, 39]\n",
      "Regulado: cadBA\n",
      "The H - NS protein modulates the activation of the ilvIH operon of Escherichia coli K12 by Lrp , the leucine regulatory protein . The H - NS protein modulates the activation of the @Regulated$ operon of Escherichia coli K12 by @TF$ , the leucine regulatory protein .\n",
      "10\n",
      "17\n",
      "Start y end del regulador: [91, 94]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [51, 56]\n",
      "Regulado: ilvIH\n",
      "Auto - activation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli . Auto - activation of the @Regulated$ multiple antibiotic resistance operon by the @TF$ transcriptional activator in Escherichia coli .\n",
      "5\n",
      "12\n",
      "Start y end del regulador: [77, 81]\n",
      "Regulador: MarA\n",
      "Start y end del regulado: [25, 31]\n",
      "Regulado: marRAB\n",
      "NarL - phosphate must bind to multiple upstream sites to activate transcription from the narG promoter of Escherichia coli . @TF$ - phosphate must bind to multiple upstream sites to activate transcription from the @Regulated$ promoter of Escherichia coli .\n",
      "14\n",
      "0\n",
      "Start y end del regulador: [0, 4]\n",
      "Regulador: NarL\n",
      "Start y end del regulado: [89, 93]\n",
      "Regulado: narG\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of @TF$ and FNR when bound in tandem .\n",
      "6\n",
      "15\n",
      "Start y end del regulador: [97, 100]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "Transcriptional co - activation at the ansB promoters : involvement of the activating regions of CRP and FNR when bound in tandem . Transcriptional co - activation at the @Regulated$ promoters : involvement of the activating regions of CRP and @TF$ when bound in tandem .\n",
      "6\n",
      "17\n",
      "Start y end del regulador: [105, 108]\n",
      "Regulador: FNR\n",
      "Start y end del regulado: [39, 43]\n",
      "Regulado: ansB\n",
      "Identification of the DNAbinding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting . Identification of the DNAbinding domain of the @TF$ protein required for transcriptional activation of the ompF and @Regulated$ genes of Escherichia coli by in vivo DNA footprinting .\n",
      "17\n",
      "7\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [116, 120]\n",
      "Regulado: ompC\n",
      "Identification of the DNAbinding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting . Identification of the DNAbinding domain of the @TF$ protein required for transcriptional activation of the @Regulated$ and ompC genes of Escherichia coli by in vivo DNA footprinting .\n",
      "15\n",
      "7\n",
      "Start y end del regulador: [47, 51]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [107, 111]\n",
      "Regulado: ompF\n",
      "Competitive activation of the Escherichia coli argO gene coding for an arginine exporter by the transcriptional regulators Lrp and ArgP . Competitive activation of the Escherichia coli @Regulated$ gene coding for an arginine exporter by the transcriptional regulators Lrp and @TF$ .\n",
      "6\n",
      "19\n",
      "Start y end del regulador: [131, 135]\n",
      "Regulador: ArgP\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Competitive activation of the Escherichia coli argO gene coding for an arginine exporter by the transcriptional regulators Lrp and ArgP . Competitive activation of the Escherichia coli @Regulated$ gene coding for an arginine exporter by the transcriptional regulators @TF$ and ArgP .\n",
      "6\n",
      "17\n",
      "Start y end del regulador: [123, 126]\n",
      "Regulador: Lrp\n",
      "Start y end del regulado: [47, 51]\n",
      "Regulado: argO\n",
      "Cyclic AMP receptor protein and RhaR synergistically activate transcription from the L - rhamnose - responsive rhaSR promoter in Escherichia coli . Cyclic AMP receptor protein and @TF$ synergistically activate transcription from the L - rhamnose - responsive @Regulated$ promoter in Escherichia coli .\n",
      "16\n",
      "5\n",
      "Start y end del regulador: [32, 36]\n",
      "Regulador: RhaR\n",
      "Start y end del regulado: [111, 116]\n",
      "Regulado: rhaSR\n",
      "Fis activates the RpoS - dependent stationary - phase expression of proP in Escherichia coli . Fis activates the @TF$ - dependent stationary - phase expression of @Regulated$ in Escherichia coli .\n",
      "11\n",
      "3\n",
      "Start y end del regulador: [18, 22]\n",
      "Regulador: RpoS\n",
      "Start y end del regulado: [68, 72]\n",
      "Regulado: proP\n",
      "Characterization of OmpR binding sequences in the upstream region of the ompF promoter essential for transcriptional activation . Characterization of @TF$ binding sequences in the upstream region of the @Regulated$ promoter essential for transcriptional activation .\n",
      "11\n",
      "2\n",
      "Start y end del regulador: [20, 24]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [73, 77]\n",
      "Regulado: ompF\n",
      "Identification of the DNA - binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting . Identification of the DNA - binding domain of the @TF$ protein required for transcriptional activation of the ompF and @Regulated$ genes of Escherichia coli by in vivo DNA footprinting .\n",
      "19\n",
      "9\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [119, 123]\n",
      "Regulado: ompC\n",
      "Identification of the DNA - binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting . Identification of the DNA - binding domain of the @TF$ protein required for transcriptional activation of the @Regulated$ and ompC genes of Escherichia coli by in vivo DNA footprinting .\n",
      "17\n",
      "9\n",
      "Start y end del regulador: [50, 54]\n",
      "Regulador: OmpR\n",
      "Start y end del regulado: [110, 114]\n",
      "Regulado: ompF\n",
      "DNA - protein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli deoCABD promoters by the DeoR repressor . DNA - protein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli @Regulated$ promoters by the @TF$ repressor .\n",
      "21\n",
      "25\n",
      "Start y end del regulador: [173, 177]\n",
      "Regulador: DeoR\n",
      "Start y end del regulado: [148, 155]\n",
      "Regulado: deoCABD\n",
      "CRP / cAMP and CytR regulated promoters in Escherichia coli K12 : the cdd promoter . @TF$ / cAMP and CytR regulated promoters in Escherichia coli K12 : the @Regulated$ promoter .\n",
      "13\n",
      "0\n",
      "Start y end del regulador: [0, 3]\n",
      "Regulador: CRP\n",
      "Start y end del regulado: [70, 73]\n",
      "Regulado: cdd\n",
      "CRP / cAMP and CytR regulated promoters in Escherichia coli K12 : the cdd promoter . CRP / cAMP and @TF$ regulated promoters in Escherichia coli K12 : the @Regulated$ promoter .\n",
      "13\n",
      "4\n",
      "Start y end del regulador: [15, 19]\n",
      "Regulador: CytR\n",
      "Start y end del regulado: [70, 73]\n",
      "Regulado: cdd\n"
     ]
    }
   ],
   "source": [
    "span_list = []\n",
    "regulator_list = []\n",
    "regulated_list = []\n",
    "for (sentence_o, sentence_t) in zip(data[\"SENTENCE\"], data[\"sentence_tagged\"]):\n",
    "    print(sentence_o, sentence_t)\n",
    "    list_sent_o = sentence_o.split()\n",
    "    list_sent_t = sentence_t.split()\n",
    "    regulator_pos = list_sent_t.index(\"@TF$\")\n",
    "    regulated_pos = list_sent_t.index(\"@Regulated$\")\n",
    "    print(regulated_pos)\n",
    "    print(regulator_pos)\n",
    "    # Get regulator TF idx inside the sentence \n",
    "    span_regulator= get_start_end(regulator_pos, list_sent_o)\n",
    "    start_regulator = span_regulator[0]\n",
    "    end_regulator = span_regulator[1]\n",
    "    print(\"Start y end del regulador: {}\".format(span_regulator))\n",
    "    print(\"Regulador: {}\".format(sentence_o[start_regulator:end_regulator]))\n",
    "    # Get regulated idx inside the sentence\n",
    "    span_regulated = get_start_end(regulated_pos, list_sent_o)\n",
    "    start_regulated = span_regulated[0]\n",
    "    end_regulated = span_regulated[1]\n",
    "    print(\"Start y end del regulado: {}\".format(span_regulated))\n",
    "    print(\"Regulado: {}\".format(sentence_o[start_regulated:end_regulated]))\n",
    "    span_list.append([span_regulator, span_regulated])\n",
    "    regulator_list.append(sentence_o[start_regulator:end_regulator])\n",
    "    regulated_list.append(sentence_o[start_regulated:end_regulated])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[[168, 171], [73, 77]], [[252, 255], [73, 77]], [[43, 46], [113, 118]], [[171, 174], [113, 118]], [[34, 38], [123, 127]], [[198, 202], [123, 127]], [[54, 57], [92, 96]], [[121, 124], [92, 96]], [[34, 38], [141, 145]], [[198, 202], [141, 145]], [[34, 38], [130, 134]], [[198, 202], [130, 134]], [[202, 206], [250, 255]], [[146, 150], [250, 255]], [[24, 27], [117, 123]], [[156, 159], [117, 123]], [[96, 99], [26, 32]], [[207, 213], [26, 32]], [[136, 140], [26, 32]], [[197, 201], [26, 32]], [[113, 117], [34, 38]], [[172, 176], [34, 38]], [[128, 132], [217, 222]], [[0, 4], [217, 222]], [[69, 73], [104, 108]], [[145, 149], [104, 108]], [[165, 169], [104, 108]], [[84, 87], [124, 130]], [[57, 60], [124, 130]], [[32, 36], [0, 4]], [[78, 82], [0, 4]], [[84, 87], [133, 137]], [[57, 60], [133, 137]], [[84, 87], [172, 176]], [[57, 60], [172, 176]], [[84, 87], [117, 121]], [[57, 60], [117, 121]], [[84, 87], [147, 151]], [[57, 60], [147, 151]], [[84, 87], [140, 144]], [[57, 60], [140, 144]], [[84, 87], [161, 165]], [[57, 60], [161, 165]], [[84, 87], [154, 158]], [[57, 60], [154, 158]], [[118, 122], [37, 40]], [[162, 166], [37, 40]], [[118, 122], [29, 32]], [[162, 166], [29, 32]], [[102, 105], [138, 144]], [[19, 22], [138, 144]], [[219, 222], [138, 144]], [[27, 30], [70, 74]], [[148, 151], [70, 74]], [[27, 30], [88, 92]], [[148, 151], [88, 92]], [[66, 70], [117, 122]], [[185, 189], [117, 122]], [[130, 134], [14, 19]], [[141, 145], [14, 19]], [[213, 217], [14, 19]], [[43, 47], [190, 194]], [[229, 233], [190, 194]], [[43, 47], [85, 89]], [[229, 233], [85, 89]], [[43, 47], [76, 80]], [[229, 233], [76, 80]], [[43, 47], [126, 130]], [[229, 233], [126, 130]], [[229, 233], [181, 185]], [[43, 47], [181, 185]], [[141, 145], [160, 164]], [[18, 22], [160, 164]], [[24, 27], [40, 46]], [[126, 129], [40, 46]], [[181, 184], [40, 46]], [[7, 11], [137, 142]], [[241, 245], [137, 142]], [[7, 11], [119, 123]], [[241, 245], [119, 123]], [[241, 245], [205, 208]], [[7, 11], [205, 208]], [[7, 11], [105, 109]], [[241, 245], [105, 109]], [[7, 11], [163, 167]], [[241, 245], [163, 167]], [[7, 11], [69, 73]], [[41, 45], [69, 73]], [[52, 56], [69, 73]], [[68, 71], [38, 44]], [[130, 133], [38, 44]], [[201, 205], [231, 239]], [[248, 252], [231, 239]], [[151, 155], [78, 82]], [[0, 4], [78, 82]], [[297, 301], [284, 288]], [[81, 85], [284, 288]], [[47, 51], [17, 21]], [[90, 94], [17, 21]], [[159, 163], [191, 195]], [[109, 113], [191, 195]], [[159, 163], [184, 188]], [[109, 113], [184, 188]], [[159, 163], [198, 202]], [[109, 113], [198, 202]], [[159, 163], [205, 209]], [[109, 113], [205, 209]], [[159, 163], [212, 216]], [[109, 113], [212, 216]], [[159, 163], [219, 223]], [[109, 113], [219, 223]], [[159, 163], [230, 234]], [[109, 113], [230, 234]], [[159, 163], [301, 305]], [[109, 113], [301, 305]], [[159, 163], [312, 316]], [[109, 113], [312, 316]], [[93, 97], [20, 24]], [[271, 275], [20, 24]], [[159, 163], [294, 298]], [[109, 113], [294, 298]], [[93, 97], [50, 56]], [[144, 148], [50, 56]], [[102, 106], [150, 155]], [[74, 78], [150, 155]], [[170, 174], [145, 149]], [[94, 98], [145, 149]], [[29, 33], [75, 79]], [[135, 139], [75, 79]], [[146, 150], [75, 79]], [[153, 157], [133, 137]], [[42, 46], [133, 137]], [[155, 159], [194, 198]], [[18, 22], [194, 198]], [[68, 72], [11, 15]], [[57, 61], [11, 15]], [[122, 126], [143, 147]], [[103, 107], [143, 147]], [[126, 130], [188, 191]], [[229, 233], [188, 191]], [[130, 134], [49, 53]], [[203, 207], [49, 53]], [[130, 134], [40, 44]], [[203, 207], [40, 44]], [[82, 86], [117, 121]], [[33, 37], [117, 121]], [[21, 25], [43, 47]], [[132, 136], [43, 47]], [[74, 78], [150, 154]], [[54, 58], [150, 154]], [[83, 87], [250, 254]], [[145, 149], [250, 254]], [[165, 169], [250, 254]], [[345, 349], [250, 254]], [[372, 376], [250, 254]], [[119, 122], [163, 167]], [[37, 40], [163, 167]], [[118, 121], [145, 149]], [[35, 38], [145, 149]], [[296, 299], [145, 149]], [[236, 240], [219, 223]], [[143, 147], [219, 223]], [[38, 42], [53, 56]], [[220, 224], [53, 56]], [[178, 181], [146, 150]], [[62, 65], [146, 150]], [[23, 27], [53, 57]], [[99, 103], [53, 57]], [[69, 72], [161, 165]], [[273, 276], [161, 165]], [[90, 94], [115, 118]], [[186, 190], [115, 118]], [[390, 394], [115, 118]], [[90, 94], [121, 125]], [[186, 190], [121, 125]], [[390, 394], [121, 125]], [[90, 94], [128, 132]], [[186, 190], [128, 132]], [[390, 394], [128, 132]], [[90, 94], [135, 139]], [[186, 190], [135, 139]], [[390, 394], [135, 139]], [[90, 94], [142, 146]], [[186, 190], [142, 146]], [[390, 394], [142, 146]], [[90, 94], [149, 153]], [[186, 190], [149, 153]], [[390, 394], [149, 153]], [[90, 94], [160, 164]], [[186, 190], [160, 164]], [[390, 394], [160, 164]], [[136, 140], [0, 4]], [[107, 111], [0, 4]], [[107, 111], [142, 146]], [[15, 19], [142, 146]], [[68, 72], [142, 146]], [[149, 153], [142, 146]], [[184, 188], [142, 146]], [[232, 236], [142, 146]], [[239, 243], [142, 146]], [[19, 22], [33, 37]], [[63, 66], [33, 37]], [[115, 118], [166, 172]], [[93, 96], [166, 172]], [[49, 53], [98, 102]], [[29, 33], [98, 102]], [[321, 325], [363, 367]], [[173, 177], [363, 367]], [[221, 225], [363, 367]], [[241, 245], [226, 229]], [[41, 45], [226, 229]], [[233, 237], [226, 229]], [[34, 38], [226, 229]], [[88, 91], [139, 144]], [[193, 196], [139, 144]], [[71, 75], [83, 86]], [[212, 216], [83, 86]], [[74, 78], [100, 104]], [[184, 188], [100, 104]], [[224, 228], [100, 104]], [[103, 107], [243, 248]], [[42, 46], [243, 248]], [[130, 133], [165, 170]], [[32, 35], [165, 170]], [[71, 75], [151, 155]], [[24, 28], [151, 155]], [[71, 75], [189, 193]], [[24, 28], [189, 193]], [[248, 252], [282, 286]], [[183, 187], [282, 286]], [[17, 21], [51, 55]], [[114, 118], [51, 55]], [[79, 83], [136, 140]], [[218, 222], [136, 140]], [[218, 222], [174, 180]], [[79, 83], [174, 180]], [[257, 261], [174, 180]], [[110, 114], [174, 180]], [[79, 83], [42, 49]], [[218, 222], [42, 49]], [[188, 191], [140, 144]], [[242, 245], [140, 144]], [[24, 28], [75, 79]], [[189, 193], [75, 79]], [[130, 133], [146, 150]], [[46, 49], [146, 150]], [[166, 170], [144, 148]], [[85, 89], [144, 148]], [[116, 119], [92, 97]], [[163, 166], [92, 97]], [[49, 53], [138, 142]], [[29, 33], [138, 142]], [[29, 32], [46, 50]], [[89, 92], [46, 50]], [[70, 73], [88, 93]], [[19, 22], [88, 93]], [[34, 38], [76, 80]], [[217, 221], [76, 80]], [[34, 38], [69, 73]], [[217, 221], [69, 73]], [[34, 38], [83, 87]], [[217, 221], [83, 87]], [[34, 38], [90, 94]], [[217, 221], [90, 94]], [[34, 38], [97, 101]], [[217, 221], [97, 101]], [[137, 141], [154, 158]], [[62, 66], [154, 158]], [[10, 14], [45, 49]], [[91, 95], [45, 49]], [[61, 65], [105, 109]], [[23, 27], [105, 109]], [[14, 18], [105, 109]], [[199, 203], [105, 109]], [[179, 183], [199, 203]], [[238, 242], [199, 203]], [[30, 34], [67, 71]], [[162, 166], [67, 71]], [[357, 361], [300, 305]], [[11, 15], [300, 305]], [[111, 115], [300, 305]], [[208, 212], [300, 305]], [[95, 99], [86, 91]], [[14, 18], [86, 91]], [[81, 85], [215, 219]], [[193, 197], [215, 219]], [[17, 21], [52, 56]], [[174, 178], [52, 56]], [[26, 30], [52, 56]], [[101, 105], [52, 56]], [[169, 173], [209, 214]], [[126, 130], [209, 214]], [[69, 73], [105, 109]], [[33, 37], [105, 109]], [[162, 166], [105, 109]], [[66, 70], [11, 15]], [[57, 61], [11, 15]], [[128, 131], [158, 162]], [[47, 50], [158, 162]], [[218, 221], [158, 162]], [[221, 224], [264, 269]], [[15, 18], [264, 269]], [[119, 122], [264, 269]], [[213, 217], [264, 269]], [[126, 130], [264, 269]], [[270, 274], [325, 328]], [[153, 157], [325, 328]], [[31, 35], [50, 55]], [[173, 177], [50, 55]], [[84, 87], [102, 107]], [[42, 45], [102, 107]], [[31, 34], [61, 66]], [[131, 134], [61, 66]], [[89, 93], [53, 59]], [[166, 170], [53, 59]], [[118, 122], [153, 157]], [[83, 87], [153, 157]], [[226, 230], [153, 157]], [[40, 43], [18, 22]], [[126, 129], [18, 22]], [[126, 129], [18, 22]], [[40, 43], [18, 22]], [[27, 30], [38, 42]], [[139, 142], [38, 42]], [[121, 124], [21, 27]], [[263, 266], [21, 27]], [[93, 96], [129, 133]], [[167, 170], [129, 133]], [[188, 192], [145, 151]], [[115, 119], [145, 151]], [[320, 324], [232, 236]], [[115, 119], [232, 236]], [[320, 324], [223, 227]], [[115, 119], [223, 227]], [[245, 249], [212, 220]], [[134, 138], [212, 220]], [[65, 68], [92, 98]], [[200, 203], [92, 98]], [[88, 91], [49, 53]], [[127, 130], [49, 53]], [[219, 223], [200, 208]], [[373, 377], [200, 208]], [[228, 231], [200, 208]], [[271, 274], [200, 208]], [[183, 187], [202, 207]], [[139, 143], [202, 207]], [[0, 4], [34, 38]], [[63, 67], [34, 38]], [[19, 22], [52, 56]], [[128, 131], [52, 56]], [[35, 39], [52, 56]], [[66, 70], [52, 56]], [[62, 66], [19, 23]], [[134, 138], [19, 23]], [[62, 66], [10, 14]], [[134, 138], [10, 14]], [[19, 23], [31, 37]], [[281, 285], [31, 37]], [[234, 238], [206, 212]], [[62, 66], [206, 212]], [[20, 23], [81, 86]], [[134, 137], [81, 86]], [[62, 66], [34, 38]], [[234, 238], [34, 38]], [[59, 63], [23, 26]], [[137, 141], [23, 26]], [[137, 141], [196, 200]], [[59, 63], [196, 200]], [[137, 141], [189, 193]], [[59, 63], [189, 193]], [[65, 69], [0, 3]], [[95, 99], [0, 3]], [[170, 174], [205, 209]], [[31, 35], [205, 209]], [[74, 78], [95, 98]], [[13, 17], [95, 98]], [[41, 45], [64, 69]], [[150, 154], [64, 69]], [[177, 180], [132, 136]], [[68, 71], [132, 136]], [[186, 190], [132, 136]], [[101, 105], [132, 136]], [[15, 19], [41, 44]], [[213, 217], [41, 44]], [[230, 233], [248, 254]], [[44, 47], [248, 254]], [[52, 56], [4, 9]], [[102, 106], [4, 9]], [[72, 75], [79, 84]], [[201, 204], [79, 84]], [[18, 21], [36, 41]], [[80, 83], [36, 41]], [[128, 132], [162, 166]], [[77, 81], [162, 166]], [[4, 8], [28, 32]], [[118, 122], [28, 32]], [[118, 122], [142, 146]], [[4, 8], [142, 146]], [[109, 113], [69, 72]], [[201, 205], [69, 72]], [[90, 94], [105, 109]], [[347, 351], [105, 109]], [[9, 12], [50, 54]], [[103, 106], [50, 54]], [[182, 185], [204, 208]], [[21, 24], [204, 208]], [[22, 26], [86, 90]], [[131, 135], [86, 90]], [[234, 238], [160, 165]], [[234, 238], [85, 90]], [[129, 133], [163, 167]], [[129, 133], [60, 64]], [[99, 103], [192, 196]], [[99, 103], [122, 126]], [[30, 34], [57, 61]], [[30, 34], [134, 138]], [[46, 49], [57, 61]], [[46, 49], [134, 138]], [[37, 41], [57, 61]], [[37, 41], [134, 138]], [[14, 17], [67, 71]], [[14, 17], [394, 398]], [[14, 17], [429, 433]], [[14, 17], [465, 469]], [[14, 17], [74, 78]], [[14, 17], [402, 406]], [[14, 17], [55, 58]], [[14, 17], [170, 173]], [[14, 17], [382, 385]], [[14, 17], [421, 424]], [[14, 17], [457, 460]], [[29, 32], [135, 139]], [[29, 32], [70, 74]], [[37, 41], [135, 139]], [[37, 41], [70, 74]], [[22, 26], [135, 139]], [[22, 26], [70, 74]], [[19, 23], [73, 77]], [[19, 23], [317, 321]], [[12, 16], [73, 77]], [[12, 16], [317, 321]], [[19, 23], [80, 84]], [[19, 23], [325, 329]], [[12, 16], [80, 84]], [[12, 16], [325, 329]], [[19, 23], [61, 64]], [[19, 23], [158, 161]], [[19, 23], [61, 64]], [[12, 16], [61, 64]], [[12, 16], [158, 161]], [[12, 16], [305, 308]], [[226, 230], [158, 162]], [[226, 230], [3, 7]], [[226, 230], [169, 173]], [[226, 230], [12, 16]], [[125, 129], [184, 188]], [[125, 129], [14, 18]], [[93, 96], [212, 217]], [[93, 96], [26, 31]], [[126, 130], [212, 217]], [[126, 130], [26, 31]], [[71, 75], [135, 139]], [[71, 75], [410, 414]], [[53, 57], [46, 49]], [[53, 57], [146, 149]], [[98, 101], [5, 9]], [[98, 101], [241, 245]], [[0, 4], [32, 37]], [[0, 4], [96, 101]], [[74, 77], [159, 163]], [[74, 77], [249, 253]], [[33, 37], [60, 66]], [[33, 37], [132, 138]], [[148, 152], [187, 191]], [[148, 152], [73, 77]], [[32, 36], [72, 78]], [[32, 36], [181, 187]], [[18, 22], [57, 61]], [[18, 22], [123, 127]], [[52, 56], [32, 36]], [[52, 56], [128, 132]], [[52, 56], [269, 273]], [[0, 4], [30, 34]], [[0, 4], [200, 204]], [[291, 295], [307, 311]], [[291, 295], [87, 91]], [[61, 69], [27, 31]], [[56, 60], [153, 157]], [[177, 178], [27, 31]], [[177, 178], [158, 168]], [[33, 37], [91, 95]], [[33, 37], [128, 132]], [[47, 51], [91, 95]], [[47, 51], [128, 132]], [[227, 231], [262, 266]], [[227, 231], [89, 93]], [[227, 231], [271, 275]], [[227, 231], [96, 100]], [[98, 102], [133, 137]], [[98, 102], [51, 55]], [[211, 215], [239, 243]], [[211, 215], [307, 311]], [[220, 224], [239, 243]], [[220, 224], [307, 311]], [[27, 31], [53, 57]], [[27, 31], [111, 115]], [[11, 15], [42, 46]], [[11, 15], [169, 173]], [[11, 15], [219, 223]], [[38, 42], [86, 90]], [[38, 42], [199, 203]], [[8, 12], [43, 47]], [[8, 12], [119, 123]], [[20, 23], [86, 90]], [[20, 23], [31, 35]], [[355, 358], [235, 237]], [[355, 358], [282, 292]], [[364, 365], [235, 237]], [[364, 365], [282, 292]], [[36, 39], [58, 64]], [[36, 39], [110, 116]], [[101, 105], [136, 140]], [[101, 105], [18, 22]], [[32, 35], [156, 169]], [[32, 35], [60, 64]], [[179, 183], [59, 62]], [[179, 183], [96, 99]], [[170, 174], [59, 62]], [[170, 174], [96, 99]], [[56, 60], [91, 96]], [[56, 60], [146, 151]], [[20, 24], [75, 79]], [[20, 24], [191, 195]], [[105, 109], [70, 74]], [[105, 109], [200, 204]], [[105, 109], [277, 281]], [[105, 109], [79, 83]], [[105, 109], [291, 295]], [[19, 23], [34, 38]], [[19, 23], [89, 93]], [[19, 23], [220, 224]], [[11, 14], [34, 38]], [[11, 14], [89, 93]], [[11, 14], [220, 224]], [[94, 98], [51, 55]], [[94, 98], [164, 168]], [[94, 98], [189, 193]], [[109, 113], [164, 168]], [[109, 113], [51, 55]], [[109, 113], [189, 193]], [[6, 10], [53, 57]], [[6, 10], [118, 122]], [[51, 54], [264, 268]], [[51, 54], [211, 215]], [[51, 54], [302, 306]], [[51, 54], [202, 206]], [[141, 144], [83, 87]], [[141, 144], [160, 164]], [[48, 52], [4, 8]], [[48, 52], [94, 98]], [[207, 211], [173, 176]], [[207, 211], [96, 99]], [[189, 193], [204, 208]], [[189, 193], [92, 96]], [[96, 100], [128, 132]], [[96, 100], [80, 84]], [[147, 151], [186, 190]], [[147, 151], [40, 44]], [[38, 42], [221, 225]], [[38, 42], [65, 69]], [[227, 231], [98, 103]], [[227, 231], [155, 160]], [[66, 69], [214, 219]], [[66, 69], [101, 106]], [[156, 160], [246, 250]], [[156, 160], [42, 46]], [[277, 281], [42, 46]], [[277, 281], [246, 250]], [[24, 27], [60, 64]], [[24, 27], [121, 125]], [[16, 19], [60, 64]], [[16, 19], [121, 125]], [[49, 53], [102, 107]], [[49, 53], [219, 224]], [[0, 4], [76, 79]], [[0, 4], [27, 30]], [[0, 4], [88, 91]], [[63, 66], [28, 32]], [[63, 66], [120, 124]], [[63, 66], [19, 23]], [[63, 66], [86, 90]], [[161, 164], [190, 195]], [[161, 164], [44, 49]], [[29, 33], [120, 124]], [[29, 33], [49, 53]], [[38, 42], [132, 136]], [[38, 42], [58, 62]], [[29, 33], [49, 53]], [[29, 33], [120, 124]], [[122, 126], [21, 25]], [[122, 126], [155, 159]], [[50, 54], [72, 76]], [[0, 4], [152, 156]], [[99, 102], [17, 21]], [[99, 102], [168, 172]], [[118, 121], [17, 21]], [[118, 121], [168, 172]], [[0, 4], [75, 79]], [[0, 4], [178, 181]], [[91, 96], [178, 181]], [[91, 96], [75, 79]], [[41, 44], [88, 94]], [[41, 44], [124, 130]], [[196, 199], [88, 94]], [[196, 199], [124, 130]], [[27, 30], [77, 81]], [[27, 30], [223, 227]], [[148, 151], [223, 227]], [[148, 151], [77, 81]], [[130, 134], [5, 9]], [[130, 134], [241, 245]], [[141, 145], [5, 9]], [[141, 145], [241, 245]], [[213, 217], [5, 9]], [[213, 217], [241, 245]], [[70, 74], [100, 104]], [[8, 12], [47, 51]], [[8, 12], [100, 104]], [[8, 12], [153, 157]], [[8, 12], [222, 226]], [[70, 74], [47, 51]], [[70, 74], [153, 157]], [[70, 74], [222, 226]], [[71, 75], [51, 54]], [[71, 75], [110, 113]], [[256, 260], [51, 54]], [[256, 260], [110, 113]], [[80, 84], [51, 54]], [[80, 84], [110, 113]], [[265, 269], [51, 54]], [[265, 269], [110, 113]], [[61, 65], [97, 101]], [[26, 30], [97, 101]], [[26, 30], [219, 223]], [[61, 65], [219, 223]], [[199, 203], [97, 101]], [[199, 203], [219, 223]], [[16, 20], [109, 113]], [[95, 98], [109, 114]], [[28, 31], [45, 50]], [[28, 31], [109, 114]], [[95, 98], [45, 50]], [[9, 13], [37, 41]], [[9, 13], [95, 99]], [[4, 8], [48, 52]], [[39, 43], [213, 217]], [[100, 104], [120, 124]], [[27, 31], [47, 51]], [[27, 31], [120, 124]], [[100, 104], [47, 51]], [[352, 356], [314, 318]], [[409, 413], [81, 85]], [[409, 413], [198, 202]], [[409, 413], [314, 318]], [[409, 413], [439, 443]], [[352, 356], [81, 85]], [[352, 356], [198, 202]], [[352, 356], [439, 443]], [[360, 363], [320, 324]], [[34, 37], [54, 58]], [[34, 37], [187, 191]], [[34, 37], [320, 324]], [[360, 363], [54, 58]], [[360, 363], [187, 191]], [[216, 220], [282, 286]], [[74, 78], [100, 104]], [[74, 78], [282, 286]], [[181, 185], [100, 104]], [[181, 185], [282, 286]], [[216, 220], [100, 104]], [[140, 144], [84, 88]], [[140, 144], [310, 314]], [[177, 181], [84, 88]], [[177, 181], [310, 314]], [[221, 225], [84, 88]], [[221, 225], [310, 314]], [[242, 246], [84, 88]], [[242, 246], [310, 314]], [[228, 232], [262, 268]], [[180, 184], [161, 167]], [[180, 184], [262, 268]], [[228, 232], [161, 167]], [[112, 115], [170, 175]], [[21, 24], [69, 74]], [[21, 24], [170, 175]], [[112, 115], [69, 74]], [[261, 264], [272, 276]], [[80, 83], [91, 95]], [[80, 83], [145, 149]], [[80, 83], [272, 276]], [[137, 140], [91, 95]], [[137, 140], [145, 149]], [[137, 140], [272, 276]], [[261, 264], [91, 95]], [[261, 264], [145, 149]], [[90, 94], [47, 51]], [[90, 94], [291, 295]], [[186, 190], [47, 51]], [[186, 190], [291, 295]], [[390, 394], [47, 51]], [[390, 394], [291, 295]], [[43, 47], [58, 62]], [[43, 47], [279, 283]], [[169, 173], [58, 62]], [[169, 173], [279, 283]], [[240, 243], [200, 204]], [[392, 395], [56, 62]], [[392, 395], [189, 195]], [[240, 243], [56, 62]], [[240, 243], [200, 204]], [[240, 243], [125, 129]], [[240, 243], [353, 357]], [[392, 395], [125, 129]], [[392, 395], [200, 204]], [[392, 395], [353, 357]], [[0, 3], [18, 21]], [[0, 3], [103, 106]], [[0, 3], [136, 139]], [[92, 95], [18, 21]], [[92, 95], [103, 106]], [[92, 95], [136, 139]], [[187, 190], [221, 226]], [[137, 140], [106, 111]], [[137, 140], [221, 226]], [[187, 190], [106, 111]], [[103, 107], [71, 75]], [[103, 107], [172, 176]], [[154, 158], [71, 75]], [[154, 158], [172, 176]], [[22, 26], [54, 59]], [[22, 26], [162, 167]], [[122, 126], [54, 59]], [[122, 126], [162, 167]], [[211, 215], [263, 267]], [[52, 56], [69, 73]], [[52, 56], [263, 267]], [[211, 215], [69, 73]], [[44, 47], [69, 73]], [[44, 47], [263, 267]], [[285, 288], [69, 73]], [[285, 288], [263, 267]], [[22, 26], [15, 19]], [[22, 26], [207, 211]], [[224, 228], [15, 19]], [[224, 228], [207, 211]], [[135, 139], [186, 190]], [[135, 139], [52, 56]], [[135, 139], [333, 337]], [[246, 250], [52, 56]], [[246, 250], [186, 190]], [[246, 250], [333, 337]], [[104, 108], [119, 122]], [[59, 63], [27, 30]], [[59, 63], [119, 122]], [[104, 108], [27, 30]], [[21, 25], [45, 49]], [[21, 25], [99, 103]], [[87, 91], [45, 49]], [[87, 91], [99, 103]], [[61, 65], [97, 101]], [[61, 65], [160, 164]], [[61, 65], [223, 227]], [[135, 139], [97, 101]], [[135, 139], [160, 164]], [[135, 139], [223, 227]], [[199, 203], [97, 101]], [[199, 203], [160, 164]], [[199, 203], [223, 227]], [[105, 109], [144, 148]], [[105, 109], [242, 246]], [[158, 162], [144, 148]], [[158, 162], [242, 246]], [[0, 4], [39, 43]], [[0, 4], [166, 170]], [[210, 214], [39, 43]], [[210, 214], [166, 170]], [[103, 107], [52, 56]], [[103, 107], [197, 201]], [[103, 107], [457, 461]], [[103, 107], [505, 509]], [[103, 107], [630, 634]], [[392, 396], [52, 56]], [[392, 396], [197, 201]], [[392, 396], [457, 461]], [[392, 396], [505, 509]], [[392, 396], [630, 634]], [[8, 11], [33, 38]], [[8, 11], [144, 149]], [[114, 117], [33, 38]], [[114, 117], [144, 149]], [[155, 159], [163, 168]], [[155, 159], [310, 315]], [[288, 292], [310, 315]], [[288, 292], [163, 168]], [[11, 15], [26, 30]], [[11, 15], [195, 199]], [[11, 15], [255, 259]], [[235, 239], [26, 30]], [[235, 239], [195, 199]], [[235, 239], [255, 259]], [[76, 80], [111, 115]], [[20, 24], [40, 44]], [[20, 24], [111, 115]], [[76, 80], [40, 44]], [[142, 146], [40, 44]], [[142, 146], [111, 115]], [[193, 197], [40, 44]], [[193, 197], [111, 115]], [[21, 24], [147, 151]], [[129, 132], [42, 46]], [[129, 132], [147, 151]], [[199, 202], [42, 46]], [[199, 202], [147, 151]], [[21, 24], [42, 46]], [[195, 199], [227, 231]], [[33, 37], [51, 55]], [[33, 37], [227, 231]], [[195, 199], [51, 55]], [[265, 268], [189, 193]], [[144, 148], [87, 91]], [[131, 135], [102, 107]], [[105, 109], [88, 92]], [[146, 150], [92, 96]], [[416, 419], [474, 480]], [[180, 184], [65, 69]], [[88, 91], [129, 135]], [[0, 4], [15, 19]], [[88, 91], [122, 126]], [[88, 91], [140, 148]], [[137, 140], [219, 222]], [[105, 109], [219, 222]], [[137, 140], [210, 214]], [[105, 109], [210, 214]], [[97, 101], [80, 84]], [[14, 17], [80, 84]], [[140, 143], [67, 74]], [[195, 199], [26, 30]], [[132, 136], [184, 190]], [[290, 294], [205, 211]], [[280, 283], [205, 211]], [[0, 4], [30, 35]], [[0, 4], [40, 44]], [[28, 32], [69, 73]], [[55, 59], [124, 128]], [[377, 381], [518, 524]], [[144, 148], [2, 7]], [[28, 32], [141, 145]], [[67, 70], [125, 130]], [[0, 4], [48, 53]], [[169, 173], [45, 49]], [[177, 181], [45, 49]], [[44, 48], [137, 143]], [[0, 4], [32, 37]], [[44, 48], [218, 222]], [[123, 127], [96, 104]], [[184, 188], [274, 278]], [[184, 188], [253, 256]], [[184, 188], [281, 285]], [[184, 188], [242, 246]], [[50, 54], [67, 71]], [[184, 188], [292, 296]], [[60, 64], [11, 15]], [[184, 188], [235, 239]], [[184, 188], [228, 232]], [[184, 188], [221, 225]], [[34, 38], [92, 96]], [[60, 64], [20, 24]], [[107, 111], [0, 4]], [[96, 100], [0, 4]], [[70, 74], [48, 52]], [[79, 82], [48, 52]], [[24, 28], [80, 84]], [[14, 17], [61, 64]], [[14, 17], [81, 84]], [[171, 175], [138, 141]], [[14, 17], [91, 95]], [[171, 175], [129, 133]], [[186, 190], [105, 110]], [[42, 46], [135, 138]], [[42, 46], [143, 146]], [[64, 68], [130, 134]], [[84, 88], [130, 134]], [[40, 44], [86, 90]], [[23, 27], [94, 98]], [[19, 23], [67, 70]], [[12, 16], [67, 70]], [[128, 132], [186, 190]], [[19, 23], [87, 90]], [[12, 16], [87, 90]], [[19, 23], [97, 101]], [[12, 16], [97, 101]], [[24, 28], [139, 145]], [[85, 89], [43, 47]], [[0, 4], [43, 47]], [[117, 121], [88, 93]], [[37, 41], [52, 56]], [[107, 110], [157, 161]], [[12, 15], [92, 96]], [[12, 15], [85, 89]], [[107, 110], [143, 147]], [[107, 110], [150, 154]], [[107, 110], [171, 175]], [[184, 187], [125, 128]], [[12, 15], [99, 103]], [[107, 110], [164, 168]], [[107, 110], [189, 193]], [[184, 187], [74, 78]], [[107, 110], [178, 182]], [[60, 64], [103, 108]], [[60, 64], [140, 144]], [[136, 139], [43, 47]], [[35, 38], [11, 15]], [[81, 85], [15, 21]], [[77, 80], [18, 22]], [[105, 109], [81, 86]], [[183, 186], [217, 222]], [[19, 22], [68, 72]], [[27, 31], [68, 72]], [[53, 57], [38, 44]], [[53, 57], [38, 44]], [[88, 92], [28, 34]], [[62, 66], [28, 34]], [[91, 95], [48, 52]], [[91, 95], [59, 63]], [[0, 4], [33, 37]], [[0, 4], [42, 46]], [[22, 26], [81, 84]], [[0, 3], [37, 41]], [[82, 86], [37, 41]], [[0, 4], [59, 64]], [[0, 4], [110, 114]], [[0, 4], [67, 71]], [[0, 4], [100, 103]], [[67, 71], [103, 107]], [[0, 3], [52, 56]], [[169, 173], [86, 89]], [[0, 4], [85, 90]], [[153, 157], [114, 117]], [[99, 103], [114, 117]], [[39, 43], [126, 130]], [[39, 43], [117, 121]], [[50, 53], [18, 22]], [[58, 62], [18, 22]], [[91, 95], [38, 42]], [[40, 44], [4, 8]], [[98, 102], [4, 12]], [[58, 62], [119, 123]], [[58, 62], [126, 129]], [[58, 62], [138, 142]], [[58, 62], [14, 18]], [[69, 73], [26, 29]], [[58, 62], [149, 153]], [[58, 62], [132, 135]], [[92, 95], [195, 199]], [[100, 104], [195, 199]], [[94, 98], [77, 81]], [[71, 75], [208, 212]], [[49, 53], [60, 63]], [[109, 113], [34, 39]], [[25, 29], [48, 51]], [[48, 52], [84, 88]], [[60, 63], [37, 41]], [[61, 65], [39, 44]], [[68, 71], [39, 44]], [[78, 82], [39, 44]], [[57, 61], [30, 34]], [[98, 101], [14, 19]], [[73, 76], [29, 35]], [[61, 65], [14, 18]], [[68, 71], [14, 18]], [[78, 82], [14, 18]], [[61, 65], [49, 53]], [[68, 71], [49, 53]], [[78, 82], [49, 53]], [[238, 242], [190, 194]], [[0, 4], [39, 44]], [[238, 242], [126, 130]], [[40, 43], [8, 14]], [[135, 139], [8, 14]], [[145, 149], [8, 14]], [[238, 242], [181, 185]], [[40, 43], [19, 29]], [[135, 139], [19, 29]], [[145, 149], [19, 29]], [[130, 134], [158, 162]], [[10, 14], [587, 592]], [[10, 14], [307, 313]], [[18, 22], [56, 60]], [[18, 22], [47, 51]], [[10, 14], [532, 536]], [[10, 14], [241, 245]], [[10, 14], [250, 254]], [[10, 14], [80, 83]], [[10, 14], [510, 514]], [[10, 14], [457, 460]], [[10, 14], [213, 217]], [[10, 14], [124, 128]], [[10, 14], [281, 284]], [[10, 14], [362, 366]], [[10, 14], [181, 185]], [[10, 14], [620, 624]], [[10, 14], [37, 41]], [[10, 14], [553, 557]], [[10, 14], [409, 412]], [[16, 20], [65, 71]], [[4, 7], [141, 145]], [[97, 101], [121, 125]], [[97, 101], [130, 134]], [[135, 139], [58, 64]], [[144, 148], [58, 64]], [[14, 18], [40, 46]], [[64, 67], [7, 11]], [[14, 18], [141, 145]], [[52, 56], [97, 103]], [[52, 56], [97, 103]], [[81, 84], [111, 115]], [[120, 123], [49, 53]], [[103, 106], [137, 141]], [[127, 131], [27, 31]], [[136, 140], [27, 31]], [[127, 131], [36, 40]], [[136, 140], [36, 40]], [[0, 4], [47, 51]], [[125, 129], [11, 15]], [[134, 137], [11, 15]], [[103, 107], [138, 142]], [[12, 16], [48, 53]], [[10, 14], [32, 35]], [[10, 14], [25, 29]], [[10, 14], [38, 41]], [[0, 4], [49, 55]], [[81, 84], [49, 55]], [[65, 68], [99, 104]], [[17, 21], [147, 151]], [[26, 30], [147, 151]], [[17, 21], [171, 175]], [[26, 30], [171, 175]], [[17, 21], [154, 158]], [[26, 30], [154, 158]], [[17, 21], [140, 144]], [[17, 21], [161, 164]], [[26, 30], [161, 164]], [[93, 97], [142, 146]], [[133, 136], [32, 37]], [[71, 75], [32, 37]], [[82, 85], [11, 15]], [[73, 77], [10, 14]], [[96, 100], [17, 21]], [[42, 46], [26, 30]], [[36, 40], [159, 163]], [[54, 58], [26, 30]], [[53, 57], [27, 31]], [[76, 80], [210, 215]], [[4, 7], [45, 50]], [[31, 34], [61, 65]], [[0, 4], [39, 43]], [[24, 28], [84, 89]], [[24, 28], [92, 98]], [[30, 34], [90, 93]], [[38, 42], [90, 93]], [[14, 18], [224, 228]], [[14, 18], [89, 93]], [[14, 18], [96, 100]], [[131, 134], [66, 70]], [[66, 69], [23, 26]], [[72, 76], [23, 26]], [[81, 85], [23, 26]], [[128, 132], [34, 38]], [[0, 3], [55, 60]], [[66, 70], [110, 116]], [[200, 203], [26, 30]], [[55, 58], [138, 142]], [[65, 69], [48, 52]], [[101, 105], [43, 47]], [[20, 24], [103, 107]], [[29, 33], [103, 107]], [[166, 170], [97, 102]], [[86, 90], [0, 4]], [[44, 47], [78, 82]], [[44, 47], [222, 227]], [[44, 47], [127, 131]], [[47, 51], [81, 84]], [[81, 85], [109, 113]], [[229, 232], [109, 113]], [[220, 224], [109, 113]], [[123, 126], [32, 36]], [[18, 22], [67, 71]], [[368, 372], [437, 441]], [[368, 372], [446, 450]], [[73, 76], [102, 105]], [[69, 73], [165, 169]], [[69, 73], [187, 191]], [[322, 326], [341, 345]], [[40, 44], [94, 99]], [[40, 44], [110, 114]], [[40, 44], [119, 124]], [[15, 18], [68, 71]], [[15, 18], [74, 78]], [[225, 228], [308, 312]], [[225, 228], [326, 330]], [[15, 18], [81, 85]], [[225, 228], [315, 319]], [[15, 18], [88, 91]], [[15, 18], [98, 102]], [[15, 18], [62, 65]], [[52, 56], [129, 132]], [[43, 47], [129, 132]], [[52, 56], [135, 139]], [[43, 47], [135, 139]], [[52, 56], [142, 146]], [[43, 47], [142, 146]], [[52, 56], [149, 152]], [[43, 47], [149, 152]], [[52, 56], [159, 163]], [[43, 47], [159, 163]], [[52, 56], [123, 126]], [[43, 47], [123, 126]], [[58, 61], [102, 107]], [[65, 69], [22, 28]], [[36, 39], [55, 59]], [[41, 45], [11, 15]], [[138, 142], [58, 62]], [[86, 90], [152, 156]], [[4, 8], [185, 189]], [[294, 298], [102, 106]], [[17, 21], [52, 56]], [[144, 148], [178, 182]], [[39, 43], [51, 55]], [[40, 43], [69, 72]], [[29, 33], [69, 72]], [[89, 93], [44, 49]], [[0, 4], [50, 54]], [[0, 4], [41, 45]], [[0, 4], [33, 38]], [[89, 93], [54, 58]], [[111, 115], [23, 27]], [[103, 106], [23, 27]], [[97, 100], [23, 27]], [[90, 94], [23, 27]], [[39, 43], [101, 105]], [[82, 86], [127, 131]], [[104, 108], [55, 58]], [[25, 29], [84, 88]], [[183, 187], [130, 135]], [[192, 196], [130, 135]], [[0, 4], [121, 125]], [[87, 91], [136, 139]], [[43, 46], [106, 109]], [[87, 91], [144, 147]], [[60, 64], [177, 183]], [[58, 62], [104, 108]], [[26, 30], [72, 76]], [[9, 13], [147, 151]], [[97, 100], [128, 133]], [[9, 13], [92, 96]], [[66, 70], [46, 50]], [[36, 40], [95, 99]], [[36, 40], [86, 90]], [[136, 140], [81, 85]], [[40, 43], [71, 75]], [[125, 129], [112, 116]], [[26, 30], [73, 77]], [[26, 30], [64, 68]], [[149, 153], [173, 177]], [[30, 33], [65, 69]], [[240, 244], [259, 263]], [[157, 161], [95, 99]], [[5, 8], [50, 54]], [[157, 161], [109, 113]], [[157, 161], [102, 106]], [[70, 73], [99, 102]], [[45, 49], [101, 105]], [[45, 49], [110, 114]], [[42, 46], [10, 14]], [[51, 55], [200, 206]], [[221, 225], [19, 23]], [[33, 37], [68, 72]], [[87, 91], [0, 4]], [[135, 138], [177, 181]], [[35, 39], [94, 98]], [[127, 131], [182, 186]], [[77, 81], [447, 450]], [[77, 81], [398, 402]], [[45, 49], [80, 84]], [[77, 81], [363, 367]], [[16, 19], [54, 58]], [[50, 54], [158, 162]], [[65, 69], [25, 29]], [[114, 118], [97, 101]], [[59, 62], [43, 49]], [[25, 28], [74, 79]], [[0, 4], [34, 38]], [[37, 41], [80, 83]], [[76, 79], [44, 48]], [[92, 96], [125, 129]], [[48, 52], [15, 19]], [[63, 67], [15, 19]], [[77, 81], [133, 137]], [[45, 49], [22, 27]], [[122, 126], [64, 68]], [[17, 21], [64, 68]], [[29, 33], [64, 68]], [[0, 4], [35, 39]], [[0, 4], [9, 14]], [[8, 11], [51, 57]], [[25, 29], [90, 94]], [[12, 16], [115, 119]], [[149, 152], [40, 44]], [[86, 90], [103, 107]], [[34, 38], [67, 71]], [[21, 24], [38, 42]], [[29, 33], [107, 110]], [[43, 47], [95, 99]], [[23, 27], [75, 79]], [[115, 119], [107, 111]], [[23, 27], [86, 90]], [[23, 27], [55, 58]], [[23, 27], [148, 152]], [[23, 27], [61, 65]], [[23, 27], [68, 72]], [[148, 151], [241, 245]], [[111, 115], [241, 245]], [[98, 102], [16, 20]], [[22, 26], [46, 50]], [[55, 58], [89, 94]], [[93, 97], [48, 52]], [[133, 137], [30, 34]], [[16, 20], [40, 44]], [[7, 11], [40, 44]], [[166, 170], [39, 43]], [[171, 175], [139, 143]], [[10, 13], [57, 62]], [[113, 117], [159, 163]], [[208, 212], [236, 240]], [[217, 221], [236, 240]], [[107, 111], [142, 146]], [[0, 4], [55, 60]], [[20, 23], [104, 109]], [[0, 4], [50, 54]], [[98, 102], [121, 125]], [[71, 74], [42, 48]], [[71, 74], [53, 57]], [[69, 73], [19, 23]], [[87, 91], [37, 40]], [[104, 108], [96, 100]], [[25, 29], [51, 55]], [[49, 53], [19, 23]], [[110, 114], [79, 82]], [[134, 138], [55, 58]], [[23, 27], [49, 53]], [[98, 102], [115, 119]], [[31, 34], [5, 9]], [[31, 34], [14, 20]], [[38, 42], [90, 94]], [[0, 4], [59, 63]], [[61, 65], [31, 35]], [[61, 65], [22, 26]], [[29, 33], [64, 68]], [[41, 44], [63, 67]], [[29, 33], [73, 77]], [[146, 150], [174, 180]], [[146, 150], [174, 180]], [[29, 33], [38, 41]], [[5, 9], [33, 36]], [[128, 132], [88, 97]], [[50, 54], [67, 71]], [[18, 22], [88, 92]], [[18, 22], [79, 83]], [[88, 92], [65, 69]], [[73, 76], [65, 69]], [[79, 83], [65, 69]], [[0, 4], [29, 34]], [[231, 234], [205, 210]], [[41, 45], [84, 88]], [[63, 67], [29, 33]], [[69, 73], [61, 65]], [[69, 73], [52, 56]], [[81, 85], [176, 180]], [[110, 113], [137, 142]], [[29, 33], [87, 91]], [[182, 186], [18, 22]], [[50, 53], [19, 24]], [[26, 30], [18, 22]], [[38, 42], [116, 122]], [[29, 33], [68, 71]], [[20, 24], [72, 78]], [[36, 40], [82, 87]], [[20, 24], [46, 50]], [[25, 28], [41, 46]], [[16, 20], [41, 46]], [[142, 146], [4, 8]], [[133, 137], [45, 49]], [[96, 100], [176, 180]], [[87, 91], [176, 180]], [[227, 231], [182, 186]], [[14, 18], [34, 40]], [[61, 65], [44, 50]], [[65, 69], [25, 29]], [[100, 104], [185, 189]], [[69, 73], [36, 40]], [[184, 188], [215, 219]], [[43, 47], [78, 82]], [[100, 104], [54, 58]], [[4, 8], [45, 51]], [[428, 432], [330, 334]], [[437, 441], [330, 334]], [[412, 416], [330, 334]], [[57, 61], [75, 79]], [[122, 126], [52, 56]], [[69, 72], [99, 103]], [[69, 72], [152, 156]], [[77, 81], [98, 102]], [[47, 51], [82, 86]], [[23, 27], [38, 42]], [[307, 311], [142, 146]], [[153, 157], [42, 46]], [[144, 148], [42, 46]], [[141, 145], [14, 18]], [[36, 39], [51, 55]], [[84, 88], [53, 57]], [[84, 88], [62, 66]], [[73, 77], [144, 149]], [[9, 13], [23, 27]], [[0, 4], [23, 27]], [[195, 199], [166, 171]], [[48, 51], [78, 82]], [[154, 158], [71, 74]], [[66, 70], [85, 89]], [[79, 83], [165, 169]], [[31, 34], [146, 150]], [[31, 34], [153, 157]], [[31, 34], [164, 168]], [[234, 238], [273, 277]], [[35, 39], [110, 114]], [[33, 36], [99, 105]], [[64, 68], [83, 87]], [[0, 4], [35, 39]], [[21, 24], [167, 171]], [[13, 17], [123, 127]], [[15, 19], [23, 27]], [[154, 157], [15, 19]], [[145, 149], [56, 60]], [[145, 149], [63, 67]], [[145, 149], [74, 78]], [[13, 17], [40, 44]], [[83, 87], [66, 70]], [[50, 54], [4, 8]], [[69, 73], [77, 83]], [[4, 8], [33, 37]], [[77, 80], [34, 40]], [[43, 47], [78, 82]], [[48, 52], [9, 13]], [[30, 33], [66, 70]], [[45, 49], [70, 74]], [[45, 49], [77, 81]], [[138, 142], [174, 178]], [[9, 13], [74, 78]], [[138, 142], [167, 171]], [[138, 142], [181, 185]], [[75, 79], [4, 8]], [[95, 98], [59, 62]], [[95, 98], [50, 54]], [[103, 107], [4, 11]], [[55, 58], [4, 11]], [[9, 13], [48, 54]], [[146, 150], [62, 66]], [[40, 44], [21, 25]], [[139, 142], [34, 40]], [[21, 25], [0, 4]], [[28, 31], [0, 4]], [[38, 42], [0, 4]], [[139, 142], [43, 47]], [[139, 142], [151, 155]], [[139, 142], [27, 31]], [[139, 142], [57, 61]], [[139, 142], [50, 54]], [[139, 142], [75, 79]], [[139, 142], [64, 68]], [[0, 3], [55, 61]], [[0, 4], [82, 86]], [[126, 130], [181, 185]], [[117, 121], [181, 185]], [[140, 144], [122, 127]], [[199, 203], [145, 151]], [[60, 64], [22, 28]], [[87, 90], [114, 118]], [[87, 90], [263, 267]], [[177, 181], [205, 209]], [[186, 190], [205, 209]], [[0, 4], [149, 154]], [[33, 37], [52, 56]], [[41, 44], [66, 71]], [[32, 36], [48, 52]], [[23, 27], [48, 52]], [[134, 138], [176, 182]], [[268, 271], [289, 295]], [[134, 138], [187, 191]], [[62, 66], [49, 55]], [[8, 11], [40, 44]], [[132, 136], [174, 179]], [[63, 66], [80, 84]], [[71, 75], [80, 84]], [[87, 91], [58, 63]], [[120, 124], [137, 140]], [[0, 4], [48, 52]], [[80, 84], [103, 108]], [[97, 101], [104, 108]], [[203, 206], [104, 108]], [[97, 101], [113, 117]], [[203, 206], [113, 117]], [[37, 41], [73, 77]], [[46, 49], [73, 77]], [[52, 55], [16, 20]], [[101, 104], [265, 274]], [[78, 82], [265, 274]], [[85, 89], [265, 274]], [[76, 79], [10, 14]], [[67, 71], [120, 127]], [[71, 75], [197, 201]], [[165, 169], [197, 201]], [[165, 169], [205, 209]], [[32, 36], [77, 81]], [[103, 107], [11, 16]], [[0, 4], [48, 53]], [[0, 4], [138, 141]], [[64, 68], [47, 51]], [[0, 4], [108, 112]], [[64, 68], [56, 60]], [[11, 14], [25, 30]], [[7, 11], [41, 45]], [[183, 187], [167, 170]], [[174, 178], [167, 170]], [[4, 8], [71, 75]], [[13, 17], [71, 75]], [[4, 8], [80, 84]], [[13, 17], [80, 84]], [[4, 8], [69, 73]], [[11, 15], [78, 82]], [[34, 38], [117, 121]], [[43, 47], [117, 121]], [[0, 3], [32, 36]], [[115, 119], [188, 192]], [[72, 76], [107, 111]], [[83, 87], [28, 33]], [[19, 22], [56, 61]], [[10, 14], [56, 61]], [[89, 93], [42, 47]], [[34, 38], [94, 99]], [[20, 24], [51, 55]], [[61, 65], [81, 84]], [[0, 4], [25, 28]], [[50, 54], [141, 145]], [[246, 249], [136, 142]], [[246, 249], [29, 33]], [[246, 249], [127, 131]], [[246, 249], [200, 208]], [[150, 154], [106, 111]], [[0, 4], [44, 48]], [[58, 62], [45, 49]], [[25, 29], [58, 61]], [[27, 30], [34, 39]], [[69, 72], [18, 24]], [[62, 66], [18, 24]], [[111, 113], [18, 24]], [[78, 82], [49, 54]], [[116, 119], [72, 76]], [[41, 44], [72, 76]], [[124, 127], [72, 76]], [[23, 27], [72, 76]], [[30, 34], [72, 76]], [[82, 86], [49, 54]], [[111, 115], [41, 45]], [[166, 170], [232, 236]], [[0, 4], [34, 40]], [[97, 101], [70, 74]], [[72, 76], [87, 91]], [[156, 160], [251, 256]], [[4, 8], [21, 25]], [[125, 129], [71, 77]], [[76, 80], [43, 47]], [[39, 42], [69, 74]], [[9, 13], [53, 57]], [[101, 105], [55, 59]], [[38, 42], [81, 85]], [[38, 42], [22, 26]], [[87, 91], [116, 120]], [[106, 109], [48, 52]], [[114, 117], [48, 52]], [[124, 128], [165, 171]], [[87, 91], [58, 63]], [[40, 44], [93, 100]], [[19, 23], [141, 145]], [[19, 23], [132, 136]], [[87, 91], [58, 63]], [[174, 177], [187, 193]], [[0, 4], [103, 107]], [[89, 93], [22, 28]], [[20, 24], [50, 54]], [[0, 4], [32, 36]], [[0, 3], [39, 44]], [[0, 3], [30, 34]], [[61, 65], [44, 48]], [[61, 65], [44, 48]], [[30, 33], [97, 101]], [[87, 91], [58, 63]], [[21, 24], [55, 59]], [[21, 25], [78, 82]], [[42, 45], [80, 85]], [[82, 86], [49, 54]], [[91, 95], [58, 63]], [[40, 44], [93, 100]], [[27, 31], [8, 12]], [[104, 108], [96, 100]], [[97, 100], [39, 43]], [[105, 108], [39, 43]], [[84, 87], [11, 15]], [[77, 81], [25, 31]], [[82, 86], [49, 54]], [[97, 100], [39, 43]], [[105, 108], [39, 43]], [[17, 21], [107, 110]], [[91, 95], [34, 38]], [[82, 86], [49, 54]], [[67, 71], [34, 38]], [[129, 133], [33, 38]], [[35, 38], [73, 77]], [[43, 47], [73, 77]], [[40, 44], [93, 100]], [[9, 12], [41, 46]], [[9, 12], [41, 46]], [[35, 38], [77, 81]], [[26, 30], [77, 81]], [[52, 56], [35, 39]], [[170, 174], [145, 152]], [[0, 4], [34, 39]], [[91, 94], [51, 56]], [[77, 81], [25, 31]], [[0, 4], [89, 93]], [[97, 100], [39, 43]], [[105, 108], [39, 43]], [[47, 51], [116, 120]], [[47, 51], [107, 111]], [[131, 135], [47, 51]], [[123, 126], [47, 51]], [[32, 36], [111, 116]], [[18, 22], [68, 72]], [[20, 24], [73, 77]], [[50, 54], [119, 123]], [[50, 54], [110, 114]], [[173, 177], [148, 155]], [[0, 3], [70, 73]], [[15, 19], [70, 73]]]\n"
     ]
    }
   ],
   "source": [
    "print(span_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Lrp', 'Lrp', 'CRP', 'CRP', 'NsrR', 'NsrR', 'Mlc', 'Mlc', 'NsrR', 'NsrR', 'NsrR', 'NsrR', 'RhaR', 'RhaR', 'CRP', 'CRP', 'CRP', 'mutant', 'FucR', 'FucR', 'NarL', 'NarL', 'MelR', 'MelR', 'IciA', 'IciA', 'IciA', 'Rob', 'Rob', 'MarR', 'MarR', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'GntR', 'GntR', 'GntR', 'GntR', 'Fnr', 'Fnr', 'Fnr', 'Rob', 'Rob', 'Rob', 'Rob', 'MelR', 'MelR', 'GntR', 'GntR', 'GntR', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'GalS', 'GalS', 'Fnr', 'Fnr', 'Fnr', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'IciA', 'IciA', 'IciA', 'CRP', 'CRP', 'RbsR', 'RbsR', 'MarA', 'MarA', 'IciA', 'IciA', 'MelR', 'MelR', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'MarA', 'MarA', 'AraC', 'AraC', 'AraC', 'AraC', 'RhaR', 'RhaR', 'MetR', 'MetR', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'ArgP', 'ArgP', 'MelR', 'MelR', 'MarA', 'MarA', 'SoxS', 'SoxS', 'MarA', 'MarA', 'MarA', 'MarA', 'MelR', 'MelR', 'MarA', 'MarA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'Lrp', 'Lrp', 'CAP', 'CAP', 'CAP', 'MelR', 'MelR', 'SoxS', 'SoxS', 'Fnr', 'Fnr', 'ArgP', 'ArgP', 'Fis', 'Fis', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'GntR', 'GntR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'IHF', 'IHF', 'Rob', 'Rob', 'IciA', 'IciA', 'ArgP', 'ArgP', 'ArgP', 'MarA', 'MarA', 'SoxS', 'SoxS', 'CRP', 'CRP', 'MarA', 'MarA', 'DnaA', 'DnaA', 'DnaA', 'MelR', 'MelR', 'CRP', 'CRP', 'MetR', 'MetR', 'MetR', 'MetR', 'MarA', 'MarA', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarP', 'NarP', 'NarL', 'NarL', 'Mlc', 'Mlc', 'DnaA', 'DnaA', 'Fnr', 'Fnr', 'ArgP', 'ArgP', 'CRP', 'CRP', 'IciA', 'IciA', 'Fnr', 'Fnr', 'CRP', 'CRP', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'AraC', 'NarL', 'NarL', 'MelR', 'MelR', 'ChbR', 'ChbR', 'NagC', 'NagC', 'NagC', 'NagC', 'DcuS', 'DcuS', 'NagC', 'NagC', 'NagC', 'NagC', 'DcuR', 'DcuR', 'NagC', 'NagC', 'ChbR', 'ChbR', 'NagC', 'NagC', 'MelR', 'MelR', 'NarL', 'NarL', 'NarL', 'MelR', 'MelR', 'Fnr', 'Fnr', 'Fnr', 'Mlc', 'Mlc', 'Mlc', 'NagC', 'NagC', 'MarA', 'MarA', 'NagC', 'NagC', 'CRP', 'CRP', 'CRP', 'CRP', 'ChbR', 'ChbR', 'MelR', 'MelR', 'MelR', 'Mlc', 'Mlc', 'Mlc', 'Mlc', 'Lrp', 'Lrp', 'CRP', 'CRP', 'CAP', 'CAP', 'NarL', 'NarL', 'GalS', 'GalS', 'GalS', 'GalS', 'RbsR', 'RbsR', 'CRP', 'CRP', 'FNR', 'FNR', 'GlcC', 'GlcC', 'IHF', 'IHF', 'RhaR', 'RhaR', 'NarL', 'NarL', 'FNR', 'FNR', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'AraC', 'AraC', 'NarP', 'NarP', 'CRP', 'CRP', 'NarP', 'NarP', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'NarL', 'MelR', 'MelR', 'NarP', 'NarP', 'NagC', 'NagC', 'Mlc', 'Mlc', 'NagC', 'NagC', 'NarP', 'NarP', 'CRP', 'CRP', 'NagC', 'NagC', 'CRP', 'CRP', 'CRP', 'CRP', 'NsrR', 'NsrR', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'NagC', 'NagC', 'ArgP', 'ArgP', 'FNR', 'FNR', 'Ada', 'Ada', 'OmpR', 'OmpR', 'GntR', 'GntR', 'GntR', 'GntR', 'DnaA', 'DnaA', 'MarA', 'MarA', 'Rob', 'Rob', 'SoxS', 'SoxS', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'IHF', 'IHF', 'NagC', 'NagC', 'NanR', 'NanR', 'MarA', 'MarA', 'SoxS', 'SoxS', 'MarA', 'MarA', 'SoxS', 'SoxS', 'MarA', 'MarA', 'MarA', 'SoxS', 'SoxS', 'SoxS', 'NsrR', 'NsrR', 'NsrR', 'NsrR', 'IciA', 'IciA', 'CRP', 'CRP', 'MelR', 'MelR', 'ArgP', 'ArgP', 'SoxS', 'SoxS', 'CRP', 'CRP', 'RhaR', 'RhaR', 'Fnr', 'Fnr', 'RhaS', 'RhaS', 'ArgP', 'ArgP', 'MarR', 'MarR', 'MarA', 'MarA', 'ChiX', 'ChiX', 'ChiX', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'observed', 'MarA', '(', '(', 'MarA', 'MarA', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'MelR', 'MelR', 'GalR', 'GalR', 'GalS', 'GalS', 'GntR', 'GntR', 'MelR', 'MelR', 'MelR', 'ArgP', 'ArgP', 'MelR', 'MelR', 'Rob', 'Rob', 'and', 'and', '.', '.', 'CRP', 'CRP', 'MelR', 'MelR', 'Fnr', 'Fnr', 'MarA', 'MarA', 'SoxS', 'SoxS', 'MelR', 'MelR', 'ArgP', 'ArgP', 'MarA', 'MarA', 'MarA', 'MarA', 'MarA', 'ArgP', 'ArgP', 'ArgP', 'Lrp', 'Lrp', 'Lrp', 'DnaA', 'DnaA', 'DnaA', 'IciA', 'IciA', 'IciA', 'NarL', 'NarL', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'TreR', 'TreR', 'OxyR', 'OxyR', 'NanR', 'NanR', 'NanR', 'NanR', 'GntR', 'GntR', 'ArgP', 'ArgP', 'MelR', 'MelR', 'CRP', 'CRP', 'NarL', 'NarL', 'NarP', 'NarP', 'CRP', 'CRP', 'FNR', 'FNR', 'MelR', 'MelR', 'CsrA', 'CsrA', 'CsrA', 'CAP', 'CAP', 'CAP', 'CAP', 'Lrp', 'Lrp', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'DnaA', ' Dna', 'CRP', 'CRP', 'CRP', 'CRP', ' Mel', ' Mel', ' whil', ' whil', 'CRP', 'CRP', 'CRP', 'CRP', 'Rob', 'Rob', 'Rob', 'Rob', 'GntR', 'GntR', 'GntR', 'GntR', 'GntR', 'GntR', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'NarL', 'NarL', 'NarL', 'NarL', 'NarP', 'NarP', 'NarP', 'NarP', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'ArgP', 'FNR', 'FNR', 'FNR', 'FNR', ' Nar', 'NarL', 'ArgP', 'ArgP', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'MarA', 'MarA', 'MarA', 'MarA', 'MarA', 'MarA', 'MarA', 'MarA', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'IciA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'MarA', 'MarA', 'MarA', 'MarA', 'CRP', 'CRP', 'CRP', 'CRP', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'Fis', 'Fis', 'Fis', 'Fis', 'Fis', 'Fis', 'CRP', 'CRP', 'CRP', 'CRP', 'DnaA', 'DnaA', 'DnaA', 'DnaA', 'MelR', 'MelR', 'MelR', 'MelR', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'Lrp', 'Lrp', 'Lrp', 'Lrp', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArcA', 'ArcA', 'ArcA', 'ArcA', 'ArcA', 'ArcA', 'MarA', 'MarA', 'MarA', 'MarA', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'RhaS', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'LysR', 'LysR', 'LysR', 'LysR', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'CRP', 'CRP', 'CRP', 'CRP', 'RhaR', 'RhaR', 'RhaR', 'RhaR', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'MelR', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'CAP', 'NarL', 'NarL', 'NarL', 'NarL', 'Lrp', 'IciA', 'MelR', 'MelR', 'MelR', 'FNR', 'GntR', 'Mlc', 'NagC', 'Mlc', 'Mlc', 'Fis', 'NarL', 'Fis', 'NarL', 'ArgP', 'Lrp', 'Fnr', 'NarL', 'AraC', 'AraC', 'CAP', 'GntR', 'GntR', 'MelR', 'GntH', 'AraC', 'MelR', 'IciA', 'CRP', 'MelR', 'NarL', 'NarP', 'RhaS', 'RhaR', 'RhaS', 'NarL', 'ArgP', 'ArgP', 'ArgP', 'ArgP', 'NanR', 'ArgP', 'MarA', 'ArgP', 'ArgP', 'ArgP', 'NagC', 'MarA', 'CpxR', 'OmpR', 'ArgP', 'Lrp', 'ArgP', 'Rob', 'Rob', 'NarL', 'Rob', 'NarL', 'GntR', 'RbsR', 'RbsR', 'RhaR', 'RhaS', 'ArgP', 'MarA', 'MarA', 'SoxS', 'MarA', 'MarA', 'SoxS', 'MarA', 'SoxS', 'AraC', 'ChiX', 'NagC', 'MelR', 'RpoS', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Fis', 'Rob', 'Rob', 'Rob', 'Fis', 'Rob', 'MelR', 'MelR', 'Fnr', 'Rob', 'AraC', 'CAP', 'MelR', 'CRP', 'Fnr', 'NarP', 'AraC', 'AraC', 'MarA', 'MarR', 'MarR', 'MarR', 'MelR', 'MelR', 'NarP', 'CRP', 'MalT', 'MarA', 'MarA', 'MarA', 'MarA', 'AraC', 'FNR', 'NarL', 'RhaR', 'MarA', 'SoxS', 'MarA', 'MarA', 'Fis', 'RpoS', 'MarA', 'MelR', 'RbsR', 'SoxS', 'SoxS', 'SoxS', 'MarA', 'NagC', 'SoxS', 'SoxS', 'Fnr', 'NarL', 'AraC', 'ArgP', 'NsrR', 'MelR', 'NarL', 'SoxR', 'Rob', 'MarA', 'Rob', 'SoxS', 'DnaA', 'CRP', 'Fis', 'MarA', 'Rob', 'SoxS', 'MarA', 'Rob', 'SoxS', 'NarP', 'MelR', 'NarP', 'Fnr', 'NarL', 'NarP', 'NarP', 'Fnr', 'NarL', 'NarP', 'GalR', 'SoxS', 'SoxS', 'MarA', 'MarA', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'AraC', 'Ada', 'NarL', 'NarL', 'GalR', 'GalS', 'MarA', 'Fis', 'MarA', 'AraC', 'AraC', 'Fnr', 'CRP', 'Fnr', 'NarL', 'NarP', 'NarL', 'NarP', 'RhaS', 'AraC', 'CRP', 'MelR', 'RhaR', 'MarA', 'MarA', 'MarA', 'AraC', 'CRP', 'CRP', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'MarA', 'SoxS', 'NarL', 'CRP', 'MelR', 'Fnr', 'SoxR', 'MarA', 'ArgP', 'ChiX', 'NanR', 'ArgP', 'MelR', 'CRP', 'Fnr', 'IciA', 'XylR', 'XylR', 'NarL', 'NarP', 'DcuR', 'DcuR', 'DcuR', 'CAP', 'FNR', 'NarL', 'NarP', 'NarP', 'Lrp', 'RhaS', 'CRP', 'CRP', 'MarA', 'ArgP', 'DnaA', 'IciA', 'MelR', 'SoxS', 'Rob', 'CRP', 'CRP', 'NagC', 'MarA', 'Rob', 'SoxS', 'CRP', 'IciA', 'SoxS', 'SoxS', 'Fnr', 'ArgP', 'ArgP', 'MarA', 'EvgA', 'EvgA', 'EvgA', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'SoxS', 'MarA', 'SoxS', 'CRP', 'EmrR', 'Lrp', 'NarL', 'MarA', 'NagC', 'ArgP', 'NagC', 'MelR', 'NagC', 'NarL', 'Rob', 'SoxS', 'MarA', 'ExuR', 'ExuR', 'ExuR', 'MarA', 'ArcA', 'CRP', 'FNR', 'SoxS', 'IciA', 'MarA', 'MarA', 'IciA', 'UidR', 'UxuR', 'GntR', 'SoxS', 'Rob', 'SoxS', 'MarR', 'DnaA', 'MarA', 'DnaA', 'CRP', 'DnaA', 'IciA', 'MarA', 'MarA', 'MarA', 'Lrp', 'IciA', 'MarA', 'MarA', 'ArgP', 'Rob', 'ArgP', 'NsrR', 'Fnr', 'NsrR', 'NsrR', 'Fnr', 'MarA', 'MarA', 'MetJ', 'AraC', 'NsrR', 'MelR', 'CpxR', 'CRP', 'ArgP', 'IciA', 'NsrR', 'NsrR', 'MelR', 'NsrR', 'Lrp', 'MelR', 'MarR', 'MelR', 'Rob', 'Lrp', 'ArgP', 'RelA', 'CRP', 'MelR', 'DnaA', 'IciA', 'MetR', 'MarA', 'DnaA', 'IciA', 'MelR', 'MelR', 'ArgP', 'CRP', 'IciA', 'MelR', 'CRP', 'NanR', 'ArgP', 'Fnr', 'NarP', 'MarA', 'SoxS', 'MarA', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'SoxS', 'IHF', 'NagC', 'LysR', 'NagC', 'CRP', 'DnaA', 'MarA', 'NagC', 'NanR', 'NanR', 'MelR', 'CRP', 'MetR', 'GalR', 'GalS', 'MelR', 'RhaR', 'CRP', 'IciA', 'PurR', 'CAP', 'CAP', 'ChiX', 'NsrR', 'GntR', 'NarP', 'NsrR', 'NagC', 'NarL', 'ArgP', 'NarL', 'Rob', 'Rob', 'MelR', 'NsrR', 'NagC', 'NagC', 'ArgP', 'IHF', 'ArgP', 'AraC', 'AraC', 'MarA', 'MarA', 'NagC', 'NanR', 'MetJ', 'MetJ', 'MarA', 'Rob', 'SoxS', 'MelR', 'CRP', 'NsrR', 'DcuR', 'MarA', 'MarA', 'MetJ', 'CRP', 'ArgP', 'NanR', 'CRP', 'MarA', 'AraC', 'MarA', 'AraC', 'MelR', 'NarP', 'CRP', 'MelR', 'MetJ', 'NagC', 'NagC', 'NanR', 'NsrR', 'AraC', 'AraC', 'DcuR', 'NagC', 'NagC', 'NanR', 'MelR', 'MarA', 'AraC', 'GalR', 'GalS', 'NagC', 'GntR', 'MelR', 'Rob', 'Rob', 'NsrR', 'MelR', 'OmpR', 'NagC', 'NagC', 'NanR', 'NarP', 'IHF', 'RfaH', 'RfaH', 'MelR', 'NagC', 'NanR', 'MelR', 'CRP', 'NarL', 'NarL', 'NagC', 'Rob', 'Rob', 'Rob', 'ArgP', 'ArgP', 'CRP', 'ChiX', 'MelR', 'IHF', 'GalS', 'ArgP', 'Fnr', 'MarA', 'MarA', 'MarA', 'ArgP', 'MelR', 'SoxS', 'RhaS', 'ArgP', 'CRP', 'MelR', 'TreR', 'Fis', 'MarA', 'MarA', 'NarL', 'NagC', 'NarL', 'NarL', 'NarP', 'Fis', 'Fis', 'ArgR', 'Lrp', 'AraC', 'NarP', 'NanR', 'Rob', 'MarA', 'Rob', 'SoxS', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'Rob', 'AraC', 'NagC', 'NanR', 'MelR', 'NarP', 'NagC', 'Rob', 'Rob', 'GalR', 'GalS', 'NagC', 'NagC', 'CRP', 'CpxR', 'OmpR', 'RhaS', 'CRP', 'RhaS', 'AraC', 'Rob', 'RhaR', 'CAP', 'ChbR', 'MelR', 'NarP', 'NarP', 'GntR', 'ArgP', 'Lrp', 'ArgP', 'Lrp', 'ArgP', 'Lrp', 'Rob', 'CAP', 'ChbR', 'NagC', 'Fis', 'NarL', 'NanR', 'NarL', 'NarL', 'NarP', 'NarL', 'NorR', 'NorR', 'OmpR', 'NorR', 'OmpR', 'CRP', 'NarL', 'OxyR', 'SoxS', 'NarL', 'NarP', 'NarL', 'NarP', 'NarL', 'NarP', 'NarL', 'NarP', 'Lrp', 'AraC', 'MelR', 'XylR', 'CRP', 'MelR', 'MelR', 'RhaS', 'NarL', 'SoxS', 'CsrA', 'YjgS', 'Mlc', 'Mlc', 'Mlc', 'Mlc', 'GntR', 'NagC', 'GntR', 'NarP', 'CRP', 'CAP', 'GalR', 'HU', 'MelR', 'Fis', 'FNR', 'IHF', 'NarL', 'NarP', 'MelR', 'GntR', 'GntR', 'RhaS', 'RhaR', 'RpiR', 'GntR', 'NagC', 'RhaS', 'DnaA', 'Lrp', 'DnaA', 'ArgP', 'AraC', 'AraC', 'ArgP', 'CRP', 'FNR', 'AraC', 'MelR', 'NarL', 'AraC', 'AraC', 'MelR', 'CRP', 'AraC', 'AraC', 'AraC', 'GalR', 'Lrp', 'Lrp', 'MelR', 'MelR', 'Lrp', 'MelR', 'Lrp', 'AraC', 'Lrp', 'MelR', 'MelR', 'NarL', 'AraC', 'GntR', 'CRP', 'FNR', 'Lrp', 'MarA', 'MelR', 'CRP', 'FNR', 'CytR', 'IciA', 'MelR', 'DnaA', 'RhaR', 'Mlc', 'NagC', 'NarL', 'Lrp', 'Lrp', 'CAP', 'UhpA', 'MelR', 'DeoR', 'CadC', 'Lrp', 'MarA', 'NarL', 'CRP', 'FNR', 'OmpR', 'OmpR', 'ArgP', 'Lrp', 'RhaR', 'RpoS', 'OmpR', 'OmpR', 'OmpR', 'DeoR', 'CRP', 'CytR']\n"
     ]
    }
   ],
   "source": [
    "print(regulator_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['argO', 'argO', 'rhaSR', 'rhaSR', 'hmpA', 'hmpA', 'manX', 'manX', 'ygbA', 'ygbA', 'ytfE', 'ytfE', 'rhaSR', 'rhaSR', 'fucPIK', 'fucPIK', 'fucPIK', 'fucPIK', 'fucPIK', 'fucPIK', 'ydhY', 'ydhY', 'melAB', 'melAB', 'dnaA', 'dnaA', 'dnaA', 'marRAB', 'marRAB', 'marA', 'marA', 'aslB', 'aslB', 'galT', 'galT', 'inaA', 'inaA', 'mdlA', 'mdlA', 'ybaO', 'ybaO', 'ybiS', 'ybiS', 'yfhD', 'yfhD', 'eda', 'eda', 'edd', 'edd', 'dmsABC', 'dmsABC', 'dmsABC', 'fumC', 'fumC', 'inaA', 'inaA', 'melAB', 'melAB', 'gntKU', 'gntKU', 'gntKU', 'fdnG', 'fdnG', 'frdA', 'frdA', 'narG', 'narG', 'nirB', 'nirB', 'nrfA', 'nrfA', 'galP', 'galP', 'dmsABC', 'dmsABC', 'dmsABC', 'guaBA', 'guaBA', 'mioC', 'mioC', 'nrd', 'nrd', 'rpoH', 'rpoH', 'uvrB', 'uvrB', 'dnaA', 'dnaA', 'dnaA', 'rhaBAD', 'rhaBAD', 'rbsDACBK', 'rbsDACBK', 'nfsA', 'nfsA', 'dnaA', 'dnaA', 'melR', 'melR', 'araA', 'araA', 'araB', 'araB', 'araD', 'araD', 'araE', 'araE', 'araF', 'araF', 'araG', 'araG', 'araH', 'araH', 'araT', 'araT', 'araU', 'araU', 'nfnB', 'nfnB', 'ytfQ', 'ytfQ', 'araBAD', 'araBAD', 'rhaSR', 'rhaSR', 'metE', 'metE', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'argO', 'argO', 'melR', 'melR', 'nfnB', 'nfnB', 'fpr', 'fpr', 'hdeA', 'hdeA', 'purA', 'purA', 'melR', 'melR', 'nfnB', 'nfnB', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'argO', 'argO', 'araC', 'araC', 'araC', 'melR', 'melR', 'mar', 'mar', 'napF', 'napF', 'dapB', 'dapB', 'nirB', 'nirB', 'asd', 'asd', 'asd', 'dapB', 'dapB', 'dapB', 'dapD', 'dapD', 'dapD', 'gdhA', 'gdhA', 'gdhA', 'lysA', 'lysA', 'lysA', 'lysC', 'lysC', 'lysC', 'lysP', 'lysP', 'lysP', 'gntR', 'gntR', 'melR', 'melR', 'melR', 'melR', 'melR', 'melR', 'melR', 'fimB', 'fimB', 'marRAB', 'marRAB', 'dnaA', 'dnaA', 'dapB', 'dapB', 'dapB', 'zwf', 'zwf', 'zwf', 'zwf', 'melAB', 'melAB', 'zwf', 'zwf', 'dnaA', 'dnaA', 'dnaA', 'melAB', 'melAB', 'melAB', 'melAB', 'metE', 'metE', 'metR', 'metR', 'nfnB', 'nfnB', 'nrfA', 'nrfA', 'cydD', 'cydD', 'fdnGHI', 'fdnGHI', 'fdnGHI', 'fdnGHI', 'narGHJI', 'narGHJI', 'manX', 'manX', 'dnaA', 'dnaA', 'dmsA', 'dmsA', 'nrdA', 'nrdA', 'rhaSR', 'rhaSR', 'dnaA', 'dnaA', 'napF', 'napF', 'rhaSR', 'rhaSR', 'araA', 'araA', 'araB', 'araB', 'araD', 'araD', 'araE', 'araE', 'araF', 'araF', 'fdnG', 'fdnG', 'melR', 'melR', 'chbB', 'chbB', 'chbB', 'chbB', 'glmU', 'glmU', 'citC', 'citC', 'nagBE', 'nagBE', 'nagBE', 'nagBE', 'citAB', 'citAB', 'fimB', 'fimB', 'chbB', 'chbB', 'chbB', 'chbB', 'melAB', 'melAB', 'nrfA', 'nrfA', 'nrfA', 'melR', 'melR', 'napF', 'napF', 'napF', 'nagBE', 'nagBE', 'nagBE', 'nagBE', 'nagBE', 'zwf', 'zwf', 'nagBE', 'nagBE', 'rhaSR', 'rhaSR', 'rhaSR', 'rhaSR', 'chbCBA', 'chbCBA', 'melR', 'melR', 'melR', 'ptsG', 'ptsG', 'ptsG', 'ptsG', 'argO', 'argO', 'malEFG', 'malEFG', 'malT', 'malT', 'fdnGHI', 'fdnGHI', 'galS', 'galS', 'mglB', 'mglB', 'rbsDABCK', 'rbsDABCK', 'rhaBAD', 'rhaBAD', 'hemA', 'hemA', 'glcDEFGB', 'glcDEFGB', 'glcDEFGB', 'glcDEFGB', 'rhaSR', 'rhaSR', 'frdA', 'frdA', 'dmsA', 'dmsA', 'dmsA', 'dmsA', 'frdA', 'frdA', 'narG', 'narG', 'araBAD', 'araBAD', 'fdnGHI', 'fdnGHI', 'rhaSR', 'rhaSR', 'ydhY', 'ydhY', 'aeg', 'aeg', 'dmsA', 'dmsA', 'frdA', 'frdA', 'aeg', 'aeg', 'melR', 'melR', 'aeg', 'aeg', 'glmUS', 'glmUS', 'nagE', 'nagE', 'nagE', 'nagE', 'aeg', 'aeg', 'rhaBAD', 'rhaBAD', 'glmUS', 'glmUS', 'rhaSR', 'rhaSR', 'rhaSR', 'rhaSR', 'ytfE', 'ytfE', 'argO', 'argO', 'lysP', 'lysP', 'nag', 'nag', 'argO', 'argO', 'dmsA', 'dmsA', 'alkA', 'alkA', 'ompC', 'ompC', 'gntKU', 'gntKU', 'gntT', 'gntT', 'dnaA', 'dnaA', 'tolC', 'tolC', 'tolC', 'tolC', 'tolC', 'tolC', 'fumC', 'fumC', 'fumC', 'fumC', 'micF', 'micF', 'zwf', 'zwf', 'zwf', 'zwf', 'zwf', 'fimB', 'fimB', 'fimB', 'fimB', 'fimB', 'fimB', 'fumC', 'not ', 'fumC', 'not ', 'micF', ' or ', 'micF', ' or ', 'zwf', 'e z', 'zwf', 'zwf', 'e z', ' of', 'hmpA', 'hmpA', 'ytfE', 'ytfE', 'dnaA', 'dnaA', 'melAB', 'melAB', 'melAB', 'melAB', 'argK', 'argK', 'rob', 'rob', 'gntT', 'gntT', 'rhaSR', 'rhaSR', 'dmsA', 'dmsA', 'rhaBAD', 'rhaBAD', 'yggA', 'yggA', 'marRAB', 'marRAB', 'nfnB', 'nfnB', 'chiP', 'chiP', 'chiP', 'argO', 'argO', 'argO', 'argO', 'nfnB', 'nfnB', 'nfnB', 'expression', 'inaA', 'inaA', 'inaA', 'inaA', 'fumC', 'fumC', 'micF', 'micF', 'melR', 'melR', 'galP', 'galP', 'galP', 'galP', 'gntT', 'gntT', 'melR', 'melR', 'melR', 'dapB', 'dapB', 'melR', 'melR', 'micF', 'micF', 'by', 'expression', 'by', 'expression', 'rhaBAD', 'rhaBAD', 'melR', 'melR', 'transcription', 'dmsA', 'zwf', 'zwf', 'zwf', 'zwf', 'melAB', 'melAB', 'lysP', 'lysP', 'gatA', 'gatA', 'gatA', 'gatC', 'gatC', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'nrfA', 'nrfA', 'nagB', 'nagB', 'nagE', 'nagE', 'nagE', 'nagE', 'treB', 'treB', 'hcp', 'hcp', 'fimB', 'fimB', 'fimB', 'fimB', 'gntV', 'gntV', 'lysP', 'lysP', 'melAB', 'melAB', 'rhaSR', 'rhaSR', 'ydhY', 'ydhY', 'ydhY', 'ydhY', 'caiF', 'caiF', 'caiF', 'caiF', 'melAB', 'melAB', 'eda', 'eda', 'eda', 'nagB', 'nagB', 'nagE', 'nagE', 'cadBA', 'cadBA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', ' dna', 'gntT', 'gntT', 'gntT', 'gntT', ' mel', ' th', ' th', ' mel', 'rhaBAD', 'rhaBAD', 'rhaBAD', 'rhaBAD', 'sodA', 'sodA', 'sodA', 'sodA', 'gntT', 'gntT', 'gntT', 'gntT', 'gntT', 'gntT', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'hcp', 'hcp', 'hcp', 'hcp', 'hcp', 'hcp', 'hcp', 'hcp', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'argO', 'acsP1', 'acsP1', 'acsP1', 'acsP1', ' yea', 'yeaR', 'argK', 'argK', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'rimK', 'rimK', 'rimK', 'rimK', 'rimK', 'rimK', 'rimK', 'rimK', 'nagE', 'nagE', 'nagE', 'nagE', 'nagE', 'nagE', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'marRAB', 'marRAB', 'marRAB', 'marRAB', 'melAB', 'melAB', 'melAB', 'melAB', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'lysP', 'lysP', 'lysP', 'lysP', 'araC', 'araBAD', 'araBAD', 'araBAD', 'araC', 'araC', 'araC', 'araC', 'araC', 'araC', 'ogt', 'ogt', 'ogt', 'ogt', 'ogt', 'ogt', 'melAB', 'melAB', 'melAB', 'melAB', 'dnaA', 'dnaA', 'dnaA', 'dnaA', 'melAB', 'melAB', 'melAB', 'melAB', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'argO', 'dcuB', 'dcuB', 'dcuB', 'dcuB', 'dcuB', 'dcuB', 'rob', 'rob', 'rob', 'rob', 'lysP', 'lysP', 'lysP', 'lysP', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'rhaT', 'yggA', 'yggA', 'yggA', 'yggA', 'lysA', 'lysA', 'lysA', 'lysA', 'argK', 'argK', 'argK', 'argK', 'argK', 'argK', 'argK', 'argK', 'argK', 'argK', 'melAB', 'melAB', 'melAB', 'melAB', 'rhaSR', 'rhaSR', 'rhaSR', 'rhaSR', 'lysP', 'lysP', 'lysP', 'lysP', 'lysP', 'lysP', 'melR', 'melR', 'melR', 'melR', 'melR', 'melR', 'melR', 'melR', 'nagE', 'nagE', 'nagE', 'nagE', 'nagE', 'nagE', 'dcuB', 'dcuB', 'dcuB', 'dcuB', 'argO', 'dnaA', 'melAB', 'melR', 'melR', 'fdnGHI', 'gntT', 'manXYZ', 'nagE', 'ptsG', 'ptsHIcrr', 'ogt', 'ogt', 'yeaR', 'yeaR', 'argO', 'argO', 'narGHJI', 'narK', 'araBAD', 'araFGH', 'araFGH', 'gntKU', 'gntT', 'melR', 'gntV', 'araBAD', 'melAB', 'dnaA', 'melAB', 'melAB', 'nrfA', 'nrfA', 'rhaBAD', 'rhaSR', 'rhaT', 'ynfEFGHI', 'argO', 'asd', 'dapB', 'dapD', 'fimB', 'gdhA', 'hdeA', 'lysA', 'lysC', 'lysP', 'manX', 'purA', 'nanC', 'nanC', 'argO', 'argO', 'dapB', 'fpr', 'nfo', 'ogt', 'sodA', 'yeaR', 'gntKU', 'add', 'udk', 'thcb', 'thcb', 'dapB', 'hdeA', 'fpr', 'fpr', 'nfnB', 'nfo', 'nfo', 'sodA', 'sodA', 'araBAD', 'chiP', 'chiP', 'melAB', 'adhE', 'aslB', 'fumC', 'inaA', 'inaA', 'marR', 'mdlA', 'ogt', 'sodA', 'ybaO', 'ybiS', 'yeaR', 'yfhD', 'melAB', 'melR', 'napF', 'galT', 'araBAD', 'araC', 'melAB', 'melAB', 'napF', 'napF', 'araBAD', 'araBAD', 'marRAB', 'marRAB', 'marA', 'marB', 'melA', 'melB', 'aeg', 'malK', 'malK', 'acrAB', 'inaA', 'micF', 'slp', 'araE', 'nirB', 'ogt', 'rhaSR', 'fpr', 'fpr', 'hdeA', 'purA', 'xylF', 'xylF', 'purA', 'melR', 'rbsDACBK', 'acnA', 'fpr', 'fumC', 'hdeA', 'mlc', 'sodA', 'zwf', 'narK', 'narK', 'araC', 'argP', 'hcp', 'melAB', 'aeg', 'soxS', 'soxS', 'acrAB', 'acrAB', 'acrAB', 'dnaA', 'gntKU', 'marRAB', 'acrZ', 'acrZ', 'acrZ', 'tolC', 'tolC', 'tolC', 'fdnG', 'melAB', 'nirB', 'nirBDC', 'nirBDC', 'nirBDC', 'nrfA', 'nrfABCDEFG', 'nrfABCDEFG', 'nrfABCDEFG', 'galP', 'acrAB', 'marRAB', 'hdeA', 'purA', 'acnA', 'fldA', 'fldB', 'fpr', 'fumC', 'fur', 'inaA', 'micF', 'nfo', 'nfsA', 'ribA', 'rimK', 'sodA', 'tolC', 'zwf', 'araBAD', 'alkA', 'fdnG', 'narK', 'galETK', 'galETK', 'marRAB', 'nirB', 'inaA', 'araBAD', 'araBAD', 'dmsA', 'gntT', 'cydA', 'nirB', 'nirB', 'nrfA', 'nrfA', 'rhaT', 'araE', 'araE', 'melR', 'rhaSR', 'fpr', 'fumC', 'nfo', 'araBAD', 'araBAD', 'rhaSR', 'fumC', 'fumC', 'inaA', 'inaA', 'micF', 'micF', 'sodA', 'zwf', 'zwf', 'ynfE', 'melAB', 'melAB', 'dmsA', 'soxS', 'nfnB', 'yggA', 'chiP', 'yjhA', 'argO', 'melAB', 'rhaSR', 'dmsA', 'dnaA', 'xylAB', 'xylFGH', 'hcp', 'hcp', 'dctA', 'dcuB', 'fumB', 'nagE', 'hcp', 'hcp', 'hcp', 'napF', 'cadBA', 'rhaBAD', 'dcuB', 'melR', 'inaA', 'argO', 'dnaA', 'dnaA', 'melAB', 'mdaA', 'micF', 'melAB', 'melR', 'nag', 'nfnB', 'nfnB', 'nfnB', 'melR', 'dnaA', 'fumC', 'micF', 'aeg', 'argO', 'dapB', 'nfnB', 'acrAB', 'mdfA', 'yhiUV', 'fpr', 'fumC', 'fumC', 'inaA', 'micF', 'micF', 'nfo', 'sodA', 'zwf', 'fpr', 'fpr', 'fumC', 'fumC', 'micF', 'micF', 'nfo', 'nfo', 'sodA', 'sodA', 'zwf', 'zwf', 'melAB', 'emrRAB', 'argO', 'dmsA', 'hdeA', 'fimB', 'argP', 'fimB', 'melR', 'galP', 'ynfE', 'zwf', 'zwf', 'acrAB', 'exuT', 'uxaB', 'uxaCA', 'tolC', 'acnA', 'acnA', 'acnA', 'acnA', 'dnaA', 'nfnB', 'zwf', 'dnaA', 'uidAB', 'uidAB', 'gntT', 'fpr', 'zwf', 'zwf', 'marRAB', 'dnaA', 'nfnB', 'dnaA', 'melAB', 'rpmH', 'dnaA', 'hdeA', 'purA', 'hdeA', 'argO', 'dnaA', 'hdeA', 'purA', 'dapB', 'galT', 'argO', 'hmpA', 'napF', 'ygbA', 'ytfE', 'aeg', 'hdeA', 'purA', 'metD', 'araBAD', 'ydbC', 'melR', 'nanC', 'acnA', 'dapB', 'dnaA', 'hcp', 'hmpA', 'melR', 'ytfE', 'argO', 'melR', 'marA', 'melR', 'marRAB', 'ilvIH', 'dapB', 'eda', 'gntT', 'melR', 'dnaA', 'dnaA', 'metH', 'b1452', 'dnaA', 'dnaA', 'melR', 'melR', 'cadBA', 'rhaBAD', 'dnaA', 'melR', 'araB', 'fimB', 'lysP', 'adhE', 'aeg', 'nfnB', 'fldB', 'hdeA', 'mdaA', 'nfo', 'ribA', 'rimK', 'tolC', 'fimB', 'fimB', 'lysP', 'fimB', 'melAB', 'dnaA', 'yhbW', 'fimB', 'fimB', 'fimB', 'melR', 'melAB', 'metE', 'galP', 'galP', 'melR', 'rhaSR', 'rhaSR', 'dnaA', 'purB', 'araBAD', 'araC', 'chiP', 'hcp', 'gntT', 'nrfA', 'ytfE', 'nag', 'aeg', 'lysP', 'fdnG', 'inaA', 'marRAB', 'melR', 'nrfA', 'fimB', 'nanC', 'dnaA', 'fimB', 'nrdA', 'araBAD', 'araBAD', 'rob', 'rob', 'chbBCARFG', 'fimB', 'metE', 'metR', 'micF', 'micF', 'micF', 'melAB', 'rhaSR', 'nrfA', 'dcuB', 'hdeA', 'purA', 'metE', 'rhaSR', 'lysP', 'yjhA', 'melAB', 'purA', 'araBAD', 'rob', 'araBAD', 'melAB', 'napF', 'melAB', 'melAB', 'mefA', 'chbB', 'fimB', 'fimB', 'yeaR', 'araBAD', 'araBAD', 'citC', 'fimB', 'chbB', 'fimB', 'melR', 'inaA', 'araBAD', 'galP', 'galP', 'galP', 'gntT', 'melR', 'fumC', 'micF', 'nrfA', 'melR', 'micF', 'chbB', 'fimB', 'fimB', 'napF', 'ompC', 'waaP', 'waaY', 'melAB', 'fimB', 'fimB', 'melAB', 'nanC', 'ogt', 'nrfA', 'fimB', 'fumC', 'inaA', 'sodA', 'argK', 'lysP', 'rhaBAD', 'chiP', 'melR', 'fimB', 'galE', 'yggA', 'napF', 'gltA', 'nfnB', 'ybjC', 'lysP', 'melR', 'ybjC', 'rhaBAD', 'yggA', 'fucPIK', 'melR', 'treC', 'osmE', 'hdeA', 'purA', 'fdnG', 'galP', 'narG', 'nrfA', 'napF', 'ogt', 'yeaR', 'artPIQM', 'artPIQM', 'araBAD', 'napF', 'fimB', 'marRAB', 'acrZ', 'acrZ', 'acrZ', 'aslB', 'galT', 'inaA', 'mdlA', 'ybaO', 'ybiS', 'yfhD', 'marRAB', 'araC', 'fimB', 'fimB', 'melAB', 'fdnGHI', 'manXYZ', 'mdlA', 'micF', 'galP', 'galP', 'glmUS', 'fimB', 'melAB', 'nanC', 'nanC', 'rhaBAD', 'rhaBAD', 'rhaT', 'araBAD', 'marA', 'rhaSR', 'chbB', 'chbB', 'melAB', 'aeg', 'fdnG', 'gntKU', 'argO', 'argO', 'lysP', 'lysP', 'argO', 'argO', 'galT', 'chbBCARFG', 'chbBCARFG', 'chbBCARFG', 'yeaR', 'narGHJI', 'fimB', 'nirB', 'nrfA', 'ydhY', 'dcuSR', 'norVW', 'hmp', 'micF', 'norB', 'ompC', 'rhaSR', 'ydhY', 'fur', 'fur', 'nirB', 'nirB', 'nrfA', 'nrfA', 'nirB', 'nrfA', 'nrfA', 'nrfA', 'lysP', 'xylA', 'melR', 'xylAB', 'melAB', 'melAB', 'melAB', 'rhaSR', 'nirB', 'fpr', 'eda', 'gntT', 'manXYZ', 'nagE', 'ptsG', 'ptsHIcrr', 'gntKU', 'chbB', 'gntT', 'aeg', 'rhaSR', 'galETK', 'galETK', 'galETK', 'melAB', 'nirB', 'nirB', 'nirB', 'nirB', 'nirB', 'melAB', 'gntT', 'gntT', 'rhaBAD', 'rhaS', 'rpiB', 'gntKU', 'nagB', 'rhaBAD', 'dnaA', 'cadBA', 'rpoH', 'dapB', 'araJ', 'xylA', 'dapB', 'ansB', 'ansB', 'araBAD', 'melAB', 'narGHJI', 'polB', 'ygeA', 'melAB', 'rhaBAD', 'araC', 'araBAD', 'ytfQ', 'ytfQ', 'cadBA', 'lysP', 'melR', 'melR', 'argO', 'melAB', 'lysP', 'ytfQ', 'cadBA', 'melAB', 'melAB', 'narGHJI', 'polB', 'gntT', 'ansB', 'ansB', 'leuE', 'marRAB', 'melAB', 'ansB', 'ansB', 'udp', 'dnaA', 'melAB', 'dnaA', 'rhaSR', 'nagE', 'nagE', 'narGHJI', 'dadAX', 'dadAX', 'uhpT', 'uhpT', 'melR', 'deoCABC', 'cadBA', 'ilvIH', 'marRAB', 'narG', 'ansB', 'ansB', 'ompC', 'ompF', 'argO', 'argO', 'rhaSR', 'proP', 'ompF', 'ompC', 'ompF', 'deoCABD', 'cdd', 'cdd']\n"
     ]
    }
   ],
   "source": [
    "print(regulated_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_span_cols = data\n",
    "data_span_cols['span_regulator_regulated'] = span_list\n",
    "data_span_cols['regulator_span_found'] = regulator_list\n",
    "data_span_cols['regulated_span_found'] = regulated_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "          PMID  NUMSENT REGULATOR REGULATED REGULATED_TYPE REGULATED_GENE  \\\n",
      "0     19906180        2       Lrp      argO             TU            NaN   \n",
      "1     19906180        2       Lrp      argO             TU            NaN   \n",
      "2     11073923        8       CRP     rhaSR             TU            NaN   \n",
      "3     11073923        8       CRP     rhaSR             TU            NaN   \n",
      "4     17449618        9      NsrR      hmpA             TU            NaN   \n",
      "...        ...      ...       ...       ...            ...            ...   \n",
      "1557  11601842      854      OmpR      ompC           GENE           ompC   \n",
      "1558  11601842      854      OmpR      ompF           GENE           ompF   \n",
      "1559   1848637      861      DeoR   deoCABD             TU            NaN   \n",
      "1560   1848637      866       CRP       cdd             TU            NaN   \n",
      "1561   1848637      866      CytR       cdd             TU            NaN   \n",
      "\n",
      "     REGULATED_TU      EFFECT NORMALIZED_EFFECT  \\\n",
      "0            argO   regulated         regulator   \n",
      "1            argO   regulated       no_relation   \n",
      "2           rhaSR  activation         activator   \n",
      "3           rhaSR  activation       no_relation   \n",
      "4            hmpA   represses         repressor   \n",
      "...           ...         ...               ...   \n",
      "1557          NaN  activation         activator   \n",
      "1558          NaN  activation         activator   \n",
      "1559      deoCABD  repression         repressor   \n",
      "1560          cdd   regulated         regulator   \n",
      "1561          cdd   regulated         regulator   \n",
      "\n",
      "                                               SENTENCE  ... regulated_count  \\\n",
      "0     Summary In vivo and in vitro analyses indicate...  ...               1   \n",
      "1     Summary In vivo and in vitro analyses indicate...  ...               1   \n",
      "2     To address the mechanisms of activation by CRP...  ...               1   \n",
      "3     To address the mechanisms of activation by CRP...  ...               1   \n",
      "4     Microarray analysis revealed that NsrR repress...  ...               1   \n",
      "...                                                 ...  ...             ...   \n",
      "1557  Identification of the DNA - binding domain of ...  ...               1   \n",
      "1558  Identification of the DNA - binding domain of ...  ...               1   \n",
      "1559  DNA - protein recognition : demonstration of t...  ...               1   \n",
      "1560  CRP / cAMP and CytR regulated promoters in Esc...  ...               1   \n",
      "1561  CRP / cAMP and CytR regulated promoters in Esc...  ...               1   \n",
      "\n",
      "     regulator_plus_one  regulated_plus_one  regulator_regulated_plus_one  \\\n",
      "0                  True               False                          True   \n",
      "1                  True               False                          True   \n",
      "2                  True               False                          True   \n",
      "3                  True               False                          True   \n",
      "4                  True               False                          True   \n",
      "...                 ...                 ...                           ...   \n",
      "1557              False               False                         False   \n",
      "1558              False               False                         False   \n",
      "1559              False               False                         False   \n",
      "1560              False               False                         False   \n",
      "1561              False               False                         False   \n",
      "\n",
      "      regulated_tag  regulator_tag  \\\n",
      "0       @Regulated$           @TF$   \n",
      "1       @Regulated$           @TF$   \n",
      "2       @Regulated$           @TF$   \n",
      "3       @Regulated$           @TF$   \n",
      "4       @Regulated$           @TF$   \n",
      "...             ...            ...   \n",
      "1557    @Regulated$           @TF$   \n",
      "1558    @Regulated$           @TF$   \n",
      "1559    @Regulated$           @TF$   \n",
      "1560    @Regulated$           @TF$   \n",
      "1561    @Regulated$           @TF$   \n",
      "\n",
      "                                        sentence_tagged  \\\n",
      "0     Summary In vivo and in vitro analyses indicate...   \n",
      "1     Summary In vivo and in vitro analyses indicate...   \n",
      "2     To address the mechanisms of activation by @TF...   \n",
      "3     To address the mechanisms of activation by CRP...   \n",
      "4     Microarray analysis revealed that @TF$ repress...   \n",
      "...                                                 ...   \n",
      "1557  Identification of the DNA - binding domain of ...   \n",
      "1558  Identification of the DNA - binding domain of ...   \n",
      "1559  DNA - protein recognition : demonstration of t...   \n",
      "1560  @TF$ / cAMP and CytR regulated promoters in Es...   \n",
      "1561  CRP / cAMP and @TF$ regulated promoters in Esc...   \n",
      "\n",
      "      span_regulator_regulated regulator_span_found regulated_span_found  \n",
      "0       [[168, 171], [73, 77]]                  Lrp                 argO  \n",
      "1       [[252, 255], [73, 77]]                  Lrp                 argO  \n",
      "2       [[43, 46], [113, 118]]                  CRP                rhaSR  \n",
      "3     [[171, 174], [113, 118]]                  CRP                rhaSR  \n",
      "4       [[34, 38], [123, 127]]                 NsrR                 hmpA  \n",
      "...                        ...                  ...                  ...  \n",
      "1557    [[50, 54], [119, 123]]                 OmpR                 ompC  \n",
      "1558    [[50, 54], [110, 114]]                 OmpR                 ompF  \n",
      "1559  [[173, 177], [148, 155]]                 DeoR              deoCABD  \n",
      "1560        [[0, 3], [70, 73]]                  CRP                  cdd  \n",
      "1561      [[15, 19], [70, 73]]                 CytR                  cdd  \n",
      "\n",
      "[1562 rows x 23 columns]\n",
      "<class 'list'>\n",
      "<class 'pandas.core.frame.DataFrame'>\n"
     ]
    }
   ],
   "source": [
    "print(data_span_cols)\n",
    "print(type(data_span_cols.iloc[:,20][0]))\n",
    "print(type(data_span_cols))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# only one regulator - one regulated file \n",
    "#data_span_cols.to_csv(\"../results/ECO/one_regulator_one_regulated/ECO_dataset_one_regulator_one_regulated_whole_tagging_info_with_span_data_v1.tsv\",sep = \"\\t\",index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [],
   "source": [
    "# All curated but multi regulator -  multi regulated\n",
    "#data_span_cols.to_csv(\"../results/ECO/curated_master_dataset/ECO_dataset_master_curated_whole_tagging_info_with_span_data_v1.tsv\",sep=\"\\t\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "# All curated including multi regulator - multi regulated \n",
    "data_span_cols.to_csv(\"../results/ECO/curated_master_dataset/ECO_dataset_master_curated_whole_tagging_info_with_span_data_v2.tsv\",sep=\"\\t\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# First pikle try using pandas library\n",
    "#data_span_cols.to_pickle(\"../results/ECO/one_regulator_one_regulated/ECO_dataset_one_regulator_one_regulated_whole_tagging_info_with_span_data_v1.pkl\",protocol=4)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Creating a pickle compatible with ascend server which has a lower version of pandas and python 3.7 meaning protocol 4 and not 5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pickle"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "# All curated including multi regulator - mulit regulated using pandas 1.3.5 and protocol 4 of serialization with pickle library\n",
    "with open(\"../results/ECO/curated_master_dataset/ECO_dataset_master_curated_whole_tagging_info_with_span_data_serialized_4_v2.pkl\",\"wb\") as f:\n",
    "    pickle.dump(data_span_cols,f,protocol=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Only one regulator - one regulated subset using pandas 1.3.5 and protocol 4 of serialization with pickle library\n",
    "#with open(\"../results/ECO/one_regulator_one_regulated/ECO_dataset_one_regulator_one_regulated_whole_tagging_info_with_span_data_serialized_4_v1.pkl\",\"wb\") as f:\n",
    "#    pickle.dump(data_span_cols,f,protocol=4)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'1.3.5'"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.__version__"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "31f2aee4e71d21fbe5cf8b01ff0e069b9275f58929596ceb00d14d90e3e16cd6"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
