{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Error analysis of the best model \n",
    "- model: LUKE\n",
    "- batch_size: 32\n",
    "- learning_rate: 0.00001\n",
    "- epoch: 12"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Deep learning framework\n",
    "import torch\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "# Download existing tokenizer for the selected model\n",
    "from transformers import LukeTokenizer\n",
    "# For creating the batches for training \n",
    "from torch.utils.data import Dataset, DataLoader\n",
    "#For spliting training, validation and test groups\n",
    "from sklearn.model_selection import train_test_split\n",
    "from pytorch_lightning import Trainer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Model construction framework\n",
    "import pytorch_lightning as pl\n",
    "# Download the pretrained model with a classification head on top\n",
    "from transformers import LukeForEntityPairClassification\n",
    "# Meassuring performance of the model F1-score\n",
    "from torchmetrics.classification import MulticlassF1Score\n",
    "# Meassuring actual vs predicted groups\n",
    "from torchmetrics.classification import ConfusionMatrix\n",
    "import torchmetrics\n",
    "# Live performance metrics of the model\n",
    "import wandb\n",
    "from pytorch_lightning.loggers import WandbLogger"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Seeding everything as in the training of the model for comparisson porpouses "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<torch._C.Generator at 0x10d5e98b0>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Setting a seed for random operations reproducibility \n",
    "torch.manual_seed(12222)\n",
    "## Getting the dataset splits "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Getting the dataset splits "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['PMID', 'NUMSENT', 'REGULATOR', 'REGULATED', 'REGULATED_TYPE',\n",
       "       'REGULATED_GENE', 'REGULATED_TU', 'EFFECT', 'NORMALIZED_EFFECT',\n",
       "       'SENTENCE', 'regulator_regex', 'regulated_regex', 'regulator_count',\n",
       "       'regulated_count', 'regulator_plus_one', 'regulated_plus_one',\n",
       "       'regulator_regulated_plus_one', 'regulated_tag', 'regulator_tag',\n",
       "       'sentence_tagged', 'span_regulator_regulated', 'regulator_span_found',\n",
       "       'regulated_span_found'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# complete data with multi regultor - multi regulated\n",
    "data = pd.read_pickle(\"../../01_preprocessing/results/ECO/curated_master_dataset/ECO_dataset_master_curated_whole_tagging_info_with_span_data_serialized_4_v2.pkl\")\n",
    "data.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                               sentence  \\\n",
      "0     Summary In vivo and in vitro analyses indicate...   \n",
      "1     Summary In vivo and in vitro analyses indicate...   \n",
      "2     To address the mechanisms of activation by CRP...   \n",
      "3     To address the mechanisms of activation by CRP...   \n",
      "4     Microarray analysis revealed that NsrR repress...   \n",
      "...                                                 ...   \n",
      "1557  Identification of the DNA - binding domain of ...   \n",
      "1558  Identification of the DNA - binding domain of ...   \n",
      "1559  DNA - protein recognition : demonstration of t...   \n",
      "1560  CRP / cAMP and CytR regulated promoters in Esc...   \n",
      "1561  CRP / cAMP and CytR regulated promoters in Esc...   \n",
      "\n",
      "                  entity_spans        label  \\\n",
      "0       [[168, 171], [73, 77]]    regulator   \n",
      "1       [[252, 255], [73, 77]]  no_relation   \n",
      "2       [[43, 46], [113, 118]]    activator   \n",
      "3     [[171, 174], [113, 118]]  no_relation   \n",
      "4       [[34, 38], [123, 127]]    repressor   \n",
      "...                        ...          ...   \n",
      "1557    [[50, 54], [119, 123]]    activator   \n",
      "1558    [[50, 54], [110, 114]]    activator   \n",
      "1559  [[173, 177], [148, 155]]    repressor   \n",
      "1560        [[0, 3], [70, 73]]    regulator   \n",
      "1561      [[15, 19], [70, 73]]    regulator   \n",
      "\n",
      "                                        sentence_tagged regulator regulated  \n",
      "0     Summary In vivo and in vitro analyses indicate...       Lrp      argO  \n",
      "1     Summary In vivo and in vitro analyses indicate...       Lrp      argO  \n",
      "2     To address the mechanisms of activation by @TF...       CRP     rhaSR  \n",
      "3     To address the mechanisms of activation by CRP...       CRP     rhaSR  \n",
      "4     Microarray analysis revealed that @TF$ repress...      NsrR      hmpA  \n",
      "...                                                 ...       ...       ...  \n",
      "1557  Identification of the DNA - binding domain of ...      OmpR      ompC  \n",
      "1558  Identification of the DNA - binding domain of ...      OmpR      ompF  \n",
      "1559  DNA - protein recognition : demonstration of t...      DeoR   deoCABD  \n",
      "1560  @TF$ / cAMP and CytR regulated promoters in Es...       CRP       cdd  \n",
      "1561  CRP / cAMP and @TF$ regulated promoters in Esc...      CytR       cdd  \n",
      "\n",
      "[1562 rows x 6 columns]\n"
     ]
    }
   ],
   "source": [
    "# Creating a smaller dataframe with the 3 columns that we need \n",
    "df = data.filter(['SENTENCE','span_regulator_regulated','NORMALIZED_EFFECT','sentence_tagged','REGULATOR','REGULATED'], axis=1)\n",
    "\n",
    "df.rename(columns = {\"SENTENCE\":\"sentence\",\"sentence_tagged\":\"sentence_tagged\",\"REGULATOR\":\"regulator\",\"REGULATED\":\"regulated\",\"span_regulator_regulated\":\"entity_spans\",\"NORMALIZED_EFFECT\":\"label\"},\n",
    "               inplace = True)\n",
    "print(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sentence example: \n",
      " Summary In vivo and in vitro analyses indicate that transcription of the argO gene coding for an arginine exporter is regulated by the global transcriptional regulator Lrp , an effect that went by unnoticed in previous genome - scale screenings of the Lrp regulatory network in Escherichia coli .\n",
      "Labels: \n",
      " ['regulator' 'no_relation' 'activator' 'repressor']\n",
      "Label distribution: \n",
      " activator      593\n",
      "no_relation    493\n",
      "repressor      269\n",
      "regulator      207\n",
      "Name: label, dtype: int64\n"
     ]
    }
   ],
   "source": [
    "print(\"Sentence example: \\n {}\".format(df.sentence[0]))\n",
    "print(\"Labels: \\n {}\".format(df.label.unique()))\n",
    "print(\"Label distribution: \\n {}\".format(df.label.value_counts()))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Changing data type of entity spans to numpy array"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'str'>\n",
      "<class 'list'>\n",
      "<class 'str'>\n"
     ]
    }
   ],
   "source": [
    "#print(data.dtypes)\n",
    "print(type(df.iloc[:,0][0]))\n",
    "print(type(df.iloc[:,1][1]))\n",
    "print(type(df.iloc[:,2][2]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "df['entity_spans'] = df['entity_spans'].apply(lambda x: np.array(list(x)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                               sentence  \\\n",
      "0     Summary In vivo and in vitro analyses indicate...   \n",
      "1     Summary In vivo and in vitro analyses indicate...   \n",
      "2     To address the mechanisms of activation by CRP...   \n",
      "3     To address the mechanisms of activation by CRP...   \n",
      "4     Microarray analysis revealed that NsrR repress...   \n",
      "...                                                 ...   \n",
      "1557  Identification of the DNA - binding domain of ...   \n",
      "1558  Identification of the DNA - binding domain of ...   \n",
      "1559  DNA - protein recognition : demonstration of t...   \n",
      "1560  CRP / cAMP and CytR regulated promoters in Esc...   \n",
      "1561  CRP / cAMP and CytR regulated promoters in Esc...   \n",
      "\n",
      "                  entity_spans        label  \\\n",
      "0       [[168, 171], [73, 77]]    regulator   \n",
      "1       [[252, 255], [73, 77]]  no_relation   \n",
      "2       [[43, 46], [113, 118]]    activator   \n",
      "3     [[171, 174], [113, 118]]  no_relation   \n",
      "4       [[34, 38], [123, 127]]    repressor   \n",
      "...                        ...          ...   \n",
      "1557    [[50, 54], [119, 123]]    activator   \n",
      "1558    [[50, 54], [110, 114]]    activator   \n",
      "1559  [[173, 177], [148, 155]]    repressor   \n",
      "1560        [[0, 3], [70, 73]]    regulator   \n",
      "1561      [[15, 19], [70, 73]]    regulator   \n",
      "\n",
      "                                        sentence_tagged regulator regulated  \n",
      "0     Summary In vivo and in vitro analyses indicate...       Lrp      argO  \n",
      "1     Summary In vivo and in vitro analyses indicate...       Lrp      argO  \n",
      "2     To address the mechanisms of activation by @TF...       CRP     rhaSR  \n",
      "3     To address the mechanisms of activation by CRP...       CRP     rhaSR  \n",
      "4     Microarray analysis revealed that @TF$ repress...      NsrR      hmpA  \n",
      "...                                                 ...       ...       ...  \n",
      "1557  Identification of the DNA - binding domain of ...      OmpR      ompC  \n",
      "1558  Identification of the DNA - binding domain of ...      OmpR      ompF  \n",
      "1559  DNA - protein recognition : demonstration of t...      DeoR   deoCABD  \n",
      "1560  @TF$ / cAMP and CytR regulated promoters in Es...       CRP       cdd  \n",
      "1561  CRP / cAMP and @TF$ regulated promoters in Esc...      CytR       cdd  \n",
      "\n",
      "[1562 rows x 6 columns]\n"
     ]
    }
   ],
   "source": [
    "print(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'str'>\n",
      "<class 'numpy.ndarray'>\n",
      "<class 'str'>\n"
     ]
    }
   ],
   "source": [
    "#print(data.dtypes)\n",
    "print(type(df.iloc[:,0][0]))\n",
    "print(type(df.iloc[:,1][1]))\n",
    "print(type(df.iloc[:,2][2]))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Creating 2 dictionaries \n",
    "We have 4 labels: activator, repressor, regulator, no_relation\n",
    "1. id2label : maps each label to a unique integer \n",
    "2. label2id: maps a unique integer to each label "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{0: 'activator', 1: 'no_relation', 2: 'repressor', 3: 'regulator'}\n"
     ]
    }
   ],
   "source": [
    "id2label = dict()\n",
    "for idx, label in enumerate(df.label.value_counts().index):\n",
    "  id2label[idx] = label\n",
    "print(id2label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['activator', 'no_relation', 'repressor', 'regulator']\n"
     ]
    }
   ],
   "source": [
    "# For test confusion matrix\n",
    "class_conf_labels = []\n",
    "for keys, values in id2label.items():\n",
    "    class_conf_labels.append(values)\n",
    "\n",
    "print(class_conf_labels)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'activator': 0, 'no_relation': 1, 'repressor': 2, 'regulator': 3}\n"
     ]
    }
   ],
   "source": [
    "label2id = {v:k for k,v in id2label.items()}\n",
    "print(label2id)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Define the PyTorch dataset and dataloaders \n",
    "In PyTorch you need to define a Dataset class and Dataloaders.\n",
    "**Dataset class**: This class stores the samples and their corresponding labels.\n",
    "**Dataloaders class**: wraps an iterable around the Dataset to enable easy access to the samples.\n",
    "\n",
    "\n",
    "\n",
    "## 3 Methods must be implemented \n",
    "1. *init* method: itnitializing the dataset with data.\n",
    "2. *len* method: returns the number of elements in the dataset.\n",
    "3. *getitem()* method: returns a single item from the dataset. \n",
    "\n",
    "## Each item of the dataset we will use must have 3 columns \n",
    "1. Sentence \n",
    "2. Spans of the 2 entities. Eg. [(0, 3), (70, 73)]\n",
    "3. label of the relationship corresponding to the \"normalized effect\" column.  Eg. activator\n",
    "\n",
    "# Tokenization \n",
    "## The expected input for the model consists of: \n",
    "1. input_ids\n",
    "   - Tensor: Indices of input sequence tokens in the vocabulary. They are numerical representations of tokens that build the input sequence. \n",
    "     - The shape of the tensor is [batch_size, sequence_length].\n",
    "2. entity_ids (particular from Luke)\n",
    "3. attention_mask\n",
    "  - Indicates whether a token should be attended to or not.\n",
    "4. entity_attention_mask (particular from Luke)\n",
    "5. entity_position_ids (particular from Luke)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "# We use this tokenizer from transformer library to turn the dataset into the inputs expected by the model \n",
    "tokenizer = LukeTokenizer.from_pretrained(\"studio-ousia/luke-base\", task=\"entity_pair_classification\")\n",
    "\n",
    "\n",
    "class RelationExtractionDataset(Dataset):\n",
    "    \"\"\"Relation extraction dataset.\"\"\"\n",
    "\n",
    "    def __init__(self, data):\n",
    "        \"\"\"\n",
    "        Args:\n",
    "            data : Pandas dataframe.\n",
    "        \"\"\"\n",
    "        self.data = data\n",
    "\n",
    "    def __len__(self):\n",
    "        return len(self.data)\n",
    "\n",
    "    def __getitem__(self, idx):\n",
    "        item = self.data.iloc[idx]\n",
    "\n",
    "        sentence = item.sentence\n",
    "        sentence_tagged = item.sentence_tagged\n",
    "        regulator = item.regulator\n",
    "        regulated = item.regulated\n",
    "        entity_spans = [tuple(x) for x in item.entity_spans]\n",
    "        \n",
    "#### Tokenization step\n",
    "# return_tensors: Specify the type of tensors we want to get back (PyTorch, TensorFlow, or plain NumPy)\n",
    "# padding: Tensors, the input of the model need to have a uniform shape but the sentences are not the same length with this we add a special token to the sentences that are shorter to ensure tensors are rectangular\n",
    "# truncation: Sometimes a sequence may be too long for a model to handle. In this case we truncate the sentence to a shorter length. True, truncate a sequence to the maximum length accepted by the model \n",
    "        encoding = tokenizer(sentence, entity_spans=entity_spans, padding=\"max_length\", truncation=True, return_tensors=\"pt\")\n",
    "\n",
    "        for k,v in encoding.items():\n",
    "          encoding[k] = encoding[k].squeeze()\n",
    "\n",
    "        encoding[\"label\"] = torch.tensor(label2id[item.label])\n",
    "\n",
    "       # Return both the tokenized data and the original sentence\n",
    "        return {\n",
    "            \"tokenized_data\": encoding,\n",
    "            # store the original sentence\n",
    "            \"sentence\": sentence,\n",
    "            \"sentence_tagged\":sentence_tagged,\n",
    "            \"regulator\":regulator,\n",
    "            \"regulated\":regulated\n",
    "            \n",
    "        } "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Spliting training and test groups and making the instances of the RelationExtractionDataset class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "length train dataset : 1249\n",
      "length test dataset: 313\n"
     ]
    }
   ],
   "source": [
    "# 80% train and 20% test\n",
    "train_df, test_df = train_test_split(df, test_size=0.2, random_state=42, shuffle=True)\n",
    "\n",
    "print(\"length train dataset : {}\".format(len(train_df)))\n",
    "print(\"length test dataset: {}\".format(len(test_df)))\n",
    "\n",
    "# define the dataset\n",
    "train_dataset = RelationExtractionDataset(data=train_df)\n",
    "test_dataset = RelationExtractionDataset(data=test_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "train_dataset: <__main__.RelationExtractionDataset object at 0x16de30e10>\n",
      "test_dataset: <__main__.RelationExtractionDataset object at 0x16dd350d0>\n"
     ]
    }
   ],
   "source": [
    "print(\"train_dataset: {}\".format(train_dataset))\n",
    "print(\"test_dataset: {}\".format(test_dataset) )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "# We will use 10,16,32 and 64\n",
    "# Shuffle = True indicates that the data will ensure a different order of samples in each epoch\n",
    "train_dataloader = DataLoader(train_dataset, batch_size=32, shuffle=True)\n",
    "test_dataloader = DataLoader(test_dataset, batch_size=32)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "length train dataloader : 40\n",
      "length test dataloader: 10\n"
     ]
    }
   ],
   "source": [
    "print(\"length train dataloader : {}\".format(len(train_dataloader)))\n",
    "print(\"length test dataloader: {}\".format(len(test_dataloader)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Original Sentence: Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with the C - terminal domain of the RNA polymerase alpha subunit .\n",
      "Original Sentence tagged: Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with the C - terminal domain of the RNA polymerase alpha subunit .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for Fnr - dependent repression at the cydA P1 promoter [ 15 ] .\n",
      "Original Sentence tagged: This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for @TF$ - dependent repression at the @Regulated$ P1 promoter [ 15 ] .\n",
      "Original regulator: Fnr\n",
      "Original regulated: cydA\n",
      "Original Sentence: However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original Sentence tagged: However , @Regulated$ and frdA operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original regulator: NarL\n",
      "Original regulated: narG\n",
      "Original Sentence: There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "Original Sentence tagged: There is no obvious effect of the @TF$ + 6 mutation on @Regulated$ expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "Original regulator: CAP\n",
      "Original regulated: nagE\n",
      "Original Sentence: One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) .\n",
      "Original Sentence tagged: One site , designated the @TF$ I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the @Regulated$ promoter region ( Fig . 3 ) .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent dmsA promoter in vitro ( data not shown ) .\n",
      "Original Sentence tagged: In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the @TF$ - dependent @Regulated$ promoter in vitro ( data not shown ) .\n",
      "Original regulator: Fnr\n",
      "Original regulated: dmsA\n",
      "Original Sentence: Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original Sentence tagged: Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntKU\n",
      "Original Sentence: The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P\n",
      "Original Sentence tagged: The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the @Regulated$ promoter 1P\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "Original Sentence tagged: For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to @TF$ as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "Original regulator: NarP\n",
      "Original regulated: fdnGHI\n",
      "Original Sentence: Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "Original Sentence tagged: Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , @TF$ and GalS repression at @Regulated$ 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .\n",
      "Original regulator: GalR\n",
      "Original regulated: galP\n",
      "Original Sentence: marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .\n",
      "Original Sentence tagged:  @Regulated$ expression is repressed by MarR and is derepressed by the interaction of @TF$ with various phenolic compounds such as salicylate .\n",
      "Original regulator: MarR\n",
      "Original regulated: marA\n",
      "Original Sentence: However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original Sentence tagged: However , narG and @Regulated$ operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original regulator: NarL\n",
      "Original regulated: frdA\n",
      "Original Sentence: RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "Original Sentence tagged: RhaR activates transcription of rhaSR , and @TF$ activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , @Regulated$ , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "Original regulator: RhaS\n",
      "Original regulated: rhaBAD\n",
      "Original Sentence: Transcriptional activation of the dnaA gene encoding the initiator for oriC replication by IciA protein , an inhibitor of in vitro oriC replication in Escherichia coli .\n",
      "Original Sentence tagged: Transcriptional activation of the @Regulated$ gene encoding the initiator for oriC replication by @TF$ protein , an inhibitor of in vitro oriC replication in Escherichia coli .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Because previous studies have shown that the MarA - binding sites of two other repressed promoters ( hdeA and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .\n",
      "Original Sentence tagged: Because previous studies have shown that the @TF$ - binding sites of two other repressed promoters ( hdeA and @Regulated$ ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .\n",
      "Original regulator: MarA\n",
      "Original regulated: purA\n",
      "Original Sentence: To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "Original Sentence tagged: To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either @TF$ or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .\n",
      "Original regulator: NagC\n",
      "Original regulated: nagBE\n",
      "Original Sentence: In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .\n",
      "Original Sentence tagged: In addition , the repression of chiP translation by @TF$ induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of @Regulated$ and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .\n",
      "Original regulator: ChiX\n",
      "Original regulated: chiP\n",
      "Original Sentence: The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original Sentence tagged: The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , @Regulated$ , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original regulator: MarA\n",
      "Original regulated: zwf\n",
      "Original Sentence: ( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop .\n",
      "Original Sentence tagged: ( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR .\n",
      "Original Sentence tagged: The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of @Regulated$ expression , as would be expected if it prevents activation by both NagC and @TF$ .\n",
      "Original regulator: NanR\n",
      "Original regulated: fimB\n",
      "Original Sentence: Since our findings indicated that ArgP is needed for activation of lysP transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .\n",
      "Original Sentence tagged: Since our findings indicated that @TF$ is needed for activation of @Regulated$ transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .\n",
      "Original regulator: ArgP\n",
      "Original regulated: lysP\n",
      "Original Sentence: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original Sentence tagged: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "Original regulator: ArgP\n",
      "Original regulated: dapB\n",
      "Original Sentence: This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type melAB promoter .\n",
      "Original Sentence tagged: This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 @TF$ - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type @Regulated$ promoter .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: The ArgP protein enhances the expression of the argK gene\n",
      "Original Sentence tagged: The @TF$ protein enhances the expression of the @Regulated$ gene\n",
      "Original regulator: ArgP\n",
      "Original regulated: argK\n",
      "Original Sentence: Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .\n",
      "Original Sentence tagged: Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and @Regulated$ ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .\n",
      "Original regulator: EvgA\n",
      "Original regulated: yhiUV\n",
      "Original Sentence: However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC .\n",
      "Original Sentence tagged: However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced @TF$ , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than NagC .\n",
      "Original regulator: NagC\n",
      "Original regulated: nagE\n",
      "Original Sentence: Expression of the operon , consisting of marR , marA , and marB , is normally repressed by MarR but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .\n",
      "Original Sentence tagged: Expression of the operon , consisting of marR , @Regulated$ , and marB , is normally repressed by @TF$ but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .\n",
      "Original regulator: MarR\n",
      "Original regulated: marA\n",
      "Original Sentence: Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "Original Sentence tagged: Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate @Regulated$ by itself ; this could be because @TF$ occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "Original regulator: NagC\n",
      "Original regulated: glmU\n",
      "Original Sentence: First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] .\n",
      "Original Sentence tagged: First , repression of nanC and @Regulated$ divergent transcripts by @TF$ has been observed both in vivo and in vitro [ 32 ] .\n",
      "Original regulator: NagC\n",
      "Original regulated: fimB\n",
      "Original Sentence: A revised model for NarL - dependent activation of nirB gene expression is shown in Fig . 9 .\n",
      "Original Sentence tagged: A revised model for @TF$ - dependent activation of @Regulated$ gene expression is shown in Fig . 9 .\n",
      "Original regulator: NarL\n",
      "Original regulated: nirB\n",
      "Original Sentence: Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original Sentence tagged: Sufficiency of @TF$ for in vitro transcriptional activation of zwf , @Regulated$ , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original regulator: Rob\n",
      "Original regulated: fpr\n",
      "Original Sentence: We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter .\n",
      "Original Sentence tagged: We identified amino acid residues in the C - terminal domain of 70 and in @TF$ that are important for RhaR - mediated transcription activation at the @Regulated$ promoter .\n",
      "Original regulator: RhaR\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "Original Sentence tagged: In this study we confirmed the location of the @TF$ - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "Original regulator: Fnr\n",
      "Original regulated: napF\n",
      "Original Sentence: However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that Lrp stimulates expression of cadBA .\n",
      "Original Sentence tagged: However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that @TF$ stimulates expression of @Regulated$ .\n",
      "Original regulator: Lrp\n",
      "Original regulated: cadBA\n",
      "Original Sentence: Coincidence of the sites for SoxS - and Rob - mediated activation of zwf transcription in vitro .\n",
      "Original Sentence tagged: Coincidence of the sites for SoxS - and @TF$ - mediated activation of @Regulated$ transcription in vitro .\n",
      "Original regulator: Rob\n",
      "Original regulated: zwf\n",
      "Original Sentence: Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven .\n",
      "Original Sentence tagged: Two other regulators , @TF$ and CpxR , activate @Regulated$ expression , although a direct control has not been proven .\n",
      "Original regulator: OmpR\n",
      "Original regulated: nanC\n",
      "Original Sentence: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original Sentence tagged: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and @TF$ proteins .\n",
      "Original regulator: NarP\n",
      "Original regulated: nirB\n",
      "Original Sentence: Thus , if the model proposed above is correct , it seems probable that NanR must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate fimB expression in the wild - type background as well .\n",
      "Original Sentence tagged: Thus , if the model proposed above is correct , it seems probable that @TF$ must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate @Regulated$ expression in the wild - type background as well .\n",
      "Original regulator: NanR\n",
      "Original regulated: fimB\n",
      "Original Sentence: In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region .\n",
      "Original Sentence tagged: In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of @TF$ binding to the @Regulated$ operator region .\n",
      "Original regulator: ArgP\n",
      "Original regulated: lysP\n",
      "Original Sentence: This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .\n",
      "Original Sentence tagged: This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an @Regulated$ mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by @TF$ .\n",
      "Original regulator: OxyR\n",
      "Original regulated: hcp\n",
      "Original Sentence: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original Sentence tagged: Some of these operons are regulated by the @TF$ protein alone , such as the @Regulated$ and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original regulator: NarL\n",
      "Original regulated: narG\n",
      "Original Sentence: These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA .\n",
      "Original Sentence tagged: These results show that the decrease in purA and @Regulated$ expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant @TF$ .\n",
      "Original regulator: MarA\n",
      "Original regulated: hdeA\n",
      "Original Sentence: Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "Original Sentence tagged: Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) @Regulated$ encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "Original regulator: ArgP\n",
      "Original regulated: yggA\n",
      "Original Sentence: The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "Original Sentence tagged: The @TF$ - binding site for @Regulated$ extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "Original Sentence tagged: Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow @TF$ to to activate @Regulated$ by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .\n",
      "Original regulator: NagC\n",
      "Original regulated: glmU\n",
      "Original Sentence: With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "Original Sentence tagged: With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters @Regulated$ and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "Original regulator: SoxS\n",
      "Original regulated: fumC\n",
      "Original Sentence: Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "Original Sentence tagged: Results @TF$ - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of @Regulated$ transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "Original Sentence tagged: Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that @TF$ plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "Original regulator: ArcA\n",
      "Original regulated: dcuB\n",
      "Original Sentence: However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original Sentence tagged: However , narG and @Regulated$ operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the @TF$ binding sites in these control regions are not efficiently recognized by the NarP protein .\n",
      "Original regulator: NarL\n",
      "Original regulated: frdA\n",
      "Original Sentence: Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "Original Sentence tagged: Miller et al . ( 28 ) have found that @TF$ activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: mar\n",
      "Original Sentence: While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor NagC requires the wild - type segment of D7 DNA in order to be able to activate fimB expression .\n",
      "Original Sentence tagged: While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor @TF$ requires the wild - type segment of D7 DNA in order to be able to activate @Regulated$ expression .\n",
      "Original regulator: NagC\n",
      "Original regulated: fimB\n",
      "Original Sentence: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original Sentence tagged: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , @Regulated$ , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .\n",
      "Original regulator: ArgP\n",
      "Original regulated: lysA\n",
      "Original Sentence: In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .\n",
      "Original Sentence tagged: In vitro studies of MelR binding at the @Regulated$ promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by @TF$ binding to upstream sites 2 , 1 and 1 0 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: Thus , binding of MetJ to this area could serve to simultaneously repress both metR and metE expression .\n",
      "Original Sentence tagged: Thus , binding of @TF$ to this area could serve to simultaneously repress both metR and @Regulated$ expression .\n",
      "Original regulator: MetJ\n",
      "Original regulated: metE\n",
      "Original Sentence: For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by Rob , the presence of Rob increases the transcription of the chromosomal marRAB promoter and therefore only increases the repression of the mutant fusions .\n",
      "Original Sentence tagged: For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by @TF$ , the presence of Rob increases the transcription of the chromosomal @Regulated$ promoter and therefore only increases the repression of the mutant fusions .\n",
      "Original regulator: Rob\n",
      "Original regulated: marRAB\n",
      "Original Sentence: In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .\n",
      "Original Sentence tagged: In vitro binding of @TF$ to the @Regulated$ promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .\n",
      "Original regulator: Rob\n",
      "Original regulated: micF\n",
      "Original Sentence: LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) \n",
      "Original Sentence tagged:  @TF$ also mediates Lys repression ( of @Regulated$ ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) \n",
      "Original regulator: LysR\n",
      "Original regulated: lysA\n",
      "Original Sentence: Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "Original Sentence tagged: Moreover , they found that @TF$ overexpression activates expression of fumC , @Regulated$ , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: sodA\n",
      "Original Sentence: NagC is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the glmUS operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) .\n",
      "Original Sentence tagged: @TF$ is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the @Regulated$ operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) .\n",
      "Original regulator: NagC\n",
      "Original regulated: glmUS\n",
      "Original Sentence: Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "Original Sentence tagged: Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , @Regulated$ , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "Original regulator: MarA\n",
      "Original regulated: fumC\n",
      "Original Sentence: As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "Original Sentence tagged: As expected , expression of known AraC - regulated genes , i . e . , araB , araA , @Regulated$ , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "Original regulator: AraC\n",
      "Original regulated: araD\n",
      "Original Sentence: Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE .\n",
      "Original Sentence tagged: Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at @Regulated$ .\n",
      "Original regulator: CAP\n",
      "Original regulated: nagE\n",
      "Original Sentence: MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose metabolism ( 7 ) .\n",
      "Original Sentence tagged:  @TF$ is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the @Regulated$ operon that encodes products essential for melibiose metabolism ( 7 ) .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: The physiological significance of the repression of purA and hdeA by MarA is not known .\n",
      "Original Sentence tagged: The physiological significance of the repression of purA and @Regulated$ by @TF$ is not known .\n",
      "Original regulator: MarA\n",
      "Original regulated: hdeA\n",
      "Original Sentence: The ArgP protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( argP ) gene ( ThoÈny et al . , 1991 ) .\n",
      "Original Sentence tagged: The @TF$ protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( @Regulated$ ) gene ( ThoÈny et al . , 1991 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argP\n",
      "Original Sentence: Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .\n",
      "Original Sentence tagged: Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of @TF$ in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .\n",
      "Original regulator: Fnr\n",
      "Original regulated: dmsABC\n",
      "Original Sentence: We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for zwf .\n",
      "Original Sentence tagged: We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by @TF$ requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for @Regulated$ .\n",
      "Original regulator: MarA\n",
      "Original regulated: zwf\n",
      "Original Sentence: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original Sentence tagged: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , @Regulated$ , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araA\n",
      "Original Sentence: Identification of the DNA - binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .\n",
      "Original Sentence tagged: Identification of the DNA - binding domain of the @TF$ protein required for transcriptional activation of the ompF and @Regulated$ genes of Escherichia coli by in vivo DNA footprinting .\n",
      "Original regulator: OmpR\n",
      "Original regulated: ompC\n",
      "Original Sentence: A melAB promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both MelR - binding sites .\n",
      "Original Sentence tagged: A @Regulated$ promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both @TF$ - binding sites .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: CRP / cAMP and CytR regulated promoters in Escherichia coli K12 : the cdd promoter .\n",
      "Original Sentence tagged: @TF$ / cAMP and CytR regulated promoters in Escherichia coli K12 : the @Regulated$ promoter .\n",
      "Original regulator: CRP\n",
      "Original regulated: cdd\n",
      "Original Sentence: We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original Sentence tagged: We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of @Regulated$ but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original regulator: Rob\n",
      "Original regulated: zwf\n",
      "Original Sentence: Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "Original Sentence tagged: Substitutions within @TF$ that affect transcription activation at the @Regulated$ promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "Original regulator: CRP\n",
      "Original regulated: melAB\n",
      "Original Sentence: 20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces SoxS - dependent transcription activation of the fpr and zwf promoters .\n",
      "Original Sentence tagged: 20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces @TF$ - dependent transcription activation of the @Regulated$ and zwf promoters .\n",
      "Original regulator: SoxS\n",
      "Original regulated: fpr\n",
      "Original Sentence: Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "Original Sentence tagged: Presumably , uninduced levels of MarA and / or @TF$ mediate a low - level induction of the inaA gene ; the lower basal level of @Regulated$ expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: inaA\n",
      "Original Sentence: MelR is essential for induction of the melAB operon that is responsible for melibiose metabolism .\n",
      "Original Sentence tagged: @TF$ is essential for induction of the @Regulated$ operon that is responsible for melibiose metabolism .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with Fis binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli nirB promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .\n",
      "Original Sentence tagged: Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with @TF$ binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli @Regulated$ promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .\n",
      "Original regulator: Fis\n",
      "Original regulated: nirB\n",
      "Original Sentence: To confirm the role of ArgP in the regulation of lysP transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) .\n",
      "Original Sentence tagged: To confirm the role of @TF$ in the regulation of @Regulated$ transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: lysP\n",
      "Original Sentence: Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "Original Sentence tagged: Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , @Regulated$ , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "Original regulator: MarA\n",
      "Original regulated: gatA\n",
      "Original Sentence: The NarL and NarP proteins compete for the 44.5 binding site ; binding of NarP further induces aeg - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .\n",
      "Original Sentence tagged: The NarL and NarP proteins compete for the 44.5 binding site ; binding of @TF$ further induces @Regulated$ - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .\n",
      "Original regulator: NarP\n",
      "Original regulated: aeg\n",
      "Original Sentence: Recent genetic data show that Rob represses transcription of the galT gene ( M . H . J . B . and B . D . , unpublished results ) .\n",
      "Original Sentence tagged: Recent genetic data show that @TF$ represses transcription of the @Regulated$ gene ( M . H . J . B . and B . D . , unpublished results ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: galT\n",
      "Original Sentence: This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression .\n",
      "Original Sentence tagged: This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates @Regulated$ expression .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntV\n",
      "Original Sentence: Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .\n",
      "Original Sentence tagged: Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , @TF$ and NarP ( Figure 1 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: hcp\n",
      "Original Sentence: We show here that MarA directly represses both purA and hdeA .\n",
      "Original Sentence tagged: We show here that @TF$ directly represses both purA and @Regulated$ .\n",
      "Original regulator: MarA\n",
      "Original regulated: hdeA\n",
      "Original Sentence: These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "Original Sentence tagged: These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of @TF$ protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "Original Sentence tagged: For example , the nitrate reductase genes @Regulated$ are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by @TF$ but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: narGHJI\n",
      "Original Sentence: Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .\n",
      "Original Sentence tagged: Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the @TF$ recognition site at the P1 promoter .\n",
      "Original regulator: Fnr\n",
      "Original regulated: dmsABC\n",
      "Original Sentence: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original Sentence tagged: Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , @Regulated$ and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original regulator: NarL\n",
      "Original regulated: nrfA\n",
      "Original Sentence: Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "Original Sentence tagged: Results @TF$ - mediated regulation of E . coli @Regulated$ Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "Original Sentence tagged: Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the @Regulated$ operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "Original regulator: DnaA\n",
      "Original regulated: guaBA\n",
      "Original Sentence: In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) .\n",
      "Original Sentence tagged: In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to @Regulated$ , dapB , and gdhA ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence:  MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .\n",
      "Original Sentence tagged:  @TF$ carrying each of the single substitutions is less able to repress the @Regulated$ promoter, while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter.\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "Original Sentence tagged: @TF$ activates transcription of @Regulated$ , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .\n",
      "Original regulator: RhaR\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original Sentence tagged: Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original regulator: ArgP\n",
      "Original regulated: dapB\n",
      "Original Sentence: In a gntR deletion mutant , the expression of a chromosomal gntT : : lacZ fusion is both high and constitutive , confirming that GntR is the negative regulator of gntT .\n",
      "Original Sentence tagged: In a gntR deletion mutant , the expression of a chromosomal @Regulated$ : : lacZ fusion is both high and constitutive , confirming that @TF$ is the negative regulator of gntT .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntT\n",
      "Original Sentence: Lrp stimulates transcription of lysP by direct binding to its control region .\n",
      "Original Sentence tagged: @TF$ stimulates transcription of @Regulated$ by direct binding to its control region .\n",
      "Original regulator: Lrp\n",
      "Original regulated: lysP\n",
      "Original Sentence: Thus , the melR promoter is not efficiently repressed by MelR , and MelR is over - expressed .\n",
      "Original Sentence tagged: Thus , the @Regulated$ promoter is not efficiently repressed by @TF$ , and MelR is over - expressed .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original Sentence tagged: The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , @Regulated$ ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: fur\n",
      "Original Sentence: DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli deoCABC promoters by the DeoR repressor .\n",
      "Original Sentence tagged: DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli @Regulated$ promoters by the @TF$ repressor .\n",
      "Original regulator: DeoR\n",
      "Original regulated: deoCABC\n",
      "Original Sentence: We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .\n",
      "Original Sentence tagged: We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by GlcC and @TF$ is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .\n",
      "Original regulator: IHF\n",
      "Original regulated: glcDEFGB\n",
      "Original Sentence: The active ( oxidized or nitrosylated ) form of SoxR activates transcription of the soxS gene up to 100 - fold .\n",
      "Original Sentence tagged: The active ( oxidized or nitrosylated ) form of @TF$ activates transcription of the @Regulated$ gene up to 100 - fold .\n",
      "Original regulator: SoxR\n",
      "Original regulated: soxS\n",
      "Original Sentence: Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .\n",
      "Original Sentence tagged: Since only the NarP protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of @TF$ activation .\n",
      "Original regulator: NarP\n",
      "Original regulated: aeg\n",
      "Original Sentence: Identification of the DNAbinding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .\n",
      "Original Sentence tagged: Identification of the DNAbinding domain of the @TF$ protein required for transcriptional activation of the @Regulated$ and ompC genes of Escherichia coli by in vivo DNA footprinting .\n",
      "Original regulator: OmpR\n",
      "Original regulated: ompF\n",
      "Original Sentence: Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "Original Sentence tagged: Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of @Regulated$ , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .\n",
      "Original regulator: ArgP\n",
      "Original regulated: yggA\n",
      "Original Sentence: ( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that full repression requires the formation of a repression loop .\n",
      "Original Sentence tagged: ( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that full repression requires the formation of a repression loop .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter .\n",
      "Original Sentence tagged: We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for @TF$ - mediated transcription activation at the @Regulated$ promoter .\n",
      "Original regulator: RhaR\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original Sentence tagged: Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with @Regulated$ ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: In order to simplify our analysis , we exploited an engineered derivative of the melAB promoter in which MelR binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation .\n",
      "Original Sentence tagged: In order to simplify our analysis , we exploited an engineered derivative of the @Regulated$ promoter in which @TF$ binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .\n",
      "Original Sentence tagged:  @TF$ - mediated regulation of @Regulated$ transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: The transcription of the marA gene is regulated by the repressor MarR ( Alekshun and Levy , 1997 ) .\n",
      "Original Sentence tagged: The transcription of the @Regulated$ gene is regulated by the repressor @TF$ ( Alekshun and Levy , 1997 ) .\n",
      "Original regulator: MarR\n",
      "Original regulated: marA\n",
      "Original Sentence: ( 1987 ) Transcriptional repression of the dnaA gene of Escherichia coli by DnaA protein .\n",
      "Original Sentence tagged: ( 1987 ) Transcriptional repression of the @Regulated$ gene of Escherichia coli by @TF$ protein .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "Original Sentence tagged: Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to @Regulated$ protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "Original regulator: CAP\n",
      "Original regulated: araC\n",
      "Original Sentence: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "Original Sentence tagged: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the @Regulated$ promoter 1P .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) .\n",
      "Original Sentence tagged: Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by @TF$ and CRP ( Stoner &amp; Schleif , 1983 ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araE\n",
      "Original Sentence: The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters .\n",
      "Original Sentence tagged: The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , @Regulated$ , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the fumC and micF promoters .\n",
      "Original regulator: SoxS\n",
      "Original regulated: micF\n",
      "Original Sentence: However , in vitro experiments show IHF to repress ompC transcription from two of its three promoters .\n",
      "Original Sentence tagged: However , in vitro experiments show @TF$ to repress @Regulated$ transcription from two of its three promoters .\n",
      "Original regulator: IHF\n",
      "Original regulated: ompC\n",
      "Original Sentence: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original Sentence tagged: Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original regulator: NarL\n",
      "Original regulated: nirB\n",
      "Original Sentence: The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original Sentence tagged: The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , @Regulated$ and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: fldA\n",
      "Original Sentence: marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .\n",
      "Original Sentence tagged:  @Regulated$ expression is repressed by @TF$ and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .\n",
      "Original regulator: MarR\n",
      "Original regulated: marA\n",
      "Original Sentence: In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .\n",
      "Original Sentence tagged: In the first point , such a case has been reported in the @Regulated$ ( M ) operon concerning galactose utilization , which is regulated by @TF$ and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .\n",
      "Original regulator: GalR\n",
      "Original regulated: galETK\n",
      "Original Sentence: Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the MarA activation of nfnB .\n",
      "Original Sentence tagged: Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the @TF$ activation of @Regulated$ .\n",
      "Original regulator: MarA\n",
      "Original regulated: nfnB\n",
      "Original Sentence: To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and NarL we have extended our studies to the c / s - acting regions required for expression of the narK gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .\n",
      "Original Sentence tagged: To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and @TF$ we have extended our studies to the c / s - acting regions required for expression of the @Regulated$ gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: narK\n",
      "Original Sentence: In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "Original Sentence tagged: In silico analyses and experimental verification of potential binding sites for Lrp in the @Regulated$ control region The formation of multiple @TF$ â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .\n",
      "Original regulator: Lrp\n",
      "Original regulated: argO\n",
      "Original Sentence: DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "Original Sentence tagged: DNA sequence inspection and previous results indicating that RhaS was required for activation of @Regulated$ expression ( 40 ) suggested that @TF$ binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .\n",
      "Original regulator: RhaS\n",
      "Original regulated: rhaT\n",
      "Original Sentence: Schematic representation of regulation of araBAD and araC promoters by CAP , P1 , and P2 . negative control of Pc by P1 and P2 .\n",
      "Original Sentence tagged: Schematic representation of regulation of araBAD and @Regulated$ promoters by @TF$ , P1 , and P2 . negative control of Pc by P1 and P2 .\n",
      "Original regulator: CAP\n",
      "Original regulated: araC\n",
      "Original Sentence: These results show that the NarP protein activates F ( aeg - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the NarL protein antagonizes this activation .\n",
      "Original Sentence tagged: These results show that the NarP protein activates F ( @Regulated$ - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the @TF$ protein antagonizes this activation .\n",
      "Original regulator: NarL\n",
      "Original regulated: aeg\n",
      "Original Sentence: The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "Original Sentence tagged: The relative degrees of repression of the @Regulated$ ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that @TF$ - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .\n",
      "Original regulator: NarP\n",
      "Original regulated: ydhY\n",
      "Original Sentence: Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original Sentence tagged: Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , @Regulated$ , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original regulator: Rob\n",
      "Original regulated: nfo\n",
      "Original Sentence: AraC - dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .\n",
      "Original Sentence tagged: AraC - dependent transcription initiation at the @Regulated$ promoter is increased by @TF$ , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .\n",
      "Original regulator: CRP\n",
      "Original regulated: araBAD\n",
      "Original Sentence: Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of argO ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either ArgP binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .\n",
      "Original Sentence tagged: Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of @Regulated$ ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either @TF$ binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "Original Sentence tagged: Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of @TF$ at the ogt promoter , expression of the @Regulated$ : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .\n",
      "Original regulator: Fis\n",
      "Original regulated: ogt\n",
      "Original Sentence: We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original Sentence tagged: We found that @TF$ ( i ) activates the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original regulator: Rob\n",
      "Original regulated: fumC\n",
      "Original Sentence: The expression of araC is repressed by its own product and stimulated by the CAP system ( 5 ) .\n",
      "Original Sentence tagged: The expression of @Regulated$ is repressed by its own product and stimulated by the @TF$ system ( 5 ) .\n",
      "Original regulator: CAP\n",
      "Original regulated: araC\n",
      "Original Sentence: Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .\n",
      "Original Sentence tagged: Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the @TF$ protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .\n",
      "Original regulator: NarL\n",
      "Original regulated: dmsA\n",
      "Original Sentence: The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the NarL protein ( e . g . , the narG , fdnG , nrfA , and nirB promoters ) .\n",
      "Original Sentence tagged: The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the @TF$ protein ( e . g . , the narG , @Regulated$ , nrfA , and nirB promoters ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: fdnG\n",
      "Original Sentence: Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "Original Sentence tagged: Transcriptional activation by @TF$ , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for @Regulated$ in Escherichia coli .\n",
      "Original regulator: MarA\n",
      "Original regulated: tolC\n",
      "Original Sentence: Coeffectors of ArgP in yggA regulation .\n",
      "Original Sentence tagged: Coeffectors of @TF$ in @Regulated$ regulation .\n",
      "Original regulator: ArgP\n",
      "Original regulated: yggA\n",
      "Original Sentence: Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "Original Sentence tagged: Similarly , at the FNR - activated @TF$ - repressed @Regulated$ promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: dmsA\n",
      "Original Sentence: However , the napF control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by NarP being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) .\n",
      "Original Sentence tagged: However , the @Regulated$ control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by @TF$ being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) .\n",
      "Original regulator: NarP\n",
      "Original regulated: napF\n",
      "Original Sentence: Therefore , the induction of dcuB depends on functional DcuS / DcuR .\n",
      "Original Sentence tagged: Therefore , the induction of @Regulated$ depends on functional DcuS / @TF$ .\n",
      "Original regulator: DcuR\n",
      "Original regulated: dcuB\n",
      "Original Sentence: Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "Original Sentence tagged: Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( inaA , @Regulated$ , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "Original regulator: Rob\n",
      "Original regulated: fumC\n",
      "Original Sentence: Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "Original Sentence tagged: Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + @TF$ TB22 Figure 2 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .\n",
      "Original Sentence tagged: NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while @TF$ represses three PTS operons , ptsG , manXYZ and @Regulated$ , involved in the uptake of glucose .\n",
      "Original regulator: Mlc\n",
      "Original regulated: ptsHIcrr\n",
      "Original Sentence: We favor the idea that the repression of gntT is the result of DNA looping through interaction between the two GntR molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) .\n",
      "Original Sentence tagged: We favor the idea that the repression of @Regulated$ is the result of DNA looping through interaction between the two @TF$ molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntT\n",
      "Original Sentence: Only one NagC operator has been identified as important for repression at galP .\n",
      "Original Sentence tagged: Only one @TF$ operator has been identified as important for repression at @Regulated$ .\n",
      "Original regulator: NagC\n",
      "Original regulated: galP\n",
      "Original Sentence: Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how MelR - dependent repression of the melR promoter is modulated by the elements at the upstream melAB promoter .\n",
      "Original Sentence tagged: Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how @TF$ - dependent repression of the @Regulated$ promoter is modulated by the elements at the upstream melAB promoter .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "Original Sentence tagged: In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by @TF$ of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .\n",
      "Original Sentence tagged: Previously , overexpression of @TF$ was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of @Regulated$ , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: fumC\n",
      "Original Sentence: Lrp was also found to be involved in the regulation of cadBA expression .\n",
      "Original Sentence tagged: @TF$ was also found to be involved in the regulation of @Regulated$ expression .\n",
      "Original regulator: Lrp\n",
      "Original regulated: cadBA\n",
      "Original Sentence: The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "Original Sentence tagged: The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .\n",
      "Original Sentence tagged: In vitro binding of @TF$ to the micF promoter 6,22 , and transcriptional activation of @Regulated$ in vitro 9 have been demonstrated .\n",
      "Original regulator: Rob\n",
      "Original regulated: micF\n",
      "Original Sentence: For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "Original Sentence tagged: For the nirBDC and @Regulated$ operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - @TF$ protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .\n",
      "Original regulator: NarP\n",
      "Original regulated: nrfABCDEFG\n",
      "Original Sentence: IciA protein binds to two sites in the dnaA promoter region .\n",
      "Original Sentence tagged: @TF$ protein binds to two sites in the @Regulated$ promoter region .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "Original Sentence tagged: Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated @Regulated$ , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .\n",
      "Original regulator: MarA\n",
      "Original regulated: gatA\n",
      "Original Sentence: A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the manX â€“ lacZ fusion , while a plasmid in which Mlc was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .\n",
      "Original Sentence tagged: A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the @Regulated$ â€“ lacZ fusion , while a plasmid in which @TF$ was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .\n",
      "Original regulator: Mlc\n",
      "Original regulated: manX\n",
      "Original Sentence: Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original Sentence tagged: Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: yfhD\n",
      "Original Sentence: Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli .\n",
      "Original Sentence tagged: Discussion The results described here show that three transcription factors , ChbR , @TF$ and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .\n",
      "Original regulator: NagC\n",
      "Original regulated: chbBCARFG\n",
      "Original Sentence: Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( ArgP ) and employed an RNase protection assay to show that nrdA transcription is ArgP regulated in vivo .\n",
      "Original Sentence tagged: Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( @TF$ ) and employed an RNase protection assay to show that @Regulated$ transcription is ArgP regulated in vivo .\n",
      "Original regulator: ArgP\n",
      "Original regulated: nrdA\n",
      "Original Sentence: Model of AraC induction by L - arabinose at the araBAD promoter and relevant sites at the divergent araC promoter .\n",
      "Original Sentence tagged: Model of @TF$ induction by L - arabinose at the @Regulated$ promoter and relevant sites at the divergent araC promoter .\n",
      "Original regulator: AraC\n",
      "Original regulated: araBAD\n",
      "Original Sentence: NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression .\n",
      "Original Sentence tagged: NarX and / or NarQ sensors , both NarL and @TF$ are phosphorylated where either response regulator can then activate @Regulated$ gene expression .\n",
      "Original regulator: NarP\n",
      "Original regulated: nrfA\n",
      "Original Sentence: To determine whether the repression by MarA was direct or occurred by an indirect action , we focused on two genes , purA and hdeA .\n",
      "Original Sentence tagged: To determine whether the repression by @TF$ was direct or occurred by an indirect action , we focused on two genes , @Regulated$ and hdeA .\n",
      "Original regulator: MarA\n",
      "Original regulated: purA\n",
      "Original Sentence: Thus , it appears that NarL and NarP adopt overlapping mechanisms to inhibit ydhY – T expression .\n",
      "Original Sentence tagged: Thus , it appears that NarL and @TF$ adopt overlapping mechanisms to inhibit @Regulated$ – T expression .\n",
      "Original regulator: NarP\n",
      "Original regulated: ydhY\n",
      "Original Sentence: Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "Original Sentence tagged: Presumably , uninduced levels of @TF$ and / or SoxS mediate a low - level induction of the @Regulated$ gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: inaA\n",
      "Original Sentence: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original Sentence tagged: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and @Regulated$ ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araU\n",
      "Original Sentence: Melville and Gunsalus ( 1996 ) reported that the hemA gene , which is also repressed by FNR , contains only one half - site of FNR consensus sequence .\n",
      "Original Sentence tagged: Melville and Gunsalus ( 1996 ) reported that the @Regulated$ gene , which is also repressed by FNR , contains only one half - site of @TF$ consensus sequence .\n",
      "Original regulator: FNR\n",
      "Original regulated: hemA\n",
      "Original Sentence: Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original Sentence tagged: Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , @Regulated$ , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .\n",
      "Original regulator: ArgP\n",
      "Original regulated: lysC\n",
      "Original Sentence: With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "Original Sentence tagged: With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters fumC and @Regulated$ , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .\n",
      "Original regulator: SoxS\n",
      "Original regulated: micF\n",
      "Original Sentence: Transcription of nfnB is greatly increased by induced or constitutive chromosomal expression of MarA but is unaltered by constitutive expression of chromosomal SoxS and Rob .\n",
      "Original Sentence tagged: Transcription of @Regulated$ is greatly increased by induced or constitutive chromosomal expression of @TF$ but is unaltered by constitutive expression of chromosomal SoxS and Rob .\n",
      "Original regulator: MarA\n",
      "Original regulated: nfnB\n",
      "Original Sentence: Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .\n",
      "Original Sentence tagged: Expression of both @Regulated$ and nagB was shown to be stimulated by @TF$ in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .\n",
      "Original regulator: CAP\n",
      "Original regulated: nagE\n",
      "Original Sentence: They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA .\n",
      "Original Sentence tagged: They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and @TF$ .\n",
      "Original regulator: ArcA\n",
      "Original regulated: acnA\n",
      "Original Sentence: The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the melR promoter by MelR differs from the repression found with the longer KK81 fragment .\n",
      "Original Sentence tagged: The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the @Regulated$ promoter by @TF$ differs from the repression found with the longer KK81 fragment .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "Original Sentence tagged: Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , @Regulated$ ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .\n",
      "Original regulator: DnaA\n",
      "Original regulated: mioC\n",
      "Original Sentence: It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .\n",
      "Original Sentence tagged: It is unlikely that CRP binding to site 4 contributes directly to an increase in @Regulated$ expression , since transcription activation by @TF$ requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Thus , NarL - mediated repression of the ydhY promoter follows the general pattern established by the frdA and dmsA promoters .\n",
      "Original Sentence tagged: Thus , @TF$ - mediated repression of the @Regulated$ promoter follows the general pattern established by the frdA and dmsA promoters .\n",
      "Original regulator: NarL\n",
      "Original regulated: ydhY\n",
      "Original Sentence: ( 2004 ) Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase s subunit .\n",
      "Original Sentence tagged: ( 2004 ) Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase s subunit .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) .\n",
      "Original Sentence tagged: When RNAP was added to the @Regulated$ promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , @TF$ inhibited rob transcription ( see Fig . 6B ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: rob\n",
      "Original Sentence: ( 1997 ) The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P .\n",
      "Original Sentence tagged: ( 1997 ) The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) \n",
      "Original Sentence tagged: LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for @Regulated$ ) and that the latter â€™ s binding to @TF$ converts it into its activator conformation ( 30 , 36 ) \n",
      "Original regulator: LysR\n",
      "Original regulated: lysA\n",
      "Original Sentence: We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type fucPIK promoters , and we identify the CRP - binding site that is required for this activity .\n",
      "Original Sentence tagged: We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type @Regulated$ promoters , and we identify the @TF$ - binding site that is required for this activity .\n",
      "Original regulator: CRP\n",
      "Original regulated: fucPIK\n",
      "Original Sentence: For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the fdnGHI operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) .\n",
      "Original Sentence tagged: For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the @Regulated$ operon is controlled principally by NarL , but @TF$ also effects a degree of control ( observed with a narL mutant ) ( 21 ) .\n",
      "Original regulator: NarP\n",
      "Original regulated: fdnGHI\n",
      "Original Sentence: ArgP and cadBA regulation .\n",
      "Original Sentence tagged: @TF$ and @Regulated$ regulation .\n",
      "Original regulator: ArgP\n",
      "Original regulated: cadBA\n",
      "Original Sentence: Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .\n",
      "Original Sentence tagged: Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 @TF$ + MelR TB22 Figure 2 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: The results suggest that the NarP protein specifically interacts with the 44.5 region in order to activate aeg - 46.5 operon expression .\n",
      "Original Sentence tagged: The results suggest that the @TF$ protein specifically interacts with the 44.5 region in order to activate @Regulated$ - 46.5 operon expression .\n",
      "Original regulator: NarP\n",
      "Original regulated: aeg\n",
      "Original Sentence: Some operons such as malEFG are under the transcriptional activation by the response regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP .\n",
      "Original Sentence tagged: Some operons such as @Regulated$ are under the transcriptional activation by the response regulators in Group 1 as well as by @TF$ , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP .\n",
      "Original regulator: CRP\n",
      "Original regulated: malEFG\n",
      "Original Sentence: Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "Original Sentence tagged: Our previous experiments have shown that when both @Regulated$ promoters 1P and 2P were repressed by DnaA protein , @TF$ protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Taken together , our results suggest that CRP site 3 is the major site required for CRP activation of rhaSR expression .\n",
      "Original Sentence tagged: Taken together , our results suggest that CRP site 3 is the major site required for @TF$ activation of @Regulated$ expression .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: We also identified AraC - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and polB ) .\n",
      "Original Sentence tagged: We also identified @TF$ - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and @Regulated$ ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: polB\n",
      "Original Sentence: We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original Sentence tagged: We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends @Regulated$ and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original regulator: Rob\n",
      "Original regulated: zwf\n",
      "Original Sentence: Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli melAB promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by MelR .\n",
      "Original Sentence tagged: Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli @Regulated$ promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by @TF$ .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: In the same experiment we also examined the expression of hdeA , which had been observed in another macroarray study to be repressed upon MarA induction in cells ( 5 ) .\n",
      "Original Sentence tagged: In the same experiment we also examined the expression of @Regulated$ , which had been observed in another macroarray study to be repressed upon @TF$ induction in cells ( 5 ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: hdeA\n",
      "Original Sentence: This results in the repression of transcription , suggesting that the extent of oligomerization of DnaA proteins over two dnaA promoters contributes to the autoregulation of expression of the dnaA gene .\n",
      "Original Sentence tagged: This results in the repression of transcription , suggesting that the extent of oligomerization of @TF$ proteins over two @Regulated$ promoters contributes to the autoregulation of expression of the dnaA gene .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression .\n",
      "Original Sentence tagged: NarX and / or NarQ sensors , both @TF$ and NarP are phosphorylated where either response regulator can then activate @Regulated$ gene expression .\n",
      "Original regulator: NarL\n",
      "Original regulated: nrfA\n",
      "Original Sentence: As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .\n",
      "Original Sentence tagged: As expected , expression of known AraC - regulated genes , i . e . , @Regulated$ , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .\n",
      "Original regulator: AraC\n",
      "Original regulated: araB\n",
      "Original Sentence: Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .\n",
      "Original Sentence tagged: Negative regulation of aeg - 46.5 operon expression by the @TF$ protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .\n",
      "Original regulator: NarL\n",
      "Original regulated: dmsA\n",
      "Original Sentence: Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .\n",
      "Original Sentence tagged: Summary Expression of the narK gene of Escherichia coli , like the @Regulated$ operon , is positively regulated by two frans - acting factors : @TF$ , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .\n",
      "Original regulator: Fnr\n",
      "Original regulated: narGHJI\n",
      "Original Sentence: Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) .\n",
      "Original Sentence tagged: Mutational analysis of the nirB and @Regulated$ operon control regions has identified cis - acting regions that are required for both @TF$ and NarP - dependent induction ( Tyson et al . , 1994 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: nrfA\n",
      "Original Sentence: Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli .\n",
      "Original Sentence tagged: Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / @TF$ , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .\n",
      "Original regulator: CAP\n",
      "Original regulated: chbBCARFG\n",
      "Original Sentence: The results in this report also demonstrate that gntT is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which was found be essential for full induction .\n",
      "Original Sentence tagged: The results in this report also demonstrate that @Regulated$ is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex , which was found be essential for full induction .\n",
      "Original regulator: CRP\n",
      "Original regulated: gntT\n",
      "Original Sentence: The resulting recombinant plasmids were used to monitor MelR - dependent activation at the melAB promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .\n",
      "Original Sentence tagged: The resulting recombinant plasmids were used to monitor @TF$ - dependent activation at the @Regulated$ promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: We also extended the previous studies by Bhende and Egan ( 4 ) of RhaS - mediated transcription activation at rhaBAD .\n",
      "Original Sentence tagged: We also extended the previous studies by Bhende and Egan ( 4 ) of @TF$ - mediated transcription activation at @Regulated$ .\n",
      "Original regulator: RhaS\n",
      "Original regulated: rhaBAD\n",
      "Original Sentence: Recent studies ( 11 ) have shown that the occupation of site 2 by MelR is the most crucial for the activation of the melAB promoter , but that , of the four target sites , site 2 binds MelR most weakly .\n",
      "Original Sentence tagged: Recent studies ( 11 ) have shown that the occupation of site 2 by @TF$ is the most crucial for the activation of the @Regulated$ promoter , but that , of the four target sites , site 2 binds MelR most weakly .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: The enhanced ability of Fnr to activate dmsABC gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .\n",
      "Original Sentence tagged: The enhanced ability of @TF$ to activate @Regulated$ gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .\n",
      "Original regulator: Fnr\n",
      "Original regulated: dmsABC\n",
      "Original Sentence: We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) .\n",
      "Original Sentence tagged: We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , @Regulated$ , and ybiS ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: yfhD\n",
      "Original Sentence: For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "Original Sentence tagged: For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell @Regulated$ regulation cannot be activated by Rob , although the gene responds to SoxS and @TF$ ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: zwf\n",
      "Original Sentence: The fourth promoter predicted to be regulated by NsrR was P hcp , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR .\n",
      "Original Sentence tagged: The fourth promoter predicted to be regulated by @TF$ was P @Regulated$ , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR .\n",
      "Original regulator: NsrR\n",
      "Original regulated: hcp\n",
      "Original Sentence: Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .\n",
      "Original Sentence tagged: Since only the @TF$ protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .\n",
      "Original regulator: NarP\n",
      "Original regulated: aeg\n",
      "Original Sentence: Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original Sentence tagged: Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: yfhD\n",
      "Original Sentence: The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P\n",
      "Original Sentence tagged: The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "Original Sentence tagged: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: This results in the formation of a repression loop and the repression of the araC promoter by AraC ( reviewed by Schleif , 1996 ) .\n",
      "Original Sentence tagged: This results in the formation of a repression loop and the repression of the @Regulated$ promoter by @TF$ ( reviewed by Schleif , 1996 ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araC\n",
      "Original Sentence: In this study , we show that the global transcription factor FNR is a class II activator at the ynfEFGHI promoter and that NarL represses activation by binding to a single target that overlaps the promoter 10 element .\n",
      "Original Sentence tagged: In this study , we show that the global transcription factor FNR is a class II activator at the @Regulated$ promoter and that @TF$ represses activation by binding to a single target that overlaps the promoter 10 element .\n",
      "Original regulator: NarL\n",
      "Original regulated: ynfEFGHI\n",
      "Original Sentence: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original Sentence tagged: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( @Regulated$ , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araB\n",
      "Original Sentence: Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "Original Sentence tagged: Based on these four sites , the . known start point for @Regulated$ transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\n",
      "Original regulator: CAP\n",
      "Original regulated: araBAD\n",
      "Original Sentence: These indicate that DnaA and IciA proteins independently bind to their respective binding sites on the dnaA # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .\n",
      "Original Sentence tagged: These indicate that DnaA and @TF$ proteins independently bind to their respective binding sites on the @Regulated$ # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original Sentence tagged: Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , @Regulated$ , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araF\n",
      "Original Sentence: On the other hand , activation of the P araB promoter was delayed when glucose was present , consistent with the regulation of this promoter also by CRP ( 6 ) .\n",
      "Original Sentence tagged: On the other hand , activation of the P @Regulated$ promoter was delayed when glucose was present , consistent with the regulation of this promoter also by @TF$ ( 6 ) .\n",
      "Original regulator: CRP\n",
      "Original regulated: araB\n",
      "Original Sentence: The shaded boxes denote 22 bp DNA sites for CRP : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .\n",
      "Original Sentence tagged: The shaded boxes denote 22 bp DNA sites for @TF$ : one site centered at position 241.5 is responsible for the activation of the @Regulated$ promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .\n",
      "Original regulator: CRP\n",
      "Original regulated: melR\n",
      "Original Sentence: As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .\n",
      "Original Sentence tagged: As the DNA binding site for @TF$ overlaps the @Regulated$ - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .\n",
      "Original regulator: FNR\n",
      "Original regulated: acsP1\n",
      "Original Sentence: Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "Original Sentence tagged: Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , @Regulated$ , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .\n",
      "Original regulator: SoxS\n",
      "Original regulated: micF\n",
      "Original Sentence: It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .\n",
      "Original Sentence tagged: It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for @TF$ than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and @Regulated$ operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .\n",
      "Original regulator: GalS\n",
      "Original regulated: galS\n",
      "Original Sentence: Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "Original Sentence tagged: Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that @TF$ is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .\n",
      "Original regulator: ArcA\n",
      "Original regulated: dcuB\n",
      "Original Sentence: Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "Original Sentence tagged: Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of @TF$ that are important in transcription activation at the @Regulated$ promoter , a library of randomly mutated crp genes was created using error - prone PCR .\n",
      "Original regulator: CRP\n",
      "Original regulated: melAB\n",
      "Original Sentence: In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "Original Sentence tagged: In the case of argO , @TF$ mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: In contrast , IHF does not enhance the ability of NanR to activate fimB expression in the wild - type background .\n",
      "Original Sentence tagged: In contrast , IHF does not enhance the ability of @TF$ to activate @Regulated$ expression in the wild - type background .\n",
      "Original regulator: NanR\n",
      "Original regulated: fimB\n",
      "Original Sentence: As shown previously ( 16 ) , IciA protein activated transcription from the dnaA promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .\n",
      "Original Sentence tagged: As shown previously ( 16 ) , IciA protein activated transcription from the @Regulated$ promoter 1P in the 488 bp EcoRI / HindIII fragment ( + @TF$ I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .\n",
      "Original Sentence tagged: Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of @Regulated$ in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .\n",
      "Original regulator: MarA\n",
      "Original regulated: rimK\n",
      "Original Sentence: The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original Sentence tagged: The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , @Regulated$ ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: nfo\n",
      "Original Sentence: Model for MelR and CRP binding during activation at the melAB promoter .\n",
      "Original Sentence tagged: Model for MelR and @TF$ binding during activation at the @Regulated$ promoter .\n",
      "Original regulator: CRP\n",
      "Original regulated: melAB\n",
      "Original Sentence: Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed .\n",
      "Original Sentence tagged: Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "Original regulator: Rob\n",
      "Original regulated: yfhD\n",
      "Original Sentence: The location of the AraC binding site upstream of ytfQ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP .\n",
      "Original Sentence tagged: The location of the @TF$ binding site upstream of @Regulated$ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP .\n",
      "Original regulator: AraC\n",
      "Original regulated: ytfQ\n",
      "Original Sentence: Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .\n",
      "Original Sentence tagged: Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli @Regulated$ promoter 345 Fig . 10 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "Original Sentence tagged: In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of @TF$ - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .\n",
      "Original regulator: Fnr\n",
      "Original regulated: napF\n",
      "Original Sentence: Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the melR promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .\n",
      "Original Sentence tagged: Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between @TF$ site 1 Other determinants for MelR - dependent repression of the @Regulated$ promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "Original Sentence tagged: Similarly , at the FNR - activated NarL - repressed @Regulated$ promoter @TF$ protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .\n",
      "Original regulator: NarL\n",
      "Original regulated: dmsA\n",
      "Original Sentence: Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .\n",
      "Original Sentence tagged: Repression by NagC and ChbR in vivo A plasmid overexpressing @TF$ produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .\n",
      "Original regulator: ChbR\n",
      "Original regulated: chbB\n",
      "Original Sentence: Two approaches were used to confirm that NsrR is this sixth factor that regulates P nrfA .\n",
      "Original Sentence tagged: Two approaches were used to confirm that @TF$ is this sixth factor that regulates P @Regulated$ .\n",
      "Original regulator: NsrR\n",
      "Original regulated: nrfA\n",
      "Original Sentence: This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression .\n",
      "Original Sentence tagged: This and the finding that expression of @Regulated$ - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates gntV expression .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntV\n",
      "Original Sentence: The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original Sentence tagged: The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , @Regulated$ , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original regulator: SoxS\n",
      "Original regulated: micF\n",
      "Original Sentence: In the case of the ydbC promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by NsrR .\n",
      "Original Sentence tagged: In the case of the @Regulated$ promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by @TF$ .\n",
      "Original regulator: NsrR\n",
      "Original regulated: ydbC\n",
      "Original Sentence: Apparently , only the NarP protein is competent to activate transcription of the aeg - 46.5 operon when bound to the 44.5 region .\n",
      "Original Sentence tagged: Apparently , only the @TF$ protein is competent to activate transcription of the @Regulated$ - 46.5 operon when bound to the 44.5 region .\n",
      "Original regulator: NarP\n",
      "Original regulated: aeg\n",
      "Original Sentence: Only one other novel AraC - regulated gene identified in this work , E . coli ytfQ , has a known connection to arabinose metabolism ( 39 ) .\n",
      "Original Sentence tagged: Only one other novel @TF$ - regulated gene identified in this work , E . coli @Regulated$ , has a known connection to arabinose metabolism ( 39 ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: ytfQ\n",
      "Original Sentence: It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .\n",
      "Original Sentence tagged: It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the @Regulated$ and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by @TF$ .\n",
      "Original regulator: GalS\n",
      "Original regulated: mglB\n",
      "Original Sentence: Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original Sentence tagged: Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , @Regulated$ , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: aslB\n",
      "Original Sentence: Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , AraC , the activator - repressor that regulates the araBAD operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another .\n",
      "Original Sentence tagged: Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , @TF$ , the activator - repressor that regulates the @Regulated$ operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another .\n",
      "Original regulator: AraC\n",
      "Original regulated: araBAD\n",
      "Original Sentence: Repression of the melR promoter carried on different KK81 fragments . The figure shows the degree of MelR - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .\n",
      "Original Sentence tagged: Repression of the @Regulated$  promoter carried on different KK81 fragments . The figure shows the degree of @TF$ - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) .\n",
      "Original Sentence tagged: In contrast , the repression of both soxS and rob by @TF$ , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and @Regulated$ promoter regions ( 15 , 16 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: rob\n",
      "Original Sentence: ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .\n",
      "Original Sentence tagged:  @TF$ - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the @Regulated$ region from 293 to + 109 ( Fig . 1C ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression .\n",
      "Original Sentence tagged: However , placing the NanR , @TF$ and NagC binding sites closer to the @Regulated$ promoter enhances the ability of the regulators to activate fimB expression .\n",
      "Original regulator: IHF\n",
      "Original regulated: fimB\n",
      "Original Sentence: To measure MelR - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .\n",
      "Original Sentence tagged: To measure @TF$ - dependent repression of @Regulated$ promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: aeg - 46.5 operon expression is also negatively regulated by the NarL protein but in this case NarL does not appear to act as a classical repressor .\n",
      "Original Sentence tagged:  @Regulated$ - 46.5 operon expression is also negatively regulated by the NarL protein but in this case @TF$ does not appear to act as a classical repressor .\n",
      "Original regulator: NarL\n",
      "Original regulated: aeg\n",
      "Original Sentence: Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .\n",
      "Original Sentence tagged: Repression by NagC and @TF$ in vivo A plasmid overexpressing ChbR produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing @TF$ produces less than twofold repression ( Table 5 ) .\n",
      "Original regulator: ChbR\n",
      "Original regulated: chbB\n",
      "Original Sentence: Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "Original Sentence tagged: Transcriptional activation by MarA , SoxS and @TF$ of two @Regulated$ promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .\n",
      "Original regulator: Rob\n",
      "Original regulated: tolC\n",
      "Original Sentence: RhaR activates transcription of the operon that encodes the two activator proteins , rhaSR ( 37 , 38 ) .\n",
      "Original Sentence tagged: @TF$ activates transcription of the operon that encodes the two activator proteins , @Regulated$ ( 37 , 38 ) .\n",
      "Original regulator: RhaR\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Finally , either NarL or NarP was able to activate nrfA - lacZ expression independently of the other , while only NarL could repress expression .\n",
      "Original Sentence tagged: Finally , either NarL or NarP was able to activate @Regulated$ - lacZ expression independently of the other , while only @TF$ could repress expression .\n",
      "Original regulator: NarL\n",
      "Original regulated: nrfA\n",
      "Original Sentence: Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD .\n",
      "Original Sentence tagged: Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate @Regulated$ expression in a rhaS strain , it was of interest to test whether @TF$ could activate transcription in combination with RhaS - CTD .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaBAD\n",
      "Original Sentence: In the second part of the present study , we investigated the E . coli ogt promoter , which Constantinidou et al . [ 11 ] had also found to be induced by NarL independently of FNR .\n",
      "Original Sentence tagged: In the second part of the present study , we investigated the E . coli @Regulated$ promoter , which Constantinidou et al . [ 11 ] had also found to be induced by @TF$ independently of FNR .\n",
      "Original regulator: NarL\n",
      "Original regulated: ogt\n",
      "Original Sentence: Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene gntT by GntR and the cyclic AMP ( cAMP ) - cAMP receptor protein complex .\n",
      "Original Sentence tagged: Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene @Regulated$ by @TF$ and the cyclic AMP ( cAMP ) - cAMP receptor protein complex .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntT\n",
      "Original Sentence: Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "Original Sentence tagged: Miller et al . ( 28 ) have found that SoxS activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - @TF$ or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: mar\n",
      "Original Sentence: Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original Sentence tagged: Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntT\n",
      "Original Sentence: Based on data from in vitro studies , Celis suggested that wild - type ArgP both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) .\n",
      "Original Sentence tagged: Based on data from in vitro studies , Celis suggested that wild - type @TF$ both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including @Regulated$ ( 12 ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argK\n",
      "Original Sentence: The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .\n",
      "Original Sentence tagged: The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P @Regulated$ as compared with Lrp .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argO\n",
      "Original Sentence: Thus , NanR and NagC appear to activate fimB expression independently .\n",
      "Original Sentence tagged: Thus , NanR and @TF$ appear to activate @Regulated$ expression independently .\n",
      "Original regulator: NagC\n",
      "Original regulated: fimB\n",
      "Original Sentence: gntR gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by GntR negatively regulated by GntR .\n",
      "Original Sentence tagged:  @Regulated$ gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by @TF$ negatively regulated by GntR .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntR\n",
      "Original Sentence: For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "Original Sentence tagged: For instance , Rob has been shown to bind and activate the @Regulated$ promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to @TF$ and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .\n",
      "Original regulator: SoxS\n",
      "Original regulated: zwf\n",
      "Original Sentence: Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original Sentence tagged: Some of these operons are regulated by the @TF$ protein alone , such as the narG and @Regulated$ operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .\n",
      "Original regulator: NarL\n",
      "Original regulated: frdA\n",
      "Original Sentence: Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the galP gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and GalS and including at least 280 bp of DNA .\n",
      "Original Sentence tagged: Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the @Regulated$ gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as @TF$ and GalS and including at least 280 bp of DNA .\n",
      "Original regulator: GalR\n",
      "Original regulated: galP\n",
      "Original Sentence: The repression of purA by MarA was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) .\n",
      "Original Sentence tagged: The repression of @Regulated$ by @TF$ was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: purA\n",
      "Original Sentence: Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors .\n",
      "Original Sentence tagged: Expression of acrZ is coregulated with @Regulated$ and tolC by the MarA , Rob , and @TF$ transcription factors .\n",
      "Original regulator: SoxS\n",
      "Original regulated: acrAB\n",
      "Original Sentence: As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .\n",
      "Original Sentence tagged: As the DNA binding site for @TF$ overlaps the acsP1 - 35 element , it is easy to understand how FNR represses @Regulated$ at the same time as it activates the nrf promoter .\n",
      "Original regulator: FNR\n",
      "Original regulated: acsP1\n",
      "Original Sentence: Studies of the araBAD promoter , p BAD , show that to activate transcription the AraC protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) .\n",
      "Original Sentence tagged: Studies of the @Regulated$ promoter , p BAD , show that to activate transcription the @TF$ protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) .\n",
      "Original regulator: AraC\n",
      "Original regulated: araBAD\n",
      "Original Sentence: Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "Original Sentence tagged: Although the role of CAP in repression of @Regulated$ â€“ B is likely to be purely architectural , it is clear that a major part of the @TF$ activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .\n",
      "Original regulator: CAP\n",
      "Original regulated: nagE\n",
      "Original Sentence: Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "Original Sentence tagged: Effect of excess ChbR and NagC on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( @TF$ ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .\n",
      "Original regulator: NagC\n",
      "Original regulated: chbB\n",
      "Original Sentence:  DnaA protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two dnaA promoters , 1P and 2P .\n",
      "Original Sentence tagged:  @TF$ protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two @Regulated$ promoters , 1P and 2P .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the dnaA promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .\n",
      "Original Sentence tagged: DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the mode of binding of @TF$ protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original Sentence tagged: Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and @Regulated$ , promoters known to be activated by MalE - SoxS and MarA .\n",
      "Original regulator: Rob\n",
      "Original regulated: sodA\n",
      "Original Sentence: Both inaA and marRAB are known Rob - regulated genes ( 5 , 21 ) .\n",
      "Original Sentence tagged: Both inaA and @Regulated$ are known @TF$ - regulated genes ( 5 , 21 ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: marRAB\n",
      "Original Sentence: The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original Sentence tagged: The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including @Regulated$ , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .\n",
      "Original regulator: MarA\n",
      "Original regulated: sodA\n",
      "Original Sentence: One gene , galT , was repressed by Rob .\n",
      "Original Sentence tagged: One gene , @Regulated$ , was repressed by @TF$ .\n",
      "Original regulator: Rob\n",
      "Original regulated: galT\n",
      "Original Sentence: By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate fimB expression .\n",
      "Original Sentence tagged: By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither @TF$ nor NagC requires this segment of DNA in order to activate @Regulated$ expression .\n",
      "Original regulator: NanR\n",
      "Original regulated: fimB\n",
      "Original Sentence: We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two CRP sites was required for rhaSR activation .\n",
      "Original Sentence tagged: We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two @TF$ sites was required for @Regulated$ activation .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for RhaR activation at rhaSR .\n",
      "Original Sentence tagged: Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for @TF$ activation at @Regulated$ .\n",
      "Original regulator: RhaR\n",
      "Original regulated: rhaSR\n",
      "Original Sentence: Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "Original Sentence tagged: Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "Original regulator: SoxS\n",
      "Original regulated: fumC\n",
      "Original Sentence: There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .\n",
      "Original Sentence tagged: There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in @Regulated$ expression due to a lack of cAMP / @TF$ activation ( 5 ; see also below ) .\n",
      "Original regulator: CAP\n",
      "Original regulated: nagE\n",
      "Original Sentence: Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "Original Sentence tagged: Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( @Regulated$ , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .\n",
      "Original regulator: Rob\n",
      "Original regulated: inaA\n",
      "Original Sentence: Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .\n",
      "Original Sentence tagged: Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the @Regulated$ pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .\n",
      "Original regulator: MelR\n",
      "Original regulated: melAB\n",
      "Original Sentence: Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "Original Sentence tagged: Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for @Regulated$ ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argK\n",
      "Original Sentence: The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression .\n",
      "Original Sentence tagged: The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that @TF$ , but not NarP , represses @Regulated$ expression .\n",
      "Original regulator: NarL\n",
      "Original regulated: dcuB\n",
      "Original Sentence: MelR - dependent repression of the melR promoter carried by each derivative was then measured , as above .\n",
      "Original Sentence tagged: @TF$ - dependent repression of the @Regulated$ promoter carried by each derivative was then measured , as above .\n",
      "Original regulator: MelR\n",
      "Original regulated: melR\n",
      "Original Sentence: This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .\n",
      "Original Sentence tagged: This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that @Regulated$ transcription is regulated by @TF$ .\n",
      "Original regulator: OxyR\n",
      "Original regulated: hcp\n",
      "Original Sentence: Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter .\n",
      "Original Sentence tagged: Role of CRP in activation of the @Regulated$ promoter The results from the in vitro studies reported above suggest that @TF$ interacts directly at the melAB promoter .\n",
      "Original regulator: CRP\n",
      "Original regulated: melAB\n",
      "Original Sentence: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .\n",
      "Original Sentence tagged: DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the @TF$ I site and how this binding activates transcription from the @Regulated$ promoter 1P .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "Original Sentence tagged: The strong upregulation of @Regulated$ by the plasmid specified @TF$ observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .\n",
      "Original regulator: MarA\n",
      "Original regulated: nfnB\n",
      "Original Sentence: We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original Sentence tagged: We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and @Regulated$ DNA , and ( vi ) binds zwf and fumC DNA as a monomer .\n",
      "Original regulator: Rob\n",
      "Original regulated: fumC\n",
      "Original Sentence: Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced Rob dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and inaA ) ( Fig . 2a – c ) .\n",
      "Original Sentence tagged: Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced @TF$ dependent transcription activation of the three class II promoters ( i . e . , fumC , @Regulated$ , and inaA ) ( Fig . 2a – c ) .\n",
      "Original regulator: Rob\n",
      "Original regulated: micF\n",
      "Original Sentence: Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) .\n",
      "Original Sentence tagged: Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and @TF$ ( Stoner &amp; Schleif , 1983 ) .\n",
      "Original regulator: CRP\n",
      "Original regulated: araE\n",
      "Original Sentence: However , the inactivation of DcuS ( IMW553 ) led to a decrease of citC - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and DcuS .\n",
      "Original Sentence tagged: However , the inactivation of DcuS ( IMW553 ) led to a decrease of @Regulated$ - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and @TF$ .\n",
      "Original regulator: DcuS\n",
      "Original regulated: citC\n",
      "Original Sentence: RESULTS CRP both directly and indirectly activates rhaBAD .\n",
      "Original Sentence tagged: RESULTS @TF$ both directly and indirectly activates @Regulated$ .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaBAD\n",
      "Original Sentence: NagC repression is critical in maintaining chiP mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .\n",
      "Original Sentence tagged: @TF$ repression is critical in maintaining @Regulated$ mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .\n",
      "Original regulator: NagC\n",
      "Original regulated: chiP\n",
      "Original Sentence: Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "Original Sentence tagged: Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected @Regulated$ runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .\n",
      "Original regulator: ArgP\n",
      "Original regulated: argK\n",
      "Original Sentence: For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .\n",
      "Original Sentence tagged: For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to @TF$ ( 33 , 39 ) .\n",
      "Original regulator: NarP\n",
      "Original regulated: fdnGHI\n",
      "Original Sentence: The treB treC operon is negatively regulated by TreR , whose gene treR is located upstream of treB but is not part of the operon .\n",
      "Original Sentence tagged: The treB @Regulated$ operon is negatively regulated by @TF$ , whose gene treR is located upstream of treB but is not part of the operon .\n",
      "Original regulator: TreR\n",
      "Original regulated: treC\n",
      "Original Sentence: Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed .\n",
      "Original Sentence tagged: Our analysis revealed that inaA , marRAB , aslB , ybaO , @Regulated$ , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .\n",
      "Original regulator: Rob\n",
      "Original regulated: mdlA\n",
      "Original Sentence: In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the CRP in the P rhaBAD promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) .\n",
      "Original Sentence tagged: In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the @TF$ in the P @Regulated$ promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) .\n",
      "Original regulator: CRP\n",
      "Original regulated: rhaBAD\n",
      "Original Sentence: A 3.4 - fold decrease of citC - lacZ induction was observed when DcuR was inactivated ( IMW554 ) .\n",
      "Original Sentence tagged: A 3.4 - fold decrease of @Regulated$ - lacZ induction was observed when @TF$ was inactivated ( IMW554 ) .\n",
      "Original regulator: DcuR\n",
      "Original regulated: citC\n",
      "Original Sentence: According to previous results ( 16 ) , IciA protein was able to activate in vitro transcription from dnaA promoter 1P in the presence of limiting amounts of RNA polymerase .\n",
      "Original Sentence tagged: According to previous results ( 16 ) , @TF$ protein was able to activate in vitro transcription from @Regulated$ promoter 1P in the presence of limiting amounts of RNA polymerase .\n",
      "Original regulator: IciA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the dapB promoter region .\n",
      "Original Sentence tagged: Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of @TF$ gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the @Regulated$ promoter region .\n",
      "Original regulator: ArgP\n",
      "Original regulated: dapB\n",
      "Original Sentence: The Escherichia coli Ada protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the alkA promoter .\n",
      "Original Sentence tagged: The Escherichia coli @TF$ protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the @Regulated$ promoter .\n",
      "Original regulator: Ada\n",
      "Original regulated: alkA\n",
      "Original Sentence: Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .\n",
      "Original Sentence tagged: Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not @Regulated$ or micF .\n",
      "Original regulator: MarA\n",
      "Original regulated: fumC\n",
      "Original Sentence: The mefA operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four MetJ dimers , compared with two in the minimal complexes formed on the synthetic sites .\n",
      "Original Sentence tagged: The @Regulated$ operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four @TF$ dimers , compared with two in the minimal complexes formed on the synthetic sites .\n",
      "Original regulator: MetJ\n",
      "Original regulated: mefA\n",
      "Original Sentence: Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original Sentence tagged: Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by @TF$ ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .\n",
      "Original regulator: GntR\n",
      "Original regulated: gntKU\n",
      "Original Sentence: Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "Original Sentence tagged: Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by @TF$ protein is promoted by the oligomerization of DnaA protein over @Regulated$ promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .\n",
      "Original regulator: DnaA\n",
      "Original regulated: dnaA\n",
      "Original Sentence: Furthermore , as NagC binding to O NC1 prevents CRP - dependent activation of nanC ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .\n",
      "Original Sentence tagged: Furthermore , as NagC binding to O NC1 prevents @TF$ - dependent activation of @Regulated$ ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .\n",
      "Original regulator: CRP\n",
      "Original regulated: nanC\n",
      "Original Sentence: Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .\n",
      "Original Sentence tagged: Summary Expression of the @Regulated$ gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and @TF$ , which is activated by the presence of nitrate .\n",
      "Original regulator: NarL\n",
      "Original regulated: narK\n"
     ]
    }
   ],
   "source": [
    "# Access the real sentences\n",
    "for idx in range(len(test_dataset)):\n",
    "    sample = test_dataset[idx]\n",
    "    sentence = sample[\"sentence\"]\n",
    "    sentence_tagged = sample[\"sentence_tagged\"]\n",
    "    sentence_regulator = sample[\"regulator\"]\n",
    "    sentence_regulated = sample[\"regulated\"]\n",
    "\n",
    "    # Now you have access to the original sentence\n",
    "    print(\"Original Sentence:\", sentence)\n",
    "    print(\"Original Sentence tagged:\", sentence_tagged)\n",
    "    print(\"Original regulator:\", sentence_regulator)\n",
    "    print(\"Original regulated:\", sentence_regulated)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input IDs: tensor([[    0, 19163, 33112,  ...,     1,     1,     1],\n",
      "        [    0,   713,    16,  ...,     1,     1,     1],\n",
      "        [    0, 10462,  2156,  ...,     1,     1,     1],\n",
      "        ...,\n",
      "        [    0,   250,  7692,  ...,     1,     1,     1],\n",
      "        [    0,   104,  5865,  ...,     1,     1,     1],\n",
      "        [    0,   170,  2006,  ...,     1,     1,     1]])\n",
      "Sentences: ['Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with the C - terminal domain of the RNA polymerase alpha subunit .', 'This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for Fnr - dependent repression at the cydA P1 promoter [ 15 ] .', 'However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .', 'One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) .', 'In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent dmsA promoter in vitro ( data not shown ) .', 'Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .', 'marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .', 'However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Transcriptional activation of the dnaA gene encoding the initiator for oriC replication by IciA protein , an inhibitor of in vitro oriC replication in Escherichia coli .', 'Because previous studies have shown that the MarA - binding sites of two other repressed promoters ( hdeA and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .', 'To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .', 'In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .', 'The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', '( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop .', 'The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR .', 'Since our findings indicated that ArgP is needed for activation of lysP transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type melAB promoter .', 'The ArgP protein enhances the expression of the argK gene', 'Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .', 'However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC .', 'Expression of the operon , consisting of marR , marA , and marB , is normally repressed by MarR but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] .', 'A revised model for NarL - dependent activation of nirB gene expression is shown in Fig . 9 .', 'Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter .']\n",
      "Sentences tagged ['Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with the C - terminal domain of the RNA polymerase alpha subunit .', 'This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for @TF$ - dependent repression at the @Regulated$ P1 promoter [ 15 ] .', 'However , @Regulated$ and frdA operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'There is no obvious effect of the @TF$ + 6 mutation on @Regulated$ expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .', 'One site , designated the @TF$ I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the @Regulated$ promoter region ( Fig . 3 ) .', 'In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the @TF$ - dependent @Regulated$ promoter in vitro ( data not shown ) .', 'Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the @Regulated$ promoter 1P', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to @TF$ as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , @TF$ and GalS repression at @Regulated$ 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .', ' @Regulated$ expression is repressed by MarR and is derepressed by the interaction of @TF$ with various phenolic compounds such as salicylate .', 'However , narG and @Regulated$ operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'RhaR activates transcription of rhaSR , and @TF$ activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , @Regulated$ , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Transcriptional activation of the @Regulated$ gene encoding the initiator for oriC replication by @TF$ protein , an inhibitor of in vitro oriC replication in Escherichia coli .', 'Because previous studies have shown that the @TF$ - binding sites of two other repressed promoters ( hdeA and @Regulated$ ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .', 'To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either @TF$ or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .', 'In addition , the repression of chiP translation by @TF$ induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of @Regulated$ and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .', 'The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , @Regulated$ , and inaA , suggesting broadly overlapping activities of these three regulators .', '( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop .', 'The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of @Regulated$ expression , as would be expected if it prevents activation by both NagC and @TF$ .', 'Since our findings indicated that @TF$ is needed for activation of @Regulated$ transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .', 'This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 @TF$ - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type @Regulated$ promoter .', 'The @TF$ protein enhances the expression of the @Regulated$ gene', 'Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and @Regulated$ ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .', 'However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced @TF$ , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than NagC .', 'Expression of the operon , consisting of marR , @Regulated$ , and marB , is normally repressed by @TF$ but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate @Regulated$ by itself ; this could be because @TF$ occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'First , repression of nanC and @Regulated$ divergent transcripts by @TF$ has been observed both in vivo and in vitro [ 32 ] .', 'A revised model for @TF$ - dependent activation of @Regulated$ gene expression is shown in Fig . 9 .', 'Sufficiency of @TF$ for in vitro transcriptional activation of zwf , @Regulated$ , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'We identified amino acid residues in the C - terminal domain of 70 and in @TF$ that are important for RhaR - mediated transcription activation at the @Regulated$ promoter .']\n",
      "Regulators ['MelR', 'Fnr', 'NarL', 'CAP', 'IciA', 'Fnr', 'GntR', 'IciA', 'NarP', 'GalR', 'MarR', 'NarL', 'RhaS', 'IciA', 'MarA', 'NagC', 'ChiX', 'MarA', 'MelR', 'NanR', 'ArgP', 'ArgP', 'MelR', 'ArgP', 'EvgA', 'NagC', 'MarR', 'NagC', 'NagC', 'NarL', 'Rob', 'RhaR']\n",
      "Regulateds ['melAB', 'cydA', 'narG', 'nagE', 'dnaA', 'dmsA', 'gntKU', 'dnaA', 'fdnGHI', 'galP', 'marA', 'frdA', 'rhaBAD', 'dnaA', 'purA', 'nagBE', 'chiP', 'zwf', 'melR', 'fimB', 'lysP', 'dapB', 'melAB', 'argK', 'yhiUV', 'nagE', 'marA', 'glmU', 'fimB', 'nirB', 'fpr', 'rhaSR']\n"
     ]
    }
   ],
   "source": [
    "for batch in test_dataloader:\n",
    "    input_ids = batch['tokenized_data']['input_ids']\n",
    "    sentences = batch['sentence'] \n",
    "    sentences_tagged = batch['sentence_tagged']\n",
    "    sentences_regulators = batch['regulator']\n",
    "    sentences_regulateds = batch['regulated'] \n",
    "\n",
    "    # Print the data in the current batch\n",
    "    print(\"Input IDs:\", input_ids)\n",
    "    print(\"Sentences:\", sentences)\n",
    "    print(\"Sentences tagged\",sentences_tagged)\n",
    "    print(\"Regulators\",sentences_regulators)\n",
    "    print(\"Regulateds\",sentences_regulateds)\n",
    "    \n",
    "    # You can process and print more details about the data as needed\n",
    "\n",
    "    # Break the loop if you only want to inspect the first batch\n",
    "    break"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loading trained model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Adding get_test_predictions\n",
    "class LUKE(pl.LightningModule):\n",
    "\n",
    "    def __init__(self, num_classes):\n",
    "        super().__init__()\n",
    "        self.model = LukeForEntityPairClassification.from_pretrained(\"studio-ousia/luke-base\", num_labels=len(label2id))\n",
    "        # Multiclass classification loss\n",
    "        # Cross Entropy loss combines softmax activation function and negative log likelihood loss into a single operation\n",
    "        # Softmax will ensure that the sum of the probabilities of each class is equal to 1 by doing 3 things:\n",
    "        # Calculates the exponential of each logit \n",
    "        # Sum up the exponential values of each logit for getting the denominator of the division \n",
    "        # Divides each exponential by the sum of the exponential values to ensure the probabilites of each class sum up to 1\n",
    "        self.criterion = torch.nn.CrossEntropyLoss()\n",
    "        # Accuracy in every step \n",
    "        self.train_acc = torchmetrics.Accuracy(task = 'multiclass', num_classes = num_classes)\n",
    "        self.val_acc = torchmetrics.Accuracy(task = 'multiclass', num_classes = num_classes)\n",
    "        self.test_acc = torchmetrics.Accuracy(task = 'multiclass', num_classes = num_classes)\n",
    "        # Multi class F1 score in every step \n",
    "        self.train_f1 = MulticlassF1Score(num_classes = num_classes, average='macro')\n",
    "        self.val_f1 =  MulticlassF1Score(num_classes = num_classes, average='macro')\n",
    "        self.test_f1 = MulticlassF1Score(num_classes = num_classes, average='macro')\n",
    "\n",
    "        self.confmat = ConfusionMatrix(task=\"multiclass\", num_classes=4)\n",
    "        self.class_names = label2id \n",
    "        self.class_conf_labels = class_conf_labels\n",
    "\n",
    "        self.test_preds_list = []\n",
    "        self.test_targets_list = []\n",
    "\n",
    "    def forward(self, input_ids, entity_ids, entity_position_ids, attention_mask, entity_attention_mask):     \n",
    "        outputs = self.model(input_ids=input_ids, attention_mask=attention_mask, entity_ids=entity_ids, entity_attention_mask=entity_attention_mask, entity_position_ids=entity_position_ids)\n",
    "        return outputs.logits\n",
    "\n",
    "### training_step\n",
    "# The training logic goes in here \n",
    "# Use self.log to send any metric to TensorBoard or your preffered logger for visualization and tooling needed for machine learning experimentation\n",
    "# in self.log(on_epoch= True) add param on_epoch= True to calculate epoch-level metrics\n",
    "# return loss  or a dictionary of predictions if you want to use them latter in training\n",
    "    def training_step(self, batch, batch_idx):\n",
    "        input_ids = batch['input_ids']\n",
    "        attention_mask = batch['attention_mask']\n",
    "        train_labels = batch['label']\n",
    "        # particular parameters for LUKE model\n",
    "        entity_ids = batch['entity_ids']\n",
    "        entity_attention_mask = batch['entity_attention_mask']\n",
    "        entity_position_ids = batch['entity_position_ids']\n",
    "\n",
    "        train_logits = self.forward(input_ids=input_ids, attention_mask=attention_mask, entity_ids=entity_ids, entity_attention_mask=entity_attention_mask, entity_position_ids=entity_position_ids)\n",
    "        # Quantifies the dissimilarity between the predicted probabilities and the target distribution\n",
    "        train_loss = self.criterion(train_logits, train_labels)\n",
    "        # argmax operation is performed along the last dimension of the tensor which is typically the class dimension\n",
    "        # it returns the indices of the maximum values along the specified dimension in this case, the index of the class with the highest logit value\n",
    "        train_preds = train_logits.argmax(-1)\n",
    "        # Log metrics\n",
    "        self.log(\"training_loss\", train_loss, on_step=True, on_epoch=True, prog_bar=True)\n",
    "        self.log(\"training_accuracy\", self.train_acc(train_preds, train_labels), on_step=True, on_epoch=True, prog_bar=True)\n",
    "        self.log(\"train_f1\", self.train_f1(train_preds,train_labels), on_step=True, on_epoch=True, prog_bar=True)\n",
    "        ####\n",
    "\n",
    "        return train_loss\n",
    "\n",
    "### validation_step\n",
    "# Here goes the validation logic \n",
    "# self.log calling it will automatically accumulate and log at the end of the epoch\n",
    "    def validation_step(self, batch, batch_idx):\n",
    "        input_ids = batch['input_ids']\n",
    "        attention_mask = batch['attention_mask']\n",
    "        val_labels = batch['label']\n",
    "        # particular parameters for LUKE model\n",
    "        entity_ids = batch['entity_ids']\n",
    "        entity_attention_mask = batch['entity_attention_mask']\n",
    "        entity_position_ids = batch['entity_position_ids']\n",
    "\n",
    "        val_logits = self.forward(input_ids=input_ids, attention_mask=attention_mask, entity_ids=entity_ids, entity_attention_mask=entity_attention_mask, entity_position_ids=entity_position_ids)\n",
    "        # Getting the predictions \n",
    "        val_preds = val_logits.argmax(-1)\n",
    "\n",
    "        # Calculating metrics \n",
    "        val_loss = self.criterion(val_logits, val_labels)\n",
    "        self.val_acc(val_preds, val_labels)\n",
    "        self.val_f1(val_preds, val_labels)\n",
    "\n",
    "        # Log metrics\n",
    "        self.log(\"validation_loss\", val_loss, on_step=True, on_epoch=True, prog_bar=True)\n",
    "        self.log(\"validation_accuracy\", self.val_acc, on_step=True, on_epoch=True, prog_bar=True)\n",
    "        self.log(\"validation_f1\", self.val_f1, on_step=True, on_epoch=True, prog_bar=True)\n",
    "        \n",
    "        self.confmat.update(val_preds, val_labels)\n",
    "\n",
    "        return val_loss\n",
    "\n",
    "    def on_validation_epoch_end(self):\n",
    "            confmat = self.confmat.compute()\n",
    "            class_names = self.class_names\n",
    "            df_cm = pd.DataFrame(confmat.cpu().numpy() , index = [i for i in class_names], columns = [i for i in class_names])\n",
    "            print('Num of val samples: {}. Check this aligns with the numbers from the dataloader'.format(df_cm.sum(axis=1).sum()))\n",
    "            #log to wandb\n",
    "            f, ax = plt.subplots(figsize = (15,10)) \n",
    "            sn.heatmap(df_cm, annot=True, ax=ax)\n",
    "            plt.xlabel(\"Predicted\")\n",
    "            plt.ylabel(\"True\")\n",
    "            plt.title(\"Confusion Matrix - Validation\")\n",
    "            wandb.log({\"plot\": wandb.Image(f) })\n",
    "            # For not stacking the results after each epoch \n",
    "            self.confmat.reset() \n",
    "\n",
    "    def test_step(self, batch, batch_idx):\n",
    "        #self.test_preds_list = []\n",
    "        #self.test_targets_list = []\n",
    "        input_ids = batch['tokenized_data']['input_ids']\n",
    "        attention_mask = batch['tokenized_data']['attention_mask']\n",
    "        test_labels = batch['tokenized_data']['label']\n",
    "        # particular parameters for LUKE model\n",
    "        entity_ids = batch['tokenized_data']['entity_ids']\n",
    "        entity_attention_mask = batch['tokenized_data']['entity_attention_mask']\n",
    "        entity_position_ids = batch['tokenized_data']['entity_position_ids']\n",
    "        sentences = batch['sentence']\n",
    "        sentences_tagged = batch['sentence_tagged']\n",
    "        sentences_regulators = batch['regulator']\n",
    "        sentences_regulateds = batch['regulated']\n",
    "\n",
    "        test_logits = self.forward(input_ids=input_ids, attention_mask=attention_mask, entity_ids=entity_ids, entity_attention_mask=entity_attention_mask, entity_position_ids=entity_position_ids)\n",
    "        # Getting the predictions\n",
    "        test_preds = test_logits.argmax(-1)\n",
    "        # Calculating metrics\n",
    "        test_loss = self.criterion(test_logits, test_labels)\n",
    "        self.test_acc(test_preds, test_labels)\n",
    "        self.test_f1(test_preds,test_labels)\n",
    "        # Log metrics\n",
    "        #self.log(\"test_loss\", test_loss, on_step=True, on_epoch=True, prog_bar=True)\n",
    "        #self.log(\"test_accuracy\", self.test_acc, on_step=True, on_epoch=True, prog_bar=True)   \n",
    "        #self.log(\"test_f1\", self.test_f1, on_step=True, on_epoch=True, prog_bar=True)\n",
    "\n",
    "        self.test_preds_list.extend(test_preds.cpu().detach().numpy())\n",
    "        self.test_targets_list.extend(test_labels.cpu().detach().numpy())\n",
    "        return {\n",
    "            \"input_ids\": input_ids,\n",
    "            \"attention_mask\": attention_mask,\n",
    "            \"test_labels\":test_labels,\n",
    "            \"test_preds\":test_preds,\n",
    "            \"sentences\":sentences,\n",
    "            \"sentences_tagged\":sentences_tagged,\n",
    "            \"sentences_regulators\":sentences_regulators,\n",
    "            \"sentences_regulateds\":sentences_regulateds\n",
    "        }\n",
    "\n",
    "    def get_test_predictions(self, test_loader):\n",
    "        self.test_preds_analysis_list = []\n",
    "        self.test_targets_analysis_list = []\n",
    "        self.test_sentences_analysis_list = []\n",
    "        self.test_sentences_tagged_analysis_list = []\n",
    "        self.test_sentences_regulators_analysis_list = []\n",
    "        self.test_sentences_regulateds_analysis_list = []\n",
    "\n",
    "\n",
    "\n",
    "        self.eval()\n",
    "\n",
    "        for batch in test_loader:\n",
    "            results = self.test_step(batch, 0)  # 0 is a placeholder for batch index\n",
    "            self.test_preds_analysis_list.extend(results[\"test_preds\"].cpu().detach().numpy())\n",
    "            self.test_targets_analysis_list.extend(results[\"test_labels\"].cpu().detach().numpy())\n",
    "\n",
    "           # Include the original sentences and entity positions in the results\n",
    "            self.test_sentences_analysis_list.extend(results[\"sentences\"])\n",
    "            self.test_sentences_tagged_analysis_list.extend(results[\"sentences_tagged\"])\n",
    "            self.test_sentences_regulators_analysis_list.extend(results[\"sentences_regulators\"])\n",
    "            self.test_sentences_regulateds_analysis_list.extend(results[\"sentences_regulateds\"])\n",
    "\n",
    "\n",
    "        return self.test_sentences_analysis_list,self.test_sentences_tagged_analysis_list,self.test_sentences_regulators_analysis_list,self.test_sentences_regulateds_analysis_list, self.test_targets_analysis_list, self.test_preds_analysis_list\n",
    "\n",
    "\n",
    "    #def on_test_epoch_end(self):\n",
    "        #wandb.log({\"confusion_matrix-test\":wandb.plot.confusion_matrix(probs=None, y_true= self.test_targets_list, preds = self.test_preds_list, class_names=self.class_conf_labels)})\n",
    "\n",
    "    def configure_optimizers(self):\n",
    "        optimizer = AdamW(self.parameters(), lr=0.00001)\n",
    "        return optimizer\n",
    "\n",
    "    def train_dataloader(self):\n",
    "        return train_dataloader\n",
    "\n",
    "    def val_dataloader(self):\n",
    "        return valid_dataloader\n",
    "\n",
    "    def test_dataloader(self):\n",
    "        return test_dataloader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/avarela/miniconda3/envs/torch-m1-mps/lib/python3.11/site-packages/pytorch_lightning/utilities/migration/utils.py:49: PossibleUserWarning: The loaded checkpoint was produced with Lightning v2.0.6, which is newer than your current Lightning version: v2.0.2\n",
      "  rank_zero_warn(\n",
      "Some weights of the model checkpoint at studio-ousia/luke-base were not used when initializing LukeForEntityPairClassification: ['entity_predictions.bias', 'lm_head.layer_norm.bias', 'entity_predictions.transform.LayerNorm.weight', 'lm_head.dense.bias', 'entity_predictions.transform.dense.bias', 'lm_head.layer_norm.weight', 'lm_head.dense.weight', 'entity_predictions.transform.LayerNorm.bias', 'entity_predictions.transform.dense.weight', 'lm_head.bias']\n",
      "- This IS expected if you are initializing LukeForEntityPairClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
      "- This IS NOT expected if you are initializing LukeForEntityPairClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n",
      "Some weights of LukeForEntityPairClassification were not initialized from the model checkpoint at studio-ousia/luke-base and are newly initialized: ['classifier.weight']\n",
      "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
     ]
    }
   ],
   "source": [
    "model = LUKE.load_from_checkpoint(\n",
    "    checkpoint_path=\"deep_for_bio_nlp/7gzyvoli/checkpoints/epoch=10-step=415.ckpt\", num_classes = 4\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract real sentences and predictions from the results\n",
    "input_sentences, input_sentences_tagged,input_regulators,input_regulateds, true_labels, predicted_labels = model.get_test_predictions(test_dataloader)\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with the C - terminal domain of the RNA polymerase alpha subunit .', 'This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for Fnr - dependent repression at the cydA P1 promoter [ 15 ] .', 'However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .', 'One site , designated the IciA I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the dnaA promoter region ( Fig . 3 ) .', 'In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the Fnr - dependent dmsA promoter in vitro ( data not shown ) .', 'Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , GalR and GalS repression at galP 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .', 'marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .', 'However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Transcriptional activation of the dnaA gene encoding the initiator for oriC replication by IciA protein , an inhibitor of in vitro oriC replication in Escherichia coli .', 'Because previous studies have shown that the MarA - binding sites of two other repressed promoters ( hdeA and purA ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .', 'To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .', 'In addition , the repression of chiP translation by ChiX induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of chiP and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .', 'The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', '( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that efficient repression requires the formation of a repression loop .', 'The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of fimB expression , as would be expected if it prevents activation by both NagC and NanR .', 'Since our findings indicated that ArgP is needed for activation of lysP transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 MelR - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type melAB promoter .', 'The ArgP protein enhances the expression of the argK gene', 'Concomitant with upregulation of tolC , EvgA upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and yhiUV ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .', 'However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC .', 'Expression of the operon , consisting of marR , marA , and marB , is normally repressed by MarR but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] .', 'A revised model for NarL - dependent activation of nirB gene expression is shown in Fig . 9 .', 'Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter .', 'In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .', 'However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that Lrp stimulates expression of cadBA .', 'Coincidence of the sites for SoxS - and Rob - mediated activation of zwf transcription in vitro .', 'Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'Thus , if the model proposed above is correct , it seems probable that NanR must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate fimB expression in the wild - type background as well .', 'In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of ArgP binding to the lysP operator region .', 'This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'These results show that the decrease in purA and hdeA expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant MarA .', 'Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .', 'The ArgP - binding site for argO extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .', 'Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .', 'Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .', 'However , narG and frdA operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .', 'While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor NagC requires the wild - type segment of D7 DNA in order to be able to activate fimB expression .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'In vitro studies of MelR binding at the melR promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by MelR binding to upstream sites 2 , 1 and 1 0 .', 'Thus , binding of MetJ to this area could serve to simultaneously repress both metR and metE expression .', 'For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by Rob , the presence of Rob increases the transcription of the chromosomal marRAB promoter and therefore only increases the repression of the mutant fusions .', 'In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .', 'LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) ', 'Moreover , they found that Rob overexpression activates expression of fumC , sodA , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .', 'NagC is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the glmUS operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) .', 'Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .', 'As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .', 'Despite the fact that the residual activity of the CAP pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at nagE .', 'MelR is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the melAB operon that encodes products essential for melibiose metabolism ( 7 ) .', 'The physiological significance of the repression of purA and hdeA by MarA is not known .', 'The ArgP protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( argP ) gene ( ThoÈny et al . , 1991 ) .', 'Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .', 'We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for zwf .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', 'Identification of the DNA - binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .', 'A melAB promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both MelR - binding sites .', 'CRP / cAMP and CytR regulated promoters in Escherichia coli K12 : the cdd promoter .', 'We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .', '20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces SoxS - dependent transcription activation of the fpr and zwf promoters .', 'Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .', 'MelR is essential for induction of the melAB operon that is responsible for melibiose metabolism .', 'Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with Fis binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli nirB promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .', 'To confirm the role of ArgP in the regulation of lysP transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) .', 'Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .', 'The NarL and NarP proteins compete for the 44.5 binding site ; binding of NarP further induces aeg - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .', 'Recent genetic data show that Rob represses transcription of the galT gene ( M . H . J . B . and B . D . , unpublished results ) .', 'This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression .', 'Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , NarL and NarP ( Figure 1 ) .', 'We show here that MarA directly represses both purA and hdeA .', 'These observations indicate that the inhibition of binding of RNA polymerase to the dnaA promoters is dependent upon the oligomerization of DnaA protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of dmsABC expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'Results ArgP - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .', 'Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .', 'In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are ArgP - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to argO , dapB , and gdhA ) .', ' MelR carrying each of the single substitutions is less able to repress the melR promoter , while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter .', 'RhaR activates transcription of rhaSR , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'In a gntR deletion mutant , the expression of a chromosomal gntT : : lacZ fusion is both high and constitutive , confirming that GntR is the negative regulator of gntT .', 'Lrp stimulates transcription of lysP by direct binding to its control region .', 'Thus , the melR promoter is not efficiently repressed by MelR , and MelR is over - expressed .', 'The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', 'DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli deoCABC promoters by the DeoR repressor .', 'We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the glcDEFGB operon by GlcC and IHF is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .', 'The active ( oxidized or nitrosylated ) form of SoxR activates transcription of the soxS gene up to 100 - fold .', 'Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .', 'Identification of the DNAbinding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .', 'Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) ArgP â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .', '( 2000 ) Repression of the Escherichia coli melR promoter by MelR : evidence that full repression requires the formation of a repression loop .', 'We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for RhaR - mediated transcription activation at the rhaSR promoter .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'In order to simplify our analysis , we exploited an engineered derivative of the melAB promoter in which MelR binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation .', 'ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .', 'The transcription of the marA gene is regulated by the repressor MarR ( Alekshun and Levy , 1997 ) .', '( 1987 ) Transcriptional repression of the dnaA gene of Escherichia coli by DnaA protein .', \"Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\", 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .', 'Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) .', 'The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , micF , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during SoxS - dependent activation of the fumC and micF promoters .', 'However , in vitro experiments show IHF to repress ompC transcription from two of its three promoters .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', 'marA expression is repressed by MarR and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .', 'In the first point , such a case has been reported in the galETK ( M ) operon concerning galactose utilization , which is regulated by GalR and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .', 'Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the MarA activation of nfnB .', 'To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and NarL we have extended our studies to the c / s - acting regions required for expression of the narK gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .', 'In silico analyses and experimental verification of potential binding sites for Lrp in the argO control region The formation of multiple Lrp â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .', 'DNA sequence inspection and previous results indicating that RhaS was required for activation of rhaT expression ( 40 ) suggested that RhaS binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .', 'Schematic representation of regulation of araBAD and araC promoters by CAP , P1 , and P2 . negative control of Pc by P1 and P2 .', 'These results show that the NarP protein activates F ( aeg - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the NarL protein antagonizes this activation .', 'The relative degrees of repression of the ydhY ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that NarP - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .', 'Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'AraC - dependent transcription initiation at the araBAD promoter is increased by CRP , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .', 'Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of argO ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either ArgP binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .', 'Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of Fis at the ogt promoter , expression of the ogt : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .', 'We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'The expression of araC is repressed by its own product and stimulated by the CAP system ( 5 ) .', 'Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .', 'The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the NarL protein ( e . g . , the narG , fdnG , nrfA , and nirB promoters ) .', 'Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .', 'Coeffectors of ArgP in yggA regulation .', 'Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .', 'However , the napF control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by NarP being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) .', 'Therefore , the induction of dcuB depends on functional DcuS / DcuR .', 'Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .', 'Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .', 'NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .', 'We favor the idea that the repression of gntT is the result of DNA looping through interaction between the two GntR molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) .', 'Only one NagC operator has been identified as important for repression at galP .', 'Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how MelR - dependent repression of the melR promoter is modulated by the elements at the upstream melAB promoter .', 'In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .', 'Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .', 'Lrp was also found to be involved in the regulation of cadBA expression .', 'The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .', 'In vitro binding of Rob to the micF promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .', 'For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .', 'IciA protein binds to two sites in the dnaA promoter region .', 'Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to MarA expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .', 'A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the manX â€“ lacZ fusion , while a plasmid in which Mlc was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .', 'Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli .', 'Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( ArgP ) and employed an RNase protection assay to show that nrdA transcription is ArgP regulated in vivo .', 'Model of AraC induction by L - arabinose at the araBAD promoter and relevant sites at the divergent araC promoter .', 'NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression .', 'To determine whether the repression by MarA was direct or occurred by an indirect action , we focused on two genes , purA and hdeA .', 'Thus , it appears that NarL and NarP adopt overlapping mechanisms to inhibit ydhY – T expression .', 'Presumably , uninduced levels of MarA and / or SoxS mediate a low - level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', 'Melville and Gunsalus ( 1996 ) reported that the hemA gene , which is also repressed by FNR , contains only one half - site of FNR consensus sequence .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for SoxS - dependent activation of transcription from class II promoters fumC and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .', 'Transcription of nfnB is greatly increased by induced or constitutive chromosomal expression of MarA but is unaltered by constitutive expression of chromosomal SoxS and Rob .', 'Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .', 'They demonstrated that acnA has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and ArcA .', 'The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the melR promoter by MelR differs from the repression found with the longer KK81 fragment .', 'Also , DnaA protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .', 'It is unlikely that CRP binding to site 4 contributes directly to an increase in rhaSR expression , since transcription activation by CRP requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .', 'Thus , NarL - mediated repression of the ydhY promoter follows the general pattern established by the frdA and dmsA promoters .', '( 2004 ) Transcription activation at the Escherichia coli melAB promoter : interactions of MelR with its DNA target site and with domain 4 of the RNA polymerase s subunit .', 'When RNAP was added to the rob promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , MarA inhibited rob transcription ( see Fig . 6B ) .', '( 1997 ) The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P .', 'LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) ', 'We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type fucPIK promoters , and we identify the CRP - binding site that is required for this activity .', 'For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the fdnGHI operon is controlled principally by NarL , but NarP also effects a degree of control ( observed with a narL mutant ) ( 21 ) .', 'ArgP and cadBA regulation .', 'Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the melR A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + MelR TB22 Figure 2 .', 'The results suggest that the NarP protein specifically interacts with the 44.5 region in order to activate aeg - 46.5 operon expression .', 'Some operons such as malEFG are under the transcriptional activation by the response regulators in Group 1 as well as by CRP , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP .', 'Our previous experiments have shown that when both dnaA promoters 1P and 2P were repressed by DnaA protein , IciA protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .', 'Taken together , our results suggest that CRP site 3 is the major site required for CRP activation of rhaSR expression .', 'We also identified AraC - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and polB ) .', 'We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli melAB promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by MelR .', 'In the same experiment we also examined the expression of hdeA , which had been observed in another macroarray study to be repressed upon MarA induction in cells ( 5 ) .', 'This results in the repression of transcription , suggesting that the extent of oligomerization of DnaA proteins over two dnaA promoters contributes to the autoregulation of expression of the dnaA gene .', 'NarX and / or NarQ sensors , both NarL and NarP are phosphorylated where either response regulator can then activate nrfA gene expression .', 'As expected , expression of known AraC - regulated genes , i . e . , araB , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli AraC target regions identified by ChIP - chip .', 'Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , dmsA and nrfA operons in response to nitrate availability .', 'Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .', 'Mutational analysis of the nirB and nrfA operon control regions has identified cis - acting regions that are required for both NarL and NarP - dependent induction ( Tyson et al . , 1994 ) .', 'Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the chbBCARFG operon in E . coli .', 'The results in this report also demonstrate that gntT is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex , which was found be essential for full induction .', 'The resulting recombinant plasmids were used to monitor MelR - dependent activation at the melAB promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .', 'We also extended the previous studies by Bhende and Egan ( 4 ) of RhaS - mediated transcription activation at rhaBAD .', 'Recent studies ( 11 ) have shown that the occupation of site 2 by MelR is the most crucial for the activation of the melAB promoter , but that , of the four target sites , site 2 binds MelR most weakly .', 'The enhanced ability of Fnr to activate dmsABC gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .', 'We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of Rob from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , yfhD , and ybiS ) .', 'For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .', 'The fourth promoter predicted to be regulated by NsrR was P hcp , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR .', 'Since only the NarP protein can activate aeg - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .', 'Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'The binding of two dimers of IciA protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P', 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .', 'This results in the formation of a repression loop and the repression of the araC promoter by AraC ( reviewed by Schleif , 1996 ) .', 'In this study , we show that the global transcription factor FNR is a class II activator at the ynfEFGHI promoter and that NarL represses activation by binding to a single target that overlaps the promoter 10 element .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', \"Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\", 'These indicate that DnaA and IciA proteins independently bind to their respective binding sites on the dnaA # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described AraC - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', 'On the other hand , activation of the P araB promoter was delayed when glucose was present , consistent with the regulation of this promoter also by CRP ( 6 ) .', 'The shaded boxes denote 22 bp DNA sites for CRP : one site centered at position 241.5 is responsible for the activation of the melR promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .', 'As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .', 'Since we have shown previously that MalE - SoxS and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .', 'It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .', 'Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating dcuB expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .', 'Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .', 'In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the argO promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .', 'In contrast , IHF does not enhance the ability of NanR to activate fimB expression in the wild - type background .', 'As shown previously ( 16 ) , IciA protein activated transcription from the dnaA promoter 1P in the 488 bp EcoRI / HindIII fragment ( + IciA I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .', 'Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of rimK in their microarray analysis for MarA - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .', 'The known SoxS - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', 'Model for MelR and CRP binding during activation at the melAB promoter .', 'Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed .', 'The location of the AraC binding site upstream of ytfQ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP .', 'Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .', 'In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of napF expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .', 'Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between MelR site 1 Other determinants for MelR - dependent repression of the melR promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .', 'Similarly , at the FNR - activated NarL - repressed dmsA promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .', 'Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .', 'Two approaches were used to confirm that NsrR is this sixth factor that regulates P nrfA .', 'This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of GntR to all 3 sites slightly activates gntV expression .', 'The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', 'In the case of the ydbC promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by NsrR .', 'Apparently , only the NarP protein is competent to activate transcription of the aeg - 46.5 operon when bound to the 44.5 region .', 'Only one other novel AraC - regulated gene identified in this work , E . coli ytfQ , has a known connection to arabinose metabolism ( 39 ) .', 'It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .', 'Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , AraC , the activator - repressor that regulates the araBAD operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another .', 'Repression of the melR promoter carried on different KK81 fragments . The figure shows the degree of MelR - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .', 'In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) .', 'ArgP - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .', 'However , placing the NanR , IHF and NagC binding sites closer to the fimB promoter enhances the ability of the regulators to activate fimB expression .', 'To measure MelR - dependent repression of melR promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .', 'aeg - 46.5 operon expression is also negatively regulated by the NarL protein but in this case NarL does not appear to act as a classical repressor .', 'Repression by NagC and ChbR in vivo A plasmid overexpressing ChbR produces very strong repression of the chbB â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .', 'Transcriptional activation by MarA , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .', 'RhaR activates transcription of the operon that encodes the two activator proteins , rhaSR ( 37 , 38 ) .', 'Finally , either NarL or NarP was able to activate nrfA - lacZ expression independently of the other , while only NarL could repress expression .', 'Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate rhaBAD expression in a rhaS strain , it was of interest to test whether CRP could activate transcription in combination with RhaS - CTD .', 'In the second part of the present study , we investigated the E . coli ogt promoter , which Constantinidou et al . [ 11 ] had also found to be induced by NarL independently of FNR .', 'Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene gntT by GntR and the cyclic AMP ( cAMP ) - cAMP receptor protein complex .', 'Miller et al . ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .', 'Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'Based on data from in vitro studies , Celis suggested that wild - type ArgP both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including argK ( 12 ) .', 'The major conclusions are that : ( i ) both Lrp and ArgP stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .', 'Thus , NanR and NagC appear to activate fimB expression independently .', 'gntR gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by GntR negatively regulated by GntR .', 'For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the galP gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as GalR and GalS and including at least 280 bp of DNA .', 'The repression of purA by MarA was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) .', 'Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors .', 'As the DNA binding site for FNR overlaps the acsP1 - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .', 'Studies of the araBAD promoter , p BAD , show that to activate transcription the AraC protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) .', 'Although the role of CAP in repression of nagE â€“ B is likely to be purely architectural , it is clear that a major part of the CAP activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .', 'Effect of excess ChbR and NagC on repression of the chbB â€“ lacZ fusion . wild type nagC chbR pTZ ( NagC ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .', ' DnaA protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two dnaA promoters , 1P and 2P .', 'DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the dnaA promoter were isolated and analyzed in order to address the mode of binding of DnaA protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .', 'Sufficiency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'Both inaA and marRAB are known Rob - regulated genes ( 5 , 21 ) .', 'The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', 'One gene , galT , was repressed by Rob .', 'By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate fimB expression .', 'We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two CRP sites was required for rhaSR activation .', 'Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for RhaR activation at rhaSR .', 'Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .', 'There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .', 'Both intact Rob and the N - terminal fragment activated expression of stress genes ( inaA , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .', 'Figure 10 illustrates the likely organization of MelR subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .', 'Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .', 'The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that NarL , but not NarP , represses dcuB expression .', 'MelR - dependent repression of the melR promoter carried by each derivative was then measured , as above .', 'This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .', 'Role of CRP in activation of the melAB promoter The results from the in vitro studies reported above suggest that CRP interacts directly at the melAB promoter .', 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .', 'The strong upregulation of nfnB by the plasmidspecified MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .', 'We found that Rob ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced Rob dependent transcription activation of the three class II promoters ( i . e . , fumC , micF , and inaA ) ( Fig . 2a – c ) .', 'Studies of araE transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and CRP ( Stoner &amp; Schleif , 1983 ) .', 'However , the inactivation of DcuS ( IMW553 ) led to a decrease of citC - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and DcuS .', 'RESULTS CRP both directly and indirectly activates rhaBAD .', 'NagC repression is critical in maintaining chiP mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .', 'Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'The treB treC operon is negatively regulated by TreR , whose gene treR is located upstream of treB but is not part of the operon .', 'Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , yfhD , and ybiS were transcriptionally activated by constitutive levels of Rob , while galT expression was repressed .', 'In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the CRP in the P rhaBAD promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) .', 'A 3.4 - fold decrease of citC - lacZ induction was observed when DcuR was inactivated ( IMW554 ) .', 'According to previous results ( 16 ) , IciA protein was able to activate in vitro transcription from dnaA promoter 1P in the presence of limiting amounts of RNA polymerase .', 'Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of ArgP gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the dapB promoter region .', 'The Escherichia coli Ada protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the alkA promoter .', 'Like MalE - SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .', 'The mefA operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four MetJ dimers , compared with two in the minimal complexes formed on the synthetic sites .', 'Both gntT and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by DnaA protein is promoted by the oligomerization of DnaA protein over dnaA promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .', 'Furthermore , as NagC binding to O NC1 prevents CRP - dependent activation of nanC ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .', 'Summary Expression of the narK gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .']\n",
      "['Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with the C - terminal domain of the RNA polymerase alpha subunit .', 'This is in contrast to the requirement for the left half - site of the Fnr1 regulatory site needed for @TF$ - dependent repression at the @Regulated$ P1 promoter [ 15 ] .', 'However , @Regulated$ and frdA operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'There is no obvious effect of the @TF$ + 6 mutation on @Regulated$ expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in nagE expression due to a lack of cAMP / CAP activation ( 5 ; see also below ) .', 'One site , designated the @TF$ I site , was located upstream IciA protein binds two sites in the dnaA promoter region DNase I - protection experiments were performed to identify binding sites of the IciA protein in the @Regulated$ promoter region ( Fig . 3 ) .', 'In a control experiment , the activity of the Fnr ( D154A ) protein was confirmed by its ability to activate transcription of the @TF$ - dependent @Regulated$ promoter in vitro ( data not shown ) .', 'Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'The binding of two dimers of @TF$ protein to the dnaA promoter 1P element enhances the binding of RNA polymerase to the @Regulated$ promoter 1P', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to @TF$ as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .', 'Note that 0.5 mM GlcNAc or glucose , when present alone , is depleted by A 650 = 0.12 . ┬® 2008 The Authors Journal compilation ┬® 2008 Blackwell Publishing Ltd , Molecular Microbiology , 71 , 146 ÔÇô 157NagC , @TF$ and GalS repression at @Regulated$ 149 corresponds to the strength of the galactose - derepressed galP promoter and implies that the long fusion is better repressed than the short fusion .', ' @Regulated$ expression is repressed by MarR and is derepressed by the interaction of @TF$ with various phenolic compounds such as salicylate .', 'However , narG and @Regulated$ operon expression is regulated by the @TF$ protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the NarL binding sites in these control regions are not efficiently recognized by the NarP protein .', 'RhaR activates transcription of rhaSR , and @TF$ activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , @Regulated$ , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Transcriptional activation of the @Regulated$ gene encoding the initiator for oriC replication by @TF$ protein , an inhibitor of in vitro oriC replication in Escherichia coli .', 'Because previous studies have shown that the @TF$ - binding sites of two other repressed promoters ( hdeA and @Regulated$ ) lie in a backward orientation ( 20 ) and either partially or completely overlap the - 35 hexamer , we examined whether the putative rob marbox might be in a similar configuration and position .', 'To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC . Plasmids expressing either @TF$ or Mlc produce considerable repression of the @Regulated$ â€“ lacZ looped fusion , reducing expression 88 - and 56 - fold ( Table 2 ) .', 'In addition , the repression of chiP translation by @TF$ induces premature Rho - dependent transcription termination within the chiP gene , preventing expression of the chiQ gene ( encoding a putative lipoprotein of unknown function ) located immediately downstream of @Regulated$ and normally cotranscribed with the latter ( Bossi et al . , 2012 ) .', 'The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , @Regulated$ , and inaA , suggesting broadly overlapping activities of these three regulators .', '( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that efficient repression requires the formation of a repression loop .', 'The wild - type ( W + ) and mutant strains indicated were analysed as in the legend to Fig . 2A . ┬® 2007 The Authors Journal compilation ┬® 2007 Blackwell Publishing Ltd , Molecular Microbiology , 63 , 1223 ÔÇô 1236 Regulation of fimB by Neu 5Ac and GlcNAc 1229 lower level of @Regulated$ expression , as would be expected if it prevents activation by both NagC and @TF$ .', 'Since our findings indicated that @TF$ is needed for activation of @Regulated$ transcription , we tested whether cadBA expression would be rendered Lys independent even in the argP mutant .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .', 'This shows that we had failed to identify MelR mutants with improved specificity for the changed KK433 @TF$ - binding sequences ; mutants with specificity for the KK433 sequence would have given lower levels of activation with the wild - type @Regulated$ promoter .', 'The @TF$ protein enhances the expression of the @Regulated$ gene', 'Concomitant with upregulation of tolC , @TF$ upregulated several multidrug resistance genes ( acrAB , emrKY , mdfA and @Regulated$ ) and showed increased antibiotic resistance that was fully dependent on TolC but only partially dependent on AcrA .', 'However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced @TF$ , shows that the isolated @Regulated$ operator site has a higher affinity for Mlc than NagC .', 'Expression of the operon , consisting of marR , @Regulated$ , and marB , is normally repressed by @TF$ but can be induced by diverse compounds such as tetracycline , chloramphenicol , menadione , and salicylates ( 6 , 16 , 28 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate @Regulated$ by itself ; this could be because @TF$ occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'First , repression of nanC and @Regulated$ divergent transcripts by @TF$ has been observed both in vivo and in vitro [ 32 ] .', 'A revised model for @TF$ - dependent activation of @Regulated$ gene expression is shown in Fig . 9 .', 'Sufficiency of @TF$ for in vitro transcriptional activation of zwf , @Regulated$ , fumC , micF , nfo , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'We identified amino acid residues in the C - terminal domain of 70 and in @TF$ that are important for RhaR - mediated transcription activation at the @Regulated$ promoter .', 'In this study we confirmed the location of the @TF$ - binding site at position 64.5 and supported the idea that the mechanism of Fnr - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .', 'However , under inducing conditions , the P cadBA activity in the lrp : : Km r mutant strain was 2 - fold lower than that in the parent strain , suggesting that @TF$ stimulates expression of @Regulated$ .', 'Coincidence of the sites for SoxS - and @TF$ - mediated activation of @Regulated$ transcription in vitro .', 'Two other regulators , @TF$ and CpxR , activate @Regulated$ expression , although a direct control has not been proven .', 'Some of these operons are regulated by the NarL protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and @TF$ proteins .', 'Thus , if the model proposed above is correct , it seems probable that @TF$ must make contact with RNA polymerase , or another regulatory factor bound closer to the promoter , in order to activate @Regulated$ expression in the wild - type background as well .', 'In vitro , ArgP binds the lysP regulatory region with a K d of around 55 nM and the binding affinity is diminished upon the addition of Lys ( to a K d of 140 nM ) , suggesting that Lys represses lysP in vivo by engendering the loss of @TF$ binding to the @Regulated$ operator region .', 'This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an @Regulated$ mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by @TF$ .', 'Some of these operons are regulated by the @TF$ protein alone , such as the @Regulated$ and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'These results show that the decrease in purA and @Regulated$ expression was caused , directly or indirectly , by transiently synthesized MarA rather than by a stress reaction due to constitutively overabundant @TF$ .', 'Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of yggA , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) @Regulated$ encodes an Arg ( and CAN ) exporter in E . coli .', 'The @TF$ - binding site for @Regulated$ extends up to nucleotide 20 and thereby overlaps the 35 promoter motif , whereas the ArgP - binding site for lysP extends up to position 47 , which is upstream of the 35 promoter site ( Fig . 2 ) .', 'Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado - Vides et al . , 1991 ; Gralla , 1991 ) , it is not sufficient to allow @TF$ to to activate @Regulated$ by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .', 'With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters @Regulated$ and micF , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .', 'Results @TF$ - mediated regulation of E . coli argO Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of @Regulated$ transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .', 'Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that @TF$ plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that ArcA is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .', 'However , narG and @Regulated$ operon expression is regulated by the NarL protein alone ( Rabin &amp; Stewart , 1993 ) , suggesting that the @TF$ binding sites in these control regions are not efficiently recognized by the NarP protein .', 'Miller et al . ( 28 ) have found that @TF$ activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .', 'While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor @TF$ requires the wild - type segment of D7 DNA in order to be able to activate @Regulated$ expression .', 'Under identical experimental conditions as for argO , the regulatory regions of the other ArgP - regulated genes ( asd , dapB , dapD , gdhA , @Regulated$ , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that @TF$ binding in these instances is Lys sensitive .', 'In vitro studies of MelR binding at the @Regulated$ promoter Our results show that MelR - dependent repression of the melR promoter is contingent on MelR binding to site R and is somehow modulated by @TF$ binding to upstream sites 2 , 1 and 1 0 .', 'Thus , binding of @TF$ to this area could serve to simultaneously repress both metR and @Regulated$ expression .', 'For the marbox transversion mutants ( no . 7 and 8 ) which are insensitive to stimulation by @TF$ , the presence of Rob increases the transcription of the chromosomal @Regulated$ promoter and therefore only increases the repression of the mutant fusions .', 'In vitro binding of @TF$ to the @Regulated$ promoter 6,22 , and transcriptional activation of micF in vitro 9 have been demonstrated .', ' @TF$ also mediates Lys repression ( of @Regulated$ ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for lysA ) and that the latter â€™ s binding to LysR converts it into its activator conformation ( 30 , 36 ) ', 'Moreover , they found that @TF$ overexpression activates expression of fumC , @Regulated$ , and inaA ( unknown function ) , but not zwf , and that purified Rob forms DNA - protein complexes with DNA fragments containing the micF , sodA , nfo , and zwf promoters ( 3 ) .', '@TF$ is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the @Regulated$ operon promoters albeit by binding at the more conventional position of - 47 relative to the transcriptional start site ( Plumbridge , 1995 ) .', 'Since we have shown previously that MalE - SoxS and @TF$ are sufficient for the in vitro transcriptional activation of the zwf , fpr , @Regulated$ , micF , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .', 'As expected , expression of known AraC - regulated genes , i . e . , araB , araA , @Regulated$ , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .', 'Despite the fact that the residual activity of the @TF$ pc proteins at the two promoters is different in the induced and uninduced conditions , there is a clear qualitative difference in the response of nagE and nagB to the mutant CAPs : 50 ÔÇô 100 % activation at nagB and maximally 20 % activation at @Regulated$ .', ' @TF$ is a member of the AraC - XylS family of bacterial gene regulatory proteins ( 6 ) and our previous studies have shown that MelR , together with the cyclic AMP receptor protein , CRP , regulates expression of the @Regulated$ operon that encodes products essential for melibiose metabolism ( 7 ) .', 'The physiological significance of the repression of purA and @Regulated$ by @TF$ is not known .', 'The @TF$ protein represses its own synthesis Sequences consistent with the consensus for E . coli promoters ( 35 and 10 ) have not been found upstream of the start codon for the iciA ( @Regulated$ ) gene ( ThoÈny et al . , 1991 ) .', 'Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of @TF$ in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the Fnr recognition site at the P1 promoter .', 'We surmise , from the distance of this marbox ( centered at - 61.5 ) to the putative - 35 transcriptional signal ( 17 , 18 ) , that activation of mar by @TF$ requires interaction with the carboxy - terminal domain of the alpha subunit of RNA polymerase , i . e . , that MarA is a class I activator ( 16 ) of mar as it is for @Regulated$ .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , @Regulated$ , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', 'Identification of the DNA - binding domain of the @TF$ protein required for transcriptional activation of the ompF and @Regulated$ genes of Escherichia coli by in vivo DNA footprinting .', 'A @Regulated$ promoter fragment that is very weakly activated by the meiR gene product has been made by creating multiple symmetrical changes in both @TF$ - binding sites .', '@TF$ / cAMP and CytR regulated promoters in Escherichia coli K12 : the @Regulated$ promoter .', 'We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of @Regulated$ but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Substitutions within @TF$ that affect transcription activation at the @Regulated$ promoter To identify the residues of CRP that are important in transcription activation at the melAB promoter , a library of randomly mutated crp genes was created using error - prone PCR .', '20 - 22,39 Indeed , changing C5 to any of the other three nucleotides severely reduces @TF$ - dependent transcription activation of the @Regulated$ and zwf promoters .', 'Presumably , uninduced levels of MarA and / or @TF$ mediate a low - level induction of the inaA gene ; the lower basal level of @Regulated$ expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .', '@TF$ is essential for induction of the @Regulated$ operon that is responsible for melibiose metabolism .', 'Similarly , incubation of the pnrf97 EcoRI â€“ HindIII fragment with @TF$ binds to the nrfA â€“ acs intergenic region and represses transcription At the E . coli @Regulated$ promoter , transcription initiation is repressed by the association of the nucleoid - associated factors , Fis and IHF ( Wu et al . , 1998 ; Browning et al . , 2000 ) .', 'To confirm the role of @TF$ in the regulation of @Regulated$ transcription , the argP gene was cloned into plasmid pBAD24 under the control of the arabinose - inducible promoter ( 18 ) .', 'Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated gatA , gatB and gatD upon treating cells with paraquat , and elevated gatZ , @Regulated$ , gatB , gatC and gatD upon treating cells with salicylate .', 'The NarL and NarP proteins compete for the 44.5 binding site ; binding of @TF$ further induces @Regulated$ - 46.5 operon expression , whereas binding of NarL has no significant effect on the basal anaerobic ( Fnr - dependent ) level of expression .', 'Recent genetic data show that @TF$ represses transcription of the @Regulated$ gene ( M . H . J . B . and B . D . , unpublished results ) .', 'This and the finding that expression of gntV - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates @Regulated$ expression .', 'Nitrate - and nitrite - mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the @Regulated$ regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate - activated regulators , @TF$ and NarP ( Figure 1 ) .', 'We show here that @TF$ directly represses both purA and @Regulated$ .', 'These observations indicate that the inhibition of binding of RNA polymerase to the @Regulated$ promoters is dependent upon the oligomerization of @TF$ protein to regions flanking the DnaA box and that the binding of one or two DnaA monomers to the DnaA box is not sufficient for occlusion of RNA polymerase from the dnaA promoters .', 'For example , the nitrate reductase genes @Regulated$ are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by @TF$ but is also responsive in part to NarP ( 33 , 39 ) .', 'Mutagenesis of the Fnr recognition sequence at the dmsA promoter region confirmed the contribution of Fnr in the 100 - fold activation of @Regulated$ expression and explored the DNA sequence and spacing requirements for the @TF$ recognition site at the P1 promoter .', 'Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , @Regulated$ and fdnG operons is controlled by both the NarL and NarP proteins .', 'Results @TF$ - mediated regulation of E . coli @Regulated$ Reconstitution of ArgP - mediated regulation of argO transcription in vitro Toward reconstitution of argO transcriptional regulation in vitro , the in vivo start site of argO transcription was first mapped ( in an argP d strain where such transcription is high and constitutive ) by primer extension analysis to an A residue situated 28 bases upstream of the predicted translation start site of the structural gene ( see Supplementary Fig . S1 ) .', 'Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , mioC ( 19 ) , the @Regulated$ operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .', 'In this study , we have examined promoter - lac fusions in strains that were argP or argP or that were carrying dominant argP mutations in order to identify several new genes that are @TF$ - regulated in vivo , including lysP , lysC , lysA , dapD , and asd ( in addition to @Regulated$ , dapB , and gdhA ) .', ' @TF$ carrying each of the single substitutions is less able to repress the @Regulated$ promoter, while MelR carrying some combinations of substitutions is completely unable to repress the melR promoter.', '@TF$ activates transcription of @Regulated$ , and RhaS activates transcription of the operon that encodes the L - rhamnose catabolic enzymes , rhaBAD , as well as the operon that encodes the L - rhamnose transport protein , rhaT .', 'Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , @Regulated$ , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'In a gntR deletion mutant , the expression of a chromosomal @Regulated$ : : lacZ fusion is both high and constitutive , confirming that @TF$ is the negative regulator of gntT .', '@TF$ stimulates transcription of @Regulated$ by direct binding to its control region .', 'Thus , the @Regulated$ promoter is not efficiently repressed by @TF$ , and MelR is over - expressed .', 'The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , @Regulated$ ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', 'DNAprotein recognition : demonstration of three genetically separated operator elements that are required for repression of the Escherichia coli @Regulated$ promoters by the @TF$ repressor .', 'We note that such an effect is not unprecedented for a member of the GntR / FadR family of transcriptional regulators like NanR ( Rigali et al . , 2002 ; Kalivoda et al . , 2003 ) ; activation of the @Regulated$ operon by GlcC and @TF$ is thought to occur in this way , with IHF bound to a site between the promoter and a distant ( &gt; - 170 bp ) upstream activating site for GlcC ( Pellicer et al . , 1999 ) .', 'The active ( oxidized or nitrosylated ) form of @TF$ activates transcription of the @Regulated$ gene up to 100 - fold .', 'Since only the NarP protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of @TF$ activation .', 'Identification of the DNAbinding domain of the @TF$ protein required for transcriptional activation of the @Regulated$ and ompC genes of Escherichia coli by in vivo DNA footprinting .', 'Therefore , a straightforward interpretation of our results , as further discussed below , is that ( i ) ArgP is a transcriptional activator of @Regulated$ , ( ii ) @TF$ â€™ s activator function is enhanced by Arg and inhibited by Lys , and ( iii ) yggA encodes an Arg ( and CAN ) exporter in E . coli .', '( 2000 ) Repression of the Escherichia coli @Regulated$ promoter by @TF$ : evidence that full repression requires the formation of a repression loop .', 'We identified amino acid residues in the C - terminal domain of 70 and in RhaR that are important for @TF$ - mediated transcription activation at the @Regulated$ promoter .', 'Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , lysC , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with @Regulated$ ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'In order to simplify our analysis , we exploited an engineered derivative of the @Regulated$ promoter in which @TF$ binding to site 2 and site 2 ¢ , in the absence of CRP , is sufficient for transcription activation .', ' @TF$ - mediated regulation of @Regulated$ transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the argO region from 293 to + 109 ( Fig . 1C ) .', 'The transcription of the @Regulated$ gene is regulated by the repressor @TF$ ( Alekshun and Levy , 1997 ) .', '( 1987 ) Transcriptional repression of the @Regulated$ gene of Escherichia coli by @TF$ protein .', \"Based on these four sites , the . known start point for araBAD transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of CAP to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to @Regulated$ protein , prevents the binding of @TF$ to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\", 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the @Regulated$ promoter 1P .', 'Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by @TF$ and CRP ( Stoner &amp; Schleif , 1983 ) .', 'The experiments in Fig . 2a ÔÇô c suggest that the binding of Rob to the robboxes of the fumC , @Regulated$ , and inaA promoters occludes the binding of 70 R4 to the 35 hexamer of these class II promoters , as is also the case during @TF$ - dependent activation of the fumC and micF promoters .', 'However , in vitro experiments show @TF$ to repress @Regulated$ transcription from two of its three promoters .', 'Some of these operons are regulated by the @TF$ protein alone , such as the narG and frdA operons , whereas expression of the @Regulated$ ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , @Regulated$ and fldB ( flavodoxins A and B ) , nfo ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', ' @Regulated$ expression is repressed by @TF$ and is derepressed by the interaction of MarR with various phenolic compounds such as salicylate .', 'In the first point , such a case has been reported in the @Regulated$ ( M ) operon concerning galactose utilization , which is regulated by @TF$ and GalS [ Weickert and Adhya , 1993a b ] , but its molecular mechanism is clearly different from that of the GntII gene regulation .', 'Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 specified nt ) ( Fig . 3B ) , their location , downstream from the - 10 hexamer and 83 nt upstream from the - 35 hexamer , respectively , suggested that these marboxes were probably not functional in the @TF$ activation of @Regulated$ .', 'To explore the possible mechanisms involved in the integrated transcriptional activation by Fnr and @TF$ we have extended our studies to the c / s - acting regions required for expression of the @Regulated$ gene , which encodes another component involved in nitrate respiration , a nitrate - nitrite antiporter ( Noji et ai , 1989 ; DeMoss and Hsu , 1991 ) .', 'In silico analyses and experimental verification of potential binding sites for Lrp in the @Regulated$ control region The formation of multiple @TF$ â€“ argO operator complexes with a different apparent stoichiometry suggests the existence of several binding sites for Lrp in the argO control region .', 'DNA sequence inspection and previous results indicating that RhaS was required for activation of @Regulated$ expression ( 40 ) suggested that @TF$ binds to DNA at the rhaT promoter , but direct evidence of RhaS protein binding to rhaT promoter DNA had not been obtained .', 'Schematic representation of regulation of araBAD and @Regulated$ promoters by @TF$ , P1 , and P2 . negative control of Pc by P1 and P2 .', 'These results show that the NarP protein activates F ( @Regulated$ - 46.5 - lacZ ) expression in response to nitrate and nitrite and that the @TF$ protein antagonizes this activation .', 'The relative degrees of repression of the @Regulated$ ÔÇô lacZ fusion in the presence of nitrite ( approximately twofold ) or nitrate ( ~ 13 - fold ) suggest that NarL - mediated inhibition of FNR recruitment is a key component of the regulation of the ydhY ÔÇô T promoter , and that @TF$ - mediated inhibition of RNA polymerase binding is partially overcome by the ability of FNR to recruit RNA polymerase .', 'Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , @Regulated$ , and sodA , promoters known to be activated by MalE - SoxS and MarA .', 'AraC - dependent transcription initiation at the @Regulated$ promoter is increased by @TF$ , which binds to a single DNA site upstream of I1 and I2 ( 11 ) .', 'Although a 26 - bp near - perfect palindromic region from 102 to 77 exists upstream of @Regulated$ ( see Fig . 1A ) , neither its deletion ( designated 1 ) nor that of an overlapping 1260 GENES &amp; DEVELOPMENT upstream region from 115 to 90 ( designated 2 ) led to any significant loss of either @TF$ binding to argO in vitro ( Fig . 2B ) or to Arg - and Lys - modulated transcriptional regulation in vivo ( Fig . 1B , top panel ) .', 'Fis represses the ogt promoter by displacing upstream - bound NarL To investigate a role of @TF$ at the ogt promoter , expression of the @Regulated$ : : lac fusion was compared in strain JCB387 and in the fis mutant derivative , JCB3871 .', 'We found that @TF$ ( i ) activates the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'The expression of @Regulated$ is repressed by its own product and stimulated by the @TF$ system ( 5 ) .', 'Negative regulation of aeg - 46.5 operon expression by the NarL protein In addition to its function as a transcriptional activator , the @TF$ protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .', 'The fact that most Fnrdependent promoters have the latter architecture may be due to the fact that many of them are also activated by the @TF$ protein ( e . g . , the narG , @Regulated$ , nrfA , and nirB promoters ) .', 'Transcriptional activation by @TF$ , SoxS and Rob of two tolC promoters using one binding site : a complex promoter configuration for @Regulated$ in Escherichia coli .', 'Coeffectors of @TF$ in @Regulated$ regulation .', 'Similarly , at the FNR - activated @TF$ - repressed @Regulated$ promoter NarL protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .', 'However , the @Regulated$ control region has a very different architecture , with an Fnr - binding site centered at position 64.5 and activation by @TF$ being mediated by a binding site downstream , at position 44.5 ( Fig . 1 ) .', 'Therefore , the induction of @Regulated$ depends on functional DcuS / @TF$ .', 'Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( inaA , @Regulated$ , sodA ) but with a pattern distinct from that found for SoxS and MarA .', 'Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 MelR + @TF$ TB22 Figure 2 .', 'NagC represses nagE , encoding the N - acetylglucosamine - specific transporter , while @TF$ represses three PTS operons , ptsG , manXYZ and @Regulated$ , involved in the uptake of glucose .', 'We favor the idea that the repression of @Regulated$ is the result of DNA looping through interaction between the two @TF$ molecules as shown for many other operons in E . coli , such as the ara , gal , lac , and deo operons ( 29 ) .', 'Only one @TF$ operator has been identified as important for repression at @Regulated$ .', 'Together with the present results , this suggests a simple model ( illustrated in Fig . 6 ) to explain how @TF$ - dependent repression of the @Regulated$ promoter is modulated by the elements at the upstream melAB promoter .', 'In the case of argO , ArgP mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by @TF$ of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .', 'Previously , overexpression of @TF$ was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of @Regulated$ , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar / soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .', '@TF$ was also found to be involved in the regulation of @Regulated$ expression .', 'The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P @Regulated$ in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P argO as compared with Lrp .', 'In vitro binding of @TF$ to the micF promoter 6,22 , and transcriptional activation of @Regulated$ in vitro 9 have been demonstrated .', 'For the nirBDC and @Regulated$ operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho - NarL or - @TF$ protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .', '@TF$ protein binds to two sites in the @Regulated$ promoter region .', 'Curiously , Barbosa and Levy ( Barbosa and Levy 2000 ) found elevated gatA and gatC transcription due to @TF$ expression , and Pomposiello and colleagues ( Pomposiello et al . , 2001 ) found elevated @Regulated$ , gatB and gatD upon treating cells with paraquat , and elevated gatZ , gatA , gatB , gatC and gatD upon treating cells with salicylate .', 'A plasmid carrying the mlc gene expressed from its own promoter ( pMlc2 , a gift from S . Ichihara ) produced a 10 - fold repression of the @Regulated$ â€“ lacZ fusion , while a plasmid in which @TF$ was under the control of the lac promoter , pTZ ( Mlc ) , reduced expression of the fusion to less than measurable values ( greater than 100 - fold ; Table 5 ) .', 'Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'Discussion The results described here show that three transcription factors , ChbR , @TF$ and cAMP / CAP , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .', 'Our conclusion runs contrary to that of Han et al . ( 20 ) , who overproduced IciA ( @TF$ ) and employed an RNase protection assay to show that @Regulated$ transcription is ArgP regulated in vivo .', 'Model of @TF$ induction by L - arabinose at the @Regulated$ promoter and relevant sites at the divergent araC promoter .', 'NarX and / or NarQ sensors , both NarL and @TF$ are phosphorylated where either response regulator can then activate @Regulated$ gene expression .', 'To determine whether the repression by @TF$ was direct or occurred by an indirect action , we focused on two genes , @Regulated$ and hdeA .', 'Thus , it appears that NarL and @TF$ adopt overlapping mechanisms to inhibit @Regulated$ – T expression .', 'Presumably , uninduced levels of @TF$ and / or SoxS mediate a low - level induction of the @Regulated$ gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and @Regulated$ ) .', 'Melville and Gunsalus ( 1996 ) reported that the @Regulated$ gene , which is also repressed by FNR , contains only one half - site of @TF$ consensus sequence .', 'Under identical experimental conditions as for argO , the regulatory regions of the other @TF$ - regulated genes ( asd , dapB , dapD , gdhA , lysA , @Regulated$ , and lysP ) were also bound by ArgP , with apparent K d s ranging from 55 nM to 170 nM ; in all these cases ( unlike the situation with argO ) , the addition of Lys was associated with an increase in the apparent K d , indicating that ArgP binding in these instances is Lys sensitive .', 'With a tri - alanine scanning library of amino acid residues of 70 from positions 531 – 590 ( residues 531 – 540 lie in the distal portion of 70 R3 . 2 , residues 541 – 570 lie in 70 R4 . 1 , and residues 571 – 590 lie in the N - terminal portion of 70 R4 . 2 ) , 1,46,47 we recently determined that none of the amino acid residues within this region are required for @TF$ - dependent activation of transcription from class II promoters fumC and @Regulated$ , including some substitutions that replace amino acids that are known to interact with nucleotides of the 35 hexamer .', 'Transcription of @Regulated$ is greatly increased by induced or constitutive chromosomal expression of @TF$ but is unaltered by constitutive expression of chromosomal SoxS and Rob .', 'Expression of both @Regulated$ and nagB was shown to be stimulated by @TF$ in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .', 'They demonstrated that @Regulated$ has two promoters , P1 and P2 , the latter being regulated by SoxS , FNR , CRP and @TF$ .', 'The results shown in Fig . 4 indicate that , with the shorter KK101 fragment , repression of the @Regulated$ promoter by @TF$ differs from the repression found with the longer KK81 fragment .', 'Also , @TF$ protein functions as a transcriptional repressor for the expression of other genes including rpoH ( 18 ) , @Regulated$ ( 19 ) , the guaBA operon ( 20 ) , and uvrB ( 21 ) , while the expression of the nrd gene appeared to be enhanced by DnaA protein ( 22 ) .', 'It is unlikely that CRP binding to site 4 contributes directly to an increase in @Regulated$ expression , since transcription activation by @TF$ requires that its binding site be on the same face of the DNA as the promoter ( 6 ) .', 'Thus , @TF$ - mediated repression of the @Regulated$ promoter follows the general pattern established by the frdA and dmsA promoters .', '( 2004 ) Transcription activation at the Escherichia coli @Regulated$ promoter : interactions of @TF$ with its DNA target site and with domain 4 of the RNA polymerase s subunit .', 'When RNAP was added to the @Regulated$ promoter at 0 Â° C prior to MarA addition and then transfer to 37 Â° C , @TF$ inhibited rob transcription ( see Fig . 6B ) .', '( 1997 ) The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase at to the dnaA promoter 1P .', 'LysR also mediates Lys repression ( of lysA ) , but in this case it has been suggested that the coeffector ligand is diaminopimelic acid ( which is the substrate for @Regulated$ ) and that the latter â€™ s binding to @TF$ converts it into its activator conformation ( 30 , 36 ) ', 'We show that the mutant CRP constitutively activates transcription from both the IS5 - disrupted and the wild - type @Regulated$ promoters , and we identify the @TF$ - binding site that is required for this activity .', 'For example , nitrate induction of the narGHJI operon ( 21 ) and the narK gene ( 12 ) is controlled exclusively by NarL , while induction of the @Regulated$ operon is controlled principally by NarL , but @TF$ also effects a degree of control ( observed with a narL mutant ) ( 21 ) .', '@TF$ and @Regulated$ regulation .', 'Orientation of MelR bound at site R Wade et al . ( 17 ) showed that MelR - binding site R is essential for MelR - dependent repression of the @Regulated$ A MelR RNA - 1 B TB23 TB23 TB22 TB22 MelR + + % Relative Intensity 150 100 50 0 TB23 @TF$ + MelR TB22 Figure 2 .', 'The results suggest that the @TF$ protein specifically interacts with the 44.5 region in order to activate @Regulated$ - 46.5 operon expression .', 'Some operons such as @Regulated$ are under the transcriptional activation by the response regulators in Group 1 as well as by @TF$ , suggesting that they can be transcribed in response to the sensing of respective sugars regardless of the absence or presence of cAMP - CRP .', 'Our previous experiments have shown that when both @Regulated$ promoters 1P and 2P were repressed by DnaA protein , @TF$ protein specifically activated transcription from dnaA promoter 1P , while dnaA promoter 2P was still repressed ( 25 ) .', 'Taken together , our results suggest that CRP site 3 is the major site required for @TF$ activation of @Regulated$ expression .', 'We also identified @TF$ - regulated genes that are transcribed due to read - through of inefficient Rho - independent terminators ( ygeA and @Regulated$ ) .', 'We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends @Regulated$ and fumC DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Q 2000 Blackwell Science Ltd , Molecular Microbiology , 36 , 211 ┬▒ 222Activation at the E . coli @Regulated$ promoter 219 which transcription activation at the melAB promoter is absolutely dependent on the occupation of Site 2 0 by @TF$ .', 'In the same experiment we also examined the expression of @Regulated$ , which had been observed in another macroarray study to be repressed upon @TF$ induction in cells ( 5 ) .', 'This results in the repression of transcription , suggesting that the extent of oligomerization of @TF$ proteins over two @Regulated$ promoters contributes to the autoregulation of expression of the dnaA gene .', 'NarX and / or NarQ sensors , both @TF$ and NarP are phosphorylated where either response regulator can then activate @Regulated$ gene expression .', 'As expected , expression of known AraC - regulated genes , i . e . , @Regulated$ , araA , araD , araE , araF , February 2014 Volume 196 Number 3 jb . asm . org 663Stringer et al . FIG 1 ( A ) Validation of putative E . coli @TF$ target regions identified by ChIP - chip .', 'Negative regulation of aeg - 46.5 operon expression by the @TF$ protein In addition to its function as a transcriptional activator , the NarL protein negatively regulates expression of the frdA , @Regulated$ and nrfA operons in response to nitrate availability .', 'Summary Expression of the narK gene of Escherichia coli , like the @Regulated$ operon , is positively regulated by two frans - acting factors : @TF$ , which is activated by anaerobic conditions , and NarL , which is activated by the presence of nitrate .', 'Mutational analysis of the nirB and @Regulated$ operon control regions has identified cis - acting regions that are required for both @TF$ and NarP - dependent induction ( Tyson et al . , 1994 ) .', 'Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP / @TF$ , cooperate in the © 2004 Blackwell Publishing Ltd , Molecular Microbiology , 52 , 437 – 449Regulation of the chbB promoter 445 repression and induction of the @Regulated$ operon in E . coli .', 'The results in this report also demonstrate that @Regulated$ is regulated by the cyclic AMP ( cAMP ) - cAMP receptor protein ( @TF$ ) complex , which was found be essential for full induction .', 'The resulting recombinant plasmids were used to monitor @TF$ - dependent activation at the @Regulated$ promoter ; i . e . transcripts initiating at the melAB promoter run to the oop terminator and give discrete transcripts that are easy to detect by gel electrophoresis .', 'We also extended the previous studies by Bhende and Egan ( 4 ) of @TF$ - mediated transcription activation at @Regulated$ .', 'Recent studies ( 11 ) have shown that the occupation of site 2 by @TF$ is the most crucial for the activation of the @Regulated$ promoter , but that , of the four target sites , site 2 binds MelR most weakly .', 'The enhanced ability of @TF$ to activate @Regulated$ gene expression is possibly , by analogy to Crp , due to increased affinity of Fnr at the consensus versus the native non - consensus Fnr recognition sequence at dmsA .', 'We identified eight genes and confirmed that seven are transcriptionally activated by normal expression of @TF$ from the chromosomal rob gene ( inaA , marR , aslB , ybaO , mdlA , @Regulated$ , and ybiS ) .', 'For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell @Regulated$ regulation cannot be activated by Rob , although the gene responds to SoxS and @TF$ ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .', 'The fourth promoter predicted to be regulated by @TF$ was P @Regulated$ , the promoter of a two - gene operon encoding the hybrid cluster protein , HCP , and its reductase , HCR .', 'Since only the @TF$ protein can activate @Regulated$ - 46.5 operon expression ( again , presumably by interacting with RNA polymerase ) , the competition from NarL for the binding sites would lead to an antagonization of NarP activation .', 'Transposon mutagenesis experiments aimed at defining the Rob regulon revealed eight @TF$ - regulated targets , namely inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'The binding of two dimers of @TF$ protein to the @Regulated$ promoter 1P element enhances the binding of RNA polymerase to the dnaA promoter 1P', 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the binding mode of @TF$ protein to DNA containing the IciA I site and how this binding activates transcription from the dnaA promoter 1P .', 'This results in the formation of a repression loop and the repression of the @Regulated$ promoter by @TF$ ( reviewed by Schleif , 1996 ) .', 'In this study , we show that the global transcription factor FNR is a class II activator at the @Regulated$ promoter and that @TF$ represses activation by binding to a single target that overlaps the promoter 10 element .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( @Regulated$ , araA , araD , araE , araF , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', \"Based on these four sites , the . known start point for @Regulated$ transcription at + 1 ( 6 ) , and an estimated start point for araC transcription near - 170 , Ogden et aL proposed ( i ) that araBAD and araC are stimulated by the . binding of @TF$ to distinct regions on the ' DNA - CRPBAD and CRPC , respectively , and ( ii ) that the araO , when bound to araC protein , prevents the binding of CAP to either CRPc or CRPBAD , and this in turn de - B756 Biochemistry : Lee et aL Proc .\", 'These indicate that DnaA and @TF$ proteins independently bind to their respective binding sites on the @Regulated$ # 1996 Blackwell Science Ltd , Molecular Microbiology , 19 , 389 – 396392 Y .', 'Together with a bioinformatic analysis of other Enterobacteriaceae species , these data identify a conserved AraC regulon that includes 7 previously described @TF$ - regulated genes ( araB , araA , araD , araE , @Regulated$ , araG , and araH ) as well as three novel targets identified in this work ( ytfQ , araT , and araU ) .', 'On the other hand , activation of the P @Regulated$ promoter was delayed when glucose was present , consistent with the regulation of this promoter also by @TF$ ( 6 ) .', 'The shaded boxes denote 22 bp DNA sites for @TF$ : one site centered at position 241.5 is responsible for the activation of the @Regulated$ promoter , while the other , centered at position 2155.5 is involved in activation of the melAB promoter .', 'As the DNA binding site for @TF$ overlaps the @Regulated$ - 35 element , it is easy to understand how FNR represses acsP1 at the same time as it activates the nrf promoter .', 'Since we have shown previously that MalE - @TF$ and MarA are sufficient for the in vitro transcriptional activation of the zwf , fpr , fumC , @Regulated$ , nfo , and sodA promoters ( i . e . , additional factors are not required ) ( 18 , 19 ) , we conducted similar studies with purified Rob protein .', 'It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for @TF$ than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the mglB and @Regulated$ operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by GalS .', 'Both the aerobic and anaerobic expression levels of dcuB were only ca . twofold lower in the arcA mutant ( JRG3841 ) , indicating that ArcA plays no more than a minor role in regulating @Regulated$ expression in response to oxygen ( Fig . 5B ) and that @TF$ is not responsible for the FNR - independent mechanism of anaerobic activation of dcuB transcription .', 'Substitutions within CRP that affect transcription activation at the melAB promoter To identify the residues of @TF$ that are important in transcription activation at the @Regulated$ promoter , a library of randomly mutated crp genes was created using error - prone PCR .', 'In the case of argO , @TF$ mediates its transcriptional activation by Arg , as well as its repression by Lys ( 7 , 26 , 32 , 37 ) ; the latter is achieved by a mechanism involving the active trapping at the @Regulated$ promoter by ArgP of RNA polymerase , which is then prevented from being released to engage in productive transcription ( 26 ) .', 'In contrast , IHF does not enhance the ability of @TF$ to activate @Regulated$ expression in the wild - type background .', 'As shown previously ( 16 ) , IciA protein activated transcription from the @Regulated$ promoter 1P in the 488 bp EcoRI / HindIII fragment ( + @TF$ I , + IciA II ) of plasmid pHJ1 without affecting transcription from the promoter 2P ( Fig . 2A and B ) .', 'Although not tested directly , the ybjC - mdaA operon may contain a third gene , rimK , as : ( i ) there does not appear to be either an RNA chain termination signal nor a promoter between mdaA and rimK ( they are separated by 172 bp ) ; ( ii ) Barbosa and Levy ( Barbosa and Levy 2000 ) observed the induction of @Regulated$ in their microarray analysis for @TF$ - induced genes ; and ( iii ) the nearest potential MarA binding site upstream of rimK is that identified for ybjC .', 'The known @TF$ - activated genes are sodA ( encoding Mnsuperoxide dismutase ) , fpr ( NADPH - ferredoxin oxidoreductase ) , micF ( antisense RNA , repressor of OmpF translation ) , ribA ( cyclic GMP hydrolase ) , inaA ( unknown function ) , fldA and fldB ( flavodoxins A and B ) , @Regulated$ ( endonuclease IV ) , marRAB ( multiple - antibiotic - resistance operon ) , nfsA ( also called mdaA , a nitroreductase ) , zwf ( glucose - 6 - phosphate dehydrogenase ) , fur ( an iron - binding repressor of iron uptake ) , fumC ( fumarase C ) , acnA ( aconitase ) , tolC ( outer membrane protein ) , acrAB ( drug efflux pump ) , and rimK ( a modifier of ribosomal protein S6 ) .', 'Model for MelR and @TF$ binding during activation at the @Regulated$ promoter .', 'Our analysis revealed that inaA , marRAB , aslB , ybaO , mdlA , @Regulated$ , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .', 'The location of the @TF$ binding site upstream of @Regulated$ is too far upstream of the transcription start site to repress transcription by directly occluding RNAP .', 'Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the melAB pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli @Regulated$ promoter 345 Fig . 10 .', 'In this study we confirmed the location of the Fnr - binding site at position 64.5 and supported the idea that the mechanism of @TF$ - dependent activation of @Regulated$ expression is distinct from that of promoters with the Fnr - binding site at position 41.5 .', 'Thus , TB201 is a derivative of TB20 in which the DNA site for CRP located between @TF$ site 1 Other determinants for MelR - dependent repression of the @Regulated$ promoter To investigate the effects of altering the spacing between MelR - binding site R and the upstream sites , derivatives of the TB22 fragment with 5 - or 10 - bp deletions , or with 5 - , 41 - or 91 - bp insertions were constructed ( see Figure 6A and Materials and methods section ) .', 'Similarly , at the FNR - activated NarL - repressed @Regulated$ promoter @TF$ protects a large region that includes the sites for both FNR and RNA polymerase binding ( Bearson et al . , 2002 ) .', 'Repression by NagC and ChbR in vivo A plasmid overexpressing @TF$ produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing NagC produces less than twofold repression ( Table 5 ) .', 'Two approaches were used to confirm that @TF$ is this sixth factor that regulates P @Regulated$ .', 'This and the finding that expression of @Regulated$ - lacZ fusion was relatively high even in the absence of cAMP ( table 4 ) may suggest that binding of @TF$ to all 3 sites slightly activates gntV expression .', 'The Rob homologs MarA and @TF$ can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , @Regulated$ , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', 'In the case of the @Regulated$ promoter , the differences in transcript levels between the two transformants were at least as great as those for the NsrRrepressed genes , making P ydbE a prime candidate for direct activation by @TF$ .', 'Apparently , only the @TF$ protein is competent to activate transcription of the @Regulated$ - 46.5 operon when bound to the 44.5 region .', 'Only one other novel @TF$ - regulated gene identified in this work , E . coli @Regulated$ , has a known connection to arabinose metabolism ( 39 ) .', 'It is interesting to note that the three central operators , galOE2 , OE3 and OE4 , which have higher affinity for GalS than GalR , all have A / T at positions 8 of the operators ( Fig . 1B ) which is also the case for the @Regulated$ and galS operators ( Weickert and Adhya , 1993a , b ) which are preferentially regulated by @TF$ .', 'Transposon mutagenesis experiments aimed at defining the @TF$ regulon revealed eight Rob - regulated targets , namely inaA , marRAB , @Regulated$ , ybaO , mdlA , yfhD , ybiS , and galT ( 10 ) , some of which are also known members of the mar and sox regulons ( 11 ) .', 'Charles Street Baltimore , MD 21218 , USA * Corresponding author In the absence of arabinose and interactions with other proteins , @TF$ , the activator - repressor that regulates the @Regulated$ operon in Escherichia coli , was found to prefer participating in DNA looping interactions between the two well - separated DNA half - sites , araI 1 and araO 2 at their normal separation of 211 base - pairs rather than binding to these same two half - sites when they are adjacent to one another .', 'Repression of the @Regulated$  promoter carried on different KK81 fragments . The figure shows the degree of @TF$ - dependent repression of the melR promoter carried on the starting KK81 fragment and the mutant derivatives , S2 ┬▒ S5 .', 'In contrast , the repression of both soxS and rob by @TF$ , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and @Regulated$ promoter regions ( 15 , 16 ) .', ' @TF$ - mediated regulation of argO transcription was then tested in a defined in vitro run - off transcription system using 70 - bearing RNAP holoenzyme and a linear DNA template that encompasses the @Regulated$ region from 293 to + 109 ( Fig . 1C ) .', 'However , placing the NanR , @TF$ and NagC binding sites closer to the @Regulated$ promoter enhances the ability of the regulators to activate fimB expression .', 'To measure @TF$ - dependent repression of @Regulated$ promoter activity , the pRW50 derivatives carrying the TB20 , TB22 , TB23 or TB10 fragments were transformed into WAM1321 melR lac cells containing either pJW15 , encoding melR , or empty control vector , pJWmelR and b - galactosidase activities were measured .', ' @Regulated$ - 46.5 operon expression is also negatively regulated by the NarL protein but in this case @TF$ does not appear to act as a classical repressor .', 'Repression by NagC and @TF$ in vivo A plasmid overexpressing ChbR produces very strong repression of the @Regulated$ â€“ lacZ fusion in a wild - type strain ( 37 - fold ) , whereas a plasmid overexpressing @TF$ produces less than twofold repression ( Table 5 ) .', 'Transcriptional activation by MarA , SoxS and @TF$ of two @Regulated$ promoters using one binding site : a complex promoter configuration for tolC in Escherichia coli .', '@TF$ activates transcription of the operon that encodes the two activator proteins , @Regulated$ ( 37 , 38 ) .', 'Finally , either NarL or NarP was able to activate @Regulated$ - lacZ expression independently of the other , while only @TF$ could repress expression .', 'Given that cyclic AMP receptor protein ( CRP ) , the second activator required for full rhaBAD expression , cannot activate @Regulated$ expression in a rhaS strain , it was of interest to test whether @TF$ could activate transcription in combination with RhaS - CTD .', 'In the second part of the present study , we investigated the E . coli @Regulated$ promoter , which Constantinidou et al . [ 11 ] had also found to be induced by @TF$ independently of FNR .', 'Positive and negative transcriptional regulation of the Escherichia coli gluconate regulon gene @Regulated$ by @TF$ and the cyclic AMP ( cAMP ) - cAMP receptor protein complex .', 'Miller et al . ( 28 ) have found that SoxS activates @Regulated$ transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either purified MalE - @TF$ or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .', 'Both @Regulated$ and gntKU are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by GntR ; one @TF$ - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'Based on data from in vitro studies , Celis suggested that wild - type @TF$ both represses its own transcription and activates that of argK ; he also sought to explain the Can r phenotype associated with the argP d mutation on the assumption that it is a loss - of - function ( that is , dominant - negative ) allele whose product is unable to efficiently activate the genes involved in Arg uptake , including @Regulated$ ( 12 ) .', 'The major conclusions are that : ( i ) both Lrp and @TF$ stimulate P argO in an effector - dependent manner , but to a different extent , ( ii ) the two regulators behave as competitive activators , and ( iii ) ArgP is intrinsically a more potent activator for P @Regulated$ as compared with Lrp .', 'Thus , NanR and @TF$ appear to activate @Regulated$ expression independently .', ' @Regulated$ gene ( Fig . 4A ) , suggesting that the promoters are However , no such reduction of bGal activity by @TF$ negatively regulated by GntR .', 'For instance , Rob has been shown to bind and activate the @Regulated$ promoter in vitro but whole cell zwf regulation cannot be activated by Rob , although the gene responds to @TF$ and MarA ( Ariza et al . , 1995 ; Jair et al . , 1995 1996a ; b ) .', 'Some of these operons are regulated by the @TF$ protein alone , such as the narG and @Regulated$ operons , whereas expression of the nirB ( encoding NADH - dependent nitrite reductase ) , nrfA and fdnG operons is controlled by both the NarL and NarP proteins .', 'Although a similar arrangement of sites around the transcription initiation regions of other genes of the gal regulon might have suggested that they all are regulated by a similar mechanism to galE ( Weickert and Adhya , 1993a , b ; Semsey et al . , 2007 ) , the results presented in this paper demonstrate that repression of the @Regulated$ gene , is mediated by a distinctly different nucleoprotein complex involving NagC as well as @TF$ and GalS and including at least 280 bp of DNA .', 'The repression of @Regulated$ by @TF$ was quantitatively similar in vivo and in vitro , whereas that of hdeA was greater in vivo ( ~ 6 – 25 - fold ) than in vitro ( ~ 1.7 - fold ) .', 'Expression of acrZ is coregulated with @Regulated$ and tolC by the MarA , Rob , and @TF$ transcription factors .', 'As the DNA binding site for @TF$ overlaps the acsP1 - 35 element , it is easy to understand how FNR represses @Regulated$ at the same time as it activates the nrf promoter .', 'Studies of the @Regulated$ promoter , p BAD , show that to activate transcription the @TF$ protein binding site must overlap the 35 region of the promoter by 4 bp ( 25 ) ( Fig . 1 ) .', 'Although the role of CAP in repression of @Regulated$ â€“ B is likely to be purely architectural , it is clear that a major part of the @TF$ activation of nagE is due to protein â€“ protein contacts between CAP and RNA polymerase .', 'Effect of excess ChbR and NagC on repression of the @Regulated$ â€“ lacZ fusion . wild type nagC chbR pTZ ( @TF$ ) 48 Â± 6 47 Â± 3 83 Â± 4 pTZ ( Mlc ) 42 Â± 4 62 Â± 3 56 Â± 4 pTZ ( ChbR ) 2.3 Â± 0.2 32 Â± 2 2.8 Â± 0.5 pTZ control .', ' @TF$ protein , the initiator protein for initiation of Escherichia coli chromosomal replication , has been shown to repress its own expression from two @Regulated$ promoters , 1P and 2P .', 'DISCUSSION Physically and functionally heterogeneous complexes containing DnaA protein bound to the @Regulated$ promoter were isolated and analyzed in order to address the mode of binding of @TF$ protein to DNA containing a single DnaA box and how this binding inhibits transcription by RNA polymerase .', 'Sufficiency of @TF$ for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and @Regulated$ , promoters known to be activated by MalE - SoxS and MarA .', 'Both inaA and @Regulated$ are known @TF$ - regulated genes ( 5 , 21 ) .', 'The Rob homologs @TF$ and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including @Regulated$ , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .', 'One gene , @Regulated$ , was repressed by @TF$ .', 'By replacing the 274 bp of wild - type DNA from - 467 to - 194 with heterologous sequences it was shown that neither @TF$ nor NagC requires this segment of DNA in order to activate @Regulated$ expression .', 'We assayed the plasmid - borne lacZ fusions carrying these mutations to determine whether either of these two @TF$ sites was required for @Regulated$ activation .', 'Previous results showing an interaction between RhaS D241 and 70 R599 at rhaBAD ( 4 ) led us to investigate whether an interaction between RhaR D276 and 70 R599 might be required for @TF$ activation at @Regulated$ .', 'Like MalE - @TF$ , MarA ( i ) activated the transcription of zwf , fpr , @Regulated$ , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .', 'There is no obvious effect of the CAP + 6 mutation on nagE expression but this is presumably due to two opposing effects which are of similar magnitude : an 10 - to 15 - fold increase in nagE expression due to loss of repressor binding ( equivalent to that seen for the nagB fusion ) and a 10 - to 15 - fold decrease in @Regulated$ expression due to a lack of cAMP / @TF$ activation ( 5 ; see also below ) .', 'Both intact @TF$ and the N - terminal fragment activated expression of stress genes ( @Regulated$ , fumC , sodA ) but with a pattern distinct from that found for SoxS and MarA .', 'Figure 10 illustrates the likely organization of @TF$ subunits during transcription activation at the @Regulated$ pro - ┬® 2003 Blackwell Publishing Ltd , Molecular Microbiology , 48 , 335 ÔÇô 348MelR binding at the E . coli melAB promoter 345 Fig . 10 .', 'Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by ArgP may need to be reexamined , given that ( i ) there is a discrepancy between the expected argK runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for @TF$ activation experiments ( 12 ) and ( ii ) the promoter ( for @Regulated$ ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .', 'The organization of the putative NarL sites in the dcuB operator - promoter region clearly does not conform to the preferred 7 - 2 - 7 arrangement , which is consistent with the observation that @TF$ , but not NarP , represses @Regulated$ expression .', '@TF$ - dependent repression of the @Regulated$ promoter carried by each derivative was then measured , as above .', 'This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that @Regulated$ transcription is regulated by @TF$ .', 'Role of CRP in activation of the @Regulated$ promoter The results from the in vitro studies reported above suggest that @TF$ interacts directly at the melAB promoter .', 'DISCUSSION Physically and functionally heterogeneous complexes containing IciA protein bound to the dnaA promoter were isolated and analyzed in order to address the binding mode of IciA protein to DNA containing the @TF$ I site and how this binding activates transcription from the @Regulated$ promoter 1P .', 'The strong upregulation of @Regulated$ by the plasmid specified @TF$ observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al . , 1999 ; Pomposiello et al . , 2001 ) .', 'We found that @TF$ ( i ) activates the transcription of zwf , fpr , fumC , micF , nfo , and sodA , ( ii ) requires a 21 - bp soxbox - marbox - robbox sequence to activate zwf transcription , ( iii ) protects the soxbox / marbox / robbox from attack by DNase I , ( iv ) is ambidextrous , i . e . , requires the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF , ( v ) bends zwf and @Regulated$ DNA , and ( vi ) binds zwf and fumC DNA as a monomer .', 'Again , with the use of the criterion for a significant effect of a substitution as one that reduces transcription to &lt; 80 % of that produced by wild - type 70 , none of the library ’ s substitutions significantly reduced @TF$ dependent transcription activation of the three class II promoters ( i . e . , fumC , @Regulated$ , and inaA ) ( Fig . 2a – c ) .', 'Studies of @Regulated$ transcription in vitro confirm the results in vivo and show that promoter activity is directly stimulated by AraC and @TF$ ( Stoner &amp; Schleif , 1983 ) .', 'However , the inactivation of DcuS ( IMW553 ) led to a decrease of @Regulated$ - lacZ expression by a factor of 1.5 , which could be due to interaction between CitA and @TF$ .', 'RESULTS @TF$ both directly and indirectly activates @Regulated$ .', '@TF$ repression is critical in maintaining @Regulated$ mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .', 'Furthermore , even the finding of Celis ( 12 ) that argK is a target for transcriptional activation by @TF$ may need to be reexamined , given that ( i ) there is a discrepancy between the expected @Regulated$ runoff transcript size ( approximately 310 bases ) determined from the promoter mapping data published by Celis et al . in an earlier paper ( 13 ) and that reported by him ( 215 bases ) for ArgP activation experiments ( 12 ) and ( ii ) the promoter ( for argK ) mapped by his group ( 13 ) is within the argK coding region per the genome sequence published by Blattner et al . ( 5 ) ( GenBank accession number AE000375 , wherein argK is annotated ygfD ) .', 'For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the @Regulated$ operon is regulated predominantly by NarL but is also responsive in part to @TF$ ( 33 , 39 ) .', 'The treB @Regulated$ operon is negatively regulated by @TF$ , whose gene treR is located upstream of treB but is not part of the operon .', 'Our analysis revealed that inaA , marRAB , aslB , ybaO , @Regulated$ , yfhD , and ybiS were transcriptionally activated by constitutive levels of @TF$ , while galT expression was repressed .', 'In addition to the specific regulator , transcription from certain promoters regulated by members of this family requires other proteins for maximal activity ( e . g . , the @TF$ in the P @Regulated$ promoter ) , or histone - like proteins that act as negative regulators ( e . g . , YomA in the VirF - regulated P yop promoters ) ( 52 , 124 , 242 ) .', 'A 3.4 - fold decrease of @Regulated$ - lacZ induction was observed when @TF$ was inactivated ( IMW554 ) .', 'According to previous results ( 16 ) , @TF$ protein was able to activate in vitro transcription from @Regulated$ promoter 1P in the presence of limiting amounts of RNA polymerase .', 'Samples were loaded on a polyacrylamide gel , run , and analyzed by autoradiography . gave a single retarded band of mobility similar to that observed with purified His 6 - ArgP protein , whereas crude extracts devoid of @TF$ gave no retarded band ( Fig . 3B ; see also Fig . 5A ) , indicating that no protein other than ArgP is able to bind significantly to the @Regulated$ promoter region .', 'The Escherichia coli @TF$ protein can interact with two distinct determinants in the 70 subunit of RNA polymerase according to promoter architecture : identification of the target of Ada activation at the @Regulated$ promoter .', 'Like MalE - SoxS , @TF$ ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21 - bp  soxbox  sequence to activate zwf transcription ; and ( iii ) was  ambidextrous ,  i . e . , required the C - terminal domain of the a subunit of RNA polymerase for activation of zwf but not @Regulated$ or micF .', 'The @Regulated$ operator had the highest apparent affinity in these assays , where the repression complex would be expected to contain at least four @TF$ dimers , compared with two in the minimal complexes formed on the synthetic sites .', 'Both gntT and @Regulated$ are regulated positively by the cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex and negatively by @TF$ ; one GntR - binding element for gntK has been proposed ( 12 , 34 ) , and two GntR - binding elements for gntT were experimentally defined ( 26 ) .', 'Our results indicate that the repression of transcription from the two dnaA promoters ( 1P and 2P ) by @TF$ protein is promoted by the oligomerization of DnaA protein over @Regulated$ promoters 1P and 2P and that this binding directly prevents RNA polymerase binding to the two promoters .', 'Furthermore , as NagC binding to O NC1 prevents @TF$ - dependent activation of @Regulated$ ( Condemine et al . , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .', 'Summary Expression of the @Regulated$ gene of Escherichia coli , like the narGHJI operon , is positively regulated by two frans - acting factors : Fnr , which is activated by anaerobic conditions , and @TF$ , which is activated by the presence of nitrate .']\n",
      "['MelR', 'Fnr', 'NarL', 'CAP', 'IciA', 'Fnr', 'GntR', 'IciA', 'NarP', 'GalR', 'MarR', 'NarL', 'RhaS', 'IciA', 'MarA', 'NagC', 'ChiX', 'MarA', 'MelR', 'NanR', 'ArgP', 'ArgP', 'MelR', 'ArgP', 'EvgA', 'NagC', 'MarR', 'NagC', 'NagC', 'NarL', 'Rob', 'RhaR', 'Fnr', 'Lrp', 'Rob', 'OmpR', 'NarP', 'NanR', 'ArgP', 'OxyR', 'NarL', 'MarA', 'ArgP', 'ArgP', 'NagC', 'SoxS', 'ArgP', 'ArcA', 'NarL', 'SoxS', 'NagC', 'ArgP', 'MelR', 'MetJ', 'Rob', 'Rob', 'LysR', 'Rob', 'NagC', 'MarA', 'AraC', 'CAP', 'MelR', 'MarA', 'ArgP', 'Fnr', 'MarA', 'AraC', 'OmpR', 'MelR', 'CRP', 'Rob', 'CRP', 'SoxS', 'SoxS', 'MelR', 'Fis', 'ArgP', 'MarA', 'NarP', 'Rob', 'GntR', 'NarL', 'MarA', 'DnaA', 'NarL', 'Fnr', 'NarL', 'ArgP', 'DnaA', 'ArgP', 'MelR', 'RhaR', 'ArgP', 'GntR', 'Lrp', 'MelR', 'SoxS', 'DeoR', 'IHF', 'SoxR', 'NarP', 'OmpR', 'ArgP', 'MelR', 'RhaR', 'ArgP', 'MelR', 'ArgP', 'MarR', 'DnaA', 'CAP', 'IciA', 'AraC', 'SoxS', 'IHF', 'NarL', 'SoxS', 'MarR', 'GalR', 'MarA', 'NarL', 'Lrp', 'RhaS', 'CAP', 'NarL', 'NarP', 'Rob', 'CRP', 'ArgP', 'Fis', 'Rob', 'CAP', 'NarL', 'NarL', 'MarA', 'ArgP', 'NarL', 'NarP', 'DcuR', 'Rob', 'MelR', 'Mlc', 'GntR', 'NagC', 'MelR', 'ArgP', 'Rob', 'Lrp', 'ArgP', 'Rob', 'NarP', 'IciA', 'MarA', 'Mlc', 'Rob', 'NagC', 'ArgP', 'AraC', 'NarP', 'MarA', 'NarP', 'MarA', 'AraC', 'FNR', 'ArgP', 'SoxS', 'MarA', 'CAP', 'ArcA', 'MelR', 'DnaA', 'CRP', 'NarL', 'MelR', 'MarA', 'IciA', 'LysR', 'CRP', 'NarP', 'ArgP', 'MelR', 'NarP', 'CRP', 'IciA', 'CRP', 'AraC', 'Rob', 'MelR', 'MarA', 'DnaA', 'NarL', 'AraC', 'NarL', 'Fnr', 'NarL', 'CAP', 'CRP', 'MelR', 'RhaS', 'MelR', 'Fnr', 'Rob', 'MarA', 'NsrR', 'NarP', 'Rob', 'IciA', 'IciA', 'AraC', 'NarL', 'AraC', 'CAP', 'IciA', 'AraC', 'CRP', 'CRP', 'FNR', 'SoxS', 'GalS', 'ArcA', 'CRP', 'ArgP', 'NanR', 'IciA', 'MarA', 'SoxS', 'CRP', 'Rob', 'AraC', 'MelR', 'Fnr', 'MelR', 'NarL', 'ChbR', 'NsrR', 'GntR', 'SoxS', 'NsrR', 'NarP', 'AraC', 'GalS', 'Rob', 'AraC', 'MelR', 'SoxS', 'ArgP', 'IHF', 'MelR', 'NarL', 'ChbR', 'Rob', 'RhaR', 'NarL', 'CRP', 'NarL', 'GntR', 'SoxS', 'GntR', 'ArgP', 'ArgP', 'NagC', 'GntR', 'SoxS', 'NarL', 'GalR', 'MarA', 'SoxS', 'FNR', 'AraC', 'CAP', 'NagC', 'DnaA', 'DnaA', 'Rob', 'Rob', 'MarA', 'Rob', 'NanR', 'CRP', 'RhaR', 'SoxS', 'CAP', 'Rob', 'MelR', 'ArgP', 'NarL', 'MelR', 'OxyR', 'CRP', 'IciA', 'MarA', 'Rob', 'Rob', 'CRP', 'DcuS', 'CRP', 'NagC', 'ArgP', 'NarP', 'TreR', 'Rob', 'CRP', 'DcuR', 'IciA', 'ArgP', 'Ada', 'MarA', 'MetJ', 'GntR', 'DnaA', 'CRP', 'NarL']\n",
      "['melAB', 'cydA', 'narG', 'nagE', 'dnaA', 'dmsA', 'gntKU', 'dnaA', 'fdnGHI', 'galP', 'marA', 'frdA', 'rhaBAD', 'dnaA', 'purA', 'nagBE', 'chiP', 'zwf', 'melR', 'fimB', 'lysP', 'dapB', 'melAB', 'argK', 'yhiUV', 'nagE', 'marA', 'glmU', 'fimB', 'nirB', 'fpr', 'rhaSR', 'napF', 'cadBA', 'zwf', 'nanC', 'nirB', 'fimB', 'lysP', 'hcp', 'narG', 'hdeA', 'yggA', 'argO', 'glmU', 'fumC', 'argO', 'dcuB', 'frdA', 'mar', 'fimB', 'lysA', 'melR', 'metE', 'marRAB', 'micF', 'lysA', 'sodA', 'glmUS', 'fumC', 'araD', 'nagE', 'melAB', 'hdeA', 'argP', 'dmsABC', 'zwf', 'araA', 'ompC', 'melAB', 'cdd', 'zwf', 'melAB', 'fpr', 'inaA', 'melAB', 'nirB', 'lysP', 'gatA', 'aeg', 'galT', 'gntV', 'hcp', 'hdeA', 'dnaA', 'narGHJI', 'dmsABC', 'nrfA', 'argO', 'guaBA', 'argO', 'melR', 'rhaSR', 'dapB', 'gntT', 'lysP', 'melR', 'fur', 'deoCABC', 'glcDEFGB', 'soxS', 'aeg', 'ompF', 'yggA', 'melR', 'rhaSR', 'argO', 'melAB', 'argO', 'marA', 'dnaA', 'araC', 'dnaA', 'araE', 'micF', 'ompC', 'nirB', 'fldA', 'marA', 'galETK', 'nfnB', 'narK', 'argO', 'rhaT', 'araC', 'aeg', 'ydhY', 'nfo', 'araBAD', 'argO', 'ogt', 'fumC', 'araC', 'dmsA', 'fdnG', 'tolC', 'yggA', 'dmsA', 'napF', 'dcuB', 'fumC', 'melR', 'ptsHIcrr', 'gntT', 'galP', 'melR', 'argO', 'fumC', 'cadBA', 'argO', 'micF', 'nrfABCDEFG', 'dnaA', 'gatA', 'manX', 'yfhD', 'chbBCARFG', 'nrdA', 'araBAD', 'nrfA', 'purA', 'ydhY', 'inaA', 'araU', 'hemA', 'lysC', 'micF', 'nfnB', 'nagE', 'acnA', 'melR', 'mioC', 'rhaSR', 'ydhY', 'melAB', 'rob', 'dnaA', 'lysA', 'fucPIK', 'fdnGHI', 'cadBA', 'melR', 'aeg', 'malEFG', 'dnaA', 'rhaSR', 'polB', 'zwf', 'melAB', 'hdeA', 'dnaA', 'nrfA', 'araB', 'dmsA', 'narGHJI', 'nrfA', 'chbBCARFG', 'gntT', 'melAB', 'rhaBAD', 'melAB', 'dmsABC', 'yfhD', 'zwf', 'hcp', 'aeg', 'yfhD', 'dnaA', 'dnaA', 'araC', 'ynfEFGHI', 'araB', 'araBAD', 'dnaA', 'araF', 'araB', 'melR', 'acsP1', 'micF', 'galS', 'dcuB', 'melAB', 'argO', 'fimB', 'dnaA', 'rimK', 'nfo', 'melAB', 'yfhD', 'ytfQ', 'melAB', 'napF', 'melR', 'dmsA', 'chbB', 'nrfA', 'gntV', 'micF', 'ydbC', 'aeg', 'ytfQ', 'mglB', 'aslB', 'araBAD', 'melR', 'rob', 'argO', 'fimB', 'melR', 'aeg', 'chbB', 'tolC', 'rhaSR', 'nrfA', 'rhaBAD', 'ogt', 'gntT', 'mar', 'gntT', 'argK', 'argO', 'fimB', 'gntR', 'zwf', 'frdA', 'galP', 'purA', 'acrAB', 'acsP1', 'araBAD', 'nagE', 'chbB', 'dnaA', 'dnaA', 'sodA', 'marRAB', 'sodA', 'galT', 'fimB', 'rhaSR', 'rhaSR', 'fumC', 'nagE', 'inaA', 'melAB', 'argK', 'dcuB', 'melR', 'hcp', 'melAB', 'dnaA', 'nfnB', 'fumC', 'micF', 'araE', 'citC', 'rhaBAD', 'chiP', 'argK', 'fdnGHI', 'treC', 'mdlA', 'rhaBAD', 'citC', 'dnaA', 'dapB', 'alkA', 'fumC', 'mefA', 'gntKU', 'dnaA', 'nanC', 'narK']\n",
      "[0, 2, 3, 1, 1, 0, 1, 1, 1, 2, 1, 3, 0, 0, 2, 2, 1, 0, 2, 1, 0, 1, 0, 0, 0, 1, 2, 1, 2, 0, 0, 1, 1, 0, 0, 0, 3, 0, 1, 1, 3, 1, 1, 3, 0, 0, 1, 0, 1, 0, 0, 1, 1, 2, 1, 1, 2, 0, 0, 0, 1, 0, 1, 2, 2, 0, 1, 3, 0, 0, 3, 1, 1, 0, 1, 0, 2, 3, 1, 0, 2, 0, 1, 2, 2, 1, 1, 1, 1, 2, 3, 2, 0, 3, 1, 0, 1, 0, 2, 0, 0, 1, 0, 1, 2, 0, 1, 0, 3, 3, 2, 1, 1, 0, 1, 2, 3, 0, 2, 3, 0, 0, 1, 1, 3, 2, 2, 0, 0, 1, 1, 0, 0, 2, 0, 1, 3, 2, 0, 0, 0, 1, 2, 2, 2, 2, 1, 0, 3, 1, 0, 0, 3, 1, 2, 1, 2, 1, 0, 0, 2, 2, 0, 3, 1, 3, 0, 0, 0, 3, 2, 2, 1, 2, 0, 1, 1, 2, 1, 0, 3, 1, 0, 0, 1, 0, 3, 1, 0, 2, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 3, 0, 3, 0, 1, 2, 2, 3, 1, 3, 3, 3, 0, 1, 0, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 2, 1, 0, 1, 1, 2, 3, 0, 0, 0, 0, 3, 3, 1, 3, 1, 1, 1, 1, 2, 1, 1, 0, 0, 1, 1, 0, 2, 1, 1, 1, 1, 0, 1, 3, 3, 2, 2, 3, 1, 0, 1, 1, 2, 1, 0, 3, 0, 2, 0, 0, 0, 0, 0, 0, 0, 1, 2, 2, 3, 1, 0, 0, 1, 0, 0, 1, 0, 2, 1, 3, 2, 0, 0, 0, 0, 1, 1, 1, 2, 2, 1, 0, 0]\n",
      "[0, 2, 3, 0, 1, 0, 1, 0, 1, 2, 2, 3, 0, 0, 2, 2, 1, 0, 2, 1, 0, 1, 1, 0, 0, 2, 2, 1, 2, 0, 0, 1, 1, 0, 0, 0, 3, 0, 3, 3, 3, 1, 1, 3, 0, 0, 1, 1, 1, 0, 0, 1, 1, 2, 1, 0, 2, 0, 0, 0, 1, 0, 1, 2, 2, 0, 1, 3, 0, 0, 3, 1, 0, 0, 1, 0, 1, 3, 1, 0, 2, 0, 1, 2, 2, 1, 1, 3, 3, 2, 3, 1, 0, 3, 1, 0, 2, 0, 2, 0, 0, 1, 0, 1, 2, 0, 3, 0, 3, 3, 2, 1, 1, 0, 1, 2, 3, 0, 2, 3, 0, 0, 1, 1, 3, 1, 1, 0, 0, 1, 1, 0, 2, 1, 0, 1, 3, 0, 0, 0, 0, 1, 2, 2, 2, 2, 1, 0, 3, 0, 0, 0, 3, 1, 1, 1, 2, 1, 0, 0, 2, 2, 0, 3, 1, 3, 0, 0, 0, 3, 2, 2, 0, 2, 0, 1, 0, 2, 1, 0, 3, 1, 0, 0, 1, 0, 3, 1, 0, 2, 2, 0, 1, 1, 0, 0, 2, 3, 0, 0, 0, 0, 0, 1, 3, 0, 3, 1, 1, 2, 2, 3, 1, 3, 3, 0, 0, 1, 0, 1, 1, 1, 1, 0, 1, 1, 0, 0, 0, 3, 1, 0, 1, 1, 2, 3, 1, 0, 1, 0, 3, 3, 1, 3, 1, 2, 1, 1, 2, 1, 2, 0, 0, 1, 1, 0, 2, 1, 1, 1, 1, 0, 2, 1, 3, 2, 2, 3, 1, 0, 1, 1, 2, 1, 0, 3, 0, 2, 0, 0, 0, 0, 1, 0, 0, 1, 2, 2, 3, 1, 1, 0, 1, 0, 0, 1, 0, 2, 1, 3, 2, 0, 0, 0, 0, 1, 1, 1, 2, 2, 1, 0, 0]\n"
     ]
    }
   ],
   "source": [
    "print(input_sentences)\n",
    "print(input_sentences_tagged)\n",
    "print(input_regulators)\n",
    "print(input_regulateds)\n",
    "print(true_labels)\n",
    "print(predicted_labels)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "313\n",
      "313\n",
      "313\n",
      "313\n",
      "313\n",
      "313\n"
     ]
    }
   ],
   "source": [
    "print(len(input_sentences))\n",
    "print(len(input_sentences_tagged))\n",
    "print(len(input_regulators))\n",
    "print(len(input_regulateds))\n",
    "print(len(true_labels))\n",
    "print(len(predicted_labels))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{0: 'activator', 1: 'no_relation', 2: 'repressor', 3: 'regulator'}"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "id2label"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[0, 2, 3, 1, 1, 0, 1, 1, 1, 2, 1, 3, 0, 0, 2, 2, 1, 0, 2, 1, 0, 1, 0, 0, 0, 1, 2, 1, 2, 0, 0, 1, 1, 0, 0, 0, 3, 0, 1, 1, 3, 1, 1, 3, 0, 0, 1, 0, 1, 0, 0, 1, 1, 2, 1, 1, 2, 0, 0, 0, 1, 0, 1, 2, 2, 0, 1, 3, 0, 0, 3, 1, 1, 0, 1, 0, 2, 3, 1, 0, 2, 0, 1, 2, 2, 1, 1, 1, 1, 2, 3, 2, 0, 3, 1, 0, 1, 0, 2, 0, 0, 1, 0, 1, 2, 0, 1, 0, 3, 3, 2, 1, 1, 0, 1, 2, 3, 0, 2, 3, 0, 0, 1, 1, 3, 2, 2, 0, 0, 1, 1, 0, 0, 2, 0, 1, 3, 2, 0, 0, 0, 1, 2, 2, 2, 2, 1, 0, 3, 1, 0, 0, 3, 1, 2, 1, 2, 1, 0, 0, 2, 2, 0, 3, 1, 3, 0, 0, 0, 3, 2, 2, 1, 2, 0, 1, 1, 2, 1, 0, 3, 1, 0, 0, 1, 0, 3, 1, 0, 2, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 3, 0, 3, 0, 1, 2, 2, 3, 1, 3, 3, 3, 0, 1, 0, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 2, 1, 0, 1, 1, 2, 3, 0, 0, 0, 0, 3, 3, 1, 3, 1, 1, 1, 1, 2, 1, 1, 0, 0, 1, 1, 0, 2, 1, 1, 1, 1, 0, 1, 3, 3, 2, 2, 3, 1, 0, 1, 1, 2, 1, 0, 3, 0, 2, 0, 0, 0, 0, 0, 0, 0, 1, 2, 2, 3, 1, 0, 0, 1, 0, 0, 1, 0, 2, 1, 3, 2, 0, 0, 0, 0, 1, 1, 1, 2, 2, 1, 0, 0]\n",
      "['activator', 'repressor', 'regulator', 'no_relation', 'no_relation', 'activator', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'regulator', 'activator', 'activator', 'repressor', 'repressor', 'no_relation', 'activator', 'repressor', 'no_relation', 'activator', 'no_relation', 'activator', 'activator', 'activator', 'no_relation', 'repressor', 'no_relation', 'repressor', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'activator', 'regulator', 'activator', 'no_relation', 'no_relation', 'regulator', 'no_relation', 'no_relation', 'regulator', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'activator', 'activator', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'no_relation', 'repressor', 'activator', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'repressor', 'repressor', 'activator', 'no_relation', 'regulator', 'activator', 'activator', 'regulator', 'no_relation', 'no_relation', 'activator', 'no_relation', 'activator', 'repressor', 'regulator', 'no_relation', 'activator', 'repressor', 'activator', 'no_relation', 'repressor', 'repressor', 'no_relation', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'regulator', 'repressor', 'activator', 'regulator', 'no_relation', 'activator', 'no_relation', 'activator', 'repressor', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'repressor', 'activator', 'no_relation', 'activator', 'regulator', 'regulator', 'repressor', 'no_relation', 'no_relation', 'activator', 'no_relation', 'repressor', 'regulator', 'activator', 'repressor', 'regulator', 'activator', 'activator', 'no_relation', 'no_relation', 'regulator', 'repressor', 'repressor', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'repressor', 'activator', 'no_relation', 'regulator', 'repressor', 'activator', 'activator', 'activator', 'no_relation', 'repressor', 'repressor', 'repressor', 'repressor', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'activator', 'regulator', 'no_relation', 'repressor', 'no_relation', 'repressor', 'no_relation', 'activator', 'activator', 'repressor', 'repressor', 'activator', 'regulator', 'no_relation', 'regulator', 'activator', 'activator', 'activator', 'regulator', 'repressor', 'repressor', 'no_relation', 'repressor', 'activator', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'repressor', 'no_relation', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'regulator', 'activator', 'regulator', 'activator', 'no_relation', 'repressor', 'repressor', 'regulator', 'no_relation', 'regulator', 'regulator', 'regulator', 'activator', 'no_relation', 'activator', 'no_relation', 'no_relation', 'activator', 'no_relation', 'activator', 'no_relation', 'activator', 'activator', 'activator', 'activator', 'repressor', 'no_relation', 'activator', 'no_relation', 'no_relation', 'repressor', 'regulator', 'activator', 'activator', 'activator', 'activator', 'regulator', 'regulator', 'no_relation', 'regulator', 'no_relation', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'no_relation', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'repressor', 'no_relation', 'no_relation', 'no_relation', 'no_relation', 'activator', 'no_relation', 'regulator', 'regulator', 'repressor', 'repressor', 'regulator', 'no_relation', 'activator', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'activator', 'regulator', 'activator', 'repressor', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'repressor', 'repressor', 'regulator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'repressor', 'no_relation', 'regulator', 'repressor', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'repressor', 'no_relation', 'activator', 'activator']\n",
      "[0, 2, 3, 0, 1, 0, 1, 0, 1, 2, 2, 3, 0, 0, 2, 2, 1, 0, 2, 1, 0, 1, 1, 0, 0, 2, 2, 1, 2, 0, 0, 1, 1, 0, 0, 0, 3, 0, 3, 3, 3, 1, 1, 3, 0, 0, 1, 1, 1, 0, 0, 1, 1, 2, 1, 0, 2, 0, 0, 0, 1, 0, 1, 2, 2, 0, 1, 3, 0, 0, 3, 1, 0, 0, 1, 0, 1, 3, 1, 0, 2, 0, 1, 2, 2, 1, 1, 3, 3, 2, 3, 1, 0, 3, 1, 0, 2, 0, 2, 0, 0, 1, 0, 1, 2, 0, 3, 0, 3, 3, 2, 1, 1, 0, 1, 2, 3, 0, 2, 3, 0, 0, 1, 1, 3, 1, 1, 0, 0, 1, 1, 0, 2, 1, 0, 1, 3, 0, 0, 0, 0, 1, 2, 2, 2, 2, 1, 0, 3, 0, 0, 0, 3, 1, 1, 1, 2, 1, 0, 0, 2, 2, 0, 3, 1, 3, 0, 0, 0, 3, 2, 2, 0, 2, 0, 1, 0, 2, 1, 0, 3, 1, 0, 0, 1, 0, 3, 1, 0, 2, 2, 0, 1, 1, 0, 0, 2, 3, 0, 0, 0, 0, 0, 1, 3, 0, 3, 1, 1, 2, 2, 3, 1, 3, 3, 0, 0, 1, 0, 1, 1, 1, 1, 0, 1, 1, 0, 0, 0, 3, 1, 0, 1, 1, 2, 3, 1, 0, 1, 0, 3, 3, 1, 3, 1, 2, 1, 1, 2, 1, 2, 0, 0, 1, 1, 0, 2, 1, 1, 1, 1, 0, 2, 1, 3, 2, 2, 3, 1, 0, 1, 1, 2, 1, 0, 3, 0, 2, 0, 0, 0, 0, 1, 0, 0, 1, 2, 2, 3, 1, 1, 0, 1, 0, 0, 1, 0, 2, 1, 3, 2, 0, 0, 0, 0, 1, 1, 1, 2, 2, 1, 0, 0]\n",
      "['activator', 'repressor', 'regulator', 'activator', 'no_relation', 'activator', 'no_relation', 'activator', 'no_relation', 'repressor', 'repressor', 'regulator', 'activator', 'activator', 'repressor', 'repressor', 'no_relation', 'activator', 'repressor', 'no_relation', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'repressor', 'repressor', 'no_relation', 'repressor', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'activator', 'regulator', 'activator', 'regulator', 'regulator', 'regulator', 'no_relation', 'no_relation', 'regulator', 'activator', 'activator', 'no_relation', 'no_relation', 'no_relation', 'activator', 'activator', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'activator', 'repressor', 'activator', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'repressor', 'repressor', 'activator', 'no_relation', 'regulator', 'activator', 'activator', 'regulator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'regulator', 'no_relation', 'activator', 'repressor', 'activator', 'no_relation', 'repressor', 'repressor', 'no_relation', 'no_relation', 'regulator', 'regulator', 'repressor', 'regulator', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'repressor', 'activator', 'repressor', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'repressor', 'activator', 'regulator', 'activator', 'regulator', 'regulator', 'repressor', 'no_relation', 'no_relation', 'activator', 'no_relation', 'repressor', 'regulator', 'activator', 'repressor', 'regulator', 'activator', 'activator', 'no_relation', 'no_relation', 'regulator', 'no_relation', 'no_relation', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'repressor', 'no_relation', 'activator', 'no_relation', 'regulator', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'repressor', 'repressor', 'repressor', 'repressor', 'no_relation', 'activator', 'regulator', 'activator', 'activator', 'activator', 'regulator', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'activator', 'activator', 'repressor', 'repressor', 'activator', 'regulator', 'no_relation', 'regulator', 'activator', 'activator', 'activator', 'regulator', 'repressor', 'repressor', 'activator', 'repressor', 'activator', 'no_relation', 'activator', 'repressor', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'regulator', 'no_relation', 'activator', 'repressor', 'repressor', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'repressor', 'regulator', 'activator', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'regulator', 'activator', 'regulator', 'no_relation', 'no_relation', 'repressor', 'repressor', 'regulator', 'no_relation', 'regulator', 'regulator', 'activator', 'activator', 'no_relation', 'activator', 'no_relation', 'no_relation', 'no_relation', 'no_relation', 'activator', 'no_relation', 'no_relation', 'activator', 'activator', 'activator', 'regulator', 'no_relation', 'activator', 'no_relation', 'no_relation', 'repressor', 'regulator', 'no_relation', 'activator', 'no_relation', 'activator', 'regulator', 'regulator', 'no_relation', 'regulator', 'no_relation', 'repressor', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'repressor', 'activator', 'activator', 'no_relation', 'no_relation', 'activator', 'repressor', 'no_relation', 'no_relation', 'no_relation', 'no_relation', 'activator', 'repressor', 'no_relation', 'regulator', 'repressor', 'repressor', 'regulator', 'no_relation', 'activator', 'no_relation', 'no_relation', 'repressor', 'no_relation', 'activator', 'regulator', 'activator', 'repressor', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'activator', 'activator', 'no_relation', 'repressor', 'repressor', 'regulator', 'no_relation', 'no_relation', 'activator', 'no_relation', 'activator', 'activator', 'no_relation', 'activator', 'repressor', 'no_relation', 'regulator', 'repressor', 'activator', 'activator', 'activator', 'activator', 'no_relation', 'no_relation', 'no_relation', 'repressor', 'repressor', 'no_relation', 'activator', 'activator']\n"
     ]
    }
   ],
   "source": [
    "true_labels_words = [id2label[label] for label in true_labels]\n",
    "predicted_labels_words = [id2label[predicted] for predicted in predicted_labels]\n",
    "print(true_labels)\n",
    "print(true_labels_words)\n",
    "print(predicted_labels)\n",
    "print(predicted_labels_words)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "import csv \n",
    "\n",
    "with open(\"../../02_modelling/error_analysis/luke_best_model_all_predictions_for_error_analysis.tsv\",\"w\") as output:\n",
    "    writer = csv.writer(output, delimiter='\\t')\n",
    "    writer.writerow([\"sentence\",\"sentence_tagged\",\"regulator\",\"regulated\", \"true_label\", \"predicted_label\"])\n",
    "\n",
    "    for sentence, sentence_tag, regulator, regulated, true_label, predicted_label in zip(input_sentences,input_sentences_tagged,input_regulators,input_regulateds, true_labels_words, predicted_labels_words):\n",
    "        writer.writerow([sentence, sentence_tag, regulator, regulated, true_label, predicted_label])"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "torch-m1-mps",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
